{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting Main Function\n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'Y156H', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'0', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18559266', evidence_text:\"Thus, the two mutations in the catalytic cleft, Y156H and G160V, render Aurora B resistant to four classes of inhibitor, namely, ZM447439, VX680, Hesperadin, and MLN8054.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'Y156#', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'1', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18559266', evidence_text:\"Thus, the two mutations in the catalytic cleft, Y156H and G160V, render Aurora B resistant to four classes of inhibitor, namely, ZM447439, VX680, Hesperadin, and MLN8054.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'G160V', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'2', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18559266', evidence_text:\"Thus, the two mutations in the catalytic cleft, Y156H and G160V, render Aurora B resistant to four classes of inhibitor, namely, ZM447439, VX680, Hesperadin, and MLN8054.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'G160#', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'3', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18559266', evidence_text:\"Thus, the two mutations in the catalytic cleft, Y156H and G160V, render Aurora B resistant to four classes of inhibitor, namely, ZM447439, VX680, Hesperadin, and MLN8054.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'4', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:\"Results revealed that MLN8237 inhibits the phosphorylation of both AURKA and AURKB, though it is more specific to AURKA. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'5', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:\"Results revealed that MLN8237 inhibits the phosphorylation of both AURKA and AURKB, though it is more specific to AURKA. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HMMR', entrez:'3161'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'3161', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'6', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23328114', evidence_text:\"Silencing of HMMR/RHAMM, but not TPX2, augments AURKA activity and sensitizes MPNST cells to AKI.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HMMR', entrez:'3161'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'3161', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'7', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23328114', evidence_text:\"Silencing of HMMR/RHAMM, but not TPX2, augments AURKA activity and sensitizes MPNST cells to AKI.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'snv', value:'T217D', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'8', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20426425', evidence_text:\"Using inducible human cell lines, we show that cells expressing near-physiological levels of a functional but partially drug-resistant Aurora A T217D mutant survive in the presence of MLN8054 or MLN8237\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'snv', value:'T217#', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'9', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20426425', evidence_text:\"Using inducible human cell lines, we show that cells expressing near-physiological levels of a functional but partially drug-resistant Aurora A T217D mutant survive in the presence of MLN8054 or MLN8237\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'10', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:\"Here, using a mouse model with orthotopic implantation of metastatic melanoma tumours taken from 19 patients, we observed that targeting aurora kinases with MLN8054/MLN8237 impaired mitosis, induced senescence and markedly blocked proliferation in patient tumour implants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Disease:  Melanoma\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'11', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:\"Here, using a mouse model with orthotopic implantation of metastatic melanoma tumours taken from 19 patients, we observed that targeting aurora kinases with MLN8054/MLN8237 impaired mitosis, induced senescence and markedly blocked proliferation in patient tumour implants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Disease:  Melanoma\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'12', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:\"Here, using a mouse model with orthotopic implantation of metastatic melanoma tumours taken from 19 patients, we observed that targeting aurora kinases with MLN8054/MLN8237 impaired mitosis, induced senescence and markedly blocked proliferation in patient tumour implants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Disease:  Melanoma |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'13', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:\"Results revealed that MLN8237 inhibits the phosphorylation of both AURKA and AURKB, though it is more specific to AURKA. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note: Some mutations in AURKB cause resistance to related drug MLN8054.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'14', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:\"We show that MET inhibition induces complete growth arrest of MET-amplified gastric cancer cells. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'D1228#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'15', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'D1228A', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'16', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'D1228L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'17', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'D1228N', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'18', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'E168#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'19', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19723643', evidence_text:\"In contrast, the cells expressing E168D mutant showed higher susceptibility to the c-Met inhibitor at both concentrations tested (Fig. 3C). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Lung Cancer. |Annotator Note: Drug class relation with SU11274. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'E168D', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'20', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19723643', evidence_text:\"In contrast, the cells expressing E168D mutant showed higher susceptibility to the c-Met inhibitor at both concentrations tested (Fig. 3C). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Lung Cancer. |Annotator Note: Drug class relation with SU11274. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'F1200#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'21', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'F1200I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'22', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'F1200L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'23', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'G1163#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'24', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'G1163E', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'25', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'G1163V', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'26', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'27', evidence:'direct', pmid:'http://bit.ly/1ziTsf4', evidence_text:\"Responses were observed in a subset of pts with MET-amplified tumors. A dose-expansion phase will enroll up to 50 pts at the MTD (300 mg QD). |Evidence Level:  Phase I |Disease:  gastroesophageal cancer. |Annotator Note: Some studies suggest amplficiation of MET as an acquired resistance mechanism upon MET inhibitor treatment.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1094#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'28', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23213094', evidence_text:\"The presence of a germline MET mutation was highly predictive of a response (five of 10 v five of 57 patients with and without germline MET mutations, respectively). |Evidence Level:  Phase II |Disease:  papillary renal cell carcinoma |Annotator Note:  Drug class relation with Foretinib. H1094R in this study was a germline mutation.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1094R', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'29', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23213094', evidence_text:\"The presence of a germline MET mutation was highly predictive of a response (five of 10 v five of 57 patients with and without germline MET mutations, respectively). |Evidence Level:  Phase II |Disease:  papillary renal cell carcinoma |Annotator Note:  Drug class relation with Foretinib. H1094R in this study was a germline mutation.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1112#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'30', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"Remarkably, five days of EMD1214063 treatment resulted in a complete regression of the sensitive H1112L-derived tumors. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1112L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'31', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"Remarkably, five days of EMD1214063 treatment resulted in a complete regression of the sensitive H1112L-derived tumors. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1112Y', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'32', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"Contrary to the observations with M1268T, H1112Y, H1112L, V1238I, and V1110I, EMD1214063 doses up to 75 nmol/L did not display any inhibition of the MET enzymatic activity in cells expressing the Y1248H, L1213V, and V1206L mutants as judged from the levels of phosphorylated MET. |Evidence Level:  Human. Pre-clinical.  Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H112#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'33', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"Contrary to the observations with M1268T, H1112Y, H1112L, V1238I, and V1110I, EMD1214063 doses up to 75 nmol/L did not display any inhibition of the MET enzymatic activity in cells expressing the Y1248H, L1213V, and V1206L mutants as judged from the levels of phosphorylated MET. |Evidence Level:  Human. Pre-clinical.  Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'K1193#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'34', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'K1193I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'35', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1195#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'36', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1195I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'37', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1195V', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'38', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1213#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'39', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"On the other hand, no changes in cell-cycle progression in terms of G1 and S-phase populations were observed in Y1248H and L1213V cells treated with the EMD1214063, indicating a resistance of these MET variants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1213V', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'40', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"On the other hand, no changes in cell-cycle progression in terms of G1 and S-phase populations were observed in Y1248H and L1213V cells treated with the EMD1214063, indicating a resistance of these MET variants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1211#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'41', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1211L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'42', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1250#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'43', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1250T', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'44', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1268#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'45', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23610116', evidence_text:\"The patient was confirmed as having a heterozygous MET mutation at M1268T. The patient had a reduction of 35% in the sum of one-dimensional measurements of target lesions after receiving treatment for 53 weeks, achieving a confirmed partial response by RECIST version 1.04. |Evidence Level:  Case Study |Disease:  papillary renal cell carcinoma\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1268T', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'46', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23610116', evidence_text:\"The patient was confirmed as having a heterozygous MET mutation at M1268T. The patient had a reduction of 35% in the sum of one-dimensional measurements of target lesions after receiving treatment for 53 weeks, achieving a confirmed partial response by RECIST version 1.04. |Evidence Level:  Case Study |Disease:  papillary renal cell carcinoma\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'N375#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'47', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19723643', evidence_text:\"The MET-N375S mutation seems to confer resistance to MET inhibition based on hepatocyte growth factor ligand binding, molecular modeling, and apoptotic susceptibility to MET inhibitor studies. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Lung Cancer. |Annotator Note: Drug class relation with SU11274. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'N375S', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'48', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19723643', evidence_text:\"The MET-N375S mutation seems to confer resistance to MET inhibition based on hepatocyte growth factor ligand binding, molecular modeling, and apoptotic susceptibility to MET inhibitor studies. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Lung Cancer. |Annotator Note: Drug class relation with SU11274. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'49', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23388504', evidence_text:\"These data suggest that c-MET may be a promising target in the treatment of HCC and that c-MET overexpression may be a predictive biomarker of response. |Evidence Level: Pre-clinical. |Disease:  Hepatocellular Cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Q1258#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'50', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Q1258L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'51', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1110#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'52', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"IC50 values for the studied mutants ranged between 2.2 and 43 nmol/L for the H1112L and V1110I variants, respectively. In this regard, EMD1214063 displays anti-MET IC50 values, which are in the range of crizotinib, a MET small molecule that is currently in phase III clinical trials.  |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1110I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'53', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"IC50 values for the studied mutants ranged between 2.2 and 43 nmol/L for the H1112L and V1110I variants, respectively. In this regard, EMD1214063 displays anti-MET IC50 values, which are in the range of crizotinib, a MET small molecule that is currently in phase III clinical trials.  |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1155#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'54', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1155L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'55', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1206#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'56', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"Of major importance is our observation that the autophosphorylation status of the MET-mutated variants Y1248H, L1213V, and V1206L displays a rather resistant-like response towards EMD1214063. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1206L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'57', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"Of major importance is our observation that the autophosphorylation status of the MET-mutated variants Y1248H, L1213V, and V1206L displays a rather resistant-like response towards EMD1214063. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1238#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'58', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"As the results of Fig. 3 suggest, variants V1238I, M1268T, and H1112L exhibited the highest sensitivity to 30 nmol/L of EMD1214063. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1238I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'59', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"As the results of Fig. 3 suggest, variants V1238I, M1268T, and H1112L exhibited the highest sensitivity to 30 nmol/L of EMD1214063. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1159#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'60', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. Not sure what class AMG 337 is exactly.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1159N', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'61', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:\"Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. Not sure what class AMG 337 is exactly.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'62', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230C', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'63', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230D', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'64', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230H', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'65', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230N', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'66', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230S', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'67', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1248#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'68', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"On the other hand, no changes in cell-cycle progression in terms of G1 and S-phase populations were observed in Y1248H and L1213V cells treated with the EMD1214063, indicating a resistance of these MET variants. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1248H', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'69', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:\"On the other hand, no changes in cell-cycle progression in terms of G1 and S-phase populations were observed in Y1248H and L1213V cells treated with the EMD1214063, indicating a resistance of these MET variants. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'70', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:\"Our data provide a strong rationale for combined inhibition of MET and HER kinase signaling in patients with MET-amplified tumors that coexpress EGFR and/or HER-3. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'del L747_S752', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'71', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:\"Transfection of GTL-16 Cells with Mutant EGFR Harboring an Exon 19 Deletion (EGFR?L747-S752) Confers Resistance to PHA-665752 treatment.  |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'72', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:\"Our data provide a strong rationale for combined inhibition of MET and HER kinase signaling in patients with MET-amplified tumors that coexpress EGFR and/or HER-3. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB3', entrez:'2065'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2065', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'73', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:\"Our data provide a strong rationale for combined inhibition of MET and HER kinase signaling in patients with MET-amplified tumors that coexpress EGFR and/or HER-3. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB3', entrez:'2065'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2065', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'74', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:\"Our data provide a strong rationale for combined inhibition of MET and HER kinase signaling in patients with MET-amplified tumors that coexpress EGFR and/or HER-3. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'75', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20841479', evidence_text:\"KRAS amplification is a general mechanism of resistance to specific MET inhibitors given that similar results were observed with two small inhibitors and in different cell lines of different histotypes. To our knowledge, this is the first report showing that overexpression of wt KRAS can overcome the inhibitory effect of a RTK inhibitor. |Evidence Level:  Human. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'76', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20841479', evidence_text:\"KRAS amplification is a general mechanism of resistance to specific MET inhibitors given that similar results were observed with two small inhibitors and in different cell lines of different histotypes. To our knowledge, this is the first report showing that overexpression of wt KRAS can overcome the inhibitory effect of a RTK inhibitor. |Evidence Level:  Human. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TGFA', entrez:'7039'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7039', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'77', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"In this study, we found that SNU638 cells adapt to MET inhibition by overexpressing the EGFR ligand TGFa to promote resistance. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Carcinoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SND1_BRAF', entrez:'EMPTY'}), (a:Aberration { type:'EMPTY', value:'EMPTY', gene_entrez:'EMPTY', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'78', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22745804', evidence_text:\"We rendered a MET amplified gastric cancer cell line, GTL16, resistant to c-Met inhibition with prolonged exposure to a c-Met inhibitor, PF-04217903 (METi). Characterization of surviving cells identified an amplified chromosomal rearrangement between 7q32 and 7q34 which overexpresses a constitutively active SND1-BRAF fusion protein. In the resistant clones, hyperactivation of the downstream MAPK pathway via SND1-BRAF conferred resistance to c-Met receptor tyrosine kinase inhibition. Combination treatment with METi and a RAF inhibitor, PF-04880594 (RAFi) inhibited ERK activation and circumvented resistance to either single agent. Alternatively, treatment with a MEK inhibitor, PD-0325901 (MEKi) alone effectively blocked ERK phosphorylation and inhibited cell growth. Our results suggest that combination of a c-Met tyrosine kinase inhibitor with a BRAF or a MEK inhibitor may be effective in treating resistant tumors that use activated BRAF to escape suppression of c-Met signaling. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PF=-4217903.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'79', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits PDGF-R-beta with IC50=2.9 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical. |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'80', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits PDGF-R-beta with IC50=2.9 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'81', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits PDGF-R-beta with IC50=2.9 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'82', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits PDGF-R-alpha with IC50=5.0 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Human. Pre-clinical. |Annotator Note:  PDGFRA SNVs can be associated either with drug response or sensitivity depending on disease and position in protein. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'83', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits PDGF-R-alpha with IC50=5.0 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Human. Pre-clinical. |Annotator Note:  'Is a target' relationship.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'84', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits PDGF-R-alpha with IC50=5.0 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Human. Pre-clinical. |Annotator Note:  'Is a target' relationship.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'85', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits KIT (c-KIT) with IC50=1.7 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'86', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits KIT (c-KIT) with IC50=1.7 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'87', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:\"Axitinib inhibits KIT (c-KIT) with IC50=1.7 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical. |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KDR', entrez:'3791'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'3791', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'88', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KDR', entrez:'3791'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3791', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'89', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KDR', entrez:'3791'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3791', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'90', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT4', entrez:'2324'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2324', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'91', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT4', entrez:'2324'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2324', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'92', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT4', entrez:'2324'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2324', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'93', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT1', entrez:'2321'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2321', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'94', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT1', entrez:'2321'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2321', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'95', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT1', entrez:'2321'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2321', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'96', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18447599', evidence_text:\"Axitinib inhibits VEGFR-1, VEGFR-2 and VEGFR-3 with picomolar potencies, and PDGFR-alpha, PDGFR-beta and c-kit with nanomolar potencies. |Evidence Level:  Human. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'C634R', gene_entrez:'5979', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'97', evidence:'direct', pmid:'http://bit.ly/19p9EVQ', evidence_text:\"Drug sensitivity was determined by inhibition of RET kinase phosphorylation and IC50s were calculated using photodensitometry. The IC50s of each TKI against the different RET mutants are presented in the Table. WT RET was most sensitive to all of the TKIs. |Evidence Level: Pre-clinical. |Annotator Note:  Axitinib is generatlly thought of as not targeting RET. Authors don't make conclusion one way of another just show biochemical evidence.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'C634#', gene_entrez:'5979', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'98', evidence:'direct', pmid:'http://bit.ly/19p9EVQ', evidence_text:\"Drug sensitivity was determined by inhibition of RET kinase phosphorylation and IC50s were calculated using photodensitometry. The IC50s of each TKI against the different RET mutants are presented in the Table. WT RET was most sensitive to all of the TKIs. |Evidence Level: Pre-clinical. |Annotator Note:  Axitinib is generatlly thought of as not targeting RET. Authors don't make conclusion one way of another just show biochemical evidence.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'V804L', gene_entrez:'5979', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'99', evidence:'direct', pmid:'http://bit.ly/19p9EVQ', evidence_text:\"Drug sensitivity was determined by inhibition of RET kinase phosphorylation and IC50s were calculated using photodensitometry. The IC50s of each TKI against the different RET mutants are presented in the Table. WT RET was most sensitive to all of the TKIs. |Evidence Level: Pre-clinical. |Annotator Note:  Axitinib is generatlly thought of as not targeting RET. Authors don't make conclusion one way of another just show biochemical evidence.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'V804#', gene_entrez:'5979', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'100', evidence:'direct', pmid:'http://bit.ly/19p9EVQ', evidence_text:\"Drug sensitivity was determined by inhibition of RET kinase phosphorylation and IC50s were calculated using photodensitometry. The IC50s of each TKI against the different RET mutants are presented in the Table. WT RET was most sensitive to all of the TKIs. |Evidence Level: Pre-clinical. |Annotator Note:  Axitinib is generatlly thought of as not targeting RET. Authors don't make conclusion one way of another just show biochemical evidence.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'M918T', gene_entrez:'5979', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'101', evidence:'direct', pmid:'http://bit.ly/19p9EVQ', evidence_text:\"Drug sensitivity was determined by inhibition of RET kinase phosphorylation and IC50s were calculated using photodensitometry. The IC50s of each TKI against the different RET mutants are presented in the Table. WT RET was most sensitive to all of the TKIs. |Evidence Level: Pre-clinical. |Annotator Note:  Axitinib is generatlly thought of as not targeting RET. Authors don't make conclusion one way of another just show biochemical evidence.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'M918#', gene_entrez:'5979', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'102', evidence:'direct', pmid:'http://bit.ly/19p9EVQ', evidence_text:\"Drug sensitivity was determined by inhibition of RET kinase phosphorylation and IC50s were calculated using photodensitometry. The IC50s of each TKI against the different RET mutants are presented in the Table. WT RET was most sensitive to all of the TKIs. |Evidence Level: Pre-clinical. |Annotator Note:  Axitinib is generatlly thought of as not targeting RET. Authors don't make conclusion one way of another just show biochemical evidence.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCB1', entrez:'5243'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5243', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'103', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21282407', evidence_text:\"We studied axitinib transport using Madin-Darby canine kidney II cells overexpressing human ABCB1 or ABCG2 or murine Abcg2. Axitinib was a good substrate of ABCB1 and Abcg2, whereas transport activity by ABCG2 was moderate. These transporters may therefore contribute to axitinib resistance in tumor cells. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCB1', entrez:'5243'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5243', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'104', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21282407', evidence_text:\"We studied axitinib transport using Madin-Darby canine kidney II cells overexpressing human ABCB1 or ABCG2 or murine Abcg2. Axitinib was a good substrate of ABCB1 and Abcg2, whereas transport activity by ABCG2 was moderate. These transporters may therefore contribute to axitinib resistance in tumor cells. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'105', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21282407', evidence_text:\"We studied axitinib transport using Madin-Darby canine kidney II cells overexpressing human ABCB1 or ABCG2 or murine Abcg2. Axitinib was a good substrate of ABCB1 and Abcg2, whereas transport activity by ABCG2 was moderate. These transporters may therefore contribute to axitinib resistance in tumor cells. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'106', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21282407', evidence_text:\"We studied axitinib transport using Madin-Darby canine kidney II cells overexpressing human ABCB1 or ABCG2 or murine Abcg2. Axitinib was a good substrate of ABCB1 and Abcg2, whereas transport activity by ABCG2 was moderate. These transporters may therefore contribute to axitinib resistance in tumor cells. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'T315I', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'107', evidence:'inferred', pmid:'http://bit.ly/1CFGZTL', evidence_text:\"Here we combine comprehensive drug sensitivity and resistance profiling of patient cells ex vivo with structural analysis to establish the VEGFR tyrosine kinase inhibitor axitinib as a selective and effective inhibitor for T315I-mutant BCR-ABL1-driven leukaemia. Axitinib potently inhibited BCR-ABL1(T315), at both biochemical and cellular levels, by binding to the active form of ABL1(T315l) in a mutation-selective binding mode.  Evidence Level:  Human. Pre-clinical. Xenograft. Disease: CML.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'T315#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'108', evidence:'inferred', pmid:'http://bit.ly/1CFGZTL', evidence_text:\"Here we combine comprehensive drug sensitivity and resistance profiling of patient cells ex vivo with structural analysis to establish the VEGFR tyrosine kinase inhibitor axitinib as a selective and effective inhibitor for T315I-mutant BCR-ABL1-driven leukaemia. Axitinib potently inhibited BCR-ABL1(T315I), at both biochemical and cellular levels, by binding to the active form of ABL1(T315I) in a mutation-selective binding mode.  |Evidence Level:  Human. Pre-clinical. Xenograft. |Disease: CML.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2264'}), (a:Aberration { type:'snv', value:'Y770fs', gene_entrez:'2264', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'109', evidence:'direct', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'110', evidence:'inferred', pmid:'http://bit.ly/1PYelHj', evidence_text:\"Preliminary analysis of efficacy data indicates tumor regressions in pts with various FGFR genetic alterations, including 4 of 5 pts with urothelial cell carcinomas (4 of which originated in the bladder) with FGFR3-activating mutations (with tumor reductions ranging from 27% to 48%). |Evidence Level: Phase 1. |Disease:  Bladder Cancer. |Annotator Note:  Results are from abstract of Phase I and mutations are not listed. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'111', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21936542', evidence_text:\"(NVP-BGJ398) was selected for in vivo evaluation and showed significant antitumor activity in RT112 bladder cancer xenografts models overexpressing wild-type FGFR3. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Bladder Cancer. |Annotator Note:  Sensitivity confirmed in this cell line in a separate study. PMID: 23002168.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'112', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21936542', evidence_text:\"(NVP-BGJ398) was selected for in vivo evaluation and showed significant antitumor activity in RT112 bladder cancer xenografts models overexpressing wild-type FGFR3. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Bladder Cancer. |Annotator Note:  Sensitivity confirmed in this cell line in a separate study. PMID: 23002168.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'Y375#', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'113', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22238366', evidence_text:\"Ponatinib inhibited growth of UMUC14 (FGFR3S249C) and MGH-U3 (FGFR3Y375C) cells with GI50 values of 103 and 181 nmol/L, respectively, but had no effect on growth of T24 cells that express wild-type FGFR3 (GI50 > 1,000 nmol/L; Fig. 3A; ref. 41). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Bladder Cancer. |Annotator Note:  Drug class relation with ponatinib and dovitinib. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'Y375C', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'114', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22238366', evidence_text:\"Ponatinib inhibited growth of UMUC14 (FGFR3S249C) and MGH-U3 (FGFR3Y375C) cells with GI50 values of 103 and 181 nmol/L, respectively, but had no effect on growth of T24 cells that express wild-type FGFR3 (GI50 > 1,000 nmol/L; Fig. 3A; ref. 41). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Bladder Cancer. |Annotator Note:  Drug class relation with ponatinib and dovitinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'R248#', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'115', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line.  |Annotator Note:  SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity. Result from abstract phase 1 trial: Preliminary analysis of efficacy data indicates tumor regressions in pts with various FGFR genetic alterations, including 4 of 5 pts with urothelial cell carcinomas (4 of which originated in the bladder) with FGFR3-activating mutations (with tumor reductions ranging from 27% to 48%). http://bit.ly/1PYelHj\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'R248C', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'116', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Result from abstract phase 1 trial: Preliminary analysis of efficacy data indicates tumor regressions in pts with various FGFR genetic alterations, including 4 of 5 pts with urothelial cell carcinomas (4 of which originated in the bladder) with FGFR3-activating mutations (with tumor reductions ranging from 27% to 48%). http://bit.ly/1PYelHj\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'S249#', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'117', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line.  |Annotator Note:  SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.  Result from abstract phase 1 trial: Preliminary analysis of efficacy data indicates tumor regressions in pts with various FGFR genetic alterations, including 4 of 5 pts with urothelial cell carcinomas (4 of which originated in the bladder) with FGFR3-activating mutations (with tumor reductions ranging from 27% to 48%). http://bit.ly/1PYelHj\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'S249C', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'118', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line.  |Annotator Note:  Result from abstract phase 1 trial: Preliminary analysis of efficacy data indicates tumor regressions in pts with various FGFR genetic alterations, including 4 of 5 pts with urothelial cell carcinomas (4 of which originated in the bladder) with FGFR3-activating mutations (with tumor reductions ranging from 27% to 48%). http://bit.ly/1PYelHj\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'V555#', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'119', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23908597', evidence_text:\"Mutations of the gatekeeper residues in FGFR1 (V561M) and FGFR3 (V555M) have been shown to result in in vitro resistance to the multi-kinase inhibitor PP58 and the FGFR inhibitor AZ12908010, respectively [24], thus indicating that mutation of the gatekeeper residue may be a general mechanism of resistance to receptor tyrosine kinase (RTK) inhibitors. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity. Drug class relation with AZ12908010 and PD173074 and AZD4547 PMID:25349422.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'V555M', gene_entrez:'2261', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'120', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23908597', evidence_text:\"Mutations of the gatekeeper residues in FGFR1 (V561M) and FGFR3 (V555M) have been shown to result in in vitro resistance to the multi-kinase inhibitor PP58 and the FGFR inhibitor AZ12908010, respectively [24], thus indicating that mutation of the gatekeeper residue may be a general mechanism of resistance to receptor tyrosine kinase (RTK) inhibitors. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with AZ12908010 and PD173074 and AZD4547 PMID:25349422\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'E556#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'121', evidence:'inferred', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'E566G', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'122', evidence:'direct', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'I548#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'123', evidence:'inferred', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'I548V', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'124', evidence:'direct', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'L618#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'125', evidence:'inferred', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'L618M', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'126', evidence:'direct', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'M536#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'127', evidence:'inferred', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'M536I', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'128', evidence:'direct', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'M538#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'129', evidence:'inferred', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'M538I', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'130', evidence:'direct', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'N550H', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'131', evidence:'direct', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'N550S', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'132', evidence:'direct', pmid:'http://bit.ly/1G9wcIt', evidence_text:\"Data presented in figure showing sensitivity to BGJ398 in BaF3 cell line model for mutation listed (10-30nM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Figure located here: http://bit.ly/1G9w7o6. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'A648#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'133', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition. SNVs at this position are associated with loss-of-function and reduced drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'A648T', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'134', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'E475#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'135', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"As anticipated, the constitutively active N549K mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including G227E, G271E, E475K, I642V, and A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A), verifying that these mutations impaired or abrogated the ability of FGFR2 to stimulate cell proliferation in response to ligand in an in vivo setting. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'E475K', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'136', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"As anticipated, the constitutively active N549K mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including G227E, G271E, E475K, I642V, and A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A), verifying that these mutations impaired or abrogated the ability of FGFR2 to stimulate cell proliferation in response to ligand in an in vivo setting. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'G227#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'137', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"As anticipated, the constitutively active N549K mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including G227E, G271E, E475K, I642V, and A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A), verifying that these mutations impaired or abrogated the ability of FGFR2 to stimulate cell proliferation in response to ligand in an in vivo setting. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition. SNVs at this position are associated with loss-of-function and reduced drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'G227E', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'138', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"As anticipated, the constitutively active N549K mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including G227E, G271E, E475K, I642V, and A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A), verifying that these mutations impaired or abrogated the ability of FGFR2 to stimulate cell proliferation in response to ligand in an in vivo setting. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'G271#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'139', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"As anticipated, the constitutively active N549K mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including G227E, G271E, E475K, I642V, and A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A), verifying that these mutations impaired or abrogated the ability of FGFR2 to stimulate cell proliferation in response to ligand in an in vivo setting. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'G271E', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'140', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"As anticipated, the constitutively active N549K mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including G227E, G271E, E475K, I642V, and A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A), verifying that these mutations impaired or abrogated the ability of FGFR2 to stimulate cell proliferation in response to ligand in an in vivo setting. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'I642#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'141', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"As anticipated, the constitutively active N549K mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including G227E, G271E, E475K, I642V, and A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A), verifying that these mutations impaired or abrogated the ability of FGFR2 to stimulate cell proliferation in response to ligand in an in vivo setting. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'I642V', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'142', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147536', evidence_text:\"As anticipated, the constitutively active N549K mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including G227E, G271E, E475K, I642V, and A648T FGFR2 resulted in reduced cell proliferation compared with wild-type FGFR2 (Fig. 6A), verifying that these mutations impaired or abrogated the ability of FGFR2 to stimulate cell proliferation in response to ligand in an in vivo setting. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  The mutations in this study were identified in a melanoma cell lines and clinical samples. They were shown to be inactivating, hence conveyed to BGJ398 as not be sensitive to inhibition.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'C383#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'143', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"Indeed, all of the FGFR2- and FGFR3-mutated cell lines...were growth inhibited by NVP-BGJ398, in line with its ability to effectively block FGFR downstream signaling in the 6 FGFR-mutated cell lines tested (Supplementary Fig. S2). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Endometrial Cancer. |Annotator Note:  This mutation was in endometrial cell line JHUEM-2. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'C383R', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'144', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"Indeed, all of the FGFR2- and FGFR3-mutated cell lines...were growth inhibited by NVP-BGJ398, in line with its ability to effectively block FGFR downstream signaling in the 6 FGFR-mutated cell lines tested (Supplementary Fig. S2). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Endometrial Cancer. |Annotator Note:  This mutation was in endometrial cell line JHUEM-2.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'145', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"All the FGFR2- and FGFR3-mutated cell lines but two and one respectively, were growth inhibited by NVP-BGJ398, in line with its ability to effectively block FGFR downstream signaling in the 6 FGFR-mutated cell lines tested. |Evidence Level:  Pre-clinical. Xenograft. Cell line. |Disease: Gastric Cancer, Esophageal Cancer, Colon Cancer. |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity. There are cases were FGFR2 mutations in melanoma have been deemed to be inactivating.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'146', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"Thus, FGFR2-amplified cell lines are sensitive to NVP- BGJ398 in vitro as well as when grown in vivo as human tumor xenografts. Hence, we envision that human gastric tumors harboring FGFR2 amplification will be responsive to NVP-BGJ398 in the clinic. Interestingly, in addition to confirming the incidence of this genetic alteration in gastric cancer, we also found FGFR2 amplification in 1 of 22 esophageal tumors, which offers a novel potential clinical opportunity for an FGFR inhibitor. |Evidence Level:  Pre-clinical. Xenograft. Cell line. |Disease: Gastric Cancer, Esophageal Cancer, Colon Cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'147', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"Thus, FGFR2-amplified cell lines are sensitive to NVP- BGJ398 in vitro as well as when grown in vivo as human tumor xenografts. Hence, we envision that human gastric tumors harboring FGFR2 amplification will be responsive to NVP-BGJ398 in the clinic. Interestingly, in addition to confirming the incidence of this genetic alteration in gastric cancer, we also found FGFR2 amplification in 1 of 22 esophageal tumors, which offers a novel potential clinical opportunity for an FGFR inhibitor. |Evidence Level:  Pre-clinical. Xenograft. Cell line. |Disease: Gastric Cancer, Esophageal Cancer, Colon Cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'K660#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'148', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line.  |Annotator Note:  SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'K660E', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'149', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'K660N', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'150', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'P235#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'151', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"He began daily treatment with 800 mg pazopanib starting on April 12, 2012. At this time, he had gross disease in the right neck (Fig. 5C, left). A follow-up visit 12 days later showed a marked reduction in tumor size (Fig. 5C, right). |Evidence Level:  Case Study |Disease:  HNSCC. |Annotator Note:  Drug class relation with pazopanib. Authors of study did show that this mutation is sensitive to BGJ398 in a cell line model. SNVs at this position are associated with activation and drug sensitivity. However, not all changes at this position may result in activation and drug sensitivity. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'P235R', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'152', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"He began daily treatment with 800 mg pazopanib starting on April 12, 2012. At this time, he had gross disease in the right neck (Fig. 5C, left). A follow-up visit 12 days later showed a marked reduction in tumor size (Fig. 5C, right). |Evidence Level:  Case Study |Disease:  HNSCC. |Annotator Note:  Drug class relation with pazopanib. Authors of study did show that this mutation is sensitive to BGJ398 in a cell line model. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'S320#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'153', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line.  |Annotator Note:  SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'S320C', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'154', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'W290#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'155', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line.  |Annotator Note:  SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'W290C', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'156', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23786770', evidence_text:\"Growth was inhibited by BGJ398 in figure 3 and figure 4. |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'N550#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'157', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23908597', evidence_text:\"Cell lines with activating FGFR2 mutations (S252W, N550K) were more sensitive to dovitinib or NVP-BGJ398 when compared with their FGFR2 wild-type counterparts (P=0.073 and P=0.021, respectively). |Evidence Level:  Pre-clinical. Cell line. |Disease: Endometrial Cancer. |Annotator Note:   Sensitivity confirmed in this cell line in a separate study. PMID: 23002168. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'N550K', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'158', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23908597', evidence_text:\"Cell lines with activating FGFR2 mutations (S252W, N550K) were more sensitive to dovitinib or NVP-BGJ398 when compared with their FGFR2 wild-type counterparts (P=0.073 and P=0.021, respectively). |Evidence Level:  Pre-clinical. Cell line. |Disease: Endometrial Cancer. |Annotator Note:  Sensitivity confirmed in this cell line in a separate study. PMID: 23002168.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'S252#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'159', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23908597', evidence_text:\"Cell lines with activating FGFR2 mutations (S252W, N550K) were more sensitive to dovitinib or NVP-BGJ398 when compared with their FGFR2 wild-type counterparts (P=0.073 and P=0.021, respectively). |Evidence Level:  Pre-clinical. Cell line. |Disease: Endometrial Cancer. |Annotator Note: Sensitivity confirmed in this cell line in a separate study. PMID: 23002168. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'S252W', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'160', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23908597', evidence_text:\"Cell lines with activating FGFR2 mutations (S252W, N550K) were more sensitive to dovitinib or NVP-BGJ398 when compared with their FGFR2 wild-type counterparts (P=0.073 and P=0.021, respectively). |Evidence Level:  Pre-clinical. Cell line. |Disease: Endometrial Cancer. |Annotator Note:  Sensitivity confirmed in this cell line in a separate study. PMID: 23002168.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'V565#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'161', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23908597', evidence_text:\"The FGFR2 V564I gatekeeper mutant was isolated as a resistant clone in a FGFR2 Ba/F3 screen of dovitinib and also was reported to confer resistance to the multi-targeted drug ponatinib. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with dovitinib and ponatinib. Conferred to BGJ398 as a gatekeeper residue. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'V565I', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'162', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23908597', evidence_text:\"The FGFR2 V564I gatekeeper mutant was isolated as a resistant clone in a FGFR2 Ba/F3 screen of dovitinib and also was reported to confer resistance to the multi-targeted drug ponatinib. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with dovitinib and ponatinib. Conferred to BGJ398 as a gatekeeper residue. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'C491#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'163', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"The C491A FGFR2 mutation was shown in Table 1 to be sensitive to BGJ398. |Evidence Level:  Pre-clinical. Cell line. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'C491A', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'164', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"The C491A FGFR2 mutation was shown in Table 1 to be sensitive to BGJ398. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'E565#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'165', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"The E565K FGFR2 mutation was shown in Table 1 to be sensitive to BGJ398. |Evidence Level:  Pre-clinical. Cell line. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'E565K', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'166', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"The E565K FGFR2 mutation was shown in Table 1 to be sensitive to BGJ398. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'K659#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'167', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"BGJ398 was efficient only against the M538I and K659N mutants. |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'K659N', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'168', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"BGJ398 was efficient only against the M538I and K659N mutants. |Evidence Level:  Pre-clinical. Cell line. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'M538#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'169', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"BGJ398 was efficient only against the M538I and K659N mutants. |Evidence Level:  Pre-clinical. Cell line. SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'M538I', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'170', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"BGJ398 was efficient only against the M538I and K659N mutants. |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'N549#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'171', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"BGJ398 showed efficiency in ANC3A endometrial cell line carrying N549K activating mutataion. |Evidence Level:  Pre-clinical. Cell line.|Disease:  Endometrial Cancer. SNVs at this position are associated with activation and drug sensitivity. However, not all changes at this position may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'N549K', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'172', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"BGJ398 showed efficiency in ANC3A endometrial cell line carrying N549K activating mutataion. |Evidence Level:  Pre-clinical. Cell line.|Disease:  Endometrial Cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'V564#', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'173', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"FIIN-2 and FIIN-3 also inhibited the FGFR2 V564M gatekeeper mutant Ba/F3 cells, with EC50s of 58 and 64 nM, respectively, whereas FIIN-1 and BGJ398 had EC50s of over 1.0 _M against this mutant. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'V564F', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'174', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"The V564F FGFR2 mutation was shown in Table 1 to be insensitive to BGJ398. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'snv', value:'V564M', gene_entrez:'2263', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'175', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"FIIN-2 and FIIN-3 also inhibited the FGFR2 V564M gatekeeper mutant Ba/F3 cells, with EC50s of 58 and 64 nM, respectively, whereas FIIN-1 and BGJ398 had EC50s of over 1.0 _M against this mutant. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR1', entrez:'2260'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2260', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'176', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"These results reveal for the first time that FGFR1 amplification occurs in osteosarcoma, confirm its prevalence in breast and lung cancer cells, and show that FGFR1 is required for cancer cell growth in these settings. Hence, FGFR1 amplification is a predictor of sensitivity to an FGFR inhibitor in these 3 lineages...Of the 13 cell lines identified with FGFR1 amplification, only 5 of 13 (38%) were found to be sensitive to NVP-BGJ398 |Evidence Level:  Pre-clinical. Cell line. |Disease:  Breast Cancer, Lung Cancer, Osteosarcoma. |Annotator Note:  Abstract for phase I trial with BGJ398 showed 1 pt with FGFR1-amplified SCC lung cancer achieved confirmed partial response (PR) while another pt achieved a PR confirmed after 27 days. http://bit.ly/1PYelHj. Additionally, In a Bladder Cancer cell lines that have high FGFR1 expression, they tend to be resitant to BGJ398 due to EMT phenotype. PMID: 23468956.  \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR1', entrez:'2260'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2260', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'177', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"The scatter plot of copy number versus transcript expression revealed that the 5 sensitive cell lines were among the highest expressers of FGFR1 mRNA within the breast, lung and osteosarcoma lineages......Of the 13 cell lines identified with FGFR1 amplification, only 5 of 13 (38%) were found to be sensitive to NVP-BGJ398 |Evidence Level:  Pre-clinical. Cell line. |Disease:  Breast Cancer, Lung Cancer, Osteosarcoma. |Annotator Note:  In a Bladder Cancer cell lines that have high FGFR1 expression, they tend to be resitant to BGJ398 due to EMT phenotype. PMID: 23468956  \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR1', entrez:'2260'}), (a:Aberration { type:'snv', value:'V561#', gene_entrez:'2260', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'178', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"To confirm that the resistance conferred by the gatekeeper mutation in FGFR2 also would be observed for the gatekeeper mutation in FGFR1 in the context of a cancer cell line, we generated FGFR1 V561M gatekeeper mutants in both H2077 and H1581 cells, two cell lines derived from a patient with a lung cancer with high- level FGFR1 amplification. These mutations caused a >50-fold shift in the EC50 of BGJ398, whereas FIIN-2 and FIIN-3 maintained good potency, with EC50s reduced less than 10-fold relative to WT.  |Evidence Level:  Pre-clinical. Cell line. |Disease:  Lung Cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR1', entrez:'2260'}), (a:Aberration { type:'snv', value:'V561M', gene_entrez:'2260', entity_class:'biomarker' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'179', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25349422', evidence_text:\"To confirm that the resistance conferred by the gatekeeper mutation in FGFR2 also would be observed for the gatekeeper mutation in FGFR1 in the context of a cancer cell line, we generated FGFR1 V561M gatekeeper mutants in both H2077 and H1581 cells, two cell lines derived from a patient with a lung cancer with high- level FGFR1 amplification. These mutations caused a >50-fold shift in the EC50 of BGJ398, whereas FIIN-2 and FIIN-3 main- tained good potency, with EC50s reduced less than 10-fold relative to WT.  |Evidence Level:  Pre-clinical. Cell line. |Disease:  Lung Cancer. |Annotator Note:  SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGF19', entrez:'9965'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9965', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'180', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"Accordingly, the Fisher exact test showed a statis- tically significant association between FGF19 copy number and response to NVP-BGJ398 for the liver cancer lineage (P = 0.01), but not when the correlation was tested across all lineages (P = 0.2)....Conversely, several cell lines that displayed sensitivity to NVP-BGJ398 did not harbor FGFR genetic lesions. Three of them, belonging to the liver cancer type, showed copy number gain for the FGFR4 ligand, FGF19, and FGF19 amplification was statistically significantly associated with response to NVP-BGJ398 when the analysis was restricted to liver cancer cell lines. Furthermore, by conditional knockdown of FGFR4, we showed dependency on this RTK in the JHH7 cells, thus supporting the concept of an FGF19/FGFR4 autocrine loop as the oncogenic driver in liver cancer with FGF19 amplification.  |Evidence Level:  Pre-clinical. Cell line. |Disease:  Liver Cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRS2', entrez:'10818'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'10818', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'181', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23393200', evidence_text:\"Consistently, a most recent study showed that high level of phosphorylated FRS2 protein was only present in liposarcoma but not in normal fat or preadipocytes (22). To determine the cellular responses to bFGF2 stimulation, these 3 cell lines were serum starved for about 72 hours and then stimulated with 10 ng/mL of basic FGF2 (bFGF2) for 30 minutes. As shown in Fig. 1D, bFGF2 stimulation promptly increased phosphorylated total FGFR (p-FGFR), FRS2 (p-FRS2), extracellular signal-regulated kinase (ERK)1/2 (p-ERK1/2), and AKT (p-AKT) in all 3 high-grade liposarcoma cell lines, especially in FU-DDLS-1 and LiSa-2, showing the presence of a functional FGFR/FRS2 signaling after bFGF2 stimulation. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Liposarcoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRS2', entrez:'10818'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'10818', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'182', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23393200', evidence_text:\"Consistently, a most recent study showed that high level of phosphorylated FRS2 protein was only present in liposarcoma but not in normal fat or preadipocytes (22). To determine the cellular responses to bFGF2 stimulation, these 3 cell lines were serum starved for about 72 hours and then stimulated with 10 ng/mL of basic FGF2 (bFGF2) for 30 minutes. As shown in Fig. 1D, bFGF2 stimulation promptly increased phosphorylated total FGFR (p-FGFR), FRS2 (p-FRS2), extracellular signal-regulated kinase (ERK)1/2 (p-ERK1/2), and AKT (p-AKT) in all 3 high-grade liposarcoma cell lines, especially in FU-DDLS-1 and LiSa-2, showing the presence of a functional FGFR/FRS2 signaling after bFGF2 stimulation. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Liposarcoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGF2', entrez:'2247'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2247', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'183', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24771645', evidence_text:\"Moreover, ponatinib sensitivity associated with mRNA expression of the ligands, FGF2 and FGF9.  |Evidence Level:  Pre-clinical. Cell line. |Disease:  Lung Cancer. |Annotator Note:  Drug class relation with ponatinib.  \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGF9', entrez:'2254'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2254', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'184', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24771645', evidence_text:\"Moreover, ponatinib sensitivity associated with mRNA expression of the ligands, FGF2 and FGF9.  |Evidence Level:  Pre-clinical. Cell line. |Disease:  Lung Cancer. |Annotator Note:  Drug class relation with ponatinib.  \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDH1', entrez:'999'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'999', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'185', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"We observed that mRNA levels of E-cadherin were downregulated in BGJ398 RS cells and N-cadherin and fibronectin expression were upregulated, consistent with an EMT phenotype (Figure 1C) and levels of PLAU, PLAUR, SNAIL and SLUG were slightly but not significantly increased (data not shown). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Bladder Cancer.  |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDH13', entrez:'1012'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1012', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'186', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"We observed that mRNA levels of E-cadherin were downregulated in BGJ398 RS cells and N-cadherin and fibronectin expression were upregulated, consistent with an EMT phenotype (Figure 1C) and levels of PLAU, PLAUR, SNAIL and SLUG were slightly but not significantly increased (data not shown). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Bladder Cancer.  |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRG1', entrez:'3084'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3084', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'187', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"NRG1, in contrast, is thought to be one of the principal ligands for stimulation of ERBB2/3 heterodimers35 and was upregulated in our resistant cell lines (Figure 5b). As ERBB2 and ERBB3 appeared to be upregulated both at the mRNA and protein level in resistant lines and also demonstrated greater degrees of phosphorylation, we sought to establish whether NRG1 alone could rescue BGJ398-induced death in RT112 and other FGFR-dependent cell lines. Similar to our results from the conditioned media experiments and with EGF stimulation we observed that NRG1 rescued BGJ398-induced death in the FGFR3- dependent lines RT112. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2064', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'188', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"BGJ398-resistant cell lines are selectively sensitive to tyrosine kinase inhibitors and to shRNA-mediated depletion of ERBB2 or ERBB3.  |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2065'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2065', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'189', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"BGJ398-resistant cell lines are selectively sensitive to tyrosine kinase inhibitors and to shRNA-mediated depletion of ERBB2 or ERBB3.  |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDH1', entrez:'999'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'999', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'190', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"We observed that mRNA levels of E-cadherin were downregulated in BGJ398 RS cells and N-cadherin and fibronectin expression were upregulated, consistent with an EMT phenotype (Figure 1C) and levels of PLAU, PLAUR, SNAIL and SLUG were slightly but not significantly increased (data not shown). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Bladder Cancer.  |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDH13', entrez:'1012'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1012', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'191', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"We observed that mRNA levels of E-cadherin were downregulated in BGJ398 RS cells and N-cadherin and fibronectin expression were upregulated, consistent with an EMT phenotype (Figure 1C) and levels of PLAU, PLAUR, SNAIL and SLUG were slightly but not significantly increased (data not shown). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Bladder Cancer.  |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRG1', entrez:'3084'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3084', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'192', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"NRG1, in contrast, is thought to be one of the principal ligands for stimulation of ERBB2/3 heterodimers35 and was upregulated in our resistant cell lines (Figure 5b). As ERBB2 and ERBB3 appeared to be upregulated both at the mRNA and protein level in resistant lines and also demonstrated greater degrees of phosphorylation, we sought to establish whether NRG1 alone could rescue BGJ398-induced death in RT112 and other FGFR-dependent cell lines. Similar to our results from the conditioned media experiments and with EGF stimulation we observed that NRG1 rescued BGJ398-induced death in the FGFR3- dependent lines RT112. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2064', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'193', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"BGJ398-resistant cell lines are selectively sensitive to tyrosine kinase inhibitors and to shRNA-mediated depletion of ERBB2 or ERBB3.  |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2065'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2065', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'194', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24909170', evidence_text:\"BGJ398-resistant cell lines are selectively sensitive to tyrosine kinase inhibitors and to shRNA-mediated depletion of ERBB2 or ERBB3.  |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  These associations were observed in a cell line model of acquired resistance to BGJ398 in the background of FGFR3_TACC3 fusion.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'BGJ398'} ) Merge (r:Rule {rulenum:'195', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23002168', evidence_text:\"A predictive model as one of the genomic predictors for NVP-BGJ398 insensitivity. |Evidence Level:  Pre-clinical. Cell line.  |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMD4', entrez:'5710'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5710', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'196', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21628408', evidence_text:\"PMSD4 was 1 of 3 genes common to both models. Residing on chromosome 1q21, PSMD4 expression is highly sensitive to copy number. Both higher PSMD4 expression levels and higher 1q21 copy numbers affected clinical outcome adversely. GEP80 baseline-defined high risk, high lactate dehydrogenase, and low albumin were the only independent adverse variables surviving multivariate survival model. We are investigating whether second-generation proteasome inhibitors (eg, carfilzomib) can overcome resistance associated with high PSMD4 levels\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMD4', entrez:'5710'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5710', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'197', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21628408', evidence_text:\"PMSD4 was 1 of 3 genes common to both models. Residing on chromosome 1q21, PSMD4 expression is highly sensitive to copy number. Both higher PSMD4 expression levels and higher 1q21 copy numbers affected clinical outcome adversely. GEP80 baseline-defined high risk, high lactate dehydrogenase, and low albumin were the only independent adverse variables surviving multivariate survival model. We are investigating whether second-generation proteasome inhibitors (eg, carfilzomib) can overcome resistance associated with high PSMD4 levels\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB8', entrez:'5696'}), (a:Aberration { type:'snv', value:'F50#', gene_entrez:'5696', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'198', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22216088', evidence_text:\"All resistant clones had a mutation in the propeptide region (Arg24Cys) and a mutation (Cys63Phe) within the active site region of _5. In addition, 2 out of the 3 BR200 clones had a Phe50Ile mutation in the propeptide regions of LMP7 (PSMB8). No mutations were found in _1, _7, or LMP2. Taken together, these data demonstrate that resistance to bortezomib is correlated with genomic alterations that affect gene expression levels and that specific point mutations are selected in the _5 gene.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB8', entrez:'5696'}), (a:Aberration { type:'snv', value:'F50I', gene_entrez:'5696', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'199', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22216088', evidence_text:\"All resistant clones had a mutation in the propeptide region (Arg24Cys) and a mutation (Cys63Phe) within the active site region of _5. In addition, 2 out of the 3 BR200 clones had a Phe50Ile mutation in the propeptide regions of LMP7 (PSMB8). No mutations were found in _1, _7, or LMP2. Taken together, these data demonstrate that resistance to bortezomib is correlated with genomic alterations that affect gene expression levels and that specific point mutations are selected in the _5 gene.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'A49T', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'200', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18565852', evidence_text:\"Rather, qualitative and quantitative alterations were observed at the level of the _5 proteasome subunit; these included a PSMB5 gene mutation resulting in a substitution of a crucial amino acid (Ala49Thr) in the bortezomib-binding pocket of the _5-subunit. These bortezomib-resistant cells highly overexpressed this mutant _5-subunit and displayed a marked cross-resistance to _5-targeted cytotoxic peptides but not to other classes of therapeutic drugs.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'201', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18565852', evidence_text:\"In contrast, _5 protein levels, relatively barely detectable in parental cells, were dramatically increased (up to 60-fold) in bortezomib-resistant cells\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'A49#', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'202', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19426847', evidence_text:\"JurkatB-G322A, JurkatB-C323T, and JurkatB-G322A/C326T clones displayed 22.0-fold, 39.4-fold, and 66.7-fold resistance, respectively, to bortezomib compared to Jurkat cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'A50#', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'203', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19426847', evidence_text:\"Mutations of the PSMB5 gene resulting in substitutions of Ala49 and Ala50 of PSMB5 protein can confer varying bortezomib resistance.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'A49V', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'204', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19426847', evidence_text:\"JurkatB-G322A, JurkatB-C323T, and JurkatB-G322A/C326T clones displayed 22.0-fold, 39.4-fold, and 66.7-fold resistance, respectively, to bortezomib compared to Jurkat cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'A50V', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'205', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19426847', evidence_text:\"Mutations of the PSMB5 gene resulting in substitutions of Ala49 and Ala50 of PSMB5 protein can confer varying bortezomib resistance.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'M45I', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'206', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21941364', evidence_text:\"Characterization of the various BTZ-resistant cells revealed upregulation of mutant _5 subunit of the proteasome. These newly identified _5-subunit mutations, along with previously described mutations, formed a mutation cluster region in the BTZ-binding pocket of the _5 subunit, that of the S1 specificity pocket in particular.  Substitutions at this Met45 can have a marked impact on the dynamics of BTZ binding by altering the binding pocket's specificity and flexibility. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'C52#', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'207', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22027222', evidence_text:\"This study confirmed acquired bortezomib resistance in NSCLC cell lines is associated with PSMB5 mutations resulting in Ala49Thr, Met45Val, and Cys52Phe substitutions. Sequence analysis of the bortezomib-binding pocket in the _5-subunit revealed Ala49Thr, Met45Val and Cys52Phe substitutions that were not previously described in solid tumors. acquired bortezomib resistance in NSCLC is associated with proteasome subunit overexpression and emergence of mutant _5-subunits that likely compromise bortezomib binding. _-Subunit-specific proteasome inhibitors, however, can efficiently bypass this resistance modality.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'C63#', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'208', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22027222', evidence_text:\"Our modeling results indicate that the Cys63Phe mutation affects the position of the helix and therefore likely affects the binding of bortezomib to B5.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'M45#', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'209', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22027222', evidence_text:\"Moreover, sequence analysis of the bortezomib-binding pocket encoded by exon 2 of the PSMB5 gene revealed Ala49Thr, Met45Val and Cys52Phe mutations which have never been described for solid tumor cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'C52F', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'210', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22027222', evidence_text:\"This study confirmed acquired bortezomib resistance in NSCLC cell lines is associated with PSMB5 mutations resulting in Ala49Thr, Met45Val, and Cys52Phe substitutions. Sequence analysis of the bortezomib-binding pocket in the _5-subunit revealed Ala49Thr, Met45Val and Cys52Phe substitutions that were not previously described in solid tumors. acquired bortezomib resistance in NSCLC is associated with proteasome subunit overexpression and emergence of mutant _5-subunits that likely compromise bortezomib binding. _-Subunit-specific proteasome inhibitors, however, can efficiently bypass this resistance modality.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'C63F', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'211', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22027222', evidence_text:\"Our modeling results indicate that the Cys63Phe mutation affects the position of the helix and therefore likely affects the binding of bortezomib to B5.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'M45V', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'212', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22027222', evidence_text:\"Moreover, sequence analysis of the bortezomib-binding pocket encoded by exon 2 of the PSMB5 gene revealed Ala49Thr, Met45Val and Cys52Phe mutations which have never been described for solid tumor cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'R24#', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'213', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22216088', evidence_text:\"Two novel and distinct mutations in the _5 subunit, Cys63Phe, located distal to the binding site in a helix critical for drug binding, and Arg24Cys, found in the propeptide region were found in all resistant clones.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'R24C', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'214', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22216088', evidence_text:\"Two novel and distinct mutations in the _5 subunit, Cys63Phe, located distal to the binding site in a helix critical for drug binding, and Arg24Cys, found in the propeptide region were found in all resistant clones.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'A108#', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'215', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22327403', evidence_text:\"We found that stable expression of GFP-tagged PSMB5, carry- ing either the M104V or A108T mutation, suppressed bortezomib sensitivity in an independent cell line (hTERT-RPE1; Fig. 2d). This result is consistent with previous reports that A108T confers borte- zomib resistance18,19. As both mutations in PSMB5 map to the drug's binding site, we hypothesize that they directly suppress drug inter-actions\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'M104#', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'216', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22327403', evidence_text:\"We found that stable expression of GFP-tagged PSMB5, carry- ing either the M104V or A108T mutation, suppressed bortezomib sensitivity in an independent cell line (hTERT-RPE1; Fig. 2d). This result is consistent with previous reports that A108T confers borte- zomib resistance18,19. As both mutations in PSMB5 map to the drug's binding site, we hypothesize that they directly suppress drug interactions\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'A108T', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'217', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22327403', evidence_text:\"We found that stable expression of GFP-tagged PSMB5, carry- ing either the M104V or A108T mutation, suppressed bortezomib sensitivity in an independent cell line (hTERT-RPE1; Fig. 2d). This result is consistent with previous reports that A108T confers borte- zomib resistance18,19. As both mutations in PSMB5 map to the drug's binding site, we hypothesize that they directly suppress drug inter- actions\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PSMB5', entrez:'5693'}), (a:Aberration { type:'snv', value:'M104V', gene_entrez:'5693', entity_class:'biomarker' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'218', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22327403', evidence_text:\"We found that stable expression of GFP-tagged PSMB5, carry- ing either the M104V or A108T mutation, suppressed bortezomib sensitivity in an independent cell line (hTERT-RPE1; Fig. 2d). This result is consistent with previous reports that A108T confers borte- zomib resistance18,19. As both mutations in PSMB5 map to the drug's binding site, we hypothesize that they directly suppress drug interactions\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HSPB1', entrez:'3315'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3315', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'219', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14559800', evidence_text:\"Blockade of Hsp27 overcomes bortezomib/proteasome inhibitor PS-341 resistance in lymphoma cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HSPB1', entrez:'3315'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3315', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'220', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14559800', evidence_text:\"Blocking Hsp27 using an antisense strategy restores the apoptotic response to PS-341 in DHL4 cells; conversely, ectopic expression of wild-type Hsp27 renders PS-341-sensitive DHL6 cells resistant to PS-341.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HSPB2', entrez:'3316'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3316', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'221', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14559800', evidence_text:\"Blockade of Hsp27 overcomes bortezomib/proteasome inhibitor PS-341 resistance in lymphoma cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HSPB2', entrez:'3316'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3316', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'222', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14559800', evidence_text:\"Blockade of Hsp27 overcomes bortezomib/proteasome inhibitor PS-341 resistance in lymphoma cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TRAF3', entrez:'7187'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7187', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'223', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17692805', evidence_text:\"These results suggest that constitutive activation of the noncanonical NF-_B pathway by inactivation of TRAF3 is associated with dexamethasone resistance and proteasome inhibitor sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TRAF3', entrez:'7187'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7187', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'224', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17692805', evidence_text:\"These results suggest that constitutive activation of the noncanonical NF-_B pathway by inactivation of TRAF3 is associated with dexamethasone resistance and proteasome inhibitor sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT3', entrez:'2322'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2322', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'225', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17897295', evidence_text:\"All AMLs with mutated FLT3 were in the bortezomib-sensitive group.  |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATF4', entrez:'468'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'468', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'226', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19417138', evidence_text:\"We show that ATF4 and LC3B play a critical role in activating autophagy and protecting cells from bortezomib-induced cell death.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATF4', entrez:'468'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'468', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'227', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19417138', evidence_text:\"We show that ATF4 and LC3B play a critical role in activating autophagy and protecting cells from bortezomib-induced cell death.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GPR78', entrez:'27201'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'27201', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'228', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19713465', evidence_text:\"Recombinant GRP-78 conferred bortezomib resistance to endothelial cells and OPM-2 myeloma cells. Knockdown of GRP78 gene expression in tumor cells and immunodepletion of GRP-78 protein from tumor cell supernatants restored bortezomib sensitivity. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GPR78', entrez:'27201'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'27201', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'229', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19713465', evidence_text:\"Recombinant GRP-78 conferred bortezomib resistance to endothelial cells and OPM-2 myeloma cells. Knockdown of GRP78 gene expression in tumor cells and immunodepletion of GRP-78 protein from tumor cell supernatants restored bortezomib sensitivity. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GNA12', entrez:'2768'}), (a:Aberration { type:'snv', value:'Q229#', gene_entrez:'2768', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'230', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20478922', evidence_text:\"Overexpression of an active mutant of G_12 (G_12QL) or G_13 (G_13QL) diminished the ability of bortezomib to induce cytotoxicity in Huh7 cells.  Our findings demonstrate that the cytotoxic sensitivity of hepatocellular carcinomas to bortezomib is closely linked to the expression levels of G_12/13, implying that these G proteins may enhance resistance of cancer cells to bortezomib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GNA13', entrez:'10672'}), (a:Aberration { type:'snv', value:'Q226#', gene_entrez:'10672', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'231', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20478922', evidence_text:\"Overexpression of an active mutant of G_12 (G_12QL) or G_13 (G_13QL) diminished the ability of bortezomib to induce cytotoxicity in Huh7 cells.  Our findings demonstrate that the cytotoxic sensitivity of hepatocellular carcinomas to bortezomib is closely linked to the expression levels of G_12/13, implying that these G proteins may enhance resistance of cancer cells to bortezomib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GNA12', entrez:'2768'}), (a:Aberration { type:'snv', value:'Q229L', gene_entrez:'2768', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'232', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20478922', evidence_text:\"Overexpression of an active mutant of G_12 (G_12QL) or G_13 (G_13QL) diminished the ability of bortezomib to induce cytotoxicity in Huh7 cells.  Our findings demonstrate that the cytotoxic sensitivity of hepatocellular carcinomas to bortezomib is closely linked to the expression levels of G_12/13, implying that these G proteins may enhance resistance of cancer cells to bortezomib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GNA13', entrez:'10672'}), (a:Aberration { type:'snv', value:'Q226L', gene_entrez:'10672', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'233', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20478922', evidence_text:\"Overexpression of an active mutant of G_12 (G_12QL) or G_13 (G_13QL) diminished the ability of bortezomib to induce cytotoxicity in Huh7 cells.  Our findings demonstrate that the cytotoxic sensitivity of hepatocellular carcinomas to bortezomib is closely linked to the expression levels of G_12/13, implying that these G proteins may enhance resistance of cancer cells to bortezomib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HSP90AA1', entrez:'3320'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3320', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'234', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21106982', evidence_text:\"In bortezomib-resistant cells, both BiP/Grp78 knockdown and cell pretreatment with the Hsp90 inhibitor of the ansamycin class, IPI-504, led to synergistic induction of apoptotic cell death when combined with bortezomib. Cell exposure to the IPI-504-bortezomib combination provoked the dissociation of Hsp90/BiP complexes, leading to BiP/Grp78 depletion, inhibition of unfolded protein response, and promotion of NOXA-mediated mitochondrial depolarization. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HSP90AA1', entrez:'3320'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3320', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'235', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21106982', evidence_text:\"In bortezomib-resistant cells, both BiP/Grp78 knockdown and cell pretreatment with the Hsp90 inhibitor of the ansamycin class, IPI-504, led to synergistic induction of apoptotic cell death when combined with bortezomib. Cell exposure to the IPI-504-bortezomib combination provoked the dissociation of Hsp90/BiP complexes, leading to BiP/Grp78 depletion, inhibition of unfolded protein response, and promotion of NOXA-mediated mitochondrial depolarization. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATG7', entrez:'10533'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'10533', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'236', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21270111', evidence_text:\"Knockdown of Beclin-1 or ATG7 sensitized B-RAF wild-type cells to fenretinide- or bortezomib-induced cell death, demonstrating a pro-survival function of autophagy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATG7', entrez:'10533'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'10533', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'237', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21270111', evidence_text:\"Knockdown of Beclin-1 or ATG7 sensitized B-RAF wild-type cells to fenretinide- or bortezomib-induced cell death, demonstrating a pro-survival function of autophagy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BECN1', entrez:'8678'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'8678', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'238', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21270111', evidence_text:\"Knockdown of Beclin-1 or ATG7 sensitized B-RAF wild-type cells to fenretinide- or bortezomib-induced cell death, demonstrating a pro-survival function of autophagy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BECN1', entrez:'8678'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'8678', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'239', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21270111', evidence_text:\"Knockdown of Beclin-1 or ATG7 sensitized B-RAF wild-type cells to fenretinide- or bortezomib-induced cell death, demonstrating a pro-survival function of autophagy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'K650#', gene_entrez:'2261', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'240', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21273588', evidence_text:\"This study indicated that FGFR3K650E is associated with bortezomib sensitivity in malignant plasma cells via ER stress pathways\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'K650E', gene_entrez:'2261', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'241', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21273588', evidence_text:\"This study indicated that FGFR3K650E is associated with bortezomib sensitivity in malignant plasma cells via ER stress pathways\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDK5', entrez:'1020'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'1020', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'242', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21289309', evidence_text:\"Viral shRNA knockdown of CDK5 consistently sensitized 5 genetically variable MM cell lines to proteasome inhibitors (bortezomib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDK5', entrez:'1020'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'1020', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'243', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21289309', evidence_text:\"Viral shRNA knockdown of CDK5 consistently sensitized 5 genetically variable MM cell lines to proteasome inhibitors (bortezomib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RRAD', entrez:'6236'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6236', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'244', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22658652', evidence_text:\"Microarray analysis revealed that the mRNA levels of Rad (Ras associated with diabetes) were higher in the bortezomib-resistant Jurkat (Jurkat-R) cells than in the parental control cells.  Rad knockdown overcame bortezomib resistance and induced mitochondrial apoptosis via Noxa/Bcl-2 modulation. Additional analyses revealed that Rad is over expressed in leukemia/lymphoma cell lines, and enhances resistance to bortezomib-induced apoptosis. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RRAD', entrez:'6236'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6236', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'245', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22658652', evidence_text:\"Microarray analysis revealed that the mRNA levels of Rad (Ras associated with diabetes) were higher in the bortezomib-resistant Jurkat (Jurkat-R) cells than in the parental control cells.  Rad knockdown overcame bortezomib resistance and induced mitochondrial apoptosis via Noxa/Bcl-2 modulation. Additional analyses revealed that Rad is over expressed in leukemia/lymphoma cell lines, and enhances resistance to bortezomib-induced apoptosis. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGF1', entrez:'3479'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3479', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'246', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22932796', evidence_text:\"Exogenous IGF-1 reduced cellular sensitivity to bortezomib, whereas pharmacologic or small hairpin RNA-mediated IGF-1R suppression enhanced bortezomib sensitivity in cell lines and patient samples. In vitro studies with OSI-906, a clinically relevant dual IGF-1R and insulin receptor inhibitor, showed it acted synergistically with bortezomib, and potently resensitized bortezomib-resistant cell lines and patient samples to bortezomib. Importantly, OSI-906 in combination with bortezomib also overcame bortezomib resistance in an in vivo model of myeloma. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGF1', entrez:'3479'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3479', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'247', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22932796', evidence_text:\"Exogenous IGF-1 reduced cellular sensitivity to bortezomib, whereas pharmacologic or small hairpin RNA-mediated IGF-1R suppression enhanced bortezomib sensitivity in cell lines and patient samples. In vitro studies with OSI-906, a clinically relevant dual IGF-1R and insulin receptor inhibitor, showed it acted synergistically with bortezomib, and potently resensitized bortezomib-resistant cell lines and patient samples to bortezomib. Importantly, OSI-906 in combination with bortezomib also overcame bortezomib resistance in an in vivo model of myeloma. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGFR1', entrez:'443515'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'443515', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'248', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22932796', evidence_text:\"Exogenous IGF-1 reduced cellular sensitivity to bortezomib, whereas pharmacologic or small hairpin RNA-mediated IGF-1R suppression enhanced bortezomib sensitivity in cell lines and patient samples. In vitro studies with OSI-906, a clinically relevant dual IGF-1R and insulin receptor inhibitor, showed it acted synergistically with bortezomib, and potently resensitized bortezomib-resistant cell lines and patient samples to bortezomib. Importantly, OSI-906 in combination with bortezomib also overcame bortezomib resistance in an in vivo model of myeloma. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGFR1', entrez:'443515'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'443515', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'249', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22932796', evidence_text:\"Exogenous IGF-1 reduced cellular sensitivity to bortezomib, whereas pharmacologic or small hairpin RNA-mediated IGF-1R suppression enhanced bortezomib sensitivity in cell lines and patient samples. In vitro studies with OSI-906, a clinically relevant dual IGF-1R and insulin receptor inhibitor, showed it acted synergistically with bortezomib, and potently resensitized bortezomib-resistant cell lines and patient samples to bortezomib. Importantly, OSI-906 in combination with bortezomib also overcame bortezomib resistance in an in vivo model of myeloma. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MIR33B', entrez:'693120'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'693120', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'250', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22983447', evidence_text:\"Overexpression of miR33b decreased MM cell viability, migration, colony formation, and increased apoptosis and sensitivity of MM cells to MLN2238 treatment. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MIR33B', entrez:'693120'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'693120', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'251', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22983447', evidence_text:\"Overexpression of miR33b decreased MM cell viability, migration, colony formation, and increased apoptosis and sensitivity of MM cells to MLN2238 treatment. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIM1', entrez:'5292'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5292', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'252', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22983447', evidence_text:\"Moreover, PIM-1 overexpression led to significant abrogation of miR33b- or MLN2238-induced cell death.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIM1', entrez:'5292'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5292', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'253', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22983447', evidence_text:\"Moreover, PIM-1 overexpression led to significant abrogation of miR33b- or MLN2238-induced cell death.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HSPA1B', entrez:'3304'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3304', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'254', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23874968', evidence_text:\"Overexpression of Hsp72 promoted bortezomib resistance in the UM-UC10 and UM-UC13 cells, whereas transient knockdown of HSPA1B further sensitized these cells to bortezomib, and exposure to the chemical HSF1 inhibitor KNK-437 promoted bortezomib sensitivity in the 253J B-V cells. Finally, shRNA-mediated stable knockdown of Hsp72 in 253J B-V promoted sensitivity to bortezomib in vitro and in tumor xenografts in vivo. Together, our results provide proof-of-concept for using Hsp72 inhibitors to promote bortezomib sensitivity in bladder cancers and suggest that selective targeting of HSPA1B could produce synthetic lethality in tumors that display\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HSPA1B', entrez:'3304'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3304', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'255', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23874968', evidence_text:\"Overexpression of Hsp72 promoted bortezomib resistance in the UM-UC10 and UM-UC13 cells, whereas transient knockdown of HSPA1B further sensitized these cells to bortezomib, and exposure to the chemical HSF1 inhibitor KNK-437 promoted bortezomib sensitivity in the 253J B-V cells. Finally, shRNA-mediated stable knockdown of Hsp72 in 253J B-V promoted sensitivity to bortezomib in vitro and in tumor xenografts in vivo. Together, our results provide proof-of-concept for using Hsp72 inhibitors to promote bortezomib sensitivity in bladder cancers and suggest that selective targeting of HSPA1B could produce synthetic lethality in tumors that display\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XBP1', entrez:'7494'}), (a:Aberration { type:'snv', value:'L167#', gene_entrez:'7494', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'256', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"constitutive expression of the mutant XBP1s-P326R or XBP1u-L167I CDS in MM cells failed to resen- sitize shXBP1-treated MM cells to BTZ, indicating that both mutations are functionally inactivating and promote BTZ resis- tance. These studies indicate that primary MM tumors tolerate inactivating mutations in XBP1 and that PI therapy may exert a selective pressure for the emergence of cells with suppression of this pathway.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XBP1', entrez:'7494'}), (a:Aberration { type:'snv', value:'P326#', gene_entrez:'7494', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'257', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"constitutive expression of the mutant XBP1s-P326R or XBP1u-L167I CDS in MM cells failed to resen- sitize shXBP1-treated MM cells to BTZ, indicating that both mutations are functionally inactivating and promote BTZ resis- tance. These studies indicate that primary MM tumors tolerate inactivating mutations in XBP1 and that PI therapy may exert a selective pressure for the emergence of cells with suppression of this pathway.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERN1', entrez:'2081'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2081', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'258', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"Strikingly, however, loss of Xbp1 induced BTZ resistance in all six MM cell lines tested, mirroring Ire1 suppression \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERN1', entrez:'2081'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2081', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'259', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"Strikingly, however, loss of Xbp1 induced BTZ resistance in all six MM cell lines tested, mirroring Ire1 suppression \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XBP1', entrez:'7494'}), (a:Aberration { type:'snv', value:'L167I', gene_entrez:'7494', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'260', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"constitutive expression of the mutant XBP1s-P326R or XBP1u-L167I CDS in MM cells failed to resen- sitize shXBP1-treated MM cells to BTZ, indicating that both mutations are functionally inactivating and promote BTZ resis- tance. These studies indicate that primary MM tumors tolerate inactivating mutations in XBP1 and that PI therapy may exert a selective pressure for the emergence of cells with suppression of this pathway.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XBP1', entrez:'7494'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7494', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'261', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"Strikingly, however, loss of Xbp1 induced BTZ resistance in all six MM cell lines tested, mirroring Ire1 suppression \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XBP1', entrez:'7494'}), (a:Aberration { type:'snv', value:'P326R', gene_entrez:'7494', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'262', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"constitutive expression of the mutant XBP1s-P326R or XBP1u-L167I CDS in MM cells failed to resen- sitize shXBP1-treated MM cells to BTZ, indicating that both mutations are functionally inactivating and promote BTZ resis- tance. These studies indicate that primary MM tumors tolerate inactivating mutations in XBP1 and that PI therapy may exert a selective pressure for the emergence of cells with suppression of this pathway.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XBP1', entrez:'7494'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7494', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'263', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"Strikingly, however, loss of Xbp1 induced BTZ resistance in all six MM cell lines tested, mirroring Ire1 suppression \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CCNE1', entrez:'898'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'898', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'264', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24218601', evidence_text:\"Using data from a genome-wide shRNA synthetic lethal screen, we show that BRCA1 and members of the ubiquitin pathway are selectively required in cancers that harbor CCNE1 amplification. Furthermore, we show specific sensitivity of CCNE1-amplified tumor cells to the proteasome inhibitor bortezomib. These findings provide an explanation for the observed mutual exclusivity of CCNE1 amplification and BRCA1/2 loss in HGSC and suggest a unique therapeutic approach for treatment-resistant CCNE1-amplified tumors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CCNE1', entrez:'898'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'898', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'265', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24218601', evidence_text:\"Using data from a genome-wide shRNA synthetic lethal screen, we show that BRCA1 and members of the ubiquitin pathway are selectively required in cancers that harbor CCNE1 amplification. Furthermore, we show specific sensitivity of CCNE1-amplified tumor cells to the proteasome inhibitor bortezomib. These findings provide an explanation for the observed mutual exclusivity of CCNE1 amplification and BRCA1/2 loss in HGSC and suggest a unique therapeutic approach for treatment-resistant CCNE1-amplified tumors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'G12#', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'266', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.  |Annotator Note:  SNVs at this position are associated with activation and reduced drug sensitivity. However, not all changes at this position may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'G13#', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'267', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.  |Annotator Note:  SNVs at this position are associated with activation and reduced drug sensitivity. However, not all changes at this position may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61#', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'268', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.  |Annotator Note:  SNVs at this position are associated with activation and reduced drug sensitivity. However, not all changes at this position may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'G12C', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'269', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'G13D', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'270', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'G13R', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'271', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61H', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'272', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61K', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'273', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61L', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'274', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61R', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'275', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGFR1', entrez:'443515'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'443515', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'276', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22932796', evidence_text:\"Exogenous IGF-1 reduced cellular sensitivity to bortezomib, whereas pharmacologic or small hairpin RNA-mediated IGF-1R suppression enhanced bortezomib sensitivity in cell lines and patient samples. In vitro studies with OSI-906, a clinically relevant dual IGF-1R and insulin receptor inhibitor, showed it acted synergistically with bortezomib, and potently resensitized bortezomib-resistant cell lines and patient samples to bortezomib. Importantly, OSI-906 in combination with bortezomib also overcame bortezomib resistance in an in vivo model of myeloma.  |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATG7', entrez:'10533'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'10533', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'277', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21270111', evidence_text:\"Knockdown of Beclin-1 or ATG7 sensitized B-RAF wild-type cells to fenretinide- or bortezomib-induced cell death, demonstrating a pro-survival function of autophagy. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BECN1', entrez:'8678'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'8678', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'278', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21270111', evidence_text:\"Knockdown of Beclin-1 or ATG7 sensitized B-RAF wild-type cells to fenretinide- or bortezomib-induced cell death, demonstrating a pro-survival function of autophagy. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIM1', entrez:'5292'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5292', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'279', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22983447', evidence_text:\"Moreover, PIM-1 overexpression led to significant abrogation of miR33b- or MLN2238-induced cell death. |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'280', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24335104', evidence_text:\"NRAS mutations were associated with a significantly lower response rate to single-agent bortezomib (7% vs 53% in patients with mutant vs wild-type NRAS, P = .00116, Bonferroni-corrected P = .016), as well as shorter time to progression in bortezomib-treated patients (P = .0058, Bonferroni-corrected P = .012). In 74 patients treated with bortezomib in the APEX trial, 1 of 15 patients with NRAS mutant tumors responded (7%, 95% CI: 0%-19%) compared with 31 of 59 (53%, 95% CI: 40%-65%) with wild-type NRAS. The NRAS mutant cohort included 12 cases within codon 61 (1 response) and 3 within codon 12 (no responses). |Evidence Level: Clinical Retrospective. |Disease:  Multiple Myeloma.  |Annotator Note:  Drug class relation with erlotinib. SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GNA12', entrez:'2768'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2768', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'281', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20478922', evidence_text:\"Overexpression of an active mutant of G_12 (G_12QL) or G_13 (G_13QL) diminished the ability of bortezomib to induce cytotoxicity in Huh7 cells.  Our findings demonstrate that the cytotoxic sensitivity of hepatocellular carcinomas to bortezomib is closely linked to the expression levels of G_12/13, implying that these G proteins may enhance resistance of cancer cells to bortezomib.  |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GNA13', entrez:'10672'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'10672', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'282', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20478922', evidence_text:\"Overexpression of an active mutant of G_12 (G_12QL) or G_13 (G_13QL) diminished the ability of bortezomib to induce cytotoxicity in Huh7 cells.  Our findings demonstrate that the cytotoxic sensitivity of hepatocellular carcinomas to bortezomib is closely linked to the expression levels of G_12/13, implying that these G proteins may enhance resistance of cancer cells to bortezomib.  |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XBP1', entrez:'7494'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7494', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'283', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"Strikingly, however, loss of Xbp1 induced BTZ resistance in all six MM cell lines tested, mirroring Ire1 suppression. |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERN1', entrez:'2081'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2081', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'284', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24029229', evidence_text:\"Strikingly, however, loss of Xbp1 induced BTZ resistance in all six MM cell lines tested, mirroring Ire1 suppression. |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TRAF3', entrez:'7187'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7187', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'285', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17692805', evidence_text:\"These results suggest that constitutive activation of the noncanonical NF-_B pathway by inactivation of TRAF3 is associated with dexamethasone resistance and proteasome inhibitor sensitivity. |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR3', entrez:'2261'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2261', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'286', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21273588', evidence_text:\"This study indicated that FGFR3 K650E is associated with bortezomib sensitize in malignant plasma cells via ER stress pathways. |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDK5', entrez:'1020'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'1020', entity_class:'modifier' }), (d:Drug {name:'bortezomib'} ) Merge (r:Rule {rulenum:'287', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21289309', evidence_text:\"Viral shRNA knockdown of CDK5 consistently sensitized 5 genetically variable MM cell lines to proteasome inhibitors (bortezomib). |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SRC', entrez:'6714'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6714', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'288', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17181170', evidence_text:\"Compound 1 (SKI-606, bosutinib), a 7-alkoxy-4-[(2,4-dichloro-5-methoxyphenyl)amino]-3-quinolinecarbonitrile, is a potent inhibitor of Src kinase activity. |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SRC', entrez:'6714'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6714', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'289', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17181170', evidence_text:\"Compound 1 (SKI-606, bosutinib), a 7-alkoxy-4-[(2,4-dichloro-5-methoxyphenyl)amino]-3-quinolinecarbonitrile, is a potent inhibitor of Src kinase activity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SRC', entrez:'6714'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6714', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'290', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17181170', evidence_text:\"Compound 1 (SKI-606, bosutinib), a 7-alkoxy-4-[(2,4-dichloro-5-methoxyphenyl)amino]-3-quinolinecarbonitrile, is a potent inhibitor of Src kinase activity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LYN', entrez:'4067'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4067', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'291', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21680801', evidence_text:\"we also applied bosutinib (SKI-606), a drug that (unlike dasatinib) deactivates Lyn and Btk but does not block KIT activation.39,40 In Western blot experiments, we were able to show that bosutinib (0.01-5?M) down-regulates expression of p-Lyn and p-Btk in HMC-1 cells, but does not deactivate KIT D816V\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LYN', entrez:'4067'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4067', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'292', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21680801', evidence_text:\"we also applied bosutinib (SKI-606), a drug that (unlike dasatinib) deactivates Lyn and Btk but does not block KIT activation.39,40 In Western blot experiments, we were able to show that bosutinib (0.01-5?M) down-regulates expression of p-Lyn and p-Btk in HMC-1 cells, but does not deactivate KIT D816V\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4233', entity_class:'modifier' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'293', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20406949', evidence_text:\"Gastric cancer cell lines positive for the activation of MET were resistant to dasatinib.  Our results suggest that c-Src is a promising target for the treatment of gastric cancer and that analysis of MET amplification might optimize patient selection for treatment with c-Src inhibitors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4233', entity_class:'modifier' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'294', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20406949', evidence_text:\"Gastric cancer cell lines positive for the activation of MET were resistant to dasatinib.  Our results suggest that c-Src is a promising target for the treatment of gastric cancer and that analysis of MET amplification might optimize patient selection for treatment with c-Src inhibitors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STK11', entrez:'6794'}), (a:Aberration { type:'exp', value:'loss', gene_entrez:'6794', entity_class:'modifier' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'295', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20541700', evidence_text:\"treatment of H358 cells with dasatinib or PF573228 resulted in a dose-dependent decrease in cell adhesion that was more prominent in H358 cells with LKB1 knockdown (Figure 5B). Similar results were obtained when H358 cells with or without LKB1 expression were examined for the effects of dasatinib and PF573228 on cell migration (Figure 5C). These observations support the notion that the activation of SRC and FAK due to LKB1 loss promotes dependence upon these signaling pathways for cell migration and adhesion to extracellular matrix components as well as migration.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STK11', entrez:'6794'}), (a:Aberration { type:'cnv', value:'under', gene_entrez:'6794', entity_class:'modifier' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'296', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20541700', evidence_text:\"treatment of H358 cells with dasatinib or PF573228 resulted in a dose-dependent decrease in cell adhesion that was more prominent in H358 cells with LKB1 knockdown (Figure 5B). Similar results were obtained when H358 cells with or without LKB1 expression were examined for the effects of dasatinib and PF573228 on cell migration (Figure 5C). These observations support the notion that the activation of SRC and FAK due to LKB1 loss promotes dependence upon these signaling pathways for cell migration and adhesion to extracellular matrix components as well as migration.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NEDD9', entrez:'4739'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4739', entity_class:'modifier' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'297', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20940402', evidence_text:\"In spite of their aggressive phenotype, MMTV-PyVT;Nedd9(-/-) cells persistently have low levels of Src activation and are hypersensitive to the Src kinase inhibitor dasatinib. The absence of NEDD9 causes a persistent deficiency in Src activation and influences the susceptibility of mammary cancer cell lines to growth inhibition by dasatinib, a drug targeting Src kinase. These studies in sum should significantly inform the use of NEDD9 as a biomarker for clinical response\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NEDD9', entrez:'4739'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4739', entity_class:'modifier' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'298', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20940402', evidence_text:\"In spite of their aggressive phenotype, MMTV-PyVT;Nedd9(-/-) cells persistently have low levels of Src activation and are hypersensitive to the Src kinase inhibitor dasatinib. The absence of NEDD9 causes a persistent deficiency in Src activation and influences the susceptibility of mammary cancer cell lines to growth inhibition by dasatinib, a drug targeting Src kinase. These studies in sum should significantly inform the use of NEDD9 as a biomarker for clinical response\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BTK', entrez:'695'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'695', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'299', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21680801', evidence_text:\"we also applied bosutinib (SKI-606), a drug that (unlike dasatinib) deactivates Lyn and Btk but does not block KIT activation.39,40 In Western blot experiments, we were able to show that bosutinib (0.01-5?M) down-regulates expression of p-Lyn and p-Btk in HMC-1 cells, but does not deactivate KIT D816V\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BTK', entrez:'695'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'695', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'300', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21680801', evidence_text:\"we also applied bosutinib (SKI-606), a drug that (unlike dasatinib) deactivates Lyn and Btk but does not block KIT activation.39,40 In Western blot experiments, we were able to show that bosutinib (0.01-5?M) down-regulates expression of p-Lyn and p-Btk in HMC-1 cells, but does not deactivate KIT D816V\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AXL', entrez:'558'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'558', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'301', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18339872', evidence_text:\"AXL is a target of the Src/Abl inhibitor SKI-606. In vitro kinase assay showing AXL evidence. (Pfizer team suggest this might translate clinically.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AXL', entrez:'558'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'558', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'302', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18339872', evidence_text:\"AXL is a target of the Src/Abl inhibitor SKI-606. In vitro kinase assay showing AXL evidence. (Pfizer team suggest this might translate clinically.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'303', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/12543790', evidence_text:\"|Annotator Note:  Is target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'304', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/12543790', evidence_text:\"Together, these experiments indicate that SKI-606 is an Abl kinase inhibitor.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'305', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/12543790', evidence_text:\"Together, these experiments indicate that SKI-606 is an Abl kinase inhibitor.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'D276#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'306', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17114238', evidence_text:\"SKI-606 retained activity in cells where resistance to imatinib was caused by BCR-ABL gene amplification and in three of four Bcr-Abl point mutants tested. In vivo experiments confirmed SKI-606 activity in models where resistance was not caused by mutations as well as in cells carrying the Y253F, E255K, and D276G mutations.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'D276G', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'307', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17114238', evidence_text:\"SKI-606 retained activity in cells where resistance to imatinib was caused by BCR-ABL gene amplification and in three of four Bcr-Abl point mutants tested. In vivo experiments confirmed SKI-606 activity in models where resistance was not caused by mutations as well as in cells carrying the Y253F, E255K, and D276G mutations.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Y253F', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'308', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17114238', evidence_text:\"SKI-606 retained activity in cells where resistance to imatinib was caused by BCR-ABL gene amplification and in three of four Bcr-Abl point mutants tested. In vivo experiments confirmed SKI-606 activity in models where resistance was not caused by mutations as well as in cells carrying the Y253F, E255K, and D276G mutations.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'E279#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'309', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'G398#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'310', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Q252#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'311', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'E255#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'312', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F486#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'313', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'G250#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'314', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'L248#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'315', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'H306#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'316', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'H396#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'317', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'L384#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'318', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'M351#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'319', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'E255K', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'320', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'E255V', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'321', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'E279K', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'322', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F486S', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'323', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'G250E', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'324', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'G398R', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'325', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'H306R', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'326', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'H396P', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'327', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'L248V', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'328', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'L384M', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'329', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'M351T', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'330', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Q252H', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'331', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19075254', evidence_text:\"Fig1 listing of mutations and response to bosutinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'L284#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'332', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23044928', evidence_text:\"We report the discovery of three new BCR/ABL mutations, L248R, T315V, and F317R identified in two patients with CML (L248R and T315V) and in one patient with Ph+ acute lymphoblastic leukemia (ALL) (F317R). L248R and T315V showed high resistance to imatinib, bosutinib, dasatinib, and nilotinib, intermediate resistance to ponatinib, but were sensitive to DCC-2036. Interestingly, F317R showed a moderate resistance to imatinib and nilotinib, but is resistant/highly resistant to dasatinib, bosutinib, ponatinib, and DCC-2036.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F317R ', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'333', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23044928', evidence_text:\"We report the discovery of three new BCR/ABL mutations, L248R, T315V, and F317R identified in two patients with CML (L248R and T315V) and in one patient with Ph+ acute lymphoblastic leukemia (ALL) (F317R). L248R and T315V showed high resistance to imatinib, bosutinib, dasatinib, and nilotinib, intermediate resistance to ponatinib, but were sensitive to DCC-2036. Interestingly, F317R showed a moderate resistance to imatinib and nilotinib, but is resistant/highly resistant to dasatinib, bosutinib, ponatinib, and DCC-2036.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'L284R', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'334', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23044928', evidence_text:\"We report the discovery of three new BCR/ABL mutations, L248R, T315V, and F317R identified in two patients with CML (L248R and T315V) and in one patient with Ph+ acute lymphoblastic leukemia (ALL) (F317R). L248R and T315V showed high resistance to imatinib, bosutinib, dasatinib, and nilotinib, intermediate resistance to ponatinib, but were sensitive to DCC-2036. Interestingly, F317R showed a moderate resistance to imatinib and nilotinib, but is resistant/highly resistant to dasatinib, bosutinib, ponatinib, and DCC-2036.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'T315V', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'335', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23044928', evidence_text:\"L248R and T315V showed high resistance to imatinib, bosutinib, dasatinib, and nilotinib, intermediate resistance to ponatinib, but were sensitive to DCC-2036. Interestingly, F317R showed a moderate resistance to imatinib and nilotinib, but is resistant/highly resistant to dasatinib, bosutinib, ponatinib, and DCC-2036.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F317#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'336', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"A complete hematological response and a major cytogenetic response were observed broadly across the BCR-ABL mutants, including individuals harboring the dasatinib-resistant F317L, the nilotinib-resistant Y253H, and F359C/I/V mutations, but not T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Y253#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'337', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"A complete hematological response and a major cytogenetic response were observed broadly across the BCR-ABL mutants, including individuals harboring the dasatinib-resistant F317L, the nilotinib-resistant Y253H, and F359C/I/V mutations, but not T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'T315#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'338', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"bosutinib effectively inhibits wild-type BCR-ABL and most imatinib-resistant BCR-ABL mutations except for V299L and T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'V299#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'339', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"bosutinib effectively inhibits wild-type BCR-ABL and most imatinib-resistant BCR-ABL mutations except for V299L and T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F359#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'340', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"A complete hematological response and a major cytogenetic response were observed broadly across the BCR-ABL mutants, including individuals harboring the dasatinib-resistant F317L, the nilotinib-resistant Y253H, and F359C/I/V mutations, but not T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F317L', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'341', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"A complete hematological response and a major cytogenetic response were observed broadly across the BCR-ABL mutants, including individuals harboring the dasatinib-resistant F317L, the nilotinib-resistant Y253H, and F359C/I/V mutations, but not T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F359C', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'342', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"A complete hematological response and a major cytogenetic response were observed broadly across the BCR-ABL mutants, including individuals harboring the dasatinib-resistant F317L, the nilotinib-resistant Y253H, and F359C/I/V mutations, but not T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F359I', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'343', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"A complete hematological response and a major cytogenetic response were observed broadly across the BCR-ABL mutants, including individuals harboring the dasatinib-resistant F317L, the nilotinib-resistant Y253H, and F359C/I/V mutations, but not T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F359V', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'344', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"A complete hematological response and a major cytogenetic response were observed broadly across the BCR-ABL mutants, including individuals harboring the dasatinib-resistant F317L, the nilotinib-resistant Y253H, and F359C/I/V mutations, but not T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'T315I', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'345', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"bosutinib effectively inhibits wild-type BCR-ABL and most imatinib-resistant BCR-ABL mutations except for V299L and T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'V299L', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'346', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"bosutinib effectively inhibits wild-type BCR-ABL and most imatinib-resistant BCR-ABL mutations except for V299L and T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Y253H', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'347', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23493838', evidence_text:\"A complete hematological response and a major cytogenetic response were observed broadly across the BCR-ABL mutants, including individuals harboring the dasatinib-resistant F317L, the nilotinib-resistant Y253H, and F359C/I/V mutations, but not T315I. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STK11', entrez:'6794'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6794', entity_class:'modifier' }), (d:Drug {name:'bosutinib'} ) Merge (r:Rule {rulenum:'348', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20541700', evidence_text:\"treatment of H358 cells with dasatinib or PF573228 resulted in a dose-dependent decrease in cell adhesion that was more prominent in H358 cells with LKB1 knockdown (Figure 5B). Similar results were obtained when H358 cells with or without LKB1 expression were examined for the effects of dasatinib and PF573228 on cell migration (Figure 5C). These observations support the notion that the activation of SRC and FAK due to LKB1 loss promotes dependence upon these signaling pathways for cell migration and adhesion to extracellular matrix components as well as migration. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TEK', entrez:'7010'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7010', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'349', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2). |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TEK', entrez:'7010'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7010', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'350', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TEK', entrez:'7010'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7010', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'351', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'352', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"The prinicipal targets of XL-184 are MET, VEGFR-2 and RET, but the drug is also reported to display inhibitory activity against KIT, FLT3 and TEK...Responses to XL-184 did not appear to be dependent on the RET status of individual patients [951540], [1051828].\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'353', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"The prinicipal targets of XL-184 are MET, VEGFR-2 and RET, but the drug is also reported to display inhibitory activity against KIT, FLT3 and TEK...Responses to XL-184 did not appear to be dependent on the RET status of individual patients [951540], [1051828].\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'C634W', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'354', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21470995', evidence_text:\"MTC-TT reported to be derived from a sporadic MTC expressing a C634W RET mutation...XL184 was the most effective inhibitor of MTC-TT (IC50 = 0.04 _M. TPC-1 derived from a patient with PTC expressing a RET/PTC-1 rearrangement...we observed that XL184 was the most effective inhibitor of ...and TPC-1 (IC50 = 0.06 _M) proliferation.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'C634#', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'355', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21470995', evidence_text:\"MTC-TT reported to be derived from a sporadic MTC expressing a C634W RET mutation...XL184 was the most effective inhibitor of MTC-TT (IC50 = 0.04 _M. TPC-1 derived from a patient with PTC expressing a RET/PTC-1 rearrangement...we observed that XL184 was the most effective inhibitor of ...and TPC-1 (IC50 = 0.06 _M) proliferation.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'M918#', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'356', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21606412', evidence_text:\"Notably, evidence of durable tumor shrinkage or stable disease was observed in 12 of 15 patients with MTC with a somatic M918T mutation in RET (treatment duration range, 6 to 38 months), which has been shown to be a strong negative prognostic indicator for metastasis-free and overall survival.34 \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'M918T', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'357', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21606412', evidence_text:\"Notably, evidence of durable tumor shrinkage or stable disease was observed in 12 of 15 patients with MTC with a somatic M918T mutation in RET (treatment duration range, 6 to 38 months), which has been shown to be a strong negative prognostic indicator for metastasis-free and overall survival.34 \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'V804#', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'358', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23705946', evidence_text:\"cabozantinib was not active against the RET mutant V804L (IC50>5000nmol/L) previously shown to render resistance to other RET inhibitors (43). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'Y791#', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'359', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23705946', evidence_text:\"In biochemical assays, cabozantinib inhibited multiple forms of oncogenic RET kinase activity, including M918T and Y791F mutants.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'V804L', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'360', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23705946', evidence_text:\"cabozantinib was not active against the RET mutant V804L (IC50>5000nmol/L) previously shown to render resistance to other RET inhibitors (43). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'Y791F', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'361', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23705946', evidence_text:\"In biochemical assays, cabozantinib inhibited multiple forms of oncogenic RET kinase activity, including M918T and Y791F mutants.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'V804M', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'362', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23811235', evidence_text:\"We confirmed that V804M substitution confers resistance to vandetanib, motesanib and cabozantinib in vitro. It is important to note, however, that V804 mutations are not selected during treatment, but arise spontane- ously and are likely to confer primary resistance to the drugs. Presently, it is not known whether drug-resistant mutations will be acquired during therapy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RET', entrez:'5979'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5979', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'363', evidence:'direct', pmid:'http://www.onclive.com/publications/obtn/2013/July-2013/cabozantinib-Benefit-Greater-in-MTC-Patients-With-RET-and-RAS-Mutations', evidence_text:\"In patients with unknown RET mutation status, a benefit in PFS was also observed, with patients in the cabozantinib arm of this subgroup achieving a median PFS of 48 weeks compared with 13 weeks in the placebo arm (HR = 0.30; 95% CI, 0.16-0.57). - See more at: http://www.onclive.com/publications/obtn/2013/July-2013/cabozantinib-Benefit-Greater-in-MTC-Patients-With-RET-and-RAS-Mutations#sthash.zZ0zLgtp.dpuf |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'364', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"The prinicipal targets of XL-184 are MET, VEGFR-2 and RET, but the drug is also reported to display inhibitory activity against KIT, FLT3 and TEK.  |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'365', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21926191', evidence_text:\"SNU-5 and Hs746T cells harboring amplified MET (26) were the most sensitive to cabozantinib (IC50 1/4 19 and 9.9 nmol/L, respectively); however, SNU-1 and SNU-16 cells lacking MET amplification were more resistant (IC50 1/4 5,223 and 1,149 nmol/L, respectively). |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'366', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"The prinicipal targets of XL-184 are MET, VEGFR-2 and RET, but the drug is also reported to display inhibitory activity against KIT, FLT3 and TEK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'367', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2). |Annotator Note:  Moleculare alterations have not been reported with KIT and cabozantinib, but is listed because of 'is a target' relationship.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'368', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2). |Annotator Note:  Moleculare alterations have not been reported with KIT and cabozantinib, but is listed because of 'is a target' relationship.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'369', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2). |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KDR', entrez:'3791'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'3791', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'370', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"The prinicipal targets of XL-184 are MET, VEGFR-2 and RET, but the drug is also reported to display inhibitory activity against KIT, FLT3 and TEK.  |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KDR', entrez:'3791'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3791', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'371', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"The prinicipal targets of XL-184 are MET, VEGFR-2 and RET, but the drug is also reported to display inhibitory activity against KIT, FLT3 and TEK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KDR', entrez:'3791'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3791', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'372', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"The prinicipal targets of XL-184 are MET, VEGFR-2 and RET, but the drug is also reported to display inhibitory activity against KIT, FLT3 and TEK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT3', entrez:'2322'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2322', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'373', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2). |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT3', entrez:'2322'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2322', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'374', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT3', entrez:'2322'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2322', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'375', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20127563', evidence_text:\"XL-184 has also been reported to inhibit RTKs that have less well-defined roles in cancer, including FMS-like tyrosine kinase 3 (FLT3), mast/stem cell growth factor receptor (KIT or c-Kit) and endothelial-specific receptor tyrosine kinase (TEK; also known as Tie2)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLT3', entrez:'2322'}), (a:Aberration { type:'snv', value:'FLT3-ITD', gene_entrez:'2322', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'376', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21926191', evidence_text:\"In addition, BaF3 cells expressing human FLT3-ITD, an activating mutation in acute myelogenous leukemia (27), were sensitive to cabozantinib (IC50 1/4 15 nmol/L) when compared with wild-type BaF3 cells (IC50 1/4 9,641 nmol/L). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AXL', entrez:'558'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'558', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'377', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21613405', evidence_text:\"XL184 inhibits AXL, Tie2, KIT, FLT3, and RET but not PDGFR or RON (28, 29, 32).  |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AXL', entrez:'558'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'558', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'378', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21613405', evidence_text:\"XL184 inhibits AXL, Tie2, KIT, FLT3, and RET but not PDGFR or RON (28, 29, 32). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AXL', entrez:'558'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'558', entity_class:'biomarker' }), (d:Drug {name:'cabozantinib'} ) Merge (r:Rule {rulenum:'379', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21613405', evidence_text:\"XL184 inhibits AXL, Tie2, KIT, FLT3, and RET but not PDGFR or RON (28, 29, 32). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GSTP1', entrez:'2950'}), (a:Aberration { type:'snv', value:'I150V', gene_entrez:'2950', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'380', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19568750', evidence_text:\" the A-->G change of GSTP1 Ile105Val polymorphism significantly increased platinum-based chemotherapy response.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'WEE1', entrez:'7465'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7465', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'381', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22942255', evidence_text:\"Abrogation of the G2-M checkpoint by targeting CHK1 and WEE1 with kinase inhibitors has been shown to sensitize cancer cells to DNA-damaging drugs (14, 42). Kinase inhibitors against CHK1 and WEE1 are already in clinical trials but cells can develop resistance to the kinase inhibitors...Overexpression of the cell-cycle kinases WEE1 and CHK1 occurred commonly in cisplatin-resistant cells.  |Annotator Note:  SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity. Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'WEE1', entrez:'7465'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7465', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'382', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22942255', evidence_text:\"Abrogation of the G2-M checkpoint by targeting CHK1 and WEE1 with kinase inhibitors has been shown to sensitize cancer cells to DNA-damaging drugs (14, 42). Kinase inhibitors against CHK1 and WEE1 are already in clinical trials but cells can develop resistance to the kinase inhibitors...Overexpression of the cell-cycle kinases WEE1 and CHK1 occurred commonly in cisplatin-resistant cells.  |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'WEE1', entrez:'7465'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7465', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'383', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22942255', evidence_text:\"Abrogation of the G2-M checkpoint by targeting CHK1 and WEE1 with kinase inhibitors has been shown to sensitize cancer cells to DNA-damaging drugs (14, 42). Kinase inhibitors against CHK1 and WEE1 are already in clinical trials but cells can develop resistance to the kinase inhibitors...Overexpression of the cell-cycle kinases WEE1 and CHK1 occurred commonly in cisplatin-resistant cells.  |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK1', entrez:'1111'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'1111', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'384', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22942255', evidence_text:\"Abrogation of the G2-M checkpoint by targeting CHK1 and WEE1 with kinase inhibitors has been shown to sensitize cancer cells to DNA-damaging drugs (14, 42). Kinase inhibitors against CHK1 and WEE1 are already in clinical trials but cells can develop resistance to the kinase inhibitors...Overexpression of the cell-cycle kinases WEE1 and CHK1 occurred commonly in cisplatin-resistant cells.  |Annotator Note:  SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity. Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK1', entrez:'1111'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1111', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'385', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22942255', evidence_text:\"Abrogation of the G2-M checkpoint by targeting CHK1 and WEE1 with kinase inhibitors has been shown to sensitize cancer cells to DNA-damaging drugs (14, 42). Kinase inhibitors against CHK1 and WEE1 are already in clinical trials but cells can develop resistance to the kinase inhibitors...Overexpression of the cell-cycle kinases WEE1 and CHK1 occurred commonly in cisplatin-resistant cells.  |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK1', entrez:'1111'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1111', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'386', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22942255', evidence_text:\"Abrogation of the G2-M checkpoint by targeting CHK1 and WEE1 with kinase inhibitors has been shown to sensitize cancer cells to DNA-damaging drugs (14, 42). Kinase inhibitors against CHK1 and WEE1 are already in clinical trials but cells can develop resistance to the kinase inhibitors...Overexpression of the cell-cycle kinases WEE1 and CHK1 occurred commonly in cisplatin-resistant cells.  |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51D', entrez:'5892'}), (a:Aberration { type:'snv', value:'T194#', gene_entrez:'5892', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'387', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22986143', evidence_text:\"All three subjects with RAD51D mutations had a complete response to platinum based chemotherapy and are currently without evidence of disease from 16 to 109 months post diagnosis. Interestingly, the individual who is currently without disease at 109 months presented initially with stage IV disease with parenchymal liver and splenic metastases. Presumably, cancer cells with RAD51D mutations would have defective homologous recombination mediated DNA repair, as evidenced by the demonstration of in vitro sensitivity to PARP inhibitors in RAD51D deficient cells [3]. The excellent therapeutic response of our RAD51D mutated cases to platinum based chemotherapy is consistent with a defect in DNA repair by homologous recombination. |Evidence Level:  Case Study. |Disease:  Ovarian Cancer. |Annotator Note:  Mutations inferred to be sensitive to platinum therapy as discussed in text of paper. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51D', entrez:'5892'}), (a:Aberration { type:'snv', value:'R232#', gene_entrez:'5892', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'388', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22986143', evidence_text:\"All three subjects with RAD51D mutations had a complete response to platinum based chemotherapy and are currently without evidence of disease from 16 to 109 months post diagnosis. Interestingly, the individual who is currently without disease at 109 months presented initially with stage IV disease with parenchymal liver and splenic metastases. Presumably, cancer cells with RAD51D mutations would have defective homologous recombination mediated DNA repair, as evidenced by the demonstration of in vitro sensitivity to PARP inhibitors in RAD51D deficient cells [3]. The excellent therapeutic response of our RAD51D mutated cases to platinum based chemotherapy is consistent with a defect in DNA repair by homologous recombination. |Evidence Level:  Case Study. |Disease:  Ovarian Cancer. |Annotator Note:  Mutations inferred to be sensitive to platinum therapy as discussed in text of paper. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51D', entrez:'5892'}), (a:Aberration { type:'snv', value:'V28#', gene_entrez:'5892', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'389', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22986143', evidence_text:\"All three subjects with RAD51D mutations had a complete response to platinum based chemotherapy and are currently without evidence of disease from 16 to 109 months post diagnosis. Interestingly, the individual who is currently without disease at 109 months presented initially with stage IV disease with parenchymal liver and splenic metastases. Presumably, cancer cells with RAD51D mutations would have defective homologous recombination mediated DNA repair, as evidenced by the demonstration of in vitro sensitivity to PARP inhibitors in RAD51D deficient cells [3]. The excellent therapeutic response of our RAD51D mutated cases to platinum based chemotherapy is consistent with a defect in DNA repair by homologous recombination. |Evidence Level:  Case Study. |Disease:  Ovarian Cancer. |Annotator Note:  Mutations inferred to be sensitive to platinum therapy as discussed in text of paper. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51D', entrez:'5892'}), (a:Aberration { type:'snv', value:'T194L', gene_entrez:'5892', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'390', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22986143', evidence_text:\"All three subjects with RAD51D mutations had a complete response to platinum based chemotherapy and are currently without evidence of disease from 16 to 109 months post diagnosis. Interestingly, the individual who is currently without disease at 109 months presented initially with stage IV disease with parenchymal liver and splenic metastases. Presumably, cancer cells with RAD51D mutations would have defective homologous recombination mediated DNA repair, as evidenced by the demonstration of in vitro sensitivity to PARP inhibitors in RAD51D deficient cells [3]. The excellent therapeutic response of our RAD51D mutated cases to platinum based chemotherapy is consistent with a defect in DNA repair by homologous recombination. |Evidence Level:  Case Study. |Disease:  Ovarian Cancer. |Annotator Note:  Mutations inferred to be sensitive to platinum therapy as discussed in text of paper. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51D', entrez:'5892'}), (a:Aberration { type:'snv', value:'R232X', gene_entrez:'5892', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'391', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22986143', evidence_text:\"All three subjects with RAD51D mutations had a complete response to platinum based chemotherapy and are currently without evidence of disease from 16 to 109 months post diagnosis. Interestingly, the individual who is currently without disease at 109 months presented initially with stage IV disease with parenchymal liver and splenic metastases. Presumably, cancer cells with RAD51D mutations would have defective homologous recombination mediated DNA repair, as evidenced by the demonstration of in vitro sensitivity to PARP inhibitors in RAD51D deficient cells [3]. The excellent therapeutic response of our RAD51D mutated cases to platinum based chemotherapy is consistent with a defect in DNA repair by homologous recombination. |Evidence Level:  Case Study. |Disease:  Ovarian Cancer. |Annotator Note:  Mutations inferred to be sensitive to platinum therapy as discussed in text of paper. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51D', entrez:'5892'}), (a:Aberration { type:'snv', value:'V28G', gene_entrez:'5892', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'392', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22986143', evidence_text:\"All three subjects with RAD51D mutations had a complete response to platinum based chemotherapy and are currently without evidence of disease from 16 to 109 months post diagnosis. Interestingly, the individual who is currently without disease at 109 months presented initially with stage IV disease with parenchymal liver and splenic metastases. Presumably, cancer cells with RAD51D mutations would have defective homologous recombination mediated DNA repair, as evidenced by the demonstration of in vitro sensitivity to PARP inhibitors in RAD51D deficient cells [3]. The excellent therapeutic response of our RAD51D mutated cases to platinum based chemotherapy is consistent with a defect in DNA repair by homologous recombination. |Evidence Level:  Case Study. |Disease:  Ovarian Cancer. |Annotator Note:  Mutations inferred to be sensitive to platinum therapy as discussed in text of paper. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC31A1', entrez:'1317'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1317', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'393', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22962276', evidence_text:\"Because hCtr1 (SLC31A1) can also transport platinum drugs, this finding underscores the important role of hCtr1 in platinum-drug sensitivity in cancer chemotherapy. Consistent with this notion is the finding that elevated hCtr1 expression was associated with favorable treatment outcomes in cisplatin-based cancer chemotherapy. Moreover, cultured cell studies showed that elevated hCtr1 expression can be induced by depleting cellular copper levels, resulting in enhanced cisplatin uptake and its cell-killing activity. A phase I clinical trial using a combination of trientine (a copper chelator) and carboplatin has been carried out with encouraging results.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC31A1', entrez:'1317'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1317', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'394', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22962276', evidence_text:\"Because hCtr1 (SLC31A1) can also transport platinum drugs, this finding underscores the important role of hCtr1 in platinum-drug sensitivity in cancer chemotherapy. Consistent with this notion is the finding that elevated hCtr1 expression was associated with favorable treatment outcomes in cisplatin-based cancer chemotherapy. Moreover, cultured cell studies showed that elevated hCtr1 expression can be induced by depleting cellular copper levels, resulting in enhanced cisplatin uptake and its cell-killing activity. A phase I clinical trial using a combination of trientine (a copper chelator) and carboplatin has been carried out with encouraging results.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XPC', entrez:'7508'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7508', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'395', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22174370', evidence_text:\"Because our results (Supplementary Data S1A and B and S2A-D) and others (6, 7) have shown that XPC-deficient cells have lower UV and cisplatin sensitivity. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XPC', entrez:'7508'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7508', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'396', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22174370', evidence_text:\"Because our results (Supplementary Data S1A and B and S2A-D) and others (6, 7) have shown that XPC-deficient cells have lower UV and cisplatin sensitivity. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XPC', entrez:'7508'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7508', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'397', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22174370', evidence_text:\"Because our results (Supplementary Data S1A and B and S2A-D) and others (6, 7) have shown that XPC-deficient cells have lower UV and cisplatin sensitivity. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'OGG1', entrez:'4968'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4968', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'398', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23806019', evidence_text:\"BER pathway to repair 8-oxoG lesion may be associated with ESCC sensitivity to cisplatin, and over-expression of hOGG1 in the nucleus can repair more 8-oxoG oxidative damage. The findings implied that over-expression of hOGG1 can protect ESCC cells from cisplatin-induced apoptosis and prolong cancer cell survival time. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'OGG1', entrez:'4968'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4968', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'399', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23806019', evidence_text:\"BER pathway to repair 8-oxoG lesion may be associated with ESCC sensitivity to cisplatin, and over-expression of hOGG1 in the nucleus can repair more 8-oxoG oxidative damage. The findings implied that over-expression of hOGG1 can protect ESCC cells from cisplatin-induced apoptosis and prolong cancer cell survival time. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TXN', entrez:'7295'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7295', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'400', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22248473', evidence_text:\"By reconstituting TRX1 protein in cisplatin-resistant cells, we increased sensitivity to cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.|Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'POLH', entrez:'5429'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5429', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'401', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22529383', evidence_text:\"Chemoresistance to cisplatin treatment is due in part to translesion synthesis by human DNA polymerase _ (hPol _). |Evidence Level:  Pre-clinical. Cell line.  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'POLH', entrez:'5429'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5429', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'402', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22529383', evidence_text:\"Chemoresistance to cisplatin treatment is due in part to translesion synthesis by human DNA polymerase _ (hPol _). |Evidence Level:  Pre-clinical. Cell line.  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'POLH', entrez:'5429'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5429', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'403', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22529383', evidence_text:\"Chemoresistance to cisplatin treatment is due in part to translesion synthesis by human DNA polymerase _ (hPol _). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity. Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_insertion', value:'6232_6238del7ins14', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'404', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18264087', evidence_text:\"Cisplatin resistance clones table S1. Secondary mutations are relevant as a mechanism of resistance only in those tumours that contain frameshift mutations in BRCA2, which occurs in only 5% of ovarian carcinomas. |Evidence Level:  Case Study. |Disease: Ovarian cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_insertion', value:'6191_6192insT', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'405', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18264087', evidence_text:\"Cisplatin resistance clones table S1. Secondary mutations are relevant as a mechanism of resistance only in those tumours that contain frameshift mutations in BRCA2, which occurs in only 5% of ovarian carcinomas. |Evidence Level:  Case Study. |Disease: Ovarian cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_deletion', value:'6006_6308del303', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'406', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18264087', evidence_text:\"Cisplatin resistance clones table S1. Secondary mutations are relevant as a mechanism of resistance only in those tumours that contain frameshift mutations in BRCA2, which occurs in only 5% of ovarian carcinomas. |Evidence Level:  Case Study. |Disease: Ovarian cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_insertion', value:'5870_6365del496ins4', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'407', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18264087', evidence_text:\"Cisplatin resistance clones table S1. Secondary mutations are relevant as a mechanism of resistance only in those tumours that contain frameshift mutations in BRCA2, which occurs in only 5% of ovarian carcinomas. |Evidence Level:  Case Study. |Disease: Ovarian cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_deletion', value:'6186_6226del41', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'408', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18264087', evidence_text:\"Cisplatin resistance clones table S1. Secondary mutations are relevant as a mechanism of resistance only in those tumours that contain frameshift mutations in BRCA2, which occurs in only 5% of ovarian carcinomas. |Evidence Level:  Case Study. |Disease: Ovarian cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_insertion', value:'6191_6192insT', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'409', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18264087', evidence_text:\"Cisplatin resistance clones table S1. Secondary mutations are relevant as a mechanism of resistance only in those tumours that contain frameshift mutations in BRCA2, which occurs in only 5% of ovarian carcinomas. |Evidence Level:  Case Study. |Disease: Ovarian cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'410', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22582397', evidence_text:\"Consistent with its effects on the biochemical functions of FANCJ, the FANCJ Q mutant alleles failed to rescue cisplatin or telomestatin sensitivity of an FA-J null cell line and exerted a dominant negative effect in a wild-type background cell line, as detected by reduced cell survival or induction of _-H2AX foci after exposure to cisplatin or TMS. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity. Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'Q25#', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'411', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22582397', evidence_text:\"Consistent with its effects on the biochemical functions of FANCJ, the FANCJ Q mutant alleles failed to rescue cisplatin or telomestatin sensitivity of an FA-J null cell line and exerted a dominant negative effect in a wild-type background cell line, as detected by reduced cell survival or induction of _-H2AX foci after exposure to cisplatin or TMS. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity. Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'412', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22582397', evidence_text:\"Consistent with its effects on the biochemical functions of FANCJ, the FANCJ Q mutant alleles failed to rescue cisplatin or telomestatin sensitivity of an FA-J null cell line and exerted a dominant negative effect in a wild-type background cell line, as detected by reduced cell survival or induction of _-H2AX foci after exposure to cisplatin or TMS. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'Q25A', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'413', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22582397', evidence_text:\"Consistent with its effects on the biochemical functions of FANCJ, the FANCJ Q mutant alleles failed to rescue cisplatin or telomestatin sensitivity of an FA-J null cell line and exerted a dominant negative effect in a wild-type background cell line, as detected by reduced cell survival or induction of _-H2AX foci after exposure to cisplatin or TMS. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'414', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22582397', evidence_text:\"Consistent with its effects on the biochemical functions of FANCJ, the FANCJ Q mutant alleles failed to rescue cisplatin or telomestatin sensitivity of an FA-J null cell line and exerted a dominant negative effect in a wild-type background cell line, as detected by reduced cell survival or induction of _-H2AX foci after exposure to cisplatin or TMS. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP73', entrez:'7161'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7161', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'415', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22340593', evidence_text:\"Consistent with the expected induction of pro-apoptotic genes by p73, cells expressing wild type and S235A mutant showed higher apoptosis than did the vector transfected cells while S235D mutant made cells least sensitive to cisplatin induced cell death. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP73', entrez:'7161'}), (a:Aberration { type:'snv', value:'S235A', gene_entrez:'7161', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'416', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22340593', evidence_text:\"Consistent with the expected induction of pro-apoptotic genes by p73, cells expressing wild type and S235A mutant showed higher apoptosis than did the vector transfected cells while S235D mutant made cells least sensitive to cisplatin induced cell death. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP73', entrez:'7161'}), (a:Aberration { type:'snv', value:'S235D', gene_entrez:'7161', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'417', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22340593', evidence_text:\"Consistent with the expected induction of pro-apoptotic genes by p73, cells expressing wild type and S235A mutant showed higher apoptosis than did the vector transfected cells while S235D mutant made cells least sensitive to cisplatin induced cell death. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP73', entrez:'7161'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7161', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'418', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22340593', evidence_text:\"Consistent with the expected induction of pro-apoptotic genes by p73, cells expressing wild type and S235A mutant showed higher apoptosis than did the vector transfected cells while S235D mutant made cells least sensitive to cisplatin induced cell death.|Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC31A2 ', entrez:'1318'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1318', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'419', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20930109', evidence_text:\"Copper transporter 2 (CTR2, SLC31A2) is one of the four copper transporters in mammalian cells that influence the cellular pharmacology of cisplatin and carboplatin.  Knockdown of CTR2 increased the tumor accumulation of cis-diamminedichloroplatinum(II) [cisplatin (cDDP)] by 9.1-fold and greatly increased its therapeutic efficacy. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC31A2 ', entrez:'1318'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1318', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'420', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20930109', evidence_text:\"Copper transporter 2 (CTR2, SLC31A2) is one of the four copper transporters in mammalian cells that influence the cellular pharmacology of cisplatin and carboplatin.  Knockdown of CTR2 increased the tumor accumulation of cis-diamminedichloroplatinum(II) [cisplatin (cDDP)] by 9.1-fold and greatly increased its therapeutic efficacy. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'HQ912715.1 (exon 4 del)', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'421', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23764753', evidence_text:\"de4 EGFR transfectants displayed significantly lower sensitivity to cisplatin than EGFR transfectants, which could be ascribed to the upregulation of Bcl-2 and downregulation of BAD in the de4 EGFR transfectants. Collectively, these results demonstrate that de4 EGFR plays an important role in the invasiveness and cisplatin resistance in epithelial ovarian cancer cells and may provide a new potential therapeutic target for epithelial ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'WRN', entrez:'7486'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7486', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'422', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'WRN', entrez:'7486'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7486', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'423', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'WRN', entrez:'7486'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'7486', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'424', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'WRN', entrez:'7486'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'7486', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'425', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BLM', entrez:'641'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'641', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'426', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BLM', entrez:'641'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'641', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'427', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BLM', entrez:'641'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'641', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'428', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BLM', entrez:'641'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'641', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'429', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'WRN', entrez:'7486'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7486', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'430', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity. Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BLM', entrez:'641'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'641', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'431', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20663905', evidence_text:\"Depletion of WRN or BLM significantly sensitize GM639 and U-2 OS cells to dose-dependent killing by CPT, cis-Pt, and 5-fluorouracil, and BLM-depleted cells to HU. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity. Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ASNA1', entrez:'439'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'439', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'432', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19724867', evidence_text:\"Down-regulated ASNA1 expression was associated with retarded growth and increased sensitivity to cisplatin, carboplatin, oxaliplatin and arsenite whereas the cisplatin resistant 2008/A overexpresses ASNA1. These observations support the hypothesis that ASNA1 is a target to overcome platinum resistance in ovarian cancer. |Disease:  Ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ASNA1', entrez:'439'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'439', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'433', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19724867', evidence_text:\"Down-regulated ASNA1 expression was associated with retarded growth and increased sensitivity to cisplatin, carboplatin, oxaliplatin and arsenite whereas the cisplatin resistant 2008/A overexpresses ASNA1. These observations support the hypothesis that ASNA1 is a target to overcome platinum resistance in ovarian cancer. |Disease:  Ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REV3L', entrez:'5980'}), (a:Aberration { type:'cnv', value:'under', gene_entrez:'5980', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'434', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19289490', evidence_text:\"Downregulation of REV3L expression significantly enhanced the sensitivity of glioma cells to cisplatin. These results suggest that REV3L may potentially contribute to gliomagenesis and play a crucial role in regulating cellular response to the DNA cross-linking agent cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REV3L', entrez:'5980'}), (a:Aberration { type:'exp', value:'loss', gene_entrez:'5980', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'435', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19289490', evidence_text:\"Downregulation of REV3L expression significantly enhanced the sensitivity of glioma cells to cisplatin. These results suggest that REV3L may potentially contribute to gliomagenesis and play a crucial role in regulating cellular response to the DNA cross-linking agent cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BIRC6', entrez:'57448'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'57448', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'436', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23287853', evidence_text:\"Elevated BIRC6 protein expression in NSCLC tissues was associated with poor 3-year relapse-free patient survival, lymph node involvement, and advanced pathological tumor, node, metastasis stage. In patient-derived lung squamous cell carcinoma xenografts, chemoresistance was associated with elevated BIRC6 expression and increased gene copy number.  Small interfering RNA-induced BIRC6 down-regulation inhibited growth of the NSCLC cells and sensitized the cells to cisplatin. |Evidence Level:  Xenograft. Pre-clinical. Cell line. |Disease:  NSCLC. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BIRC6', entrez:'57448'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'57448', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'437', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23287853', evidence_text:\"Elevated BIRC6 protein expression in NSCLC tissues was associated with poor 3-year relapse-free patient survival, lymph node involvement, and advanced pathological tumor, node, metastasis stage. In patient-derived lung squamous cell carcinoma xenografts, chemoresistance was associated with elevated BIRC6 expression and increased gene copy number.  Small interfering RNA-induced BIRC6 down-regulation inhibited growth of the NSCLC cells and sensitized the cells to cisplatin. |Evidence Level:  Xenograft. Pre-clinical. Cell line. |Disease:  NSCLC. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ENG', entrez:'2022'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2022', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'438', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23147994', evidence_text:\"Endoglin downregulation promotes apoptosis, induces significant DNA damage through modulation of numerous DNA repair genes, and improves platinum sensitivity both in vivo and in vitro. |Evidence Level:  Xenograft. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ENG', entrez:'2022'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2022', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'439', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23147994', evidence_text:\"Endoglin downregulation promotes apoptosis, induces significant DNA damage through modulation of numerous DNA repair genes, and improves platinum sensitivity both in vivo and in vitro. |Evidence Level:  Xenograft. Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC1', entrez:'2067'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2067', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'440', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18316546', evidence_text:\"Evaluation of ERCC1 mRNA levels in tumor samples taken from patients in small retrospective clinical trials of ovarian (27, 28), colorectal (7, 29), and non-small cell lung cancer (30) had shown an inverse correlation with either response to platinum therapy or survival. Data from in vitro systems have shown that suppression of ERCC1 expression enhances or restores platinum sensitivity. |Evidence Level:  Case Study |Disease:  Ovarian cancer, Colorectal cancer, NSCLC. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC1', entrez:'2067'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2067', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'441', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18316546', evidence_text:\"Evaluation of ERCC1 mRNA levels in tumor samples taken from patients in small retrospective clinical trials of ovarian (27, 28), colorectal (7, 29), and non-small cell lung cancer (30) had shown an inverse correlation with either response to platinum therapy or survival. Data from in vitro systems have shown that suppression of ERCC1 expression enhances or restores platinum sensitivity. |Evidence Level:  Case Study |Disease:  Ovarian cancer, Colorectal cancer, NSCLC. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC1', entrez:'2067'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2067', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'442', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18316546', evidence_text:\"Evaluation of ERCC1 mRNA levels in tumor samples taken from patients in small retrospective clinical trials of ovarian (27, 28), colorectal (7, 29), and non-small cell lung cancer (30) had shown an inverse correlation with either response to platinum therapy or survival. Data from in vitro systems have shown that suppression of ERCC1 expression enhances or restores platinum sensitivity. |Evidence Level:  Case Study |Disease:  Ovarian cancer, Colorectal cancer, NSCLC. |Annotator Note:  SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC22A2', entrez:'6582'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6582', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'443', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20067471', evidence_text:\"Expression of hSLC22A2 in HEK293 cells resulted in enhanced sensitivity to oxaliplatin (12-fold) and cisplatin (1.8-fold). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with oxaliplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC22A2', entrez:'6582'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6582', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'444', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20067471', evidence_text:\"Expression of hSLC22A2 in HEK293 cells resulted in enhanced sensitivity to oxaliplatin (12-fold) and cisplatin (1.8-fold). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with oxaliplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCC', entrez:'2176'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2176', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'445', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"FANCC-mutant VU- 1131 cells were clearly hypersensitive to cisplatin and oxaliplatin, while the FANCA-mutant OHSU-974 cells were unexpectedly resistant (Fig. 1A and B). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin and oxaliplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCC', entrez:'2176'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2176', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'446', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"FANCC-mutant VU- 1131 cells were clearly hypersensitive to cisplatin and oxaliplatin, while the FANCA-mutant OHSU-974 cells were unexpectedly resistant (Fig. 1A and B). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin and oxaliplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCC', entrez:'2176'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2176', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'447', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"FANCC-mutant VU- 1131 cells were clearly hypersensitive to cisplatin and oxaliplatin, while the FANCA-mutant OHSU-974 cells were unexpectedly resistant (Fig. 1A and B). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin and oxaliplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCA', entrez:'2175'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2175', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'448', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"FANCC-mutant VU- 1131 cells were clearly hypersensitive to cisplatin and oxaliplatin, while the FANCA-mutant OHSU-974 cells were unexpectedly resistant (Fig. 1A and B). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin and oxaliplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCA', entrez:'2175'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2175', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'449', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"FANCC-mutant VU- 1131 cells were clearly hypersensitive to cisplatin and oxaliplatin, while the FANCA-mutant OHSU-974 cells were unexpectedly resistant (Fig. 1A and B). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin and oxaliplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCA', entrez:'2175'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2175', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'450', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"FANCC-mutant VU- 1131 cells were clearly hypersensitive to cisplatin and oxaliplatin, while the FANCA-mutant OHSU-974 cells were unexpectedly resistant (Fig. 1A and B). |Annotator Note:  Drug class relation with cisplatin and oxaliplatin. SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MIR30A', entrez:'407029'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'407029', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'451', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22157765', evidence_text:\"Finally, an in vivo tumor implantation mouse model clearly showed that elevation of miR-30a in implanted tumor cells by administration of the recombinant lentivirus expressing miR-30a strongly enhanced cis-DDP-induced apoptosis of tumor cells. In conclusion, our results demonstrate for the first time that miR-30a can sensitize tumor cells to cis-DDP via reducing beclin 1-mediated autophagy and that increasing miR-30a level in tumor cells represents a novel approach to enhance the efficacy of chemotherapy during cancer treatment. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MIR30A', entrez:'407029'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'407029', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'452', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22157765', evidence_text:\"Finally, an in vivo tumor implantation mouse model clearly showed that elevation of miR-30a in implanted tumor cells by administration of the recombinant lentivirus expressing miR-30a strongly enhanced cis-DDP-induced apoptosis of tumor cells. In conclusion, our results demonstrate for the first time that miR-30a can sensitize tumor cells to cis-DDP via reducing beclin 1-mediated autophagy and that increasing miR-30a level in tumor cells represents a novel approach to enhance the efficacy of chemotherapy during cancer treatment. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC1', entrez:'7515'}), (a:Aberration { type:'snv', value:'R399#', gene_entrez:'7515', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'453', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16875718', evidence_text:\"Genetic polymorphism of XRCC1 R399Q is associated with response to platinum-based NAC in bulky cervical cancer, and MDR analysis documented association between gene-gene interaction of XRCC1 R399Q and treatment response. |Evidence Level:  Clinical. |Disease:  Cervical cancer. |Annotator Note:  Drug class relation with cisplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC1', entrez:'7515'}), (a:Aberration { type:'snv', value:'R399Q', gene_entrez:'7515', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'454', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16875718', evidence_text:\"Genetic polymorphism of XRCC1 R399Q is associated with response to platinum-based NAC in bulky cervical cancer, and MDR analysis documented association between gene-gene interaction of XRCC1 R399Q and treatment response. |Evidence Level:  Clinical. |Disease:  Cervical cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLCO1B3', entrez:'28234'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'28234', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'455', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23757163', evidence_text:\"Here we show that the gene encoding OATP1B3 (SLCO1B3), but not OATP1B1, is abundantly expressed in multiple human solid tumors that include hepatocellular, lung, and ovarian carcinomas. Surprisingly, OATP1B3 gene expression in a panel of 60 human tumor cell lines was linked with sensitivity to multiple cytotoxic agents, including the platinum anticancer drugs cisplatin, carboplatin, and oxaliplatin. In addition, overexpression of OATP1B3 in mammalian cells increased cellular accumulation of platinum agents and decreased cell survival. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLCO1B3', entrez:'28234'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'28234', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'456', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23757163', evidence_text:\"Here we show that the gene encoding OATP1B3 (SLCO1B3), but not OATP1B1, is abundantly expressed in multiple human solid tumors that include hepatocellular, lung, and ovarian carcinomas. Surprisingly, OATP1B3 gene expression in a panel of 60 human tumor cell lines was linked with sensitivity to multiple cytotoxic agents, including the platinum anticancer drugs cisplatin, carboplatin, and oxaliplatin. In addition, overexpression of OATP1B3 in mammalian cells increased cellular accumulation of platinum agents and decreased cell survival. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCC2', entrez:'1244'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1244', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'457', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/8797578', evidence_text:\"human cMOAT cDNA hybridized a 6.5-kb mRNA that was expressed 4- to 6-fold higher by three cisplatin-resistant cell lines derived from various human tumors exhibiting decreased drug accumulation. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCC2', entrez:'1244'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1244', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'458', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/8797578', evidence_text:\"human cMOAT cDNA hybridized a 6.5-kb mRNA that was expressed 4- to 6-fold higher by three cisplatin-resistant cell lines derived from various human tumors exhibiting decreased drug accumulation. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'POLB', entrez:'5423'}), (a:Aberration { type:'cnv ', value:'gain', gene_entrez:'5423', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'459', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/9770529', evidence_text:\"increases in pol beta mRNA and protein were observed in human ovarian, colon, and leukemia cancer cells resistant to cisplatin over the sensitive parent cell lines (22), supporting a role for pol beta in tolerance and tumor progression. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'POLB', entrez:'5423'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5423', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'460', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/9770529', evidence_text:\"increases in pol beta mRNA and protein were observed in human ovarian, colon, and leukemia cancer cells resistant to cisplatin over the sensitive parent cell lines (22), supporting a role for pol beta in tolerance and tumor progression. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'snv', value:'R188H', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'461', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17141189', evidence_text:\"Interestingly, Xrcc2_/_ DT40 cells complemented with variant XRCC2 (R188H) were more tolerant to cisplatin than those with wild-type XRCC2. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MBD1', entrez:'4152'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4152', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'462', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23588667', evidence_text:\"Knockdown of MBD1 significantly increased the sensitivity of cells to radiation and cisplatin (diamindichloridoplatin, DDP) in vitro. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MBD1', entrez:'4152'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4152', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'463', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23588667', evidence_text:\"Knockdown of MBD1 significantly increased the sensitivity of cells to radiation and cisplatin (diamindichloridoplatin, DDP) in vitro. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP1', entrez:'7150'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7150', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'464', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22344449', evidence_text:\"Moreover, low levels of TOP I RNA were shown to be linked to cisplatin response (p=0.002). In the survival analysis, a significant correlation between a better PFS with a low TOP I RNA expression. |Evidence Level:  Clinical. |Disease:  NSCLC. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP1', entrez:'7150'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7150', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'465', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22344449', evidence_text:\"Moreover, low levels of TOP I RNA were shown to be linked to cisplatin response (p=0.002). In the survival analysis, a significant correlation between a better PFS with a low TOP I RNA expression. |Evidence Level:  Clinical. |Disease:  NSCLC. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP1', entrez:'7150'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7150', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'466', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22344449', evidence_text:\"Moreover, low levels of TOP I RNA were shown to be linked to cisplatin response (p=0.002). In the survival analysis, a significant correlation between a better PFS with a low TOP I RNA expression. |Evidence Level:  Clinical. |Disease:  NSCLC. |Annotator Note:  Drug class relation with cisplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC6', entrez:'2074'}), (a:Aberration { type:'snv', value:'Q534*', gene_entrez:'2074', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'467', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC6', entrez:'2074'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2074', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'468', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC6', entrez:'2074'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2074', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'469', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC6', entrez:'2074'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'2074', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'470', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC6', entrez:'2074'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'2074', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'471', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC6', entrez:'2074'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2074', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'472', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. |Annotator Note:  SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC4', entrez:'2072'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2072', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'473', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC4', entrez:'2072'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2072', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'474', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC4', entrez:'2072'}), (a:Aberration { type:'snv', value:'A583T', gene_entrez:'2072', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'475', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC4', entrez:'2072'}), (a:Aberration { type:'snv', value:'A583#', gene_entrez:'2072', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'476', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. |Annotator Note:  SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC4', entrez:'2072'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'2072', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'477', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC4', entrez:'2072'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'2072', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'478', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC4', entrez:'2072'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2072', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'479', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/25634215', evidence_text:\"Moreover, two NER mutations (ERCC6-Q524* and ERCC4-A583T), identified in the two most platinum-sensitive tumors, were functionally associated with platinum sensitivity in vitro. |Evidence Level:  Case Study. Pre-clinical Cell line. Disease:  Ovarian cancer. |Annotator Note:  SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'C10orf90', entrez:'118611'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'118611', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'480', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22137560', evidence_text:\"mRNA level of FATS (C10orf90) was significantly downregulated in NSCLC and the tumor expression levels of FATS mRNA were predictive of tumor response to cisplatin-based chemotherapy in patients with NSCLC. Enhanced expression of FATS significantly sensitized NSCLC cells to cisplatin-induced apoptosis. High expression of FATS mRNA provides a new biomarker for a good outcome in patients receiving cisplatin-based chemotherapy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'C10orf90', entrez:'118611'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'118611', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'481', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22137560', evidence_text:\"mRNA level of FATS (C10orf90) was significantly downregulated in NSCLC and the tumor expression levels of FATS mRNA were predictive of tumor response to cisplatin-based chemotherapy in patients with NSCLC. Enhanced expression of FATS significantly sensitized NSCLC cells to cisplatin-induced apoptosis. High expression of FATS mRNA provides a new biomarker for a good outcome in patients receiving cisplatin-based chemotherapy. |Evidence Level:  Pre-clinical. Cell line. |Disease:  NSCLC. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH2', entrez:'4436'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4436', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'482', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20145178', evidence_text:\"MSH2 levels were low in 257 (38%) and high in 416 (62%) tumors. The benefit from chemotherapy was likely different according to MSH2 (interaction test, P = 0.06): there was a trend for chemotherapy to prolong overall survival when MSH2 was low [hazard ratio (HR), 0.76; 95% confidence interval (95% CI), 0.59-0.97; P = 0.03], but not when MSH2 was high (HR, 1.12; 95% CI, 0.81-1.55; P = 0.48). In the control arm, the HR was 0.66 (95% CI, 0.49-0.90; P = 0.01) when MSH2 was high. When combining MSH2 with excision repair cross-complementing group 1 (ERCC1) into four subgroups, the benefit of chemotherapy decreased with the number of markers expressed at high levels (P = 0.01). A similar decrease was noted when combining MSH2 and P27 (P = 0.01). Chemotherapy prolonged overall survival in the combined low MSH2/low ERCC1 subgroup (HR, 0.65; 95% CI, 0.47-0.91; P = 0.01) and in the combined low MSH2/low P27 subgroup (HR, 0.65; 95% CI, 0.46-0.93; P = 0.01). MSH2 expression is a borderline significant predictor of a long-term benefit from adjuvant cisplatin-based chemotherapy in patients with completely resected lung cancer. MSH2 combined with ERCC1 or P27 may identify patients most likely to benefit durably from chemotherapy. |Evidence Level:  Clinical. |Disease:  NSCLC. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH2', entrez:'4436'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4436', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'483', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20145178', evidence_text:\"MSH2 levels were low in 257 (38%) and high in 416 (62%) tumors. The benefit from chemotherapy was likely different according to MSH2 (interaction test, P = 0.06): there was a trend for chemotherapy to prolong overall survival when MSH2 was low [hazard ratio (HR), 0.76; 95% confidence interval (95% CI), 0.59-0.97; P = 0.03], but not when MSH2 was high (HR, 1.12; 95% CI, 0.81-1.55; P = 0.48). In the control arm, the HR was 0.66 (95% CI, 0.49-0.90; P = 0.01) when MSH2 was high. When combining MSH2 with excision repair cross-complementing group 1 (ERCC1) into four subgroups, the benefit of chemotherapy decreased with the number of markers expressed at high levels (P = 0.01). A similar decrease was noted when combining MSH2 and P27 (P = 0.01). Chemotherapy prolonged overall survival in the combined low MSH2/low ERCC1 subgroup (HR, 0.65; 95% CI, 0.47-0.91; P = 0.01) and in the combined low MSH2/low P27 subgroup (HR, 0.65; 95% CI, 0.46-0.93; P = 0.01). MSH2 expression is a borderline significant predictor of a long-term benefit from adjuvant cisplatin-based chemotherapy in patients with completely resected lung cancer. MSH2 combined with ERCC1 or P27 may identify patients most likely to benefit durably from chemotherapy. |Evidence Level:  Clinical. |Disease:  NSCLC. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH2', entrez:'4436'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4436', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'484', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20145178', evidence_text:\"MSH2 levels were low in 257 (38%) and high in 416 (62%) tumors. The benefit from chemotherapy was likely different according to MSH2 (interaction test, P = 0.06): there was a trend for chemotherapy to prolong overall survival when MSH2 was low [hazard ratio (HR), 0.76; 95% confidence interval (95% CI), 0.59-0.97; P = 0.03], but not when MSH2 was high (HR, 1.12; 95% CI, 0.81-1.55; P = 0.48). In the control arm, the HR was 0.66 (95% CI, 0.49-0.90; P = 0.01) when MSH2 was high. When combining MSH2 with excision repair cross-complementing group 1 (ERCC1) into four subgroups, the benefit of chemotherapy decreased with the number of markers expressed at high levels (P = 0.01). A similar decrease was noted when combining MSH2 and P27 (P = 0.01). Chemotherapy prolonged overall survival in the combined low MSH2/low ERCC1 subgroup (HR, 0.65; 95% CI, 0.47-0.91; P = 0.01) and in the combined low MSH2/low P27 subgroup (HR, 0.65; 95% CI, 0.46-0.93; P = 0.01). MSH2 expression is a borderline significant predictor of a long-term benefit from adjuvant cisplatin-based chemotherapy in patients with completely resected lung cancer. MSH2 combined with ERCC1 or P27 may identify patients most likely to benefit durably from chemotherapy. |Evidence Level:  Clinical. |Disease:  NSCLC. |Annotator Note:  SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH3', entrez:'4437'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4437', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'485', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23724141', evidence_text:\"MSH3-deficient vs proficient CRC cells showed increased sensitivity to the irinotecan metabolite SN-38 and to oxaliplatin. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Colorectal cancer. |Annotator Note:  Drug class relation with oxaliplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH3', entrez:'4437'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4437', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'486', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23724141', evidence_text:\"MSH3-deficient vs proficient CRC cells showed increased sensitivity to the irinotecan metabolite SN-38 and to oxaliplatin. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Colorectal cancer. |Annotator Note:  Drug class relation with oxaliplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH3', entrez:'4437'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4437', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'487', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23724141', evidence_text:\"MSH3-deficient vs proficient CRC cells showed increased sensitivity to the irinotecan metabolite SN-38 and to oxaliplatin. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Colorectal cancer. |Annotator Note:  Drug class relation with oxaliplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MYD88', entrez:'4615'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4615', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'488', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23766530', evidence_text:\"MyD88 inhibition leads to defective ERCC1-dependent DNA repair and to accumulation of DNA damage, resulting in cancer cell death via p53. Furthermore, we show that knocking down MyD88 sensitizes cancer cells to genotoxic agents such as platinum salts in vitro and in vivo. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MYD88', entrez:'4615'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4615', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'489', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23766530', evidence_text:\"MyD88 inhibition leads to defective ERCC1-dependent DNA repair and to accumulation of DNA damage, resulting in cancer cell death via p53. Furthermore, we show that knocking down MyD88 sensitizes cancer cells to genotoxic agents such as platinum salts in vitro and in vivo. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC5', entrez:'2073'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2073', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'490', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18187810', evidence_text:\"Of the NER proteins, XPF and XPG showed the highest correlations with drug activity across all three tissue-of-origin groups (ovarian cancer, colon cancer, and XP cell lines). The increased protein expression was directly correlated with Et-743 sensitivity and cisplatin resistance. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC5', entrez:'2073'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2073', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'491', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18187810', evidence_text:\"Of the NER proteins, XPF and XPG showed the highest correlations with drug activity across all three tissue-of-origin groups (ovarian cancer, colon cancer, and XP cell lines). The increased protein expression was directly correlated with Et-743 sensitivity and cisplatin resistance. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC5', entrez:'2073'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2073', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'492', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18187810', evidence_text:\"Of the NER proteins, XPF and XPG showed the highest correlations with drug activity across all three tissue-of-origin groups (ovarian cancer, colon cancer, and XP cell lines). The increased protein expression was directly correlated with Et-743 sensitivity and cisplatin resistance. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH5', entrez:'4439'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4439', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'493', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22401567', evidence_text:\"our current study has suggested a role for hMSH5 in the processing of cisplatin-induced DSBs, and silencing of hMSH5 may provide a new means to improve the therapeutic efficacy of cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH5', entrez:'4439'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4439', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'494', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22401567', evidence_text:\"Our current study has suggested a role for hMSH5 in the processing of cisplatin-induced DSBs, and silencing of hMSH5 may provide a new means to improve the therapeutic efficacy of cisplatin. |Annotator Note: Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH5', entrez:'4439'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4439', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'495', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22401567', evidence_text:\"Our current study has suggested a role for hMSH5 in the processing of cisplatin-induced DSBs, and silencing of hMSH5 may provide a new means to improve the therapeutic efficacy of cisplatin. |Annotator Note: Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH5', entrez:'4439'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4439', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'496', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22401567', evidence_text:\"Our current study has suggested a role for hMSH5 in the processing of cisplatin-induced DSBs, and silencing of hMSH5 may provide a new means to improve the therapeutic efficacy of cisplatin. |Annotator Note: Drug class relation with cisplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HDAC6', entrez:'10013'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'10013', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'497', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22957056', evidence_text:\"our findings suggest that HDAC6 is positively associated with cisplatin resistance in NSCLC and reveal HDAC6 as a potential novel therapeutic target for platinum refractory NSCLC. |Evidence Level:  Pre-clinical. Cell line. |Disease:  NSCLC. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HDAC6', entrez:'10013'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'10013', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'498', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22957056', evidence_text:\"our findings suggest that HDAC6 is positively associated with cisplatin resistance in NSCLC and reveal HDAC6 as a potential novel therapeutic target for platinum refractory NSCLC. |Evidence Level:  Pre-clinical. Cell line. |Disease:  NSCLC. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC2', entrez:'2068'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2068', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'499', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/12359753', evidence_text:\"Our results indicate that XPD overexpression in SK-MG-4 cells leads to cisplatin resistance without affecting the nucleotide excision repair activity or UV light sensitivity of the cell. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERCC2', entrez:'2068'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2068', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'500', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/12359753', evidence_text:\"Our results indicate that XPD overexpression in SK-MG-4 cells leads to cisplatin resistance without affecting the nucleotide excision repair activity or UV light sensitivity of the cell. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_deletion', value:'6174delT', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'501', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18264087', evidence_text:\"Platinum-refractory tumor UW3548 revealed genetic reversion of BRCA2.6174delT. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'502', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15014021', evidence_text:\"In conclusion, our findings indicated that expression of the novel ABC transporter protein BCRP, but not of Pgp, MRP1, MRP2, and MRP3, is a predictive factor of poor clinical outcome in advanced NSCLC patients treated with standard platinum-based combination chemotherapy. |Evidence Level:  Clinical. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'503', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15014021', evidence_text:\"In conclusion, our findings indicated that expression of the novel ABC transporter protein BCRP, but not of Pgp, MRP1, MRP2, and MRP3, is a predictive factor of poor clinical outcome in advanced NSCLC patients treated with standard platinum-based combination chemotherapy. |Evidence Level:  Clinical. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'snv', value:'C421A', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'504', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22112610', evidence_text:\"We found an association between ABCG2 C421A polymorphism and PFS in this population. Specifically, women with the CA+AA genotypes vs the CC genotype had improved PFS, and this association persisted after adjustment for clinical covariates. These findings may be particularly important given that this SNP occurs frequently in the population (~20% with a heterozygous or homozygous variant allele). |Evidence Level:  Clinical. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC1', entrez:'7515'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7515', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'505', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23225521', evidence_text:\"Preclinically, XRCC1 negative cells were sensitive to cisplatin compared to XRCC1 positive cells. Sensitivity to cisplatin in XRCC1 negative cells was associated with accumulation of DNA double strand breaks and G2/M cell cycle arrest. XRCC1 expression is associated with adverse clinicopathological and survival outcomes in patients. |Evidence Level:  Clinical. Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC1', entrez:'7515'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7515', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'506', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23225521', evidence_text:\"Preclinically, XRCC1 negative cells were sensitive to cisplatin compared to XRCC1 positive cells. Sensitivity to cisplatin in XRCC1 negative cells was associated with accumulation of DNA double strand breaks and G2/M cell cycle arrest. XRCC1 expression is associated with adverse clinicopathological and survival outcomes in patients. |Evidence Level:  Clinical. Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC1', entrez:'7515'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7515', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'507', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23225521', evidence_text:\"Preclinically, XRCC1 negative cells were sensitive to cisplatin compared to XRCC1 positive cells. Sensitivity to cisplatin in XRCC1 negative cells was associated with accumulation of DNA double strand breaks and G2/M cell cycle arrest. XRCC1 expression is associated with adverse clinicopathological and survival outcomes in patients. |Evidence Level:  Clinical. Pre-clinical. Cell line. |Annotator Note:  SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STAT5B', entrez:'6777'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6777', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'508', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20585448', evidence_text:\"Proteomic analysis identified RELA and STAT5 as two major proteins associated with carboplatin resistance in ovarian tumors. Our results further showed that NF-kappaB and STAT5 inhibitor could sensitize carboplatin-resistant cells and suggest that such inhibitors can be used to benefit patients with carboplatin-resistant recurrent ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STAT5B', entrez:'6777'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6777', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'509', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20585448', evidence_text:\"Proteomic analysis identified RELA and STAT5 as two major proteins associated with carboplatin resistance in ovarian tumors. Our results further showed that NF-kappaB and STAT5 inhibitor could sensitize carboplatin-resistant cells and suggest that such inhibitors can be used to benefit patients with carboplatin-resistant recurrent ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STAT5B', entrez:'6777'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6777', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'510', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20585448', evidence_text:\"Proteomic analysis identified RELA and STAT5 as two major proteins associated with carboplatin resistance in ovarian tumors. Our results further showed that NF-kappaB and STAT5 inhibitor could sensitize carboplatin-resistant cells and suggest that such inhibitors can be used to benefit patients with carboplatin-resistant recurrent ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RELA', entrez:'5970'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5970', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'511', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20585448', evidence_text:\"Proteomic analysis identified RELA and STAT5 as two major proteins associated with carboplatin resistance in ovarian tumors. Our results further showed that NF-kappaB and STAT5 inhibitor could sensitize carboplatin-resistant cells and suggest that such inhibitors can be used to benefit patients with carboplatin-resistant recurrent ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RELA', entrez:'5970'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5970', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'512', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20585448', evidence_text:\"Proteomic analysis identified RELA and STAT5 as two major proteins associated with carboplatin resistance in ovarian tumors. Our results further showed that NF-kappaB and STAT5 inhibitor could sensitize carboplatin-resistant cells and suggest that such inhibitors can be used to benefit patients with carboplatin-resistant recurrent ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RELA', entrez:'5970'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5970', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'513', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20585448', evidence_text:\"Proteomic analysis identified RELA and STAT5 as two major proteins associated with carboplatin resistance in ovarian tumors. Our results further showed that NF-kappaB and STAT5 inhibitor could sensitize carboplatin-resistant cells and suggest that such inhibitors can be used to benefit patients with carboplatin-resistant recurrent ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC3', entrez:'7517'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7517', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'514', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11283264', evidence_text:\"Remarkably, the mutant cell lines all exhibit very similar phenotypes: spontaneous chromosomal aberrations, high sensitivity to killing by cross-linking agents (mitomycin C and cisplatin). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC3', entrez:'7517'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7517', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'515', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11283264', evidence_text:\"Remarkably, the mutant cell lines all exhibit very similar phenotypes: spontaneous chromosomal aberrations, high sensitivity to killing by cross-linking agents (mitomycin C and cisplatin). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'516', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11283264', evidence_text:\"Remarkably, the mutant cell lines all exhibit very similar phenotypes: spontaneous chromosomal aberrations, high sensitivity to killing by cross-linking agents (mitomycin C and cisplatin). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'517', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11283264', evidence_text:\"Remarkably, the mutant cell lines all exhibit very similar phenotypes: spontaneous chromosomal aberrations, high sensitivity to killing by cross-linking agents (mitomycin C and cisplatin). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC3', entrez:'7517'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7517', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'518', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11283264', evidence_text:\"Remarkably, the mutant cell lines all exhibit very similar phenotypes: spontaneous chromosomal aberrations, high sensitivity to killing by cross-linking agents (mitomycin C and cisplatin). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity. Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'519', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11283264', evidence_text:\"Remarkably, the mutant cell lines all exhibit very similar phenotypes: spontaneous chromosomal aberrations, high sensitivity to killing by cross-linking agents (mitomycin C and cisplatin). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity. Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'snv', value:'R188H', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'520', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17141189', evidence_text:\"Interestingly, Xrcc2(-/-) DT40 cells complemented with variant XRCC2 (R188H) were more tolerant to cisplatin than those with wild-type XRCC2. Considering that the XRCC2 (R188H) allele reduces risk to epithelial ovarian cancer, the increased XRCC2 activity with the R188H polymorphism may have clinical benefit in preventing cancer risk. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'snv', value:'R188#', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'521', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17141189', evidence_text:\"Interestingly, Xrcc2(-/-) DT40 cells complemented with variant XRCC2 (R188H) were more tolerant to cisplatin than those with wild-type XRCC2. Considering that the XRCC2 (R188H) allele reduces risk to epithelial ovarian cancer, the increased XRCC2 activity with the R188H polymorphism may have clinical benefit in preventing cancer risk. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. SNVs at this position are associated with activation and reduced drug sensitivity. However, not all changes at this position may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP2K3', entrez:'5606'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5606', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'522', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22164285', evidence_text:\"Resistant cells express elevated levels of MKK3 which is indeed intrinsically phosphorylated. As a consequence, resistant cells show an aberrant basal activity of p38 MAPK which is controlling MKK6 by decreasing its RNA levels. As consequence of the constitutive activation of MKK3, cells are unable to respond to cDDP treatment (right), surviving after exposure to the drug. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP2K3', entrez:'5606'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5606', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'523', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22164285', evidence_text:\"Resistant cells express elevated levels of MKK3 which is indeed intrinsically phosphorylated. As a consequence, resistant cells show an aberrant basal activity of p38 MAPK which is controlling MKK6 by decreasing its RNA levels. As consequence of the constitutive activation of MKK3, cells are unable to respond to cDDP treatment (right), surviving after exposure to the drug. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP2K3', entrez:'5606'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5606', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'524', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22164285', evidence_text:\"Resistant cells express elevated levels of MKK3 which is indeed intrinsically phosphorylated. As a consequence, resistant cells show an aberrant basal activity of p38 MAPK which is controlling MKK6 by decreasing its RNA levels. As consequence of the constitutive activation of MKK3, cells are unable to respond to cDDP treatment (right), surviving after exposure to the drug. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51D', entrez:'5892'}), (a:Aberration { type:'snv', value:'E233# ', gene_entrez:'5892', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'525', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19033885', evidence_text:\"Results indicated that the E233G variant actually increased the cellular resistance to the DNA-damaging agents, mitomycin C, cisplatin, methyl methane sulfonate, and ultraviolet light as well as to taxol. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51D', entrez:'5892'}), (a:Aberration { type:'snv', value:'E233G ', gene_entrez:'5892', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'526', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19033885', evidence_text:\"Results indicated that the E233G variant actually increased the cellular resistance to the DNA-damaging agents, mitomycin C, cisplatin, methyl methane sulfonate, and ultraviolet light as well as to taxol. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SEC61B', entrez:'10952'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'10952', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'527', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22710939', evidence_text:\"Sec61_ knockdown (KD) resulted in 8-, 16.8-, and 9-fold resistance to cisplatin (cDDP), carboplatin, and oxaliplatin, respectively. Sec61_ KD reduced the cellular accumulation of cDDP to 67% of that in parental cells. Because resistance to the platinum-containing drugs is linked to copper homeostasis, we analyzed the effects of Sec61_ knockdown on the expression of copper transporters and chaperones, and we found a significant increase in the expression of ATP7A. Knockdown of Sec61_ in cells lacking ATP7A failed to alter drug sensitivity, which demonstrates a specific role for ATP7A. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SEC61B', entrez:'10952'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'10952', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'528', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22710939', evidence_text:\"Sec61_ knockdown (KD) resulted in 8-, 16.8-, and 9-fold resistance to cisplatin (cDDP), carboplatin, and oxaliplatin, respectively. Sec61_ KD reduced the cellular accumulation of cDDP to 67% of that in parental cells. Because resistance to the platinum-containing drugs is linked to copper homeostasis, we analyzed the effects of Sec61_ knockdown on the expression of copper transporters and chaperones, and we found a significant increase in the expression of ATP7A. Knockdown of Sec61_ in cells lacking ATP7A failed to alter drug sensitivity, which demonstrates a specific role for ATP7A. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'529', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19074863', evidence_text:\"Several reports suggest that ovarian cancer patients who carry BRCA1 or BRCA2 mutations gain more benefit from carboplatin compared with patients with no family history of the disease, but resistance does eventually occur. |Evidence Level:  Case Study |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA1', entrez:'672'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'672', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'530', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19074863', evidence_text:\"Several reports suggest that ovarian cancer patients who carry BRCA1 or BRCA2 mutations gain more benefit from carboplatin compared with patients with no family history of the disease, but resistance does eventually occur. |Evidence Level:  Case Study |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'531', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19074863', evidence_text:\"Several reports suggest that ovarian cancer patients who carry BRCA1 or BRCA2 mutations gain more benefit from carboplatin compared with patients with no family history of the disease, but resistance does eventually occur. |Evidence Level:  Case Study |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA1', entrez:'672'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'672', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'532', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19074863', evidence_text:\"Several reports suggest that ovarian cancer patients who carry BRCA1 or BRCA2 mutations gain more benefit from carboplatin compared with patients with no family history of the disease, but resistance does eventually occur. |Evidence Level:  Case Study |Disease:  Ovarian cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SSRP1', entrez:'6749'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'6749', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'533', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21679440', evidence_text:\"Silencing SSRP1 expression increased sensitivity to cisplatin and decreased _H2AX appearance. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SSRP1', entrez:'6749'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6749', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'534', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21679440', evidence_text:\"Silencing SSRP1 expression increased sensitivity to cisplatin and decreased _H2AX appearance. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SSRP1', entrez:'6749'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'6749', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'535', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21679440', evidence_text:\"Silencing SSRP1 expression increased sensitivity to cisplatin and decreased _H2AX appearance. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PRKDC', entrez:'5591'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5591', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'536', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21679440', evidence_text:\"Silencing the expression of the catalytic subunit of DNA-PK (DNA-PKcs) increased sensitivity to cisplatin and decreased the appearance of _H2AX after cisplatin treatment. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PRKDC', entrez:'5591'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'5591', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'537', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21679440', evidence_text:\"Silencing the expression of the catalytic subunit of DNA-PK (DNA-PKcs) increased sensitivity to cisplatin and decreased the appearance of _H2AX after cisplatin treatment. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PRKDC', entrez:'5591'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5591', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'538', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21679440', evidence_text:\"Silencing SSRP1 expression increased sensitivity to cisplatin and decreased _H2AX appearance. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EIF3J', entrez:'8669'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'8669', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'539', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21625209', evidence_text:\"Taken together, we conclude that eIF3a (EIF3J) has an important role in the CDDP response and in NER activity of NPCs by suppressing the synthesis of NER proteins. |Evidence Level:  Pre-clinical. Cell line. |Disease: Nasopharyngeal carcinoma |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EIF3J', entrez:'8669'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'8669', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'540', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21625209', evidence_text:\"Taken together, we conclude that eIF3a (EIF3J) has an important role in the CDDP response and in NER activity of NPCs by suppressing the synthesis of NER proteins. |Evidence Level:  Pre-clinical. Cell line. |Disease: Nasopharyngeal carcinoma |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC31A1', entrez:'1317'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'1317', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'541', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20159940', evidence_text:\"The absence of CTR1 expression renders yeast and mammalian cells resistant to copper, DDP, carboplatin, and oxaliplatin (Lin et al., 2002; Holzer et al., 2004b, 2006a; Larson et al., 2009), whereas overexpression of CTR1 was found to sensitize cells to the toxic effects of these agents (Holzer et al., 2004b; Noordhuis et al., 2008). |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC31A1', entrez:'1317'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'1317', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'542', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20159940', evidence_text:\"The absence of CTR1 expression renders yeast and mammalian cells resistant to copper, DDP, carboplatin, and oxaliplatin (Lin et al., 2002; Holzer et al., 2004b, 2006a; Larson et al., 2009), whereas overexpression of CTR1 was found to sensitize cells to the toxic effects of these agents (Holzer et al., 2004b; Noordhuis et al., 2008). |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REV1', entrez:'51455'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'51455', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'543', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22761336', evidence_text:\"The DNA repair protein RAD51 and the trans-lesion synthesis DNA polymerase REV1 are required for resistance to DNA interstrand cross-linking agents such as cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51', entrez:'5888'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5888', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'544', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22761336', evidence_text:\"The DNA repair protein RAD51 and the trans-lesion synthesis DNA polymerase REV1 are required for resistance to DNA interstrand cross-linking agents such as cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REV1', entrez:'51455'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'51455', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'545', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22761336', evidence_text:\"The DNA repair protein RAD51 and the trans-lesion synthesis DNA polymerase REV1 are required for resistance to DNA interstrand cross-linking agents such as cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REV1', entrez:'51455'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'51455', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'546', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22761336', evidence_text:\"The DNA repair protein RAD51 and the trans-lesion synthesis DNA polymerase REV1 are required for resistance to DNA interstrand cross-linking agents such as cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51', entrez:'5888'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5888', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'547', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22761336', evidence_text:\"The DNA repair protein RAD51 and the trans-lesion synthesis DNA polymerase REV1 are required for resistance to DNA interstrand cross-linking agents such as cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51', entrez:'5888'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'5888', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'548', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22761336', evidence_text:\"The DNA repair protein RAD51 and the trans-lesion synthesis DNA polymerase REV1 are required for resistance to DNA interstrand cross-linking agents such as cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note: Drug class relation with cisplatin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATP7B', entrez:'540'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'540', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'549', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15326162', evidence_text:\"The ectopic expression of ATP7A and ATP7B in cultured cells is associated with DDP resistance (14, 15), and the endogenous ATP7A or ATP7B proteins are over-expressed in certain DDP-resistant cell lines (16-18). It is thought that these copper transporters confer DDP resistance by transporting DDP into vesicular compartments, thereby sequestering the drug from cytotoxic targets (19). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATP7B', entrez:'540'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'540', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'550', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15326162', evidence_text:\"The ectopic expression of ATP7A and ATP7B in cultured cells is associated with DDP resistance (14, 15), and the endogenous ATP7A or ATP7B proteins are over-expressed in certain DDP-resistant cell lines (16-18). It is thought that these copper transporters confer DDP resistance by transporting DDP into vesicular compartments, thereby sequestering the drug from cytotoxic targets (19). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATP7A', entrez:'538'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'538', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'551', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15326162', evidence_text:\"The ectopic expression of ATP7A and ATP7B in cultured cells is associated with DDP resistance (14, 15), and the endogenous ATP7A or ATP7B proteins are over-expressed in certain DDP-resistant cell lines (16-18). It is thought that these copper transporters confer DDP resistance by transporting DDP into vesicular compartments, thereby sequestering the drug from cytotoxic targets (19). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATP7A', entrez:'538'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'538', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'552', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15326162', evidence_text:\"The ectopic expression of ATP7A and ATP7B in cultured cells is associated with DDP resistance (14, 15), and the endogenous ATP7A or ATP7B proteins are over-expressed in certain DDP-resistant cell lines (16-18). It is thought that these copper transporters confer DDP resistance by transporting DDP into vesicular compartments, thereby sequestering the drug from cytotoxic targets (19). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC31A2', entrez:'1318'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'1318', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'553', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19509135', evidence_text:\"The fact that modest reductions in the level of CTR2 protein were accompanied by quite large increases in sensitivity to both DDP and CBDCA suggests that CTR2 expression may also be a useful biomarker of initial responsiveness to therapy with these drugs in patients with ovarian cancer and that strategies directed at selectively modulating the expression of this transporter may be therapeutically useful. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SLC31A2', entrez:'1318'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'1318', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'554', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19509135', evidence_text:\"The fact that modest reductions in the level of CTR2 protein were accompanied by quite large increases in sensitivity to both DDP and CBDCA suggests that CTR2 expression may also be a useful biomarker of initial responsiveness to therapy with these drugs in patients with ovarian cancer and that strategies directed at selectively modulating the expression of this transporter may be therapeutically useful. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCD2', entrez:'2177'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2177', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'555', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"The observation that FANCC- and FANCDD2-deficient cell lines were significantly more sensitive to platinum drugs than FANCA-deficient cells. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCD2', entrez:'2177'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2177', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'556', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"The observation that FANCC- and FANCDD2-deficient cell lines were significantly more sensitive to platinum drugs than FANCA-deficient cells. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FANCD2', entrez:'2177'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2177', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'557', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20034732', evidence_text:\"The observation that FANCC- and FANCDD2-deficient cell lines were significantly more sensitive to platinum drugs than FANCA-deficient cells. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51C', entrez:'5889'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5889', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'558', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51C', entrez:'5889'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'5889', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'559', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51C', entrez:'5889'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5889', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'560', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PALB2', entrez:'79728'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'79728', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'561', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. Was listed as nucleotide change and not protein changed. The alteration was c.1050del4\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PALB2', entrez:'79728'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'79728', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'562', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. Was listed as nucleotide change and not protein changed. The alteration was c.1050del4\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PALB2', entrez:'79728'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'79728', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'563', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. Was listed as nucleotide change and not protein changed. The alteration was c.1050del4\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PALB2', entrez:'79728'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'79728', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'564', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NBN', entrez:'4683'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4683', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'565', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NBN', entrez:'4683'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4683', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'566', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NBN', entrez:'4683'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4683', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'567', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FAM175A', entrez:'84142'}), (a:Aberration { type:'small_insertion', value:'1106insG', gene_entrez:'84142', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'568', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FAM175A', entrez:'84142'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'84142', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'569', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FAM175A', entrez:'84142'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'84142', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'570', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FAM175A', entrez:'84142'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'84142', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'571', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK2', entrez:'11200'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'11200', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'572', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK2', entrez:'11200'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'11200', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'573', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK2', entrez:'11200'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'11200', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'574', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'575', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'576', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA1', entrez:'672'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'672', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'577', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA1', entrez:'672'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'672', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'578', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATM', entrez:'472'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'472', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'579', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATM', entrez:'472'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'472', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'580', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATM', entrez:'472'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'472', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'581', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24240112', evidence_text:\"The presence of a germline or somatic mutation in a homologous recombination gene was strongly asso- ciated with primary platinum sensitivity. Seventy-one of 85 (84%) primary carcinomas with a homologous recombination mutation (germline or somatic) demonstrated platinum sensitivity. In contrast, 95 of 158 (60%) carcinomas without an identified homologous recombination mutation had platinum sensitivity, and the remainder were either platinum resistant or refractory (P=0.0002). Germline homologous recombination mutations and somatic homologous recombination mutations were each separately predictive of platinum sensitivity com-pared with cases without homologous recombination mutations: 49 of 61 (80%) cases with a germline mutation were platinum sensitive (P=0.005), and 22 of 24 (92%) carcinomas with a somatic mutation were platinum sensitive (P=0.003; Fig. 3). Annotator note:  Mutation was taken from supplementary table 3. SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATP11B', entrez:'23200'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'23200', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'582', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23585472', evidence_text:\"The report the discovery and characterization of the role of ATP11B, a P-type ATPase membrane protein, in cisplatin resistance. We found that ATP11B expression was correlated with higher tumor grade in human ovarian cancer samples and with cisplatin resistance in human ovarian cancer cell lines. ATP11B gene silencing restored the sensitivity of ovarian cancer cell lines to cisplatin in vitro. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATP11B', entrez:'23200'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'23200', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'583', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23585472', evidence_text:\"The report the discovery and characterization of the role of ATP11B, a P-type ATPase membrane protein, in cisplatin resistance. We found that ATP11B expression was correlated with higher tumor grade in human ovarian cancer samples and with cisplatin resistance in human ovarian cancer cell lines. ATP11B gene silencing restored the sensitivity of ovarian cancer cell lines to cisplatin in vitro. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HDAC4', entrez:'9759'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9759', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'584', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21571862', evidence_text:\"Therefore, clinical selection of HDAC4-overexpressing tumor cells upon exposure to chemotherapy promotes STAT1 deacetylation and cancer cell survival. Together, our findings identify HDAC4 as a novel, therapeutically tractable target to counter platinum resistance in ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HDAC4', entrez:'9759'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9759', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'585', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21571862', evidence_text:\"Therefore, clinical selection of HDAC4-overexpressing tumor cells upon exposure to chemotherapy promotes STAT1 deacetylation and cancer cell survival. Together, our findings identify HDAC4 as a novel, therapeutically tractable target to counter platinum resistance in ovarian cancer. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Ovarian cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH2', entrez:'4436'}), (a:Aberration { type:'snv', value:'G674#', gene_entrez:'4436', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'586', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14744764', evidence_text:\"These results are consistent with the observations made in the MEF cells and demonstrate that although the Msh2G674A mutation has a severe impact on DNA mismatch repair, it still confers a wild-type-specific response to cisplatin-induced DNA damage. In this regard, Msh2G674A is a separation of function mutation. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH2', entrez:'4436'}), (a:Aberration { type:'snv', value:'G674A', gene_entrez:'4436', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'587', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14744764', evidence_text:\"These results are consistent with the observations made in the MEF cells and demonstrate that although the Msh2G674A mutation has a severe impact on DNA mismatch repair, it still confers a wild-type-specific response to cisplatin-induced DNA damage. In this regard, Msh2G674A is a separation of function mutation. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XPA', entrez:'7507'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7507', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'588', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/10074455', evidence_text:\"These results show that a low XPA level in the testis tumour cell lines is sufficient to explain their poor ability to remove cisplatin adducts from DNA and might be a major reason for the high cisplatin sensitivity of testis tumours. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Testicular cancer.   \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XPA', entrez:'7507'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7507', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'589', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/10074455', evidence_text:\"These results show that a low XPA level in the testis tumour cell lines is sufficient to explain their poor ability to remove cisplatin adducts from DNA and might be a major reason for the high cisplatin sensitivity of testis tumours. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Testicular cancer.   \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XPA', entrez:'7507'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7507', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'590', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/10074455', evidence_text:\"These results show that a low XPA level in the testis tumour cell lines is sufficient to explain their poor ability to remove cisplatin adducts from DNA and might be a major reason for the high cisplatin sensitivity of testis tumours. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Testicular cancer.   |Annotator Note:  SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GCLM', entrez:'2730'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2730', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'591', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16012725', evidence_text:\"These results suggest that the overexpression of GCLM is correlated with CDDP-resistance in SCLC xenografts in vivo. |Evidence Level:  Pre-clinical. Xenograft. |Disease:  NSCLC. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GCLM', entrez:'2730'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2730', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'592', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16012725', evidence_text:\"These results suggest that the overexpression of GCLM is correlated with CDDP-resistance in SCLC xenografts in vivo.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NOTCH3', entrez:'4854'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4854', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'593', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23019585', evidence_text:\"This study demonstrates that the Notch signaling pathway and Notch3 in particular are critical for the regulation of CSCs and tumor resistance to platinum. |Evidence Level:  Pre-clinical. Xenograft. Cell line. |Disease:  Ovarian Cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NOTCH3', entrez:'4854'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4854', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'594', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23019585', evidence_text:\"This study demonstrates that the Notch signaling pathway and Notch3 in particular are critical for the regulation of CSCs and tumor resistance to platinum. |Evidence Level:  Pre-clinical. Xenograft. Cell line. |Disease:  Ovarian Cancer. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NOTCH3', entrez:'4854'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4854', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'595', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23019585', evidence_text:\"This study demonstrates that the Notch signaling pathway and Notch3 in particular are critical for the regulation of CSCs and tumor resistance to platinum. |Evidence Level:  Pre-clinical. Xenograft. Cell line. |Disease:  Ovarian Cancer. |Annotator Note:  SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity. Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA1', entrez:'672'}), (a:Aberration { type:'snv', value:'C61G', gene_entrez:'672', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'596', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22172724', evidence_text:\"Tumors carrying the Brca1(C61G) mutation responded poorly to platinum drugs. |Evidence Level:  Pre-clinical. Mouse model. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA1', entrez:'672'}), (a:Aberration { type:'snv', value:'C61#', gene_entrez:'672', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'597', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22172724', evidence_text:\"Tumors carrying the Brca1(C61G) mutation responded poorly to platinum drugs. |Evidence Level:  Pre-clinical. Mouse model. |Annotator Note:  SNVs at this position are associated with loss-of-function and reduced drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'A349#', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'598', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20639400', evidence_text:\"We conclude that expression of the FANCJ-A349P protein encoded by the mutant allele exerts a dominant-negative effect on cell survival or DNA damage accumulation after treatment with a DNA cross-linking agent. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin. SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'R251C', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'599', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24573678', evidence_text:\"Compared with fancj+/+ cells expressing GFP-tagged FANCJ-WT or transfected with empty vector, fancj+/+ cells expressing FANCJ-R251C or -Q255H showed reduced survival as a function of cisplatin dose and were ?50-fold more sensitive to the highest concentration of cisplatin (1 uM). |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'R251#', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'600', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24573678', evidence_text:\"Compared with fancj+/+ cells expressing GFP-tagged FANCJ-WT or transfected with empty vector, fancj+/+ cells expressing FANCJ-R251C or -Q255H showed reduced survival as a function of cisplatin dose and were ?50-fold more sensitive to the highest concentration of cisplatin (1 uM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'Q255H', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'601', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24573678', evidence_text:\"Compared with fancj+/+ cells expressing GFP-tagged FANCJ-WT or transfected with empty vector, fancj+/+ cells expressing FANCJ-R251C or -Q255H showed reduced survival as a function of cisplatin dose and were ?50-fold more sensitive to the highest concentration of cisplatin (1 uM). |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'Q255#', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'602', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24573678', evidence_text:\"Compared with fancj+/+ cells expressing GFP-tagged FANCJ-WT or transfected with empty vector, fancj+/+ cells expressing FANCJ-R251C or -Q255H showed reduced survival as a function of cisplatin dose and were ?50-fold more sensitive to the highest concentration of cisplatin (1 uM). |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRIP1', entrez:'83990'}), (a:Aberration { type:'snv', value:'A349P', gene_entrez:'83990', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'603', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20639400', evidence_text:\"We conclude that expression of the FANCJ-A349P protein encoded by the mutant allele exerts a dominant-negative effect on cell survival or DNA damage accumulation after treatment with a DNA cross-linking agent.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FAU', entrez:'2197'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2197', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'604', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19830698', evidence_text:\"We demonstrate that the expression of FAU was reduced in the A2780cis (cisplatin resistant subclone of A2780) cell line compared with the A2780 ovarian cancer cell line, and was directly related to the cell line's sensitivity to carboplatin. Downregulation of FAU in the A2780 cell line by transfection with two predesigned short-interfering RNAs (siRNAs) to FAU resulted in a significant increase in resistance to carboplatin-induced cell death. Transfection of the A2780cis cell line with the pcDNA3 plasmid containing FAU was associated with increased sensitivity to carboplatin-induced apoptosis, with decreased cell viability and increased apoptosis. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FAU', entrez:'2197'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2197', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'605', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19830698', evidence_text:\"We demonstrate that the expression of FAU was reduced in the A2780cis (cisplatin resistant subclone of A2780) cell line compared with the A2780 ovarian cancer cell line, and was directly related to the cell line's sensitivity to carboplatin. Downregulation of FAU in the A2780 cell line by transfection with two predesigned short-interfering RNAs (siRNAs) to FAU resulted in a significant increase in resistance to carboplatin-induced cell death. Transfection of the A2780cis cell line with the pcDNA3 plasmid containing FAU was associated with increased sensitivity to carboplatin-induced apoptosis, with decreased cell viability and increased apoptosis. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RPS6KA3', entrez:'6197'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'6197', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'606', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108403', evidence_text:\"we found that A375 cells that stably knockdown RSK1/2 expression are more susceptible to DNA-damaging agents (Figure 7f), suggesting that RSK may simultaneously regulate cell survival and proliferation. These results were corroborated by the fact that A375 cells treated with RSK inhibitors are more susceptible to a higher dose of DTIC (30uM) (Figure 7g), indicating that RSK inhibition sensitizes melanoma cells to the cytostatic and cytotoxic effects of DNA-damaging agents. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RPS6KA3', entrez:'6197'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'6197', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'607', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108403', evidence_text:\"we found that A375 cells that stably knockdown RSK1/2 expression are more susceptible to DNA-damaging agents (Figure 7f), suggesting that RSK may simultaneously regulate cell survival and proliferation. These results were corroborated by the fact that A375 cells treated with RSK inhibitors are more susceptible to a higher dose of DTIC (30uM) (Figure 7g), indicating that RSK inhibition sensitizes melanoma cells to the cytostatic and cytotoxic effects of DNA-damaging agents. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RPS6KA1', entrez:'6195'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'6195', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'608', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108403', evidence_text:\"we found that A375 cells that stably knockdown RSK1/2 expression are more susceptible to DNA-damaging agents (Figure 7f), suggesting that RSK may simultaneously regulate cell survival and proliferation. These results were corroborated by the fact that A375 cells treated with RSK inhibitors are more susceptible to a higher dose of DTIC (30uM) (Figure 7g), indicating that RSK inhibition sensitizes melanoma cells to the cytostatic and cytotoxic effects of DNA-damaging agents. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RPS6KA1', entrez:'6195'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'6195', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'609', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108403', evidence_text:\"we found that A375 cells that stably knockdown RSK1/2 expression are more susceptible to DNA-damaging agents (Figure 7f), suggesting that RSK may simultaneously regulate cell survival and proliferation. These results were corroborated by the fact that A375 cells treated with RSK inhibitors are more susceptible to a higher dose of DTIC (30uM) (Figure 7g), indicating that RSK inhibition sensitizes melanoma cells to the cytostatic and cytotoxic effects of DNA-damaging agents. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD52', entrez:'5893'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5893', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'610', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23209746', evidence_text:\"We found that both the rs1051669 variant and RAD52 protein expression were significantly associated with carboplatin resistance (P_=_0.024 and 0.028, respectively) and rs10774474 with nedaplatin resistance (P_=_0.018). The rs1051669 variant was significantly associated with RAD52 protein expression (adjusted OR_=_4.7, 95% CI_=_1.4-16.1, P_=_0.013). When these three RAD52 variants were combined, progression-free survival was lower in patients who carried at least one (>=1) variant allele compared to those without any of the variant alleles (P_=_0.047). Therefore, both RAD52 variants and protein expression can predict platinum resistance, and RAD52 variants appeared to predict prognosis in cervical cancer patients. |Evidence Level:  Clinical. |Disease:  Cervical cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD52', entrez:'5893'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5893', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'611', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23209746', evidence_text:\"We found that both the rs1051669 variant and RAD52 protein expression were significantly associated with carboplatin resistance (P_=_0.024 and 0.028, respectively) and rs10774474 with nedaplatin resistance (P_=_0.018). The rs1051669 variant was significantly associated with RAD52 protein expression (adjusted OR_=_4.7, 95% CI_=_1.4-16.1, P_=_0.013). When these three RAD52 variants were combined, progression-free survival was lower in patients who carried at least one (>=1) variant allele compared to those without any of the variant alleles (P_=_0.047). Therefore, both RAD52 variants and protein expression can predict platinum resistance, and RAD52 variants appeared to predict prognosis in cervical cancer patients. |Evidence Level:  Clinical. |Disease:  Cervical cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD52', entrez:'5893'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5893', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'612', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23209746', evidence_text:\"We found that both the rs1051669 variant and RAD52 protein expression were significantly associated with carboplatin resistance (P_=_0.024 and 0.028, respectively) and rs10774474 with nedaplatin resistance (P_=_0.018). The rs1051669 variant was significantly associated with RAD52 protein expression (adjusted OR_=_4.7, 95% CI_=_1.4-16.1, P_=_0.013). When these three RAD52 variants were combined, progression-free survival was lower in patients who carried at least one (>=1) variant allele compared to those without any of the variant alleles (P_=_0.047). Therefore, both RAD52 variants and protein expression can predict platinum resistance, and RAD52 variants appeared to predict prognosis in cervical cancer patients. |Evidence Level:  Clinical. |Disease:  Cervical cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'COPS5', entrez:'10987'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'10987', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'613', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22797071', evidence_text:\"We found that Jab1 was overexpressed in two relatively cisplatin-, IR- and UV-resistant NPC cell lines, and knocking down its expression conferred sensitivity to cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'COPS5', entrez:'10987'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'10987', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'614', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22797071', evidence_text:\"We found that Jab1 was overexpressed in two relatively cisplatin-, IR- and UV-resistant NPC cell lines, and knocking down its expression conferred sensitivity to cisplatin. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  Drug class relation with cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLNA', entrez:'2316'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2316', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'615', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22051193', evidence_text:\"We found that reduction of filamin-A sensitizes cancer cells to chemotherapy reagents bleomycin and cisplatin, delays the repair of not only DSBs but also single strand breaks (SSBs) and interstrand crosslinks (ICLs), and increases chromosome breaks after the drug treatment. The level of filamin-A expression can be used as a biomarker to predict melanoma sensitivity to therapeutic DNA damages. We further inhibited the expression of filamin-A in melanoma cells, and found that this confers an increased sensitivity to bleomycin and cisplatin treatment in a mouse xenograft tumor model. |Evidence Level:  Pre-clinical. Xenograft. Cell line. |Disease:  Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FLNA', entrez:'2316'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2316', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'616', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22051193', evidence_text:\"We found that reduction of filamin-A sensitizes cancer cells to chemotherapy reagents bleomycin and cisplatin, delays the repair of not only DSBs but also single strand breaks (SSBs) and interstrand crosslinks (ICLs), and increases chromosome breaks after the drug treatment. The level of filamin-A expression can be used as a biomarker to predict melanoma sensitivity to therapeutic DNA damages. We further inhibited the expression of filamin-A in melanoma cells, and found that this confers an increased sensitivity to bleomycin and cisplatin treatment in a mouse xenograft tumor model. |Evidence Level:  Pre-clinical. Xenograft. Cell line. |Disease:  Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC1', entrez:'7515'}), (a:Aberration { type:'snv', value:'R194#', gene_entrez:'7515', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'617', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19157633', evidence_text:\"We found that there was a significantly increased chance of treatment response to platinum-based chemotherapy with the XRCC1 194Arg/Trp genotype (odds ratio 0.429; 95% CI 0.137-1.671; P=0.035). |Evidence Level:  Clinical. |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC1', entrez:'7515'}), (a:Aberration { type:'snv', value:'R194W', gene_entrez:'7515', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'618', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19157633', evidence_text:\"We found that there was a significantly increased chance of treatment response to platinum-based chemotherapy with the XRCC1 194Arg/Trp genotype (odds ratio 0.429; 95% CI 0.137-1.671; P=0.035). |Evidence Level:  Clinical. |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NFE2L2', entrez:'4780'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4780', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'619', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18829555', evidence_text:\"we show that constitutive activation of Nrf2 in lung cancer cells promotes tumorigenicity and contributes to chemoresistance by up-regulation of glutathione, thioredoxin... Inhibiting Nrf2 expression using naked siRNA duplexes in combination with carboplatin significantly inhibits tumor growth in a subcutaneous model of lung cancer. Thus, targeting Nrf2 activity in lung cancers, particularly those with Keap1 mutations, could be a promising strategy to inhibit tumor growth and circumvent chemoresistance. |Evidence Level:  Pre-clinical. Xenograft |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NFE2L2', entrez:'4780'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4780', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'620', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18829555', evidence_text:\"we show that constitutive activation of Nrf2 in lung cancer cells promotes tumorigenicity and contributes to chemoresistance by up-regulation of glutathione, thioredoxin... Inhibiting Nrf2 expression using naked siRNA duplexes in combination with carboplatin significantly inhibits tumor growth in a subcutaneous model of lung cancer. Thus, targeting Nrf2 activity in lung cancers, particularly those with Keap1 mutations, could be a promising strategy to inhibit tumor growth and circumvent chemoresistance. |Evidence Level:  Pre-clinical. Xenograft |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PARP1', entrez:'142'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'142', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'621', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23554447', evidence_text:\"we show that PARP1 is highly expressed and constitutively hyperactivated in a majority of human CDDP-resistant cancer cells of distinct histologic origin. Cells manifesting elevated intracellular levels of poly(ADP-ribosyl)ated proteins (PAR(high)) responded to pharmacologic PARP inhibitors as well as to PARP1-targeting siRNAs by initiating a DNA damage response that translated into cell death following the activation of the intrinsic pathway of apoptosis. |Evidence Level:  Pre-clinical. Xenograft |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PARP1', entrez:'142'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'142', entity_class:'modifier' }), (d:Drug {name:'carboplatin'} ) Merge (r:Rule {rulenum:'622', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23554447', evidence_text:\"we show that PARP1 is highly expressed and constitutively hyperactivated in a majority of human CDDP-resistant cancer cells of distinct histologic origin. Cells manifesting elevated intracellular levels of poly(ADP-ribosyl)ated proteins (PAR(high)) responded to pharmacologic PARP inhibitors as well as to PARP1-targeting siRNAs by initiating a DNA damage response that translated into cell death following the activation of the intrinsic pathway of apoptosis. |Evidence Level:  Pre-clinical. Xenograft |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ROS1', entrez:'6098'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6098', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'623', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22215748', evidence_text:\"In vitro studies assessed the responsiveness of cells with ROS1 rearrangement to the tyrosine kinase inhibitor crizotinib. |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'624', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22553343', evidence_text:\"autophosphorylation of EGF receptor (EGFR), were shown to be responsible for acquired resistance to crizotinib in ALK translocated cancers. |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ROS1', entrez:'6098'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6098', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'625', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22215748', evidence_text:\"Is target. Inferrence from work on targeting fusions with ROS1.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ROS1', entrez:'6098'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6098', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'626', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22215748', evidence_text:\"Is target. Inferrence from work on targeting fusions with ROS1.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'627', evidence:'inferred', pmid:'http://1.usa.gov/1Gb1zQ6', evidence_text:\"Crizotinib, in an ATP-competitive manner, binds to and inhibits ALK kinase and ALK fusion proteins. In addition, crizotinib inhibits c-Met kinase, and disrupts the c-Met signaling pathway. |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'628', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22042947', evidence_text:\"Two of four patients with MET-amplified tumors treated with crizotinib experienced tumor shrinkage (-30% and -16%) and experienced progression after 3.7 and 3.5 months. MET amplification defines a small and aggressive subset of GEC with indications of transient sensitivity to the targeted MET inhibitor crizotinib (PF-02341066). |Evidence Level:  Retrospective Clinical Study.  |Disease: gastroesophageal cancer.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1094#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'629', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"In these studies, PF-2341066 exhibited improved or similar activity against NIH3T3 cells engineered to express c-Met ATP-binding site mutants V1092I (IC50, 19 nmol/L) or H1094R (IC50, 2 nmol/L) or the P-loop mutant M1250T (IC50, 15 nmol/L) compared with NIH3T3 cells expressing wild-type receptor (IC50, 13 nmol/L). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1094R', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'630', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"In these studies, PF-2341066 exhibited improved or similar activity against NIH3T3 cells engineered to express c-Met ATP-binding site mutants V1092I (IC50, 19 nmol/L) or H1094R (IC50, 2 nmol/L) or the P-loop mutant M1250T (IC50, 15 nmol/L) compared with NIH3T3 cells expressing wild-type receptor (IC50, 13 nmol/L). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1250#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'631', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"In these studies, PF-2341066 exhibited improved or similar activity against NIH3T3 cells engineered to express c-Met ATP-binding site mutants V1092I (IC50, 19 nmol/L) or H1094R (IC50, 2 nmol/L) or the P-loop mutant M1250T (IC50, 15 nmol/L) compared with NIH3T3 cells expressing wild-type receptor (IC50, 13 nmol/L). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1250T', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'632', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"In these studies, PF-2341066 exhibited improved or similar activity against NIH3T3 cells engineered to express c-Met ATP-binding site mutants V1092I (IC50, 19 nmol/L) or H1094R (IC50, 2 nmol/L) or the P-loop mutant M1250T (IC50, 15 nmol/L) compared with NIH3T3 cells expressing wild-type receptor (IC50, 13 nmol/L). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'N375#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'633', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21716144', evidence_text:\"This mutation is localized to the ligand-binding semaphorin domain of MET and was found to be associated with loss of affinity of the receptor for HGF,10 although its biological consequences remain largely unclear. We have now shown that crizotinib had little effect on signal transduction or cell survival in H2122 cells, consistent with the previous observation that transient expression of the N375S mutant of MET did not result in an increased susceptibility of the transfected cells to a MET-TKI compared with that of cells expressing wild-type MET\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'N375S', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'634', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21716144', evidence_text:\"This mutation is localized to the ligand-binding semaphorin domain of MET and was found to be associated with loss of affinity of the receptor for HGF,10 although its biological consequences remain largely unclear. We have now shown that crizotinib had little effect on signal transduction or cell survival in H2122 cells, consistent with the previous observation that transient expression of the N375S mutant of MET did not result in an increased susceptibility of the transfected cells to a MET-TKI compared with that of cells expressing wild-type MET\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'635', evidence:'inferred', pmid:'http://www.cancer.gov/drugdictionary?CdrID=586080', evidence_text:\"Is Target. An orally available aminopyridine-based inhibitor of the receptor tyrosine kinase anaplastic lymphoma kinase (ALK) and the c-Met/hepatocyte growth factor receptor (HGFR) with antineoplastic activity. crizotinib, in an ATP-competitive manner, binds to and inhibits ALK kinase and ALK fusion proteins. In addition, crizotinib inhibits c-Met kinase, and disrupts the c-Met signaling pathway.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'R988#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'636', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"PF-2341066 also potently inhibited c-Met phosphor- ylation in NCI-H69 (IC50, 13 nmol/L) and HOP92 (IC50, 16 nmol/L) cells, which express the endogenous c-Met variants R988C and T1010I, respectively.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'R988C', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'637', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"PF-2341066 also potently inhibited c-Met phosphor- ylation in NCI-H69 (IC50, 13 nmol/L) and HOP92 (IC50, 16 nmol/L) cells, which express the endogenous c-Met variants R988C and T1010I, respectively.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'T1010#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'638', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"PF-2341066 also potently inhibited c-Met phosphor- ylation in NCI-H69 (IC50, 13 nmol/L) and HOP92 (IC50, 16 nmol/L) cells, which express the endogenous c-Met variants R988C and T1010I, respectively.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'T1010I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'639', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"PF-2341066 also potently inhibited c-Met phosphor- ylation in NCI-H69 (IC50, 13 nmol/L) and HOP92 (IC50, 16 nmol/L) cells, which express the endogenous c-Met variants R988C and T1010I, respectively.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1092#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'640', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"In these studies, PF-2341066 exhibited improved or similar activity against NIH3T3 cells engineered to express c-Met ATP-binding site mutants V1092I (IC50, 19 nmol/L) or H1094R (IC50, 2 nmol/L) or the P-loop mutant M1250T (IC50, 15 nmol/L) compared with NIH3T3 cells expressing wild-type receptor (IC50, 13 nmol/L). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1092I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'641', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"In these studies, PF-2341066 exhibited improved or similar activity against NIH3T3 cells engineered to express c-Met ATP-binding site mutants V1092I (IC50, 19 nmol/L) or H1094R (IC50, 2 nmol/L) or the P-loop mutant M1250T (IC50, 15 nmol/L) compared with NIH3T3 cells expressing wild-type receptor (IC50, 13 nmol/L). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'642', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"One mechanism, observed by modeling resistance both in vitro and in vivo, involved the acquisition of a mutation in the MET activation loop (Y1230). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230C', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'643', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"Assessment of single clones of cDNA iso- lated from the M1 cell lined showed 2 different mutations in Tyr1230 in the resistant cancers Y1230H (TAT ! CAT) and Y1230C (TAT ! TGT). We derived cell lines from single-cell clones from the M1 cell line and assessed 15 of the derived clones. Three clones had no mutations in MET (wt/wt), 9 harbored MET Y1230H mutations, and 3 harbored Y1230C mutations (Fig. 5E). All of the clones harboring mutations in MET maintained resistance to PHA-665752 in vitro (Supple- mentary Fig. S9A).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230H', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'644', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"One mechanism, observed by modeling resistance both in vitro and in vivo, involved the acquisition of a mutation in the MET activation loop (Y1230). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1235#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'645', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"In contrast, a marked shift in potency was observed against cells engineered to express c-Met activation loop mutants Y1230C (IC50, 127 nmol/L) and Y1235D (IC50, 92 nmol/L) compared with wild- type receptor.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1235D', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'646', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483355', evidence_text:\"In contrast, a marked shift in potency was observed against cells engineered to express c-Met activation loop mutants Y1230C (IC50, 127 nmol/L) and Y1235D (IC50, 92 nmol/L) compared with wild- type receptor.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'647', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20841479', evidence_text:\"To predict mechanisms of acquired resistance, we generated resistant cells by treating MET-addicted cells with increasing concentrations of the MET small-molecule inhibitors PHA-665752 or JNJ38877605. Resistant cells displayed MET gene amplification, leading to increased expression and constitutive phosphorylation of MET, followed by subsequent amplification and overexpression of wild-type (wt) KRAS. Cells harboring KRAS amplification progressively lost their MET dependence and acquired KRAS dependence. FISH analysis showed that the extra copies of KRAS were progressively acquired as double minutes outside chromosome 12 (where KRAS is positioned; Fig. 3D). Our results suggest that MET and KRAS amplification is a general mechanism of resistance to specific MET inhibitors given that similar results were observed with two small inhibitors and in different cell lines of different histotypes. To our knowledge, this is the first report showing that overexpression of wt KRAS can overcome the inhibitory effect of a RTK inhibitor. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TGFA', entrez:'7039'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7039', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'648', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"We observed that exogenous TGFa was indeed sufficient to promote marked resistance to MET inhibition, but resis- tance was overcome by combined inhibition of MET and EGFR \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TGFA', entrez:'7039'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7039', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'649', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:\"We observed that exogenous TGFa was indeed sufficient to promote marked resistance to MET inhibition, but resis- tance was overcome by combined inhibition of MET and EGFR \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G12#', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'650', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22235099', evidence_text:\"KRAS mutations as a mechanism for crizotinib resistance (Table 2)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G12C', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'651', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22235099', evidence_text:\"KRAS mutations as a mechanism for crizotinib resistance (Table 2)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G12V', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'652', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22235099', evidence_text:\"KRAS mutations as a mechanism for crizotinib resistance (Table 2)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3815', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'653', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22277784', evidence_text:\"In two of the 18 resistant specimens in the Massachusetts General Hospital series, we discovered high-level c-KIT gene amplification by FISH.35 Overexpression of c-KIT was confirmed by IHC. In addition, by IHC, there was also increased expression of the c-KIT ligand stem-cell factor (SCF) in the stromal cells of the solid compo- nent of the resistant specimens. Laboratory experiments confirmed that c-KIT overexpression required SCF to promote resistance, and resistance could be overcome by combined imatinib and crizotinib treatment. |Evidence Level:  Case Study. |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3815', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'654', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22277784', evidence_text:\"In two of the 18 resistant specimens in the Massachusetts General Hospital series, we discovered high-level c-KIT gene amplification by FISH.35 Overexpression of c-KIT was confirmed by IHC. In addition, by IHC, there was also increased expression of the c-KIT ligand stem-cell factor (SCF) in the stromal cells of the solid compo- nent of the resistant specimens. Laboratory experiments confirmed that c-KIT overexpression required SCF to promote resistance, and resistance could be overcome by combined imatinib and crizotinib treatment. |Evidence Level:  Case Study. |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'655', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22553343', evidence_text:\"autophosphorylation of EGF receptor (EGFR), were shown to be responsible for acquired resistance to crizotinib in ALK_translocated cancers\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'656', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22553343', evidence_text:\"autophosphorylation of EGF receptor (EGFR), were shown to be responsible for acquired resistance to crizotinib in ALK_translocated cancers\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HGF', entrez:'3082'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3082', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'657', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22683780', evidence_text:\"However, leukemic cells treated with the specific MET kinase inhibitor crizotinib developed resistance resulting from compensatory upregulation of HGF expression, leading to the restoration of MET signaling.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HGF', entrez:'3082'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3082', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'658', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22683780', evidence_text:\"However, leukemic cells treated with the specific MET kinase inhibitor crizotinib developed resistance resulting from compensatory upregulation of HGF expression, leading to the restoration of MET signaling.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ROS1', entrez:'6098'}), (a:Aberration { type:'snv', value:'C2060#', gene_entrez:'6098', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'659', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24218589', evidence_text:\"To preemptively identify ROS1 kinase domain mutations conferring resistance to crizotinib, we performed an N-ethyl-N-nitrosourea (ENU)-assisted accelerated mutagenesis screen using Ba/F3 FIG-ROS cells in the presence of increasing concentrations of crizotinib (500-2,500 nM). Six different ROS1 kinase domain point mutations were identified either alone or in combination, the two most frequent being V2098I and G1971E (Fig. 4A).  The L1947R, L1982F, and V2098I mutations map close to structural features that are implicated in inhibitor binding (nucleotide binding loop, helix alphaC, and activation loop, respectively). Importantly, mutations of L1152 in ALK, a residue homologous to L1982 in ROS1, confer crizotinib resistance and this mutation was recently identified in a patient with crizotinib-treated NSCLC with clinical resistance. Also, DFG motif-proximal mutations in ALK exhibit increased resistance to crizotinib. The mutations G1971E and C2060G map to locations that are distant to the inhibitor binding site in the N-terminal and C-terminal lobes, respectively.  We generated each of the six most frequently recovered crizotinib-resistant mutants (V2098I, G1971E, L1982F, C2060G, L1947R, and E1935G) in Ba/F3 FIG-ROS cells and confirmed their ability to confer IL-3 independence similar to wild-type FIG-ROS (Fig. S5E). As expected, all tested FIG-ROS mutants exhibited increased resistance to crizotinib (IC50 range: 350-2,450 nM; Fig. 4B and Fig. S6 A and C). IC50 values of the FIG-ROS mutants for crizotinib were close to or exceeded the clinically achievable maximum plasma concentration (411 ng/mL; 913 nM), suggesting that some of these mutants may be inefficiently inhibited at physiologically relevant concentrations. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ROS1', entrez:'6098'}), (a:Aberration { type:'snv', value:'C2060G', gene_entrez:'6098', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'660', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24218589', evidence_text:\"To preemptively identify ROS1 kinase domain mutations conferring resistance to crizotinib, we performed an N-ethyl-N-nitrosourea (ENU)-assisted accelerated mutagenesis screen using Ba/F3 FIG-ROS cells in the presence of increasing concentrations of crizotinib (500-2,500 nM). Six different ROS1 kinase domain point mutations were identified either alone or in combination, the two most frequent being V2098I and G1971E (Fig. 4A).  The L1947R, L1982F, and V2098I mutations map close to structural features that are implicated in inhibitor binding (nucleotide binding loop, helix alphaC, and activation loop, respectively). Importantly, mutations of L1152 in ALK, a residue homologous to L1982 in ROS1, confer crizotinib resistance and this mutation was recently identified in a patient with crizotinib-treated NSCLC with clinical resistance. Also, DFG motif-proximal mutations in ALK exhibit increased resistance to crizotinib. The mutations G1971E and C2060G map to locations that are distant to the inhibitor binding site in the N-terminal and C-terminal lobes, respectively.  We generated each of the six most frequently recovered crizotinib-resistant mutants (V2098I, G1971E, L1982F, C2060G, L1947R, and E1935G) in Ba/F3 FIG-ROS cells and confirmed their ability to confer IL-3 independence similar to wild-type FIG-ROS (Fig. S5E). As expected, all tested FIG-ROS mutants exhibited increased resistance to crizotinib (IC50 range: 350-2,450 nM; Fig. 4B and Fig. S6 A and C). IC50 values of the FIG-ROS mutants for crizotinib were close to or exceeded the clinically achievable maximum plasma concentration (411 ng/mL; 913 nM), suggesting that some of these mutants may be inefficiently inhibited at physiologically relevant concentrations. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'661', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21154129', evidence_text:\"|Annotator Note:  Is target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'662', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21154129', evidence_text:\"Is target.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'C1156#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'663', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21791641', evidence_text:\"To date 3 secondary mutations, all identified from crizotinib treated NSCLC or IMT patients, have been reported (14, 15). These mutations either involve the 'gatekeeper' residue (L1196) or sites away from critoztinib binding (F1174L and C1156Y) (14, 15).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'F1174#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'664', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21791641', evidence_text:\"To date 3 secondary mutations, all identified from crizotinib treated NSCLC or IMT patients, have been reported (14, 15). These mutations either involve the 'gatekeeper' residue (L1196) or sites away from critoztinib binding (F1174L and C1156Y) (14, 15).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'L1152#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'665', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21791641', evidence_text:\"Inferred from EML4-ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'C1156Y', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'666', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21791641', evidence_text:\"To date 3 secondary mutations, all identified from crizotinib treated NSCLC or IMT patients, have been reported (14, 15). These mutations either involve the 'gatekeeper' residue (L1196) or sites away from critoztinib binding (F1174L and C1156Y) (14, 15).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'F1174L', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'667', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21791641', evidence_text:\"To date 3 secondary mutations, all identified from crizotinib treated NSCLC or IMT patients, have been reported (14, 15). These mutations either involve the 'gatekeeper' residue (L1196) or sites away from critoztinib binding (F1174L and C1156Y) (14, 15).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'L1152R', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'668', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21791641', evidence_text:\"Inferred from EML4-ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'R1129#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'669', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21838707', evidence_text:\"Proliferation of all four hALK mutants was inhibited by crizotinib (Fig. 3A). Subsequent IC50 calculations of this inhibitor demonstrated that ALK F1174L and ALK R1192P required significantly high dose of crizotinib as compared to ALK G1128A and ALK R1245C.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'R1245#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'670', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21838707', evidence_text:\"Proliferation of all four hALK mutants was inhibited by crizotinib (Fig. 3A). Subsequent IC50 calculations of this inhibitor demonstrated that ALK F1174L and ALK R1192P required significantly high dose of crizotinib as compared to ALK G1128A and ALK R1245C.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'G1128#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'671', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21838707', evidence_text:\"Proliferation of all four hALK mutants was inhibited by crizotinib (Fig. 3A). Subsequent IC50 calculations of this inhibitor demonstrated that ALK F1174L and ALK R1192P required significantly high dose of crizotinib as compared to ALK G1128A and ALK R1245C.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'G1128A', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'672', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21838707', evidence_text:\"Proliferation of all four hALK mutants was inhibited by crizotinib (Fig. 3A). Subsequent IC50 calculations of this inhibitor demonstrated that ALK F1174L and ALK R1192P required significantly high dose of crizotinib as compared to ALK G1128A and ALK R1245C.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'R1192P', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'673', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21838707', evidence_text:\"Proliferation of all four hALK mutants was inhibited by crizotinib (Fig. 3A). Subsequent IC50 calculations of this inhibitor demonstrated that ALK F1174L and ALK R1192P required significantly high dose of crizotinib as compared to ALK G1128A and ALK R1245C.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'R1245C', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'674', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21838707', evidence_text:\"Proliferation of all four hALK mutants was inhibited by crizotinib (Fig. 3A). Subsequent IC50 calculations of this inhibitor demonstrated that ALK F1174L and ALK R1192P required significantly high dose of crizotinib as compared to ALK G1128A and ALK R1245C.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'D1230#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'675', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21948233', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'L1198#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'676', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21948233', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'D1203N', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'677', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21948233', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'L1198P', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'678', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21948233', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'S1206#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'679', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22034911', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'G1269S', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'680', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22034911', evidence_text:\"analogous Ba/F3 xenografts expressing L1196M, S1206R, or G1269S mutants were completely insensitive to these doses, with no statistically significant changes in tumor growth rate.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'S1206R', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'681', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22034911', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'R1275#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'682', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22072639', evidence_text:\"crizotinib inhibited proliferation of cell lines expressing either R1275Q-mutated ALK or amplified wild-type ALK.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'683', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22072639', evidence_text:\"crizotinib inhibited proliferation of cell lines expressing either R1275Q-mutated ALK or amplified wild-type ALK. (Annotator note, amplification of EML_ALK had been shown to lead to resistance PMID 22235099 Two patients, one with a resistance mutation, exhibited new onset ALK copy number gain (CNG), also in PMID: 22323827 review on crizotinib resistance, ALK amplification is associated with required resistance but also possibly as primary mechanism of resistance.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'R1275Q', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'684', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22072639', evidence_text:\"crizotinib inhibited proliferation of cell lines expressing either R1275Q-mutated ALK or amplified wild-type ALK.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'G1269#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'685', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22235099', evidence_text:\"A novel mutation in the ALK kinase domain, encoding a G1269A amino acid substitution that confers resistance to crizotinib in vitro, was identified in two of these cases\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'G1269A', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'686', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22235099', evidence_text:\"A novel mutation in the ALK kinase domain, encoding a G1269A amino acid substitution that confers resistance to crizotinib in vitro, was identified in two of these cases\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'G1202#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'687', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22277784', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'L1196#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'688', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22277784', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'small_insertion', value:'1151Tins', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'689', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22277784', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'G1202R', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'690', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22277784', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'L1196M', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'691', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22277784', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'S1206Y', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'692', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22277784', evidence_text:\"Inferred from EML4_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'I1171#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'693', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23239810', evidence_text:\"Sequencing of ALK kinase domain revealed that a single mutation became predominant at high crizotinib doses in each cell line, namely L1196Q and I1171N in Karpas-299 and SUP-M2 cells, respectively. These mutations also conferred resistance to crizotinib in Ba/F3 cells expressing human NPM- ALK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'I1171N', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'694', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23239810', evidence_text:\"Sequencing of ALK kinase domain revealed that a single mutation became predominant at high crizotinib doses in each cell line, namely L1196Q and I1171N in Karpas-299 and SUP-M2 cells, respectively. These mutations also conferred resistance to crizotinib in Ba/F3 cells expressing human NPM- ALK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'L1196Q', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'695', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23239810', evidence_text:\"Sequencing of ALK kinase domain revealed that a single mutation became predominant at high crizotinib doses in each cell line, namely L1196Q and I1171N in Karpas-299 and SUP-M2 cells, respectively. These mutations also conferred resistance to crizotinib in Ba/F3 cells expressing human NPM- ALK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'E1132#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'696', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'E1129#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'697', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'E1210#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'698', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'E1408#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'699', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'T1151#', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'700', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'E1129V', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'701', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'E1132K', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'702', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'E1210K', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'703', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'E1408K', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'704', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'F1174C', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'705', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'F1174I', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'706', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'F1174V', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'707', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ALK', entrez:'238'}), (a:Aberration { type:'snv', value:'T1151M', gene_entrez:'238', entity_class:'biomarker' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'708', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23533265', evidence_text:\"Inferred from NPM_ALK rule.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'crizotinib'} ) Merge (r:Rule {rulenum:'709', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22235099', evidence_text:\"KRAS mutations as a mechanism for crizotinib resistance (Table 2). |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'710', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21573016', evidence_text:\"found that FANCD1/BRCA2 small interference RNA efficiently enhanced cellular sensitivity toward ACNU and TMZ in human glioblastoma A172 cells. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATM', entrez:'328'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'328', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'711', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Having shown that both ATM and ATR determine the resistance of cells to temozolomide, we extended the study to glioblastoma and malignant melanoma cells, using the well-characterized lines LN229 (29) and D03 (28), respectively. This was accomplished by downregu- lation of ATM or ATR in these cells and determining the impact on their sensitivity to temozolomide. Knockdown of either ATM or ATR greatly enhanced the sensitivity of LN229 and D03 cells to temozolomide, with a significantly stronger sensitizing effect if ATR was downregulated. Simultaneous downregulation of ATM and ATR had no additional effect on the killing response. The data indicate that ATM and, even more, ATR, are targets for improving the anticancer effect of temozolomide. We should stress that the temozolomide doses applied in the experiments are in the therapeutic range, that is, up to 100 mmol/L, which corresponds to the drug serum level during ther- apy (43)...Because MGMT abolishes the increased sensitivity given by the impaired function of ATM or ATR, the therapeutic benefit is limited to patients with tumors that lack MGMT expression. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity. MGMT expression or presence should be compromised. WT MGMT will abrogate response when ATR/ATM is compromised.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATR', entrez:'545'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'545', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'712', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Having shown that both ATM and ATR determine the resistance of cells to temozolomide, we extended the study to glioblastoma and malignant melanoma cells, using the well-characterized lines LN229 (29) and D03 (28), respectively. This was accomplished by downregu- lation of ATM or ATR in these cells and determining the impact on their sensitivity to temozolomide. Knockdown of either ATM or ATR greatly enhanced the sensitivity of LN229 and D03 cells to temozolomide, with a significantly stronger sensitizing effect if ATR was downregulated. Simultaneous downregulation of ATM and ATR had no additional effect on the killing response. The data indicate that ATM and, even more, ATR, are targets for improving the anticancer effect of temozolomide. We should stress that the temozolomide doses applied in the experiments are in the therapeutic range, that is, up to 100 mmol/L, which corresponds to the drug serum level during ther- apy (43)...Because MGMT abolishes the increased sensitivity given by the impaired function of ATM or ATR, the therapeutic benefit is limited to patients with tumors that lack MGMT expression. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity. MGMT expression or presence should be compromised. WT MGMT will abrogate response when ATR/ATM is compromised.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MGMT', entrez:'4255'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4255', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'713', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11023546', evidence_text:\"MGMT(-/-) mice were considerably more sensitive to dacarbazine, a monofunctional triazene, than were wild-type mice, in terms of survival.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MGMT', entrez:'4255'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4255', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'714', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11023546', evidence_text:\"MGMT(-/-) mice were considerably more sensitive to dacarbazine, a monofunctional triazene, than were wild-type mice, in terms of survival.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MGMT', entrez:'4255'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4255', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'715', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11023546', evidence_text:\"MGMT(-/-) mice were considerably more sensitive to dacarbazine, a monofunctional triazene, than were wild-type mice, in terms of survival.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MGMT', entrez:'4255'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4255', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'716', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/11023546', evidence_text:\"MGMT(-/-) mice were considerably more sensitive to dacarbazine, a monofunctional triazene, than were wild-type mice, in terms of survival.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH2', entrez:'4436'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4436', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'717', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/9739019', evidence_text:\"Cytogenetic analysis of two Jurkat clones revealed a deletion in the short arm of chromosome region 2p15-21, indicating an allelic loss of both hMSH2 and hMSH6 genes. DNA sequencing revealed that exon 13 of the second hMSH2 allele contains a base substitution at codon 711, which changes an arginine to a termination codon (CGA-->TGA). In addition, a (C)8-->(C)7 frameshift mutation in codon 1085-1087 of the hMSH6 gene was also found. |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MLH1', entrez:'4292'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4292', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'718', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14985452', evidence_text:\"More recently, it has been observed that resistance to temozolomide (and a variety of other clinically important cytotoxic drugs) in ovarian and colon tumor xenografts could be reversed by 5-aza-CdR, which induces demethylation of the hMLH1 promoter and therefore restoration of a functional MMR (11).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MLH1', entrez:'4292'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4292', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'719', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14985452', evidence_text:\"More recently, it has been observed that resistance to temozolomide (and a variety of other clinically important cytotoxic drugs) in ovarian and colon tumor xenografts could be reversed by 5-aza-CdR, which induces demethylation of the hMLH1 promoter and therefore restoration of a functional MMR (11).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'POLB', entrez:'5423'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'5423', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'720', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16024643', evidence_text:\"RNA interference-mediated pol-beta suppression was sufficient to increase temozolomide efficacy\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'POLB', entrez:'5423'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5423', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'721', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16024643', evidence_text:\"RNA interference-mediated pol-beta suppression was sufficient to increase temozolomide efficacy\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'APEX1', entrez:'328'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'328', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'722', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16243814', evidence_text:\"Suppressing Ap endo activity in a human medulloblastoma cell line significantly increased sensitivity to 1,3-bis(2-chlororethyl)-1-nitrosourea and temozolomide\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'APEX1', entrez:'328'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'328', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'723', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16243814', evidence_text:\"Suppressing Ap endo activity in a human medulloblastoma cell line significantly increased sensitivity to 1,3-bis(2-chlororethyl)-1-nitrosourea and temozolomide\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'724', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16298473', evidence_text:\"Xrcc2 deficient cells were found to be hypersensitive to temozolomide\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'725', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16298473', evidence_text:\"Xrcc2 deficient cells were found to be hypersensitive to temozolomide\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MRE11', entrez:'4361'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4361', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'726', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17121922', evidence_text:\"Suppression of Mre11 levels also made glioma cells _2-fold more resistant to temozolomide-induced cytotoxicity. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MRE11', entrez:'4361'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4361', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'727', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17121922', evidence_text:\"Suppression of Mre11 levels also made glioma cells _2-fold more resistant to temozolomide-induced cytotoxicity. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PARP1', entrez:'142'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'142', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'728', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18440222', evidence_text:\"Stable depletion of poly (ADP-ribose) polymerase-1 reduces in vivo melanoma growth and increases chemosensitivity\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PARP1', entrez:'142'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'142', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'729', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18440222', evidence_text:\"Stable depletion of poly (ADP-ribose) polymerase-1 reduces in vivo melanoma growth and increases chemosensitivity\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP53', entrez:'7157'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7157', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'730', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19074867', evidence_text:\"Results from in vitro cell viability analysis showed that the cytotoxic activity of TMZ was substantially increased when p53 wild-type (p53(wt))\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP53', entrez:'7157'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7157', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'731', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19074867', evidence_text:\"Results from in vitro cell viability analysis showed that the cytotoxic activity of TMZ was substantially increased when p53 wild-type (p53(wt))\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH6', entrez:'2956'}), (a:Aberration { type:'snv', value:'T1219#', gene_entrez:'2956', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'732', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19584161', evidence_text:\"In vitro modeling through exposure of an MSH6 wild-type glioblastoma line to temozolomide resulted in resistant clones; one clone showed an MSH6 mutation, Thr(1219)Ile, that had been independently noted in two treated TCGA glioblastomas. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity. Drug class relation with temozolomide.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH6', entrez:'2956'}), (a:Aberration { type:'snv', value:'T1219I', gene_entrez:'2956', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'733', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19584161', evidence_text:\"In vitro modeling through exposure of an MSH6 wild-type glioblastoma line to temozolomide resulted in resistant clones; one clone showed an MSH6 mutation, Thr(1219)Ile, that had been independently noted in two treated TCGA glioblastomas. |Evidence Level:  Pre-clinical. Cell line. Drug class relation with temozolomide.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LIG4', entrez:'3981'}), (a:Aberration { type:'cnv', value:'change', gene_entrez:'3981', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'734', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19615340', evidence_text:\" In addition, it was found that small interference RNAs (siRNA) for Lig4 efficiently enhanced cell lethality induced by TMZ in human glioblastoma A172 cells. These findings suggest that down regulation of Lig4 might provide a useful tool for cell sensitization during TMZ chemotherapy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LIG4', entrez:'3981'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'3981', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'735', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19615340', evidence_text:\" In addition, it was found that small interference RNAs (siRNA) for Lig4 efficiently enhanced cell lethality induced by TMZ in human glioblastoma A172 cells. These findings suggest that down regulation of Lig4 might provide a useful tool for cell sensitization during TMZ chemotherapy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LIG4', entrez:'3981'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'3981', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'736', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19615340', evidence_text:\" In addition, it was found that small interference RNAs (siRNA) for Lig4 efficiently enhanced cell lethality induced by TMZ in human glioblastoma A172 cells. These findings suggest that down regulation of Lig4 might provide a useful tool for cell sensitization during TMZ chemotherapy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REV3L', entrez:'5980'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'5980', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'737', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19641035', evidence_text:\"These data show that Rev3L contributes to temozolomide and fotemustine resistance, thus acting in concert with O6-methylguanine-DNA methyltransferase, BER, mismatch repair, and double-strand break repair in defense against simple alkylating anticancer drugs\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REV3L', entrez:'5980'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5980', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'738', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19641035', evidence_text:\"These data show that Rev3L contributes to temozolomide and fotemustine resistance, thus acting in concert with O6-methylguanine-DNA methyltransferase, BER, mismatch repair, and double-strand break repair in defense against simple alkylating anticancer drugs\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NBN', entrez:'4683'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4683', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'739', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20729302', evidence_text:\"Down-regulation of NBN in melanoma cells by small interfering RNA rendered them more sensitive to temozolomide, suggesting that NBN is a novel modulator of temozolomide sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NBN', entrez:'4683'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4683', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'740', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20729302', evidence_text:\"Down-regulation of NBN in melanoma cells by small interfering RNA rendered them more sensitive to temozolomide, suggesting that NBN is a novel modulator of temozolomide sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51', entrez:'5888'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'5888', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'741', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21363882', evidence_text:\"Rad51 levels are inversely correlated with radiosensitivity, and downregulation markedly increases the cytotoxicity of TMZ.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51', entrez:'5888'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5888', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'742', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21363882', evidence_text:\"Rad51 levels are inversely correlated with radiosensitivity, and downregulation markedly increases the cytotoxicity of TMZ.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'H2AFX', entrez:'3014'}), (a:Aberration { type:'snv', value:'S139#', gene_entrez:'3014', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'743', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21555580', evidence_text:\"mutant (S139A) H2Ax genes, exposed to a range of agents with varied mechanisms of DNA damage. Knockout and mutant cells were sensitive to _-rays, etoposide, temozolamide\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'H2AFX', entrez:'3014'}), (a:Aberration { type:'snv', value:'S139A', gene_entrez:'3014', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'744', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21555580', evidence_text:\"mutant (S139A) H2Ax genes, exposed to a range of agents with varied mechanisms of DNA damage. Knockout and mutant cells were sensitive to _-rays, etoposide, temozolamide\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'745', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21573016', evidence_text:\"found that FANCD1/BRCA2 small interference RNA efficiently enhanced cellular sensitivity toward ACNU and TMZ in human glioblastoma A172 cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'746', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21573016', evidence_text:\"found that FANCD1/BRCA2 small interference RNA efficiently enhanced cellular sensitivity toward ACNU and TMZ in human glioblastoma A172 cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'747', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21573016', evidence_text:\"found that FANCD1/BRCA2 small interference RNA efficiently enhanced cellular sensitivity toward ACNU and TMZ in human glioblastoma A172 cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRCA2', entrez:'675'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'675', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'748', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21573016', evidence_text:\"found that FANCD1/BRCA2 small interference RNA efficiently enhanced cellular sensitivity toward ACNU and TMZ in human glioblastoma A172 cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MPG', entrez:'4350'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4350', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'749', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22156195', evidence_text:\"Reintroducing expression of APNG in TMZ-sensitive GBM lines conferred resistance to TMZ in vitro and in orthotopic xenograft mouse models. In addition, resistance was enhanced with coexpression of MGMT. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MPG', entrez:'4350'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4350', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'750', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22156195', evidence_text:\"Reintroducing expression of APNG in TMZ-sensitive GBM lines conferred resistance to TMZ in vitro and in orthotopic xenograft mouse models. In addition, resistance was enhanced with coexpression of MGMT. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K5', entrez:'4217'}), (a:Aberration { type:'snv', value:'I780#', gene_entrez:'4217', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'751', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K9', entrez:'4293'}), (a:Aberration { type:'snv', value:'K171#', gene_entrez:'4293', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'752', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K9', entrez:'4293'}), (a:Aberration { type:'snv', value:'W333#', gene_entrez:'4293', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'753', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K5', entrez:'4217'}), (a:Aberration { type:'snv', value:'I780F', gene_entrez:'4217', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'754', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K5', entrez:'4217'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4217', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'755', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K5', entrez:'4217'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4217', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'756', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K9', entrez:'4293'}), (a:Aberration { type:'snv', value:'K171A', gene_entrez:'4293', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'757', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K9', entrez:'4293'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4293', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'758', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K9', entrez:'4293'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4293', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'759', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K9', entrez:'4293'}), (a:Aberration { type:'snv', value:'W333*', gene_entrez:'4293', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'760', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STAT3', entrez:'6774'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6774', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'761', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22532597', evidence_text:\"Here, we show that upregulation of both MGMT and STAT3 was accompanied with acquisition of temozolomide resistance in the GBM cell line U87.  Finally, we showed that STAT3 inhibitor or STAT3 knockdown potentiated temozolomide efficacy in temozolomide-resistant GBM cell lines. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STAT3', entrez:'6774'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6774', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'762', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22532597', evidence_text:\"Here, we show that upregulation of both MGMT and STAT3 was accompanied with acquisition of temozolomide resistance in the GBM cell line U87.  Finally, we showed that STAT3 inhibitor or STAT3 knockdown potentiated temozolomide efficacy in temozolomide-resistant GBM cell lines. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'UNG', entrez:'7374'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7374', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'763', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23038810', evidence_text:\"increased TMZ-mediated sensitization in cells depleted of UNG (UNG-KD) was validated using a separate shRNA targeting sequence and an independent experimental design. Thus, cell sensitization to alkylating agents with UNG-KD is not because of RNAi off-target effects or artifacts of the screening protocol. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'UNG', entrez:'7374'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7374', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'764', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23038810', evidence_text:\"increased TMZ-mediated sensitization in cells depleted of UNG (UNG-KD) was validated using a separate shRNA targeting sequence and an independent experimental design. Thus, cell sensitization to alkylating agents with UNG-KD is not because of RNAi off-target effects or artifacts of the screening protocol. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RPS6KA1', entrez:'6195'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'6195', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'765', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108403', evidence_text:\"we found that A375 cells that stably knockdown RSK1/2 expression are more susceptible to DNA-damaging agents (Figure 7f), suggesting that RSK may simultaneously regulate cell survival and proliferation. These results were corroborated by the fact that A375 cells treated with RSK inhibitors are more susceptible to a higher dose of DTIC (30??M) (Figure 7g), indicating that RSK inhibition sensitizes melanoma cells to the cytostatic and cytotoxic effects of DNA-damaging agents.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RPS6KA1', entrez:'6195'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'6195', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'766', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108403', evidence_text:\"we found that A375 cells that stably knockdown RSK1/2 expression are more susceptible to DNA-damaging agents (Figure 7f), suggesting that RSK may simultaneously regulate cell survival and proliferation. These results were corroborated by the fact that A375 cells treated with RSK inhibitors are more susceptible to a higher dose of DTIC (30??M) (Figure 7g), indicating that RSK inhibition sensitizes melanoma cells to the cytostatic and cytotoxic effects of DNA-damaging agents.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RPS6KA3', entrez:'6197'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'6197', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'767', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108403', evidence_text:\"we found that A375 cells that stably knockdown RSK1/2 expression are more susceptible to DNA-damaging agents (Figure 7f), suggesting that RSK may simultaneously regulate cell survival and proliferation. These results were corroborated by the fact that A375 cells treated with RSK inhibitors are more susceptible to a higher dose of DTIC (30??M) (Figure 7g), indicating that RSK inhibition sensitizes melanoma cells to the cytostatic and cytotoxic effects of DNA-damaging agents.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RPS6KA3', entrez:'6197'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'6197', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'768', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108403', evidence_text:\"we found that A375 cells that stably knockdown RSK1/2 expression are more susceptible to DNA-damaging agents (Figure 7f), suggesting that RSK may simultaneously regulate cell survival and proliferation. These results were corroborated by the fact that A375 cells treated with RSK inhibitors are more susceptible to a higher dose of DTIC (30??M) (Figure 7g), indicating that RSK inhibition sensitizes melanoma cells to the cytostatic and cytotoxic effects of DNA-damaging agents.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GRM3', entrez:'2913'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'2913', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'769', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23175182', evidence_text:\"The DNA-alkylating agent, temozolomide, killed GSCs only if mGlu3 receptors were knocked down or pharmacologically inhibited. In contrast, mGlu3 receptor blockade did not affect the action of paclitaxel, etoposide, cis-platinum, and irinotecan. mGlu3 receptor blockade enabled temozolomide toxicity by inhibiting a phosphatidylinositol-3-kinase/nuclear factor-_B pathway that supports the expression of MGMT. 87 patients with GBM undergoing surgery followed by adjuvant chemotherapy with temozolomide survived for longer time if tumor cells expressed low levels of mGlu3 receptors. In addition, the methylation state of the MGMT gene promoter in tumor extracts influenced survival only in those patients with low expression of mGlu3 receptors in the tumor.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GRM3', entrez:'2913'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'2913', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'770', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23175182', evidence_text:\"The DNA-alkylating agent, temozolomide, killed GSCs only if mGlu3 receptors were knocked down or pharmacologically inhibited. In contrast, mGlu3 receptor blockade did not affect the action of paclitaxel, etoposide, cis-platinum, and irinotecan. mGlu3 receptor blockade enabled temozolomide toxicity by inhibiting a phosphatidylinositol-3-kinase/nuclear factor-_B pathway that supports the expression of MGMT. 87 patients with GBM undergoing surgery followed by adjuvant chemotherapy with temozolomide survived for longer time if tumor cells expressed low levels of mGlu3 receptors. In addition, the methylation state of the MGMT gene promoter in tumor extracts influenced survival only in those patients with low expression of mGlu3 receptors in the tumor.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH6', entrez:'2956'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2956', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'771', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23365123', evidence_text:\"the immunopositivity of MSH6 was positively correlated with TMZ response (P = .015, Fisher's exact test). Conclusions: This study showed that preserving MSH6 function was contributory to the effectiveness of TMZ in malignant pituitary neoplasms. |Evidence Level:  Clinical. |Annotator Note:  Relationship confirmed in an preclinical cell line model. PMID: 19584161. Drug class relation with temozolomide.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH6', entrez:'2956'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2956', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'772', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23365123', evidence_text:\"the immunopositivity of MSH6 was positively correlated with TMZ response (P = .015, Fisher's exact test). Conclusions: This study showed that preserving MSH6 function was contributory to the effectiveness of TMZ in malignant pituitary neoplasms. |Evidence Level:  Clinical. |Annotator Note:  Relationship confirmed in an preclinical cell line model. PMID: 19584161. Drug class relation with temozolomide.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REG1A', entrez:'5967'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5967', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'773', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23903855', evidence_text:\"Melanoma lines with high REG1A mRNA expression were more susceptible to dacarbazine and Cisplatin, as compared with those with low REG1A mRNA expression. In conclusion, REG1A expression status may be useful as a biomarker in melanoma patients for sensitivity to these chemotherapeutic agents.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REG1A', entrez:'5967'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5967', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'774', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23903855', evidence_text:\"Melanoma lines with high REG1A mRNA expression were more susceptible to dacarbazine and Cisplatin, as compared with those with low REG1A mRNA expression. In conclusion, REG1A expression status may be useful as a biomarker in melanoma patients for sensitivity to these chemotherapeutic agents.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATM', entrez:'328'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'328', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'775', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Having shown that both ATM and ATR determine the resistance of cells to temozolomide, we extended the study to glioblastoma and malignant melanoma cells, using the well-characterized lines LN229 (29) and D03 (28), respectively. This was accomplished by downregu- lation of ATM or ATR in these cells and determining the impact on their sensitivity to temozolomide. Knockdown of either ATM or ATR greatly enhanced the sensitivity of LN229 and D03 cells to temozolomide, with a significantly stronger sensitizing effect if ATR was downregulated. Simultaneous downregulation of ATM and ATR had no additional effect on the killing response. The data indicate that ATM and, even more, ATR, are targets for improving the anticancer effect of temozolomide. We should stress that the temozolomide doses applied in the experiments are in the therapeutic range, that is, up to 100 mmol/L, which corresponds to the drug serum level during ther- apy (43)...Because MGMT abolishes the increased sensitivity given by the impaired function of ATM or ATR, the therapeutic benefit is limited to patients with tumors that lack MGMT expression. (Annotator note:  MGMT expression or presence should be compromised. WT MGMT will abrogate response when ATR/ATM is compromised.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATM', entrez:'328'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'328', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'776', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Having shown that both ATM and ATR determine the resistance of cells to temozolomide, we extended the study to glioblastoma and malignant melanoma cells, using the well-characterized lines LN229 (29) and D03 (28), respectively. This was accomplished by downregu- lation of ATM or ATR in these cells and determining the impact on their sensitivity to temozolomide. Knockdown of either ATM or ATR greatly enhanced the sensitivity of LN229 and D03 cells to temozolomide, with a significantly stronger sensitizing effect if ATR was downregulated. Simultaneous downregulation of ATM and ATR had no additional effect on the killing response. The data indicate that ATM and, even more, ATR, are targets for improving the anticancer effect of temozolomide. We should stress that the temozolomide doses applied in the experiments are in the therapeutic range, that is, up to 100 mmol/L, which corresponds to the drug serum level during ther- apy (43)...Because MGMT abolishes the increased sensitivity given by the impaired function of ATM or ATR, the therapeutic benefit is limited to patients with tumors that lack MGMT expression. (Annotator note:  MGMT expression or presence should be compromised. WT MGMT will abrogate response when ATR/ATM is compromised.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATR', entrez:'545'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'545', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'777', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Having shown that both ATM and ATR determine the resistance of cells to temozolomide, we extended the study to glioblastoma and malignant melanoma cells, using the well-characterized lines LN229 (29) and D03 (28), respectively. This was accomplished by downregu- lation of ATM or ATR in these cells and determining the impact on their sensitivity to temozolomide. Knockdown of either ATM or ATR greatly enhanced the sensitivity of LN229 and D03 cells to temozolomide, with a significantly stronger sensitizing effect if ATR was downregulated. Simultaneous downregulation of ATM and ATR had no additional effect on the killing response. The data indicate that ATM and, even more, ATR, are targets for improving the anticancer effect of temozolomide. We should stress that the temozolomide doses applied in the experiments are in the therapeutic range, that is, up to 100 mmol/L, which corresponds to the drug serum level during ther- apy (43)...Because MGMT abolishes the increased sensitivity given by the impaired function of ATM or ATR, the therapeutic benefit is limited to patients with tumors that lack MGMT expression. (Annotator note:  MGMT expression or presence should be compromised. WT MGMT will abrogate response when ATR/ATM is compromised.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATR', entrez:'545'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'545', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'778', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Having shown that both ATM and ATR determine the resistance of cells to temozolomide, we extended the study to glioblastoma and malignant melanoma cells, using the well-characterized lines LN229 (29) and D03 (28), respectively. This was accomplished by downregu- lation of ATM or ATR in these cells and determining the impact on their sensitivity to temozolomide. Knockdown of either ATM or ATR greatly enhanced the sensitivity of LN229 and D03 cells to temozolomide, with a significantly stronger sensitizing effect if ATR was downregulated. Simultaneous downregulation of ATM and ATR had no additional effect on the killing response. The data indicate that ATM and, even more, ATR, are targets for improving the anticancer effect of temozolomide. We should stress that the temozolomide doses applied in the experiments are in the therapeutic range, that is, up to 100 mmol/L, which corresponds to the drug serum level during ther- apy (43)...Because MGMT abolishes the increased sensitivity given by the impaired function of ATM or ATR, the therapeutic benefit is limited to patients with tumors that lack MGMT expression. (Annotator note:  MGMT expression or presence should be compromised. WT MGMT will abrogate response when ATR/ATM is compromised.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK1', entrez:'1111'}), (a:Aberration { type:'exp ', value:'under', gene_entrez:'1111', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'779', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Previous studies revealed that CHK1 inhibition increases temo- zolomide cytotoxicity (59), which is in line with our data.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK1', entrez:'1111'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'1111', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'780', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Previous studies revealed that CHK1 inhibition increases temo- zolomide cytotoxicity (59), which is in line with our data.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH2', entrez:'4436'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4436', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'781', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/9739019', evidence_text:\"Cytogenetic analysis of two Jurkat clones revealed a deletion in the short arm of chromosome region 2p15-21, indicating an allelic loss of both hMSH2 and hMSH6 genes. DNA sequencing revealed that exon 13 of the second hMSH2 allele contains a base substitution at codon 711, which changes an arginine to a termination codon (CGA-->TGA). In addition, a (C)8-->(C)7 frameshift mutation in codon 1085-1087 of the hMSH6 gene was also found.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH2', entrez:'4436'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4436', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'782', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/9739019', evidence_text:\"Cytogenetic analysis of two Jurkat clones revealed a deletion in the short arm of chromosome region 2p15-21, indicating an allelic loss of both hMSH2 and hMSH6 genes. DNA sequencing revealed that exon 13 of the second hMSH2 allele contains a base substitution at codon 711, which changes an arginine to a termination codon (CGA-->TGA). In addition, a (C)8-->(C)7 frameshift mutation in codon 1085-1087 of the hMSH6 gene was also found.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STAT3', entrez:'6774'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6774', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'783', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22532597', evidence_text:\"Here, we show that upregulation of both MGMT and STAT3 was accompanied with acquisition of temozolomide resistance in the GBM cell line U87.  Finally, we showed that STAT3 inhibitor or STAT3 knockdown potentiated temozolomide efficacy in temozolomide-resistant GBM cell lines. |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MLH1', entrez:'4292'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4292', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'784', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14985452', evidence_text:\"More recently, it has been observed that resistance to temozolomide (and a variety of other clinically important cytotoxic drugs) in ovarian and colon tumor xenografts could be reversed by 5-aza-CdR, which induces demethylation of the hMLH1 promoter and therefore restoration of a functional MMR (11). |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MSH6', entrez:'2956'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2956', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'785', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19584161', evidence_text:\"In vitro modeling through exposure of an MSH6 wild-type glioblastoma line to temozolomide resulted in resistant clones; one clone showed an MSH6 mutation, Thr(1219)Ile, that had been independently noted in two treated TCGA glioblastomas. |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity. Drug class relation with temozolomide.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CHEK1', entrez:'1111'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'1111', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'786', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23960094', evidence_text:\"Previous studies revealed that CHK1 inhibition increases temo- zolomide cytotoxicity (59), which is in line with our data. |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RAD51', entrez:'5888'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5888', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'787', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21363882', evidence_text:\"Rad51 levels are inversely correlated with radiosensitivity, and downregulation markedly increases the cytotoxicity of TMZ. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'POLB', entrez:'5423'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5423', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'788', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16024643', evidence_text:\"RNA interference-mediated pol-beta suppression was sufficient to increase temozolomide efficacy. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP53', entrez:'7157'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7157', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'789', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19074867', evidence_text:\"Results from in vitro cell viability analysis showed that the cytotoxic activity of TMZ was substantially increased when p53 wild-type (p53(wt)). |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PARP1', entrez:'142'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'142', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'790', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18440222', evidence_text:\"Stable depletion of poly (ADP-ribose) polymerase-1 reduces in vivo melanoma growth and increases chemosensitivity. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'APEX1', entrez:'328'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'328', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'791', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16243814', evidence_text:\"Suppressing Ap endo activity in a human medulloblastoma cell line significantly increased sensitivity to 1,3-bis(2-chlororethyl)-1-nitrosourea and temozolomide. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K5', entrez:'4217'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4217', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'792', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma. |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K9', entrez:'4293'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4293', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'793', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22197930', evidence_text:\"The position of the mutations and the loss of heterozygosity of MAP3K5 and MAP3K9 in 85% and 67% of melanoma samples, respectively, together suggest that the mutations are likely to be inactivating. In in vitro kinase assays, MAP3K5 I780F and MAP3K9 W333* variants had reduced kinase activity. Overexpression of MAP3K5 or MAP3K9 mutants in HEK293T cells reduced the phosphorylation of downstream MAP kinases. Attenuation of MAP3K9 function in melanoma cells using siRNA led to increased cell viability after temozolomide treatment, suggesting that decreased MAP3K pathway activity can lead to chemoresistance in melanoma. |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'REV3L', entrez:'5980'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5980', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'794', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19641035', evidence_text:\"These data show that Rev3L contributes to temozolomide and fotemustine resistance, thus acting in concert with O6-methylguanine-DNA methyltransferase, BER, mismatch repair, and double-strand break repair in defense against simple alkylating anticancer drugs. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'XRCC2', entrez:'7516'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7516', entity_class:'modifier' }), (d:Drug {name:'dacarbazine'} ) Merge (r:Rule {rulenum:'795', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16298473', evidence_text:\"Xrcc2 deficient cells were found to be hypersensitive to temozolomide. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STK11', entrez:'6794'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6794', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'796', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22698401', evidence_text:\"Although the mechanism whereby loss of LKB1 kinase activity induces YES activation is not clear, the present data suggest YES as a promising therapeutic target in melanoma lacking LKB1 function. dasatinib exhibits modest antimelanoma activity in human patients (Kluger et al., 2011). Because we and others (Deguchi et al., 2008; Konecny et al., 2009) have noted an ~10-fold decrease in the potency of dasatinib for YES compared to SRC and other SFKs, these results suggest that an SFK inhibitor with greater potency against YES inhibitor would exhibit greater in vivo antimelanoma efficacy. (Annotator note: Lung cancer showed much better response to dasatinib then in melanoma. Possibly due to the amount more of dasatinib needed to hit YES, but this is based on one study.) |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CBL', entrez:'867'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'867', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'797', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22246246', evidence_text:\"Homozygous CBL mutations result in hypersensitivity to growth factors. Over expression of the WT CBL inhibited the growth of a CBL mutant cell line, consistent with the homozygous nature of CBL mutations identified in most patients with myeloid malignancies who harbor CBL mutations. dasatinib is the most effective TKI in a mutant CBL background and specifically reduced the phosphorylation of RTK and SFK. |Annotator Note: SNVs should be loss-of-function and associated with drug sensitivity. However, not all SNVs may result in loss-of-function and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'YES', entrez:'7525'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7525', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'798', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Annotator Note: (PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'YES', entrez:'7525'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7525', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'799', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. (Annotator note: PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'YES', entrez:'7525'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7525', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'800', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. (Annotator note: PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TEC', entrez:'7006'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'7006', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'801', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"dasatinib Inhibits Btk and Tec Kinase Activity in Vitro and in Cells. |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TEC', entrez:'7006'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7006', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'802', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"dasatinib Inhibits Btk and Tec Kinase Activity in Vitro and in Cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TEC', entrez:'7006'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7006', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'803', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"dasatinib Inhibits Btk and Tec Kinase Activity in Vitro and in Cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TEC', entrez:'7006'}), (a:Aberration { type:'snv', value:'T442#', gene_entrez:'7006', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'804', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"Btk T474I and Tec T442I Are Resistant to dasatinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TEC', entrez:'7006'}), (a:Aberration { type:'snv', value:'T442I', gene_entrez:'7006', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'805', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"Btk T474I and Tec T442I Are Resistant to dasatinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SRC', entrez:'6714'}), (a:Aberration { type:'snv', value:'T338#', gene_entrez:'6714', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'806', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19462975', evidence_text:\"(T338M) at the gatekeeper residue, an amino acid situated at the back of the ATP pocket that is well-known for influencing type I inhibitor affinity and selectivity profiles among kinases, of cSrc which results in resistance to dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SRC', entrez:'6714'}), (a:Aberration { type:'snv', value:'T338M', gene_entrez:'6714', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'807', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19462975', evidence_text:\"(T338M) at the gatekeeper residue, an amino acid situated at the back of the ATP pocket that is well-known for influencing type I inhibitor affinity and selectivity profiles among kinases, of cSrc which results in resistance to dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SRC', entrez:'6714'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6714', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'808', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Annotator Note: (PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SRC', entrez:'6714'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6714', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'809', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SRC', entrez:'6714'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6714', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'810', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'811', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Annotator Note: (PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'812', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'813', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'814', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18794084', evidence_text:\"All inhibitors tested exhibited a high efficacy toward the PDGFRA(DeltaDIM842-844) mutant. In contrast, ex vivo and in vitro assays revealed that only dasatinib potently inhibited the PDGFRA(D842V) isoform with an IC(50) value of 62 nmol/L. (Annotator note:  PMID:23970477 suggest that dasatinib is not effective in low grade glioma samples). |Annotator Note:  PDGFRA SNVs can be associated either with drug response or sensitivity depending on disease and position in protein. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'small_deletion', value:'D842_M844delDIM', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'815', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18794084', evidence_text:\"All second-line inhibitors tested are potent inhibitors of the growth of the PDGFRADDIM842-844-expressing Ba/F3 cells in a dose-dependent manner. imatinib, nilotinib, sorafenib, and dasatinib inhibited the cell growth of PDGFRA(DeltaDIM842-844) Ba/F3 cells with IC50s of 20, 56, 17, and 10 nmol/L, respectively. Furthermore, significant induction of apoptosis of these cells was recorded at 100 nmol/L of imatinib, sorafenib, and dasatinib, and at 500 nmol/L of nilotinib. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'snv', value:'D842#', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'816', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18794084', evidence_text:\"ex vivo and in vitro assays revealed that only dasatinib potently inhibited the PDGFRA(D842V) isoform with an IC(50) value of 62 nmol/L. Sorafenib and nilotinib were significantly less efficacious against this mutation, inhibiting the PDGFRA kinase activity at >1,000 and >5,000 nmol/L, and suppressing the proliferation of the cells expressing the PDGFRA(D842V) mutant with an IC(50) value of 239 and 1,310 nmol/L, respectively. |Evidence Level:  Pre-clinical. Cell line. |Disease:  GIST. |Annotator Note:  SNVs at this position are associated with drug sensitivity. However, not all changes at this position may result in drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'snv', value:'D842V', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'817', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18794084', evidence_text:\"ex vivo and in vitro assays revealed that only dasatinib potently inhibited the PDGFRA(D842V) isoform with an IC(50) value of 62 nmol/L. Sorafenib and nilotinib were significantly less efficacious against this mutation, inhibiting the PDGFRA kinase activity at >1,000 and >5,000 nmol/L, and suppressing the proliferation of the cells expressing the PDGFRA(D842V) mutant with an IC(50) value of 239 and 1,310 nmol/L, respectively. |Evidence Level:  Pre-clinical. Cell line. |Disease:  GIST.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'818', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"Dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  'Is a target' relationship.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'819', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"Dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  'Is a target' relationship.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LYN', entrez:'4067'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4067', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'820', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Annotator Note: (PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LYN', entrez:'4067'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4067', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'821', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LYN', entrez:'4067'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4067', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'822', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LCK', entrez:'3932'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'3932', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'823', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Annotator Note: (PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LCK', entrez:'3932'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3932', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'824', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'LCK', entrez:'3932'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3932', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'825', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'826', evidence:'direct', pmid:'http://1.usa.gov/17zbXoc', evidence_text:\"An orally bioavailable synthetic small molecule-inhibitor of SRC-family protein-tyrosine kinases. Dasatinib binds to and inhibits the growth-promoting activities of these kinases. Apparently because of its less stringent binding affinity for the BCR-ABL kinase, dasatinib has been shown to overcome the resistance to imatinib of chronic myeloid leukemia (CML) cells harboring BCR-ABL kinase domain point mutations. SRC-family protein-tyrosine kinases interact with a variety of cell-surface receptors and participate in intracellular signal transduction pathways; tumorigenic forms can occur through altered regulation or expression of the endogenous protein and by way of virally-encoded kinase genes. |Annotator Note:  Listed as a 'is target' relationship. However, KIT amplification appears not sensitive to imatinib in melanoma subytpes.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'827', evidence:'direct', pmid:'http://1.usa.gov/17zbXoc', evidence_text:\"An orally bioavailable synthetic small molecule-inhibitor of SRC-family protein-tyrosine kinases. Dasatinib binds to and inhibits the growth-promoting activities of these kinases. Apparently because of its less stringent binding affinity for the BCR-ABL kinase, dasatinib has been shown to overcome the resistance to imatinib of chronic myeloid leukemia (CML) cells harboring BCR-ABL kinase domain point mutations. SRC-family protein-tyrosine kinases interact with a variety of cell-surface receptors and participate in intracellular signal transduction pathways; tumorigenic forms can occur through altered regulation or expression of the endogenous protein and by way of virally-encoded kinase genes. |Annotator Note:  Listed as a 'is target' relationship. However, KIT amplification appears not sensitive to imatinib in melanoma subytpes.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'K642#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'828', evidence:'inferred', pmid:'http://bit.ly/1vtmITZ', evidence_text:\"One (acral lentiginous melanoma) had a 3+ c-Kit staining in 100% cells and a K642E substition mutation in exon 13. |Evidence Level:  Phase II. |Disease:  Acral melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'K642E', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'829', evidence:'direct', pmid:'http://bit.ly/1vtmITZ', evidence_text:\"One (acral lentiginous melanoma) had a 3+ c-Kit staining in 100% cells and a K642E substition mutation in exon 13. |Evidence Level:  Phase II. |Disease:  Acral melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'830', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16397263', evidence_text:\"Dasatinib (formerly BMS-354825) is a small-molecule, ATP-competitive inhibitor of SRC and ABL tyrosine kinases with potency in the low nanomolar range. Some small-molecule SRC/ABL inhibitors also have potency against WT KIT kinase. Therefore, we hypothesized that dasatinib might inhibit the kinase activity of both WT and mutant KIT isoforms. We report herein that dasatinib potently inhibits WT KIT and juxtamembrane domain mutant KIT autophosphorylation and KIT-dependent activation of downstream pathways important for cell viability and cell survival, such as Ras/mitogen-activated protein kinase, phosphoinositide 3-kinase/Akt, and Janus-activated kinase/signal transducers and activators of transcription. |Annotator Note:  Some case-reports on dasatinib working in acral/mucosal melananoma with SNVs in KIT. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'D816F', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'831', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16434489', evidence_text:\"Consistent with the results of our biochemical studies, dasatinib inhibited the proliferation of Ba/F3 KIT D816V and D816F cells with an IC50 of 100 to 150 nmol/L, and Ba/F3 KIT D816Y cells with an IC50 of 5 nmol/L. In contrast, imatinib had no significant inhibitory effect on the growth of these three cell lines. Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'D816Y', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'832', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16434489', evidence_text:\"Consistent with the results of our biochemical studies, dasatinib inhibited the proliferation of Ba/F3 KIT D816V and D816F cells with an IC50 of 100 to 150 nmol/L, and Ba/F3 KIT D816Y cells with an IC50 of 5 nmol/L. In contrast, imatinib had no significant inhibitory effect on the growth of these three cell lines. Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'833', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"Dasatinib inhibited the imatinib less-sensitive Ba/F3 KITAY502-3ins with an IC50 of 74 nmol/L.  |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'D820#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'834', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"Imatinib-resistant KITV559D/D820Y was sensitive to nilotinib, dasatinib, and sorafenib inhibition.. |Evidence Level:  Pre-clinical. Cell Line. |Annotator Note:  This inhibition was done in the background of a KIT V559D mutation. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'D820Y', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'835', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"Imatinib-resistant KITV559D/D820Y was sensitive to nilotinib, dasatinib, and sorafenib inhibition.. |Evidence Level:  Pre-clinical. Cell Line. |Annotator Note:  This inhibition was done in the background of a KIT V559D mutation. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'small_insertion', value:'Y502_Y503insSA', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'836', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"Dasatinib inhibited the imatinib less-sensitive Ba/F3 KITAY502-3ins with an IC50 of 74 nmol/L.  |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'small_insertion', value:'Y502_Y503insFA', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'837', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"Dasatinib inhibited the imatinib less-sensitive Ba/F3 KITAY502-3ins with an IC50 of 74 nmol/L.  |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'small_insertion', value:'Y502_Y503insAS', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'838', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"Dasatinib inhibited the imatinib less-sensitive Ba/F3 KITAY502-3ins with an IC50 of 74 nmol/L.  |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'small_deletion', value:'W557_K558del', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'839', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"The growth of KITWK557-8del cells was inhibited by dasatinib with an IC50 of 58 nmol/L, whereas imatinib, nilotinib, and sorafenib had an IC50 of 460, 83, and 211 nmol/L, respectively (Fig. 1 ). Furthermore, significant induction of apoptosis was noted at 100 nmol/L of dasatinib and nilotinib. |Evidence Level:  Pre-clinical. Cell line. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'V560#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'840', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"Dasatinib was previously shown to be more potent than imatinib in inhibiting KITV559D (23), KITV560G (16), and wild- type KIT (15). |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'V560G', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'841', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17699867', evidence_text:\"Dasatinib was previously shown to be more potent than imatinib in inhibiting KITV559D (23), KITV560G (16), and wild- type KIT (15). |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'D816#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'842', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18986703', evidence_text:\"Dasatinib has been reported to potently inhibit juxtamembrane domain mutant KIT(D816V) autophosphorylation and KIT-dependent activation of down stream signaling important for cell growth and survival of neoplastic cells. Additionally, dasatinib induced apoptosis in mast cell and leukemia cell lines expressing KIT(D816V). Here, we present the first case report of long-term hematologic and molecular remission achieved with combined treatment with chemotherapy and dasatinib in a patient with systemic mastocytosis (SM) and acute myeloid leukemia (AML) with mutant KIT(D816V) expression. |Evidence Level:  Case Study. |Disease AML and Mastocytosis.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'D816V', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'843', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18986703', evidence_text:\"Dasatinib has been reported to potently inhibit juxtamembrane domain mutant KIT(D816V) autophosphorylation and KIT-dependent activation of down stream signaling important for cell growth and survival of neoplastic cells. Additionally, dasatinib induced apoptosis in mast cell and leukemia cell lines expressing KIT(D816V). Here, we present the first case report of long-term hematologic and molecular remission achieved with combined treatment with chemotherapy and dasatinib in a patient with systemic mastocytosis (SM) and acute myeloid leukemia (AML) with mutant KIT(D816V) expression. |Evidence Level:  Case Study. |Disease AML and Mastocytosis.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'L576#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'844', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19671763', evidence_text:\"Mucosal melanoma. Two metastatic melanoma patients with the L576P KIT mutation were treated with dasatinib, including one patient previously treated with imatinib. Both patients had marked reduction (>50%) and elimination of tumor F18-fluorodeoxyglucose (FDG)-avidity by positron emission tomography (PET) imaging after dasatinib treatment. These data support the selective inhibitory effect of dasatinib against cells harboring the most common KIT mutation in melanoma, and thus has therapeutic implications for acrallentiginous, chronic sun-damaged, and mucosal melanomas. |Evidence Level:  Case Study. |Disease: Mucosal Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'L576P', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'845', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19671763', evidence_text:\"Two metastatic melanoma patients with the L576P KIT mutation were treated with dasatinib, including one patient previously treated with imatinib. Both patients had marked reduction (>50%) and elimination of tumor F18-fluorodeoxyglucose (FDG)-avidity by positron emission tomography (PET) imaging after dasatinib treatment. These data support the selective inhibitory effect of dasatinib against cells harboring the most common KIT mutation in melanoma, and thus has therapeutic implications for acrallentiginous, chronic sun-damaged, and mucosal melanomas. |Evidence Level:  Case Study. |Disease: Mucosal Melanoma.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'D820E', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'846', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19861442', evidence_text:\"In keeping with these observations, the three patients who developed sunitinib resistance after at least one year of clinical benefit had second site mutations in the KIT kinase activation domain (N822K, D820Y and D820E). Expression of these sunitinib-resistant double mutants in an in vitro Ba/F3 model showed sensitivity to both dasatinib and nilotinib, suggesting alternative therapeutic options for these patients. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  In a KITAY502-3ins  Kit mutant background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'N822#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'847', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19861442', evidence_text:\"In keeping with these observations, the three patients who developed sunitinib resistance after at least one year of clinical benefit had second site mutations in the KIT kinase activation domain (N822K, D820Y and D820E). Expression of these sunitinib-resistant double mutants in an in vitro Ba/F3 model showed sensitivity to both dasatinib and nilotinib, suggesting alternative therapeutic options for these patients. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  In a KITAY502-3ins  Kit mutant background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'N822K', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'848', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19861442', evidence_text:\"In keeping with these observations, the three patients who developed sunitinib resistance after at least one year of clinical benefit had second site mutations in the KIT kinase activation domain (N822K, D820Y and D820E). Expression of these sunitinib-resistant double mutants in an in vitro Ba/F3 model showed sensitivity to both dasatinib and nilotinib, suggesting alternative therapeutic options for these patients. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  In a KITAY502-3ins  Kit mutant background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'V654#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'849', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19861442', evidence_text:\"ATP-binding pocket remain sensitive to sunitinib inhibition. Inhibition of KIT activation of the Ba/F3 KIT502-3AYins/V654A double-mutant was achieved with <10 nmol/L sunitinib, whereas inhibition of cell proliferation with an IC50 of 20.6 nmol/L, concomitant with induction of apoptosis at 100 nmol/L. Dasatinib-treated cells also showed dramatic decrease of KIT kinase activity at 10 nmol/L, whereas the IC50 for pro- liferation was 334.5 nmol/L and at 1,000 nmol/L induction of apoptosis _70%. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The V654A mutant in this system was in a KIT 502-2AYins background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'V654A', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'850', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19861442', evidence_text:\"ATP-binding pocket remain sensitive to sunitinib inhibition. Inhibition of KIT activation of the Ba/F3 KIT502-3AYins/V654A double-mutant was achieved with <10 nmol/L sunitinib, whereas inhibition of cell proliferation with an IC50 of 20.6 nmol/L, concomitant with induction of apoptosis at 100 nmol/L. Dasatinib-treated cells also showed dramatic decrease of KIT kinase activity at 10 nmol/L, whereas the IC50 for pro- liferation was 334.5 nmol/L and at 1,000 nmol/L induction of apoptosis _70%. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The V654A mutant in this system was in a KIT 502-2AYins background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'V559#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'851', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21689725', evidence_text:\"dasatinib inhibited cell growth of imatinib-sensitive Ba/F3 KITV559D with an IC50 of 27 nmol/L, whereas imatinib, nilotinib, and sorafenib had an IC50 of 63, 44, and 66 nmol/L, respectively. dasatinib induced apoptosis in >20% of cells at 10 nmol/L, whereas the other drugs induced programmed cell death at 100 nmol/L. The biochemical assays showed consistent KITV559D phosphorylation inhibition by all four inhibitors tested, with dasatinib being the most potent. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'V559D ', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'852', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21689725', evidence_text:\"dasatinib inhibited cell growth of imatinib-sensitive Ba/F3 KITV559D with an IC50 of 27 nmol/L, whereas imatinib, nilotinib, and sorafenib had an IC50 of 63, 44, and 66 nmol/L, respectively. dasatinib induced apoptosis in >20% of cells at 10 nmol/L, whereas the other drugs induced programmed cell death at 100 nmol/L. The biochemical assays showed consistent KITV559D phosphorylation inhibition by all four inhibitors tested, with dasatinib being the most potent. |Evidence Level:  Pre-clinical. Cell line.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'A829#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'853', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23582185', evidence_text:\"IMR_A829P cells show greatest sensitivity to dasatinib with an IC50 of 54 nM, although this represents a 10-fold increase over the parent. Additionally, these cells retain some sensitivity to nilotinib (IC50 188 nM; a 2.4-fold increase over the parent), but are highly resistant to imatinib (IC50 > 1000 nM) and sunitinib (IC50 998 nM). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma |Annotator Note:  Found in context with L576P mutation background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'A829P', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'854', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23582185', evidence_text:\"IMR_A829P cells show greatest sensitivity to dasatinib with an IC50 of 54 nM, although this represents a 10-fold increase over the parent. Additionally, these cells retain some sensitivity to nilotinib (IC50 188 nM; a 2.4-fold increase over the parent), but are highly resistant to imatinib (IC50 > 1000 nM) and sunitinib (IC50 998 nM). |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma |Annotator Note:  Found in context with L576P mutation background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'T670#', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'855', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23582185', evidence_text:\"Sublines with an additional T670I c-Kit mutation showed resistance to imatinib, nilotinib and dasatinib, but responded to sunitinib. |Disease:  Melanoma |Annotator Note:  Found in context with L576P mutation background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIT', entrez:'3815'}), (a:Aberration { type:'snv', value:'T670I', gene_entrez:'3815', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'856', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23582185', evidence_text:\"Sublines with an additional T670I c-Kit mutation showed resistance to imatinib, nilotinib and dasatinib, but responded to sunitinib. |Disease:  Melanoma |Annotator Note:  Found in context with L576P mutation background.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ITK', entrez:'3702'}), (a:Aberration { type:'snv', value:'F435T', gene_entrez:'3702', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'857', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"Itk F435T has a similar IC50 for dasatinib as Tec WT (228 nM).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FYN', entrez:'2534'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2534', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'858', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Annotator Note: (PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FYN', entrez:'2534'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2534', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'859', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FYN', entrez:'2534'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2534', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'860', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRK', entrez:'2444'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2444', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'861', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24735922', evidence_text:\"In this study we identified recurrent somatic mutations of FRK leading to an activation of the FRK kinase activity and to constitutive STAT3 activation associated with the inflammatory phenotype of HCA. Whereas the activation of STAT3 by Src has been previously described in vitro, , we showed that FRK mutants induce STAT3 activation in vitro, in vivo, and in human tumors.  In addition, the dramatic response to the Src inhibitor dasatinib in vitro and in vivo supports FRK-activating mutations as a paradigm of a mutated gene addiction. In Ba/F3 cells, IL-3-independent growth induced by FRK mutant was abrogated by dasatinib treatment ( Figure 3C). Subcutaneous injection of Ba/F3 cells with stable expression of the VK FRK mutant in nude mice resulted in tumor growth. Furthermore daily intraperitoneal injections of dasatinib (10 mg/kg/day) after subcutaneous tumor development and during 14 days led to a complete regression of the tumors (n = 9; Figures 3F and 3G; Figure S2I). |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRK', entrez:'2444'}), (a:Aberration { type:'snv', value:'E346#', gene_entrez:'2444', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'862', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24735922', evidence_text:\"In this study we identified recurrent somatic mutations of FRK leading to an activation of the FRK kinase activity and to constitutive STAT3 activation associated with the inflammatory phenotype of HCA. Whereas the activation of STAT3 by Src has been previously described in vitro, , we showed that FRK mutants induce STAT3 activation in vitro, in vivo, and in human tumors.  In addition, the dramatic response to the Src inhibitor dasatinib in vitro and in vivo supports FRK-activating mutations as a paradigm of a mutated gene addiction. In Ba/F3 cells, IL-3-independent growth induced by FRK mutant was abrogated by dasatinib treatment ( Figure 3C). Subcutaneous injection of Ba/F3 cells with stable expression of the VK FRK mutant in nude mice resulted in tumor growth. Furthermore daily intraperitoneal injections of dasatinib (10 mg/kg/day) after subcutaneous tumor development and during 14 days led to a complete regression of the tumors (n = 9; Figures 3F and 3G; Figure S2I). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRK', entrez:'2444'}), (a:Aberration { type:'snv', value:'E346G', gene_entrez:'2444', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'863', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24735922', evidence_text:\"In this study we identified recurrent somatic mutations of FRK leading to an activation of the FRK kinase activity and to constitutive STAT3 activation associated with the inflammatory phenotype of HCA. Whereas the activation of STAT3 by Src has been previously described in vitro, , we showed that FRK mutants induce STAT3 activation in vitro, in vivo, and in human tumors.  In addition, the dramatic response to the Src inhibitor dasatinib in vitro and in vivo supports FRK-activating mutations as a paradigm of a mutated gene addiction. In Ba/F3 cells, IL-3-independent growth induced by FRK mutant was abrogated by dasatinib treatment ( Figure 3C). Subcutaneous injection of Ba/F3 cells with stable expression of the VK FRK mutant in nude mice resulted in tumor growth. Furthermore daily intraperitoneal injections of dasatinib (10 mg/kg/day) after subcutaneous tumor development and during 14 days led to a complete regression of the tumors (n = 9; Figures 3F and 3G; Figure S2I). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRK', entrez:'2444'}), (a:Aberration { type:'small_deletion', value:'F379_K380', gene_entrez:'2444', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'864', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24735922', evidence_text:\"In this study we identified recurrent somatic mutations of FRK leading to an activation of the FRK kinase activity and to constitutive STAT3 activation associated with the inflammatory phenotype of HCA. Whereas the activation of STAT3 by Src has been previously described in vitro, , we showed that FRK mutants induce STAT3 activation in vitro, in vivo, and in human tumors.  In addition, the dramatic response to the Src inhibitor dasatinib in vitro and in vivo supports FRK-activating mutations as a paradigm of a mutated gene addiction. In Ba/F3 cells, IL-3-independent growth induced by FRK mutant was abrogated by dasatinib treatment ( Figure 3C). Subcutaneous injection of Ba/F3 cells with stable expression of the VK FRK mutant in nude mice resulted in tumor growth. Furthermore daily intraperitoneal injections of dasatinib (10 mg/kg/day) after subcutaneous tumor development and during 14 days led to a complete regression of the tumors (n = 9; Figures 3F and 3G; Figure S2I). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRK', entrez:'2444'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2444', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'865', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24735922', evidence_text:\"In this study we identified recurrent somatic mutations of FRK leading to an activation of the FRK kinase activity and to constitutive STAT3 activation associated with the inflammatory phenotype of HCA. Whereas the activation of STAT3 by Src has been previously described in vitro, , we showed that FRK mutants induce STAT3 activation in vitro, in vivo, and in human tumors.  In addition, the dramatic response to the Src inhibitor dasatinib in vitro and in vivo supports FRK-activating mutations as a paradigm of a mutated gene addiction. In Ba/F3 cells, IL-3-independent growth induced by FRK mutant was abrogated by dasatinib treatment ( Figure 3C). Subcutaneous injection of Ba/F3 cells with stable expression of the VK FRK mutant in nude mice resulted in tumor growth. Furthermore daily intraperitoneal injections of dasatinib (10 mg/kg/day) after subcutaneous tumor development and during 14 days led to a complete regression of the tumors (n = 9; Figures 3F and 3G; Figure S2I). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRK', entrez:'2444'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2444', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'866', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24735922', evidence_text:\"In this study we identified recurrent somatic mutations of FRK leading to an activation of the FRK kinase activity and to constitutive STAT3 activation associated with the inflammatory phenotype of HCA. Whereas the activation of STAT3 by Src has been previously described in vitro, , we showed that FRK mutants induce STAT3 activation in vitro, in vivo, and in human tumors.  In addition, the dramatic response to the Src inhibitor dasatinib in vitro and in vivo supports FRK-activating mutations as a paradigm of a mutated gene addiction. In Ba/F3 cells, IL-3-independent growth induced by FRK mutant was abrogated by dasatinib treatment ( Figure 3C). Subcutaneous injection of Ba/F3 cells with stable expression of the VK FRK mutant in nude mice resulted in tumor growth. Furthermore daily intraperitoneal injections of dasatinib (10 mg/kg/day) after subcutaneous tumor development and during 14 days led to a complete regression of the tumors (n = 9; Figures 3F and 3G; Figure S2I). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRK', entrez:'2444'}), (a:Aberration { type:'small_deletion', value:'V378_F379', gene_entrez:'2444', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'867', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24735922', evidence_text:\"In this study we identified recurrent somatic mutations of FRK leading to an activation of the FRK kinase activity and to constitutive STAT3 activation associated with the inflammatory phenotype of HCA. Whereas the activation of STAT3 by Src has been previously described in vitro, , we showed that FRK mutants induce STAT3 activation in vitro, in vivo, and in human tumors.  In addition, the dramatic response to the Src inhibitor dasatinib in vitro and in vivo supports FRK-activating mutations as a paradigm of a mutated gene addiction. In Ba/F3 cells, IL-3-independent growth induced by FRK mutant was abrogated by dasatinib treatment ( Figure 3C). Subcutaneous injection of Ba/F3 cells with stable expression of the VK FRK mutant in nude mice resulted in tumor growth. Furthermore daily intraperitoneal injections of dasatinib (10 mg/kg/day) after subcutaneous tumor development and during 14 days led to a complete regression of the tumors (n = 9; Figures 3F and 3G; Figure S2I). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FRK', entrez:'2444'}), (a:Aberration { type:'small_deletion', value:'V378_K380', gene_entrez:'2444', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'868', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24735922', evidence_text:\"In this study we identified recurrent somatic mutations of FRK leading to an activation of the FRK kinase activity and to constitutive STAT3 activation associated with the inflammatory phenotype of HCA. Whereas the activation of STAT3 by Src has been previously described in vitro, , we showed that FRK mutants induce STAT3 activation in vitro, in vivo, and in human tumors.  In addition, the dramatic response to the Src inhibitor dasatinib in vitro and in vivo supports FRK-activating mutations as a paradigm of a mutated gene addiction. In Ba/F3 cells, IL-3-independent growth induced by FRK mutant was abrogated by dasatinib treatment ( Figure 3C). Subcutaneous injection of Ba/F3 cells with stable expression of the VK FRK mutant in nude mice resulted in tumor growth. Furthermore daily intraperitoneal injections of dasatinib (10 mg/kg/day) after subcutaneous tumor development and during 14 days led to a complete regression of the tumors (n = 9; Figures 3F and 3G; Figure S2I). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EPHA2', entrez:'1969'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'1969', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'869', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18047674', evidence_text:\"Is Target. Furthermore, EphA2, a target of dasatinib, was found to be a sensitivity biomarker. |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EPHA2', entrez:'1969'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1969', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'870', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18047674', evidence_text:\"Is Target. Furthermore, EphA2, a target of dasatinib, was found to be a sensitivity biomarker.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EPHA2', entrez:'1969'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1969', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'871', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18047674', evidence_text:\"Is Target. Furthermore, EphA2, a target of dasatinib, was found to be a sensitivity biomarker.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ITK', entrez:'3702'}), (a:Aberration { type:'snv', value:'F435#', gene_entrez:'3702', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'872', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"mutation of the corresponding residue in Itk (Phe-435) to Thr sensitized the otherwise insensitive Itk to dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ITK', entrez:'3702'}), (a:Aberration { type:'snv', value:'F435T', gene_entrez:'3702', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'873', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"mutation of the corresponding residue in Itk (Phe-435) to Thr sensitized the otherwise insensitive Itk to dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCB1', entrez:'5243'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5243', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'874', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19785662', evidence_text:\"Nilotinib and dasatinib were high-affinity substrates of ABCG2, and this protein mediated an effective resistance in cancer cells against these compounds. Nilotinib and dasatinib also interacted with ABCB1, but this transporter provided resistance only against dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCB1', entrez:'5243'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5243', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'875', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19785662', evidence_text:\"Nilotinib and dasatinib were high-affinity substrates of ABCG2, and this protein mediated an effective resistance in cancer cells against these compounds. Nilotinib and dasatinib also interacted with ABCB1, but this transporter provided resistance only against dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'876', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19785662', evidence_text:\"Nilotinib and dasatinib were high-affinity substrates of ABCG2, and this protein mediated an effective resistance in cancer cells against these compounds. Nilotinib and dasatinib also interacted with ABCB1, but this transporter provided resistance only against dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'877', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19785662', evidence_text:\"Nilotinib and dasatinib were high-affinity substrates of ABCG2, and this protein mediated an effective resistance in cancer cells against these compounds. Nilotinib and dasatinib also interacted with ABCB1, but this transporter provided resistance only against dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4233', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'878', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20406949', evidence_text:\"Gastric cancer cell lines positive for the activation of MET were resistant to dasatinib.  Our results suggest that c-Src is a promising target for the treatment of gastric cancer and that analysis of MET amplification might optimize patient selection for treatment with c-Src inhibitors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4233', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'879', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20406949', evidence_text:\"Gastric cancer cell lines positive for the activation of MET were resistant to dasatinib.  Our results suggest that c-Src is a promising target for the treatment of gastric cancer and that analysis of MET amplification might optimize patient selection for treatment with c-Src inhibitors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NEDD9', entrez:'4739'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4739', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'880', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20940402', evidence_text:\"In spite of their aggressive phenotype, MMTV-PyVT;Nedd9(-/-) cells persistently have low levels of Src activation and are hypersensitive to the Src kinase inhibitor dasatinib. The absence of NEDD9 causes a persistent deficiency in Src activation and influences the susceptibility of mammary cancer cell lines to growth inhibition by dasatinib, a drug targeting Src kinase. These studies in sum should significantly inform the use of NEDD9 as a biomarker for clinical response\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NEDD9', entrez:'4739'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4739', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'881', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20940402', evidence_text:\"In spite of their aggressive phenotype, MMTV-PyVT;Nedd9(-/-) cells persistently have low levels of Src activation and are hypersensitive to the Src kinase inhibitor dasatinib. The absence of NEDD9 causes a persistent deficiency in Src activation and influences the susceptibility of mammary cancer cell lines to growth inhibition by dasatinib, a drug targeting Src kinase. These studies in sum should significantly inform the use of NEDD9 as a biomarker for clinical response\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STAT5A', entrez:'6776'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6776', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'882', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21220747', evidence_text:\"Here we show that the expression level of the transcription factor STAT5 is another parameter that determines the sensitivity of BCR-ABL1(+) cells against tyrosine kinase inhibitors (TKIs), such as imatinib, nilotinib, or dasatinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STAT5A', entrez:'6776'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6776', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'883', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21220747', evidence_text:\"Here we show that the expression level of the transcription factor STAT5 is another parameter that determines the sensitivity of BCR-ABL1(+) cells against tyrosine kinase inhibitors (TKIs), such as imatinib, nilotinib, or dasatinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIK3CA ', entrez:'5290'}), (a:Aberration { type:'snv', value:'E545#', gene_entrez:'5290', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'884', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21299849', evidence_text:\"None of the resistant cell lines carried mutations in the kinase domain of BCR-ABL1 and all showed resistance to second generation TKI, nilotinib or dasatinib. PI3K pathway inhibitors effected dephosphorylation of RPS6 in imatinib-resistant cell lines suggesting that an oncogene other than BCR-ABL1 might be responsible for activation of the PI3K/AKT1/mTOR pathway, which would explain the TKI resistance of these cells. We show that the TKI-resistant cell line KCL-22 carries a PI3K_ E545G mutation, a site critical for the constitutive activation of the PI3K/AKT1 pathway. Mutations in PIK3CA, as observed in cell line KCL-22, or PI3K activating oncogenes may undelie TKI-resistance in these cell lines.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIK3CA ', entrez:'5290'}), (a:Aberration { type:'snv', value:'E545G', gene_entrez:'5290', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'885', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21299849', evidence_text:\"None of the resistant cell lines carried mutations in the kinase domain of BCR-ABL1 and all showed resistance to second generation TKI, nilotinib or dasatinib. PI3K pathway inhibitors effected dephosphorylation of RPS6 in imatinib-resistant cell lines suggesting that an oncogene other than BCR-ABL1 might be responsible for activation of the PI3K/AKT1/mTOR pathway, which would explain the TKI resistance of these cells. We show that the TKI-resistant cell line KCL-22 carries a PI3K_ E545G mutation, a site critical for the constitutive activation of the PI3K/AKT1 pathway. Mutations in PIK3CA, as observed in cell line KCL-22, or PI3K activating oncogenes may undelie TKI-resistance in these cell lines.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CBL', entrez:'867'}), (a:Aberration { type:'snv', value:'R420#', gene_entrez:'867', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'886', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22246246', evidence_text:\"Homozygous CBL mutations result in hypersensitivity to growth factors. Over expression of the WT CBL inhibited the growth of a CBL mutant cell line, consistent with the homozygous nature of CBL mutations identified in most patients with myeloid malignancies who harbor CBL mutations. dasatinib is the most effective TKI in a mutant CBL background and specifically reduced the phosphorylation of RTK and SFK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CBL', entrez:'867'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'867', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'887', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22246246', evidence_text:\"Homozygous CBL mutations result in hypersensitivity to growth factors. Over expression of the WT CBL inhibited the growth of a CBL mutant cell line, consistent with the homozygous nature of CBL mutations identified in most patients with myeloid malignancies who harbor CBL mutations. dasatinib is the most effective TKI in a mutant CBL background and specifically reduced the phosphorylation of RTK and SFK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CBL', entrez:'867'}), (a:Aberration { type:'snv', value:'R420Q', gene_entrez:'867', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'888', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22246246', evidence_text:\"Homozygous CBL mutations result in hypersensitivity to growth factors. Over expression of the WT CBL inhibited the growth of a CBL mutant cell line, consistent with the homozygous nature of CBL mutations identified in most patients with myeloid malignancies who harbor CBL mutations. dasatinib is the most effective TKI in a mutant CBL background and specifically reduced the phosphorylation of RTK and SFK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CBL', entrez:'867'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'867', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'889', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22246246', evidence_text:\"Homozygous CBL mutations result in hypersensitivity to growth factors. Over expression of the WT CBL inhibited the growth of a CBL mutant cell line, consistent with the homozygous nature of CBL mutations identified in most patients with myeloid malignancies who harbor CBL mutations. dasatinib is the most effective TKI in a mutant CBL background and specifically reduced the phosphorylation of RTK and SFK. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CRKL', entrez:'1399'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1399', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'890', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22591714', evidence_text:\"In this study, BMS354825, a dual inhibitor for Src and BCR-ABL kinases, but not AMN107, a BCR- ABL specific inhibitor, showed an inhibitory effect on the survival of MKN74 cells with CRKL amplification. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CRKL', entrez:'1399'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1399', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'891', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22591714', evidence_text:\"In this study, BMS354825, a dual inhibitor for Src and BCR-ABL kinases, but not AMN107, a BCR- ABL specific inhibitor, showed an inhibitory effect on the survival of MKN74 cells with CRKL amplification. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'674'}), (a:Aberration { type:'snv', value:'G466#', gene_entrez:'674', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'892', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22649091', evidence_text:\"H1666 and Cal12T cells, which have en- dogenous BRAF inactivation (G466VBRAF), and H661 cells, which we transfected with kinase-impaired BRAF, underwent apo- ptosis and not senescence when exposed to dasatinib (Fig. 3C)...Cells expressing G466VBRAF or Y472CBRAF had higher CRAF activity than did those cells expressing wtBRAF or V600EBRAF (Fig. 1E). |Evidence Level:  Pre-clinical. Cell line. |Disease:  NSCLC. |Annotator Note:  SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'674'}), (a:Aberration { type:'snv', value:'G466V', gene_entrez:'674', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'893', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22649091', evidence_text:\"H1666 and Cal12T cells, which have en- dogenous BRAF inactivation (G466VBRAF), and H661 cells, which we transfected with kinase-impaired BRAF, underwent apo- ptosis and not senescence when exposed to dasatinib (Fig. 3C)...Cells expressing G466VBRAF or Y472CBRAF had higher CRAF activity than did those cells expressing wtBRAF or V600EBRAF (Fig. 1E). |Evidence Level:  Pre-clinical. Cell line. |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'673'}), (a:Aberration { type:'snv', value:'V600#', gene_entrez:'673', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'894', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22649091', evidence_text:\"Expression of the inactivating mutations increased the cells' sensitivity to dasatinib, whereas expression of V600EBRAF induced further resistance (Figures 3A, 3B, and S8). In contrast H1666 and Cal12T cells, which have endogenous BRAF inactivation (G466VBRAF), H661 cells, which we transfected with kinase impaired BRAF, underwent apoptosis and not senescence when exposed to dasatinib (Figure 3C).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'673'}), (a:Aberration { type:'snv', value:'V600E', gene_entrez:'673', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'895', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22649091', evidence_text:\"Expression of the inactivating mutations increased the cells' sensitivity to dasatinib, whereas expression of V600EBRAF induced further resistance (Figures 3A, 3B, and S8). In contrast H1666 and Cal12T cells, which have endogenous BRAF inactivation (G466VBRAF), H661 cells, which we transfected with kinase impaired BRAF, underwent apoptosis and not senescence when exposed to dasatinib (Figure 3C).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'673'}), (a:Aberration { type:'snv', value:'Y472#', gene_entrez:'673', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'896', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22649091', evidence_text:\"We identified a patient with stage IV NSCLC with long-term disease con- trol by single-agent dasatinib therapy alone. His tumor harbored a kinase-inactivating Y472CBRAF mutation that was likely responsible for its unusual sensitivity to dasatinib... As with previously characterized kinase-inactivating BRAF mutations, Y472CBRAF expression led to CRAF, MEK, and ERK activation. Also, NSCLC cell lines with endogenous inactivating BRAF mutation underwent senescence when exposed to dasatinib; the cell lines' dasatinib sensitivity was reversed with the overexpression of active BRAF...These data and PX's pattern of clinical response are consistent with the conclusion that his tumor underwent senescence when exposed to dasatinib owing to its inactivating BRAF mutation. |Evidence Level:  Case Study. Preclinical. Cell Line. |Disease:  NSCLC. |Annotator Note:  SNVs at this position are associated with loss-of-function and drug sensitivity. However, not all changes at this position may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STK11', entrez:'6794'}), (a:Aberration { type:'cnv', value:'under', gene_entrez:'6794', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'897', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22698401', evidence_text:\"Although the mechanism whereby loss of LKB1 kinase activity induces YES activation is not clear, the present data suggest YES as a promising therapeutic target in melanoma lacking LKB1 function. dasatinib exhibits modest antimelanoma activity in human patients (Kluger et al., 2011). Because we and others (Deguchi et al., 2008; Konecny et al., 2009) have noted an ~10-fold decrease in the potency of dasatinib for YES compared to SRC and other SFKs, these results suggest that an SFK inhibitor with greater potency against YES inhibitor would exhibit greater in vivo antimelanoma efficacy. (Annotator note: Lung cancer showed much better response to dasatinib then in melanoma. Possibly due to the amount more of dasatinib needed to hit YES, but this is based on one study.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'STK11', entrez:'6794'}), (a:Aberration { type:'exp', value:'loss', gene_entrez:'6794', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'898', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22698401', evidence_text:\"Although the mechanism whereby loss of LKB1 kinase activity induces YES activation is not clear, the present data suggest YES as a promising therapeutic target in melanoma lacking LKB1 function. dasatinib exhibits modest antimelanoma activity in human patients (Kluger et al., 2011). Because we and others (Deguchi et al., 2008; Konecny et al., 2009) have noted an ~10-fold decrease in the potency of dasatinib for YES compared to SRC and other SFKs, these results suggest that an SFK inhibitor with greater potency against YES inhibitor would exhibit greater in vivo antimelanoma efficacy. (Annotator note: Lung cancer showed much better response to dasatinib then in melanoma. Possibly due to the amount more of dasatinib needed to hit YES, but this is based on one study.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GAB2', entrez:'9846'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9846', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'899', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22858987', evidence_text:\"We show for the first time that Gab2 signaling protects CML cells from various Bcr-Abl inhibitors (imatinib, nilotinib, dasatinib and GNF-2), whereas Gab2 knockdown or haploinsufficiency leads to increased TKI sensitivity. We dissected the underlying molecular mechanism using various Gab2 mutants and kinase inhibitors and identified the Shp2/Ras/ERK and the PI3K/AKT/mTOR axes as the two critical signaling pathways. Gab2-mediated TKI resistance was associated with persistent phosphorylation of Gab2 Y452, a PI3K recruitment site, and consistent with this finding, the protective effect of Gab2 was completely abolished by the combination of dasatinib with the dual PI3K/mTOR inhibitor NVP-BEZ235. The identification of Gab2 as a novel modulator of TKI sensitivity in CML suggests that Gab2 could be exploited as a biomarker and therapeutic target in TKI-resistant disease.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GAB2', entrez:'9846'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9846', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'900', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22858987', evidence_text:\"We show for the first time that Gab2 signaling protects CML cells from various Bcr-Abl inhibitors (imatinib, nilotinib, dasatinib and GNF-2), whereas Gab2 knockdown or haploinsufficiency leads to increased TKI sensitivity. We dissected the underlying molecular mechanism using various Gab2 mutants and kinase inhibitors and identified the Shp2/Ras/ERK and the PI3K/AKT/mTOR axes as the two critical signaling pathways. Gab2-mediated TKI resistance was associated with persistent phosphorylation of Gab2 Y452, a PI3K recruitment site, and consistent with this finding, the protective effect of Gab2 was completely abolished by the combination of dasatinib with the dual PI3K/mTOR inhibitor NVP-BEZ235. The identification of Gab2 as a novel modulator of TKI sensitivity in CML suggests that Gab2 could be exploited as a biomarker and therapeutic target in TKI-resistant disease.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGFBP2', entrez:'3485'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3485', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'901', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24130049', evidence_text:\"The results showed that high IGFBP2 levels were associated with resistance to dasatinib \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGFBP2', entrez:'3485'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3485', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'902', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24130049', evidence_text:\"The results showed that high IGFBP2 levels were associated with resistance to dasatinib \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'903', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24296828', evidence_text:\"Our data indicate another setting in which NF1 loss contributes to tyrosine kinase inhibitor resistance and suggests that it can drive resistance to dasatinib in sensitive cell lines with different mechanisms of sensitivity to dasatinib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'904', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24296828', evidence_text:\"We then assessed whether NF1 knockdown in parental NCI-H2286 cells impacted the sensitivity to dasatinib and observed a decrease in dasatinib sensitivity using all of the shRNA vectors (Figure 2C). In an average of three independent experiments the GI50 was 65.9 nM for the parental line expressing control shRNA targeting the lacZ gene (shLACZ) and 1.9 _M in the dasatinib resistant line. Parental NCI-H2286 cells expressing shRNA vectors targeting NF1 displayed GI50s ranging from 122 to 544 nM, a 1.85 to 8.3 fold shift in GI50 compared to the parental cell line expressing shLACZ. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'905', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24296828', evidence_text:\"Our data indicate another setting in which NF1 loss contributes to tyrosine kinase inhibitor resistance and suggests that it can drive resistance to dasatinib in sensitive cell lines with different mechanisms of sensitivity to dasatinib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'906', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24296828', evidence_text:\"Our data indicate another setting in which NF1 loss contributes to tyrosine kinase inhibitor resistance and suggests that it can drive resistance to dasatinib in sensitive cell lines with different mechanisms of sensitivity to dasatinib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCA3', entrez:'21'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'21', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'907', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed23432194', evidence_text:\"ABCA3 protected leukaemic cells from the cytotoxic effects of the tyrosine kinase inhibitors imatinib, dasatinib, and nilotinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCA3', entrez:'21'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'21', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'908', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed23432194', evidence_text:\"ABCA3 protected leukaemic cells from the cytotoxic effects of the tyrosine kinase inhibitors imatinib, dasatinib, and nilotinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'909', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18938156', evidence_text:\"Our studies demonstrate that all 3 compounds are potent inhibitors of the kinase activity of both DDR1 and DDR2. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'910', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18938156', evidence_text:\"Our studies demonstrate that all 3 compounds are potent inhibitors of the kinase activity of both DDR1 and DDR2. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'T654#', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'911', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"To validate DDR2 as a relevant target of dasatinib in SCCs, we ectopically expressed a DDR2 transgene with a threonine-to-methionine mutation at amino acid 654, a mutation site shown previously to render DDR2 insensitive to dasatinib in a manner similar to the ability of the T790M mutation in EGFR to confer acquired resistance to the tyrosine kinase inhibitors erlotinib and gefitinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'912', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control. |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'G253#', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'913', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'G505#', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'914', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'G774#', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'915', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'I638#', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'916', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'L239#', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'917', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'L63#', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'918', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'G253C', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'919', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'G505S', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'920', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'G774V', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'921', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'I638F', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'922', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'L239R', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'923', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'L63V', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'924', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"We observed that ectopic expression of all 6 DDR2 mutants identified in our primary and secondary screens led to IL-3-independent growth of Ba/F3 cells, as did high expression levels of wild-type DDR2; culture with dasatinib led to cell death in all cell lines expressing DDR2 mutants, with a mean calculated IC50 of 680 nM for the mutants and 30 _M for the control\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'T654M', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'925', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22328973', evidence_text:\"To validate DDR2 as a relevant target of dasatinib in SCCs, we ectopically expressed a DDR2 transgene with a threonine-to-methionine mutation at amino acid 654, a mutation site shown previously to render DDR2 insensitive to dasatinib in a manner similar to the ability of the T790M mutation in EGFR to confer acquired resistance to the tyrosine kinase inhibitors erlotinib and gefitinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'S768#', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'926', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23932362', evidence_text:\"We report a patient with squamous cell carcinoma (SQCC) of the lung and a discoid-receptor-2 (DDR2) kinase domain mutation that responded to dasatinib treatment. Our case report is consistent with previous publications suggesting that DDR2 mutation may confer sensitivity to dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'S768R', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'927', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23932362', evidence_text:\"We report a patient with squamous cell carcinoma (SQCC) of the lung and a discoid-receptor-2 (DDR2) kinase domain mutation that responded to dasatinib treatment. Our case report is consistent with previous publications suggesting that DDR2 mutation may confer sensitivity to dasatinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR2', entrez:'4921'}), (a:Aberration { type:'snv', value:'T654I', gene_entrez:'4921', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'928', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24296828', evidence_text:\"DDR2 T654I is a gatekeeper mutation site which confers dasatinib resistance in a manner analogous to EGFR T790M (50). We have shown previously that expression of DDR2 T654I mutants in HCC-366 and NCI-H2286 confers dasatinib resistance, with an increase in GI50 of 209- and 35-fold respectively for dasatinib in HCC-366 and NCI- H2286 (19). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR1', entrez:'780'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'780', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'929', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18938156', evidence_text:\"Our studies demonstrate that all 3 compounds are potent inhibitors of the kinase activity of both DDR1 and DDR2.  |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR1', entrez:'780'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'780', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'930', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18938156', evidence_text:\"Our studies demonstrate that all 3 compounds are potent inhibitors of the kinase activity of both DDR1 and DDR2. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DDR1', entrez:'780'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'780', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'931', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18938156', evidence_text:\"Our studies demonstrate that all 3 compounds are potent inhibitors of the kinase activity of both DDR1 and DDR2. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CSF3R', entrez:'1441'}), (a:Aberration { type:'small_deletion', value:'S783fs', gene_entrez:'1441', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'932', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23656643', evidence_text:\"Analysis of cells from Patient 3, who had CNL with the CSF3R S783fs mutation (Table S3 and Fig. S2 in the Supplementary Appendix), revealed dramatic sensitivity to the multikinase inhibitor dasatinib (Sprycel, Bristol-Myers Squibb) but no sensitivity to inhibitors of JAK family kinases (Figure 1B).   Taken together, the functional genomic data on the samples from these three patients suggest that there are two different classes of CSF3R mutations: truncation mutations, which result in dysregulation of SRC family-TNK2 kinases, and membrane proximal mutations, which result in dysregulation of JAK family kinases. The data also suggest that truncation mutations confer sensitivity to dasatinib but not to JAK kinase inhibitors, whereas the reverse is true for membrane proximal mutant cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CSF1R', entrez:'1436'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'1436', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'933', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors. |Annotator Note: (PMID article 22698401 indicates that dasatinib doesn't hit YES efficiently.)  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CSF1R', entrez:'1436'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1436', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'934', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CSF1R', entrez:'1436'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1436', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'935', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21670084', evidence_text:\"dasatinib also inhibits other SRC family kinases (SFK), such as LCK, HCK, FYN, YES, FGR, BLK, LYN, and FRK (3). Moreover, dasatinib can block with high potency the kinase activity of certain receptor tyrosine kinases (RTK), such as c-KIT, c-FMS (the receptor of the macrophage colony stimulating factor), platelet-derived growth factor receptors (PDGFR) _ and _, the discoidin domain receptor 1 (DDR1), and Ephrin receptors.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BTK', entrez:'695'}), (a:Aberration { type:'snv', value:'T474#', gene_entrez:'695', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'936', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"Btk T474I and Tec T442I Are Resistant to dasatinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BTK', entrez:'695'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'695', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'937', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"dasatinib Inhibits Btk and Tec Kinase Activity in Vitro and in Cells. |Annotator Note: Is a target. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BTK', entrez:'695'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'695', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'938', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"dasatinib Inhibits Btk and Tec Kinase Activity in Vitro and in Cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BTK', entrez:'695'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'695', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'939', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"dasatinib Inhibits Btk and Tec Kinase Activity in Vitro and in Cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BTK', entrez:'695'}), (a:Aberration { type:'snv', value:'T474I', gene_entrez:'695', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'940', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17684099', evidence_text:\"Btk T474I and Tec T442I Are Resistant to dasatinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F317I', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'941', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F317L', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'942', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F317S', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'943', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F317V', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'944', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Specifically, we would predict that the three BCR-ABL mutations recovered in >90% of the clones in our screen (T315I, T315A, and F317V) should also account for most cases of clinical resistance. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'L248R', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'945', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Q252H', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'946', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'T315A', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'947', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Specifically, we would predict that the three BCR-ABL mutations recovered in >90% of the clones in our screen (T315I, T315A, and F317V) should also account for most cases of clinical resistance. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'V299L', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'948', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'F317#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'949', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Specifically, we would predict that the three BCR-ABL mutations recovered in >90% of the clones in our screen (T315I, T315A, and F317V) should also account for most cases of clinical resistance. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'L248#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'950', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Q252#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'951', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'V299#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'952', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15705718', evidence_text:\"To examine the potential role of conformational binding properties in drug resistance, we mapped the mutations in BCR-ABL capable of conferring resistance to BMS-354825. Through saturation mutagenesis, we identified 10 such BCR-ABL mutations, 8 of which occurred at drug contact residues. Four of these six sites (Leu-248, Val-299, Thr-315, and Phe-317) are BMS-354825 contact residues, as shown by crystallographic analysis, and account for 97.5% of isolates (11). The exceptions were four clones of E255K and one clone of Q252H, mutants shown previously to have a BMS-354825 IC50 for growth <10-fold higher than WT BCR-ABL \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'953', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16203784', evidence_text:\"Is Target. dasatinib (BMS-354825), an ATP-competitive tyrosine kinase inhibitor that sensitively inhibits all members of the Src family, including c-Src, Lck, Fyn, and Yes (IC50 < 1.1 nmol/L; refs. 16-18). At higher concentrations (3 to 28 nmol/L), dasatinib also inhibits the Src kinases Abl, c-Kit, PDGFR, and EphA2\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'954', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16203784', evidence_text:\"Is Target. dasatinib (BMS-354825), an ATP-competitive tyrosine kinase inhibitor that sensitively inhibits all members of the Src family, including c-Src, Lck, Fyn, and Yes (IC50 < 1.1 nmol/L; refs. 16-18). At higher concentrations (3 to 28 nmol/L), dasatinib also inhibits the Src kinases Abl, c-Kit, PDGFR, and EphA2\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'T315I', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'955', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21888027', evidence_text:\"The T315I mutation was found to be resistant to second generation drugs such as dasatinib. The study reinforces the need for new therapeutic options which can target this mutation.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'T315#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'956', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21888027', evidence_text:\"The T315I mutation was found to be resistant to second generation drugs such as dasatinib. The study reinforces the need for new therapeutic options which can target this mutation.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'957', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23704238', evidence_text:\"To assist in deciding when to switch drugs for resistant disease, the NCCN Guidelines suggest basing the choice on mutations: consider nilotinib over dasatinib for V299L and F317L mutations; consider dasatinib over nilotinib for Y253H and E255K/V mutations. |Annotator Note: SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'E255#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'958', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23704238', evidence_text:\"To assist in deciding when to switch drugs for resistant disease, the NCCN Guidelines suggest basing the choice on mutations: consider nilotinib over dasatinib for V299L and F317L mutations; consider dasatinib over nilotinib for Y253H and E255K/V mutations. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Y253#', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'959', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23704238', evidence_text:\"To assist in deciding when to switch drugs for resistant disease, the NCCN Guidelines suggest basing the choice on mutations: consider nilotinib over dasatinib for V299L and F317L mutations; consider dasatinib over nilotinib for Y253H and E255K/V mutations. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'E255K', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'960', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23704238', evidence_text:\"To assist in deciding when to switch drugs for resistant disease, the NCCN Guidelines suggest basing the choice on mutations: consider nilotinib over dasatinib for V299L and F317L mutations; consider dasatinib over nilotinib for Y253H and E255K/V mutations. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'E255V', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'961', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23704238', evidence_text:\"To assist in deciding when to switch drugs for resistant disease, the NCCN Guidelines suggest basing the choice on mutations: consider nilotinib over dasatinib for V299L and F317L mutations; consider dasatinib over nilotinib for Y253H and E255K/V mutations. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABL1', entrez:'25'}), (a:Aberration { type:'snv', value:'Y253H', gene_entrez:'25', entity_class:'biomarker' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'962', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23704238', evidence_text:\"To assist in deciding when to switch drugs for resistant disease, the NCCN Guidelines suggest basing the choice on mutations: consider nilotinib over dasatinib for V299L and F317L mutations; consider dasatinib over nilotinib for Y253H and E255K/V mutations. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'673'}), (a:Aberration { type:'snv', value:'Y472C', gene_entrez:'673', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'963', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22649091', evidence_text:\"We identified a patient with stage IV NSCLC with long-term disease con- trol by single-agent dasatinib therapy alone. His tumor harbored a kinase-inactivating Y472CBRAF mutation that was likely responsible for its unusual sensitivity to dasatinib... As with previously characterized kinase-inactivating BRAF mutations, Y472CBRAF expression led to CRAF, MEK, and ERK activation. Also, NSCLC cell lines with endogenous inactivating BRAF mutation underwent senescence when exposed to dasatinib; the cell lines' dasatinib sensitivity was reversed with the overexpression of active BRAF...These data and PX's pattern of clinical response are consistent with the conclusion that his tumor underwent senescence when exposed to dasatinib owing to its inactivating BRAF mutation. |Evidence Level:  Case Study. Preclinical. Cell Line. |Disease:  NSCLC. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIK3CA ', entrez:'5290'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5290', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'964', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21299849', evidence_text:\"None of the resistant cell lines carried mutations in the kinase domain of BCR-ABL1 and all showed resistance to second generation TKI, nilotinib or dasatinib. PI3K pathway inhibitors effected dephosphorylation of RPS6 in imatinib-resistant cell lines suggesting that an oncogene other than BCR-ABL1 might be responsible for activation of the PI3K/AKT1/mTOR pathway, which would explain the TKI resistance of these cells. We show that the TKI-resistant cell line KCL-22 carries a PI3K_ E545G mutation, a site critical for the constitutive activation of the PI3K/AKT1 pathway. Mutations in PIK3CA, as observed in cell line KCL-22, or PI3K activating oncogenes may undelie TKI-resistance in these cell lines. |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'dasatinib'} ) Merge (r:Rule {rulenum:'965', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24296828', evidence_text:\"Our data indicate another setting in which NF1 loss contributes to tyrosine kinase inhibitor resistance and suggests that it can drive resistance to dasatinib in sensitive cell lines with different mechanisms of sensitivity to dasatinib.  |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61#', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'decarbazine'} ) Merge (r:Rule {rulenum:'966', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24941944', evidence_text:\"NRAS mutant melanoma cells are less responsive to DTIC than BRAF mutant cells. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'decarbazine'} ) Merge (r:Rule {rulenum:'967', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24941944', evidence_text:\"NRAS mutant melanoma cells are less responsive to DTIC than BRAF mutant cells. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note:  SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP2A', entrez:'7153'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7153', entity_class:'biomarker' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'968', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21189395', evidence_text:\"In a study involving nearly 5,000 breast malignancies, both test set and validation set demonstrate that TOP2A coamplification, not HER2 amplification, is the clinically useful predictive marker of an incremental response to anthracycline-based chemotherapy. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP2A', entrez:'7153'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7153', entity_class:'biomarker' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'969', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21189395', evidence_text:\"In a study involving nearly 5,000 breast malignancies, both test set and validation set demonstrate that TOP2A coamplification, not HER2 amplification, is the clinically useful predictive marker of an incremental response to anthracycline-based chemotherapy. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP2A', entrez:'7153'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7153', entity_class:'biomarker' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'970', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23122841', evidence_text:\"Resistance to doxorubicin, etoposide and mitoxantrone was associated with down-regulation of topoisomerase II.  CAL51-shTOP2A cells expressed approximately 50% less TOP2A mRNA than the corresponding control cells transfected with the empty vector (Fig. 5A). Down-regulation of TOP2A led to a notable increase in the number of doxorubicin resistant colonies (Fig. 5B). Thus, down-regulation of TOP2A leads to drug resistance in CALDOX cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP2A', entrez:'7153'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7153', entity_class:'biomarker' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'971', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23122841', evidence_text:\"Resistance to doxorubicin, etoposide and mitoxantrone was associated with down-regulation of topoisomerase II.  CAL51-shTOP2A cells expressed approximately 50% less TOP2A mRNA than the corresponding control cells transfected with the empty vector (Fig. 5A). Down-regulation of TOP2A led to a notable increase in the number of doxorubicin resistant colonies (Fig. 5B). Thus, down-regulation of TOP2A leads to drug resistance in CALDOX cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP2A', entrez:'7153'}), (a:Aberration { type:'snv', value:'R449#', gene_entrez:'7153', entity_class:'biomarker' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'972', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/2853972', evidence_text:\"The CEM/VM1 cell line (TOP2A R449Q), selected for resistance to teniposide, is cross-resistant to etoposide, doxorubicin, and actinomycin D.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TOP2A', entrez:'7153'}), (a:Aberration { type:'snv', value:'R449Q', gene_entrez:'7153', entity_class:'biomarker' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'973', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/2853972', evidence_text:\"The CEM/VM1 cell line (TOP2A R449Q), selected for resistance to teniposide, is cross-resistant to etoposide, doxorubicin, and actinomycin D.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K3', entrez:'4215'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4215', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'974', evidence:'direct', pmid:'/http://www.ncbi.nlm.nih.gov/pubmed/14662759', evidence_text:\"MEKK3 stable cells showed reduced activation of caspases 3 and 8 and poly(ADP-ribose) polymerase cleavage and dramatically increased resistance to apoptosis induced by tumor necrosis factor-related apoptosis-inducing ligand, doxorubicin, daunorubicin, camptothecin, and paclitaxel. Intriguingly, analysis of human breast and ovarian cancers showed that a significant fraction of these samples have elevated MEKK3 protein levels with corresponding increases in NFkappaB binding activities. Thus, our results established that elevated expression of MEKK3 appears to be a frequent occurrence in breast and ovarian cancers and that overexpression of MEKK3 in cells leads to increased NFkappaB activity and increased expression of cell survival factors and ultimately contributes to their resistance to apoptosis. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PTK7', entrez:'5754'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5754', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'975', evidence:'inferred', pmid:'/http://www.ncbi.nlm.nih.gov/pubmed/20558616', evidence_text:\"In vitro, expression of PTK7 in cultured leukemia cells promotes cell migration, cell survival, and resistance to anthracycline-induced apoptosis. The intracellular region of PTK7 is required for these effects. Furthermore, we efficiently sensitized primary AML blasts to anthracycline-mediated cell death using a recombinant soluble PTK7-Fc protein. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NTRK2', entrez:'4915'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4915', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'976', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/12438236', evidence_text:\"Here, we provide evidence that the TrkB-BDNF pathway is associated with enhanced survival and resistance to chemotherapy (dox, etop) in neuroblastoma. In all cases, the TrkB-expressing subclones were more resistant to treatment than the parent line. Furthermore, when the TrkB tyrosine kinase was blocked with the Trk-specific inhibitor CEP-2563, or by neutralizing antibody to BDNF, sensitize to chemotherapy was significantly increased. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NTRK2', entrez:'4915'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4915', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'977', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/12438236', evidence_text:\"Here, we provide evidence that the TrkB-BDNF pathway is associated with enhanced survival and resistance to chemotherapy (dox, etop) in neuroblastoma. In all cases, the TrkB-expressing subclones were more resistant to treatment than the parent line. Furthermore, when the TrkB tyrosine kinase was blocked with the Trk-specific inhibitor CEP-2563, or by neutralizing antibody to BDNF, sensitize to chemotherapy was significantly increased. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP3K3', entrez:'4215'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4215', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'978', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/14662759', evidence_text:\"MEKK3 stable cells showed reduced activation of caspases 3 and 8 and poly(ADP-ribose) polymerase cleavage and dramatically increased resistance to apoptosis induced by tumor necrosis factor-related apoptosis-inducing ligand, doxorubicin, daunorubicin, camptothecin, and paclitaxel. Intriguingly, analysis of human breast and ovarian cancers showed that a significant fraction of these samples have elevated MEKK3 protein levels with corresponding increases in NFkappaB binding activities. Thus, our results established that elevated expression of MEKK3 appears to be a frequent occurrence in breast and ovarian cancers and that overexpression of MEKK3 in cells leads to increased NFkappaB activity and increased expression of cell survival factors and ultimately contributes to their resistance to apoptosis. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AKR1A1', entrez:'10327'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'10327', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'979', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17482758', evidence_text:\"In the present investigation, human pancreas carcinoma cells were transfected with three important DRC reductases, namely carbonyl reductase (CBR1), aldehyde reductase (AKR1A1) and aldose reductase (AKR1B1), and levels of resistance towards DCR determined. Overexpression of all three reductases lead to a higher DRC inactivation and to an elevation of chemoresistance (7-fold for CBR1, 4.5-fold for AKR1A1 and 3.7-fold for AKR1B1), when IC(50)-values were considered. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AKR1A1', entrez:'10327'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'10327', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'980', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17482758', evidence_text:\"In the present investigation, human pancreas carcinoma cells were transfected with three important DRC reductases, namely carbonyl reductase (CBR1), aldehyde reductase (AKR1A1) and aldose reductase (AKR1B1), and levels of resistance towards DCR determined. Overexpression of all three reductases lead to a higher DRC inactivation and to an elevation of chemoresistance (7-fold for CBR1, 4.5-fold for AKR1A1 and 3.7-fold for AKR1B1), when IC(50)-values were considered. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AKR1B1', entrez:'231'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'231', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'981', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17482758', evidence_text:\"In the present investigation, human pancreas carcinoma cells were transfected with three important DRC reductases, namely carbonyl reductase (CBR1), aldehyde reductase (AKR1A1) and aldose reductase (AKR1B1), and levels of resistance towards DCR determined. Overexpression of all three reductases lead to a higher DRC inactivation and to an elevation of chemoresistance (7-fold for CBR1, 4.5-fold for AKR1A1 and 3.7-fold for AKR1B1), when IC(50)-values were considered. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AKR1B1', entrez:'231'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'231', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'982', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17482758', evidence_text:\"In the present investigation, human pancreas carcinoma cells were transfected with three important DRC reductases, namely carbonyl reductase (CBR1), aldehyde reductase (AKR1A1) and aldose reductase (AKR1B1), and levels of resistance towards DCR determined. Overexpression of all three reductases lead to a higher DRC inactivation and to an elevation of chemoresistance (7-fold for CBR1, 4.5-fold for AKR1A1 and 3.7-fold for AKR1B1), when IC(50)-values were considered. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DUSP1', entrez:'1843'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1843', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'983', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483361', evidence_text:\"Transient or stable overexpression of MKP-1 (DUSP1) reduced caspase activation and DNA fragmentation while enhancing viability in the face of treatment with alkylating agents (mechlorethamine), anthracylines (doxorubicin), and microtubule inhibitors (paclitaxel). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DUSP1', entrez:'1843'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1843', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'984', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17483361', evidence_text:\"Transient or stable overexpression of MKP-1 (DUSP1) reduced caspase activation and DNA fragmentation while enhancing viability in the face of treatment with alkylating agents (mechlorethamine), anthracylines (doxorubicin), and microtubule inhibitors (paclitaxel). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCB1', entrez:'5243'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5243', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'985', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17854304', evidence_text:\"Using a combination of microarray analysis and small interfering (si)RNA experiments, we demonstrated that T-ALL cells acquired resistance to DNR by overexpression of ABCB1. For Jurkat DNR-resistant cells, we developed a rank-ordered list (steps 3 and 4), which showed that 19 probesets were upregulated sixfold; ABCB1 (also known as MDR1, P-glycoprotein) was ranked in the first and second positions. We found that the resulting transient suppression of ABCB1 mRNA restored drug sensitivity to DNR-resistant cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCB1', entrez:'5243'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5243', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'986', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17854304', evidence_text:\"Using a combination of microarray analysis and small interfering (si)RNA experiments, we demonstrated that T-ALL cells acquired resistance to DNR by overexpression of ABCB1. For Jurkat DNR-resistant cells, we developed a rank-ordered list (steps 3 and 4), which showed that 19 probesets were upregulated sixfold; ABCB1 (also known as MDR1, P-glycoprotein) was ranked in the first and second positions. We found that the resulting transient suppression of ABCB1 mRNA restored drug sensitivity to DNR-resistant cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDK5RAP2', entrez:'55755'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'55755', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'987', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19282672', evidence_text:\"CDK5RAP2-knockdown cells have increased resistance to paclitaxel and doxorubicin, and this resistance is partially rescued upon restoration of CDK5RAP2 expression\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CDK5RAP2', entrez:'55755'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'55755', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'988', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19282672', evidence_text:\"CDK5RAP2-knockdown cells have increased resistance to paclitaxel and doxorubicin, and this resistance is partially rescued upon restoration of CDK5RAP2 expression\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CSE1L', entrez:'1434'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1434', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'989', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20020938', evidence_text:\"Our studies showed that CSE1L/CAS over-expression increased p53 accumulation and apoptosis induced by 5-fluorouracil, doxorubicin, cisplatin, and tamoxifen in HT-29 cancer cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CSE1L', entrez:'1434'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1434', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'990', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20020938', evidence_text:\"Our studies showed that CSE1L/CAS over-expression increased p53 accumulation and apoptosis induced by 5-fluorouracil, doxorubicin, cisplatin, and tamoxifen in HT-29 cancer cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PTK7', entrez:'5754'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5754', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'991', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20558616', evidence_text:\"In vitro, expression of PTK7 in cultured leukemia cells promotes cell migration, cell survival, and resistance to anthracycline-induced apoptosis. The intracellular region of PTK7 is required for these effects. Furthermore, we efficiently sensitized primary AML blasts to anthracycline-mediated cell death using a recombinant soluble PTK7-Fc protein. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCB5', entrez:'340273'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'340273', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'992', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20682318', evidence_text:\"Chemoresistant cells that expressed GEP had increased levels of ABCB5; suppression of ABCB5 sensitized the cells to doxorubicin uptake and apoptosis.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCB5', entrez:'340273'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'340273', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'993', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20682318', evidence_text:\"Chemoresistant cells that expressed GEP had increased levels of ABCB5; suppression of ABCB5 sensitized the cells to doxorubicin uptake and apoptosis.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCC1', entrez:'4363'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4363', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'994', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21143116', evidence_text:\"Cells that highly express MRP1/ABCC1 confer resistance to a variety of natural product anticancer drugs such as vinca alkaloids (e.g. vincristine), anthracyclines (e.g. etoposide) and epipodophyllotoxins (e.g. doxorubicin and mitoxantrone). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCC1', entrez:'4363'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4363', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'995', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21143116', evidence_text:\"Cells that highly express MRP1/ABCC1 confer resistance to a variety of natural product anticancer drugs such as vinca alkaloids (e.g. vincristine), anthracyclines (e.g. etoposide) and epipodophyllotoxins (e.g. doxorubicin and mitoxantrone). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGF2BP3', entrez:'10643'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'10643', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'996', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21613208', evidence_text:\"Furthermore, IMP3 knockdown also made U138 cells more sensitive to paclitaxel, temozolomide, and adriamycin. Thus, we have identified IMP3 as a GBM-specific proproliferative and proinvasive marker acting through IGF-2 resulting in the activation of oncogenic PI3K and MAPK pathways.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGF2BP3', entrez:'10643'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'10643', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'997', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21613208', evidence_text:\"Furthermore, IMP3 knockdown also made U138 cells more sensitive to paclitaxel, temozolomide, and adriamycin. Thus, we have identified IMP3 as a GBM-specific proproliferative and proinvasive marker acting through IGF-2 resulting in the activation of oncogenic PI3K and MAPK pathways.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NTHL1', entrez:'4913'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4913', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'998', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22475343', evidence_text:\"Knockdown of MBD4 and NTHL1 expression in resistant cells using siRNA decreased cell survival after treatment with H(2)O(2) and doxorubicin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NTHL1', entrez:'4913'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4913', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'999', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22475343', evidence_text:\"Knockdown of MBD4 and NTHL1 expression in resistant cells using siRNA decreased cell survival after treatment with H(2)O(2) and doxorubicin. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CBR1', entrez:'873'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'873', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1000', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22562609', evidence_text:\"Increased expression of CBR1 significantly reduced the in vitro cytotoxicity of daunorubicin and also positively correlated with intracellular DOL levels. This pilot study suggests that CBR1 RNA expression may be helpful in identifying AML patients at risk of developing resistance or toxicity to daunorubicin due to increased formation of DOL. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CBR1', entrez:'873'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'873', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1001', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22562609', evidence_text:\"Increased expression of CBR1 significantly reduced the in vitro cytotoxicity of daunorubicin and also positively correlated with intracellular DOL levels. This pilot study suggests that CBR1 RNA expression may be helpful in identifying AML patients at risk of developing resistance or toxicity to daunorubicin due to increased formation of DOL. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIAA0101', entrez:'9768'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9768', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1002', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22576474', evidence_text:\"Additionally, KIAA0101 tv1 prevented cells from apoptosis caused by ADR through suppressing the acetylation of p53 at Lys382. Immunoprecipitation analysis and mammalian two-hybrid assay indicated that KIAA0101 tv1 bound to the transactivation region (1-42 amino acids) of p53 and strongly inhibits its transcriptional activity. Taken together, our data suggest that KIAA0101 tv1 played an important role in the late stage of metastatic HCC and prevented apoptosis after chemotherapeutic drug treatment through inhibiting the transcriptional activity of the p53 gene.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIAA0101', entrez:'9768'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9768', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1003', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22576474', evidence_text:\"Additionally, KIAA0101 tv1 prevented cells from apoptosis caused by ADR through suppressing the acetylation of p53 at Lys382. Immunoprecipitation analysis and mammalian two-hybrid assay indicated that KIAA0101 tv1 bound to the transactivation region (1-42 amino acids) of p53 and strongly inhibits its transcriptional activity. Taken together, our data suggest that KIAA0101 tv1 played an important role in the late stage of metastatic HCC and prevented apoptosis after chemotherapeutic drug treatment through inhibiting the transcriptional activity of the p53 gene.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'B3GNT8', entrez:'374907'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'374907', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1004', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22579717', evidence_text:\"To further evaluate the effects of B3GNT8 or ST8SIA4 silencing on cells chemosensitivity, each cells group was treated with adriamycin, paclitaxel and vincristine, respectively. The K562/ADR-B3GNT8 siRNA cells or K562/ADR-ST8SIA4 siRNA cells showed increased sensitivity to these chemotherapeutic drugs, as compared with K562/ADR-control siRNA groups\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'B3GNT8', entrez:'374907'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'374907', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1005', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22579717', evidence_text:\"To further evaluate the effects of B3GNT8 or ST8SIA4 silencing on cells chemosensitivity, each cells group was treated with adriamycin, paclitaxel and vincristine, respectively. The K562/ADR-B3GNT8 siRNA cells or K562/ADR-ST8SIA4 siRNA cells showed increased sensitivity to these chemotherapeutic drugs, as compared with K562/ADR-control siRNA groups\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MN1', entrez:'4330'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4330', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1006', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22905229', evidence_text:\"In vitro murine AML cells that overexpressed MN1 became resistant to treatment with cytarabine and highly resistant to doxorubicin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MN1', entrez:'4330'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4330', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1007', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22905229', evidence_text:\"In vitro murine AML cells that overexpressed MN1 became resistant to treatment with cytarabine and highly resistant to doxorubicin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FOXM1', entrez:'2305'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2305', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1008', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108394', evidence_text:\"Ectopic expression of FOXM1 can increase cell viability and abrogate DSBs sustained by MCF-7 cells following epirubicin, owing to an enhancement in repair efficiency. Conversely, alkaline comet and _H2AX foci formation assays show that Foxm1-null cells are hypersensitive to DNA damage, epirubicin and _-irradiation. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FOXM1', entrez:'2305'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2305', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1009', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23108394', evidence_text:\"Ectopic expression of FOXM1 can increase cell viability and abrogate DSBs sustained by MCF-7 cells following epirubicin, owing to an enhancement in repair efficiency. Conversely, alkaline comet and _H2AX foci formation assays show that Foxm1-null cells are hypersensitive to DNA damage, epirubicin and _-irradiation. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DLX4', entrez:'1748'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1748', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1010', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23222298', evidence_text:\"Expression of DLX4 significantly reduced sensitivity of MDA-MB-468 cells to doxorubicin (P = 0.005) but did not alter sensitivity to paclitaxel (microtubule disruptor), 5-fluorouracil (pyrimidine analogue), or cyclophosphamide (alkylating agent; Supplementary Table SI). Analysis of breast cancer datasets revealed that TOP2A-high cases that also highly expressed DLX4 responded more poorly to anthracycline-based chemotherapy than TOP2A-high cases that expressed DLX4 at low levels.  Our findings indicate that DLX4 induces TOP2A expression but reduces sensitivity of tumor cells to TOP2 poisons by stimulating Ku-dependent repair of DSBs. These opposing activities of DLX4 could explain why some TOP2A-overexpressing tumors are not highly sensitive to TOP2 poisons.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DLX4', entrez:'1748'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1748', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1011', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23222298', evidence_text:\"Expression of DLX4 significantly reduced sensitivity of MDA-MB-468 cells to doxorubicin (P = 0.005) but did not alter sensitivity to paclitaxel (microtubule disruptor), 5-fluorouracil (pyrimidine analogue), or cyclophosphamide (alkylating agent; Supplementary Table SI). Analysis of breast cancer datasets revealed that TOP2A-high cases that also highly expressed DLX4 responded more poorly to anthracycline-based chemotherapy than TOP2A-high cases that expressed DLX4 at low levels.  Our findings indicate that DLX4 induces TOP2A expression but reduces sensitivity of tumor cells to TOP2 poisons by stimulating Ku-dependent repair of DSBs. These opposing activities of DLX4 could explain why some TOP2A-overexpressing tumors are not highly sensitive to TOP2 poisons.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FASN', entrez:'2194'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2194', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1012', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23613870', evidence_text:\"Caveolin-1 (Cav-1), Fatty acid synthase (FASN) and Cytochrome P450 (CYP450) protein levels were elevated in drug resistant cells. Downregulation of these molecules by respective siRNAs and/or by specific pharmacological inhibitors resensitize cells to paclitaxel. Interestingly, these drug resistant cells were also less sensitive to vinblastine, doxorubicin and methotrexate with the exception of cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FASN', entrez:'2194'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2194', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1013', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23613870', evidence_text:\"Caveolin-1 (Cav-1), Fatty acid synthase (FASN) and Cytochrome P450 (CYP450) protein levels were elevated in drug resistant cells. Downregulation of these molecules by respective siRNAs and/or by specific pharmacological inhibitors resensitize cells to paclitaxel. Interestingly, these drug resistant cells were also less sensitive to vinblastine, doxorubicin and methotrexate with the exception of cisplatin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1014', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24179544', evidence_text:\"The Eca109/ABCG2 esophageal cancer cells with ABCG2 gene overexpression were resistant to adriamycin (ADM), daunorubicin (DNR) and mitoxantrone (MIT), which indicated that ABCG2 may be associated with drug resistance in esophageal cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ABCG2', entrez:'9429'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9429', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1015', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24179544', evidence_text:\"The Eca109/ABCG2 esophageal cancer cells with ABCG2 gene overexpression were resistant to adriamycin (ADM), daunorubicin (DNR) and mitoxantrone (MIT), which indicated that ABCG2 may be associated with drug resistance in esophageal cancer. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PRKDC', entrez:'5591'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5591', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1016', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/9685611', evidence_text:\"The overexpression of DNA-PK is thus shown to be a novel cellular adaptation mechanistically contributing to the resistance of cancer cells to the anthracycline drug adriamycin, and as such, may have implications for its therapeutic use.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PRKDC', entrez:'5591'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5591', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1017', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/9685611', evidence_text:\"The overexpression of DNA-PK is thus shown to be a novel cellular adaptation mechanistically contributing to the resistance of cancer cells to the anthracycline drug adriamycin, and as such, may have implications for its therapeutic use.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ZNF217', entrez:'7764'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'7764', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1018', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed16203743', evidence_text:\"We present evidence that ZNF217 can attenuate apoptotic signals resulting from telomere dysfunction as well as from doxorubicin-induced DNA damage and that silencing ZNF217 with siRNA restores sensitize to doxorubicin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ZNF217', entrez:'7764'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7764', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1019', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed16203743', evidence_text:\"We present evidence that ZNF217 can attenuate apoptotic signals resulting from telomere dysfunction as well as from doxorubicin-induced DNA damage and that silencing ZNF217 with siRNA restores sensitize to doxorubicin.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RND3', entrez:'390'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'390', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1020', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed19101510', evidence_text:\"Our study demonstrated that RhoE may promote the multidrug resistance phenotype of gastric cancer cells by decreasing the expression of Bax at posttranscriptional level, thus inhibiting vincristine-induced apoptosis. The IC50 values of SGC7901-RhoE cells for ADR, VCR, CDDP, 5-FU, VP-16, MMC were significantly increased compared with those of SGC7901-FLAG cells\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'RND3', entrez:'390'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'390', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1021', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed19101510', evidence_text:\"Our study demonstrated that RhoE may promote the multidrug resistance phenotype of gastric cancer cells by decreasing the expression of Bax at posttranscriptional level, thus inhibiting vincristine-induced apoptosis. The IC50 values of SGC7901-RhoE cells for ADR, VCR, CDDP, 5-FU, VP-16, MMC were significantly increased compared with those of SGC7901-FLAG cells\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP53', entrez:'7157'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'7157', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1022', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed22763759', evidence_text:\"The growth-inhibitory activity of vincristine, doxorubicin, carboplatin, etoposide, and temozolomide was significantly impaired by silencing of TP53. In conclusion, these results indicate a potential predictive role of p53 in choroid plexus carcinomas. Alterations of p53 should be taken into account when evaluating the effect of anticancer agents in future clinical trials.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TP53', entrez:'7157'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'7157', entity_class:'modifier' }), (d:Drug {name:'doxorubicin'} ) Merge (r:Rule {rulenum:'1023', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed22763759', evidence_text:\"The growth-inhibitory activity of vincristine, doxorubicin, carboplatin, etoposide, and temozolomide was significantly impaired by silencing of TP53. In conclusion, these results indicate a potential predictive role of p53 in choroid plexus carcinomas. Alterations of p53 should be taken into account when evaluating the effect of anticancer agents in future clinical trials.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NFKBIA', entrez:'4792'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4792', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1024', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21430781', evidence_text:\"Because reduced I_B levels were associated with erlotinib resistance in cell lines and tumour models, we asked if I_B expression correlated with EGFR TKI response in patients with EGFR-mutant lung cancers. Low I_B expression (high-NF-_B' activation state) was predictive of worse progression-free survival (Fig. 4c and Supplementary Table 4a) and decreased overall survival (Supplementary Fig. 16a and Supplementary Table 4b). |Annotator Note:SNVs should be loss-of-function and associated with reduced drug sensitivity. However, not all SNVs may result in loss-of-function and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIK3CA', entrez:'5290'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5290', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1025', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16906227', evidence_text:\"Continued activation of PI3K signaling by the PIK3CA oncogenic mutant, p110_ E545K, was sufficient to abrogate gefitinib-induced apoptosis. |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR1', entrez:'2260'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'2260', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1026', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23536707', evidence_text:\"Expression analyses indicated that both FGFR1 and FGF2 were increased in PC9 gefitinib-resistant (PC9 GR) cells as compared with PC9 nave (PC9 na) cells. Importantly, proliferation of gefitinib-resistant cells was dependent on the FGF2 -FGFR1 pathway. Mechanistically, inhibition of either FGF2 or FGFR1 by siRNA or FGFR inhibitor (PD173074) restored gefitinib sensitivity in PC9 GR cells. These data suggest that FGF2 -FGFR1 activation through an autocrine loop is a novel mechanism of acquired resistance to EGFR-TKIs. |Annotator Note: SNVs should be activating and associated with reduced drug sensitivity. However, not all SNVs may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'small_insertion', value:'Ins 774YVMA', gene_entrez:'2064', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1027', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"n contrast, the insertion ERBB2 mutant rendered HCC827 cells resistant to gefitinib but not to PF00299804\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'small_insertion', value:'Ins G776C', gene_entrez:'2064', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1028', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"We examined NSCLC cell lines with known EGFR, K-ras, or ERBB2 mutations (Table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'small_insertion', value:'Ins G776V', gene_entrez:'2064', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1029', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"We examined NSCLC cell lines with known EGFR, K-ras, or ERBB2 mutations (Table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAPK1', entrez:'5594'}), (a:Aberration { type:'snv', value:'E322#', gene_entrez:'5594', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1030', evidence:'inferred', pmid:'http://bit.ly/QtmV66', evidence_text:\"A bi-allelic missense mutation of MAPK1(E322K) was detected in the tumor in the absence of other mutations reported to be correlated with EGFR TKI sensitivity in preclinical models or cancer patients. Combined bioinformatics analyses of genomics-drug databases revealed, for the first time, that loss-of-function of MAPK1 or MAPK1 mutation correlates with erlotinib sensitivity. Functional studies confirmed heightened sensitivity to erlotinib by loss-of-function of MAPK1 in HNSCC cells. Our preliminary findings suggest that this MAPK1 mutation, which is also found at low frequencies in HNSCC and cervical cancers, may identify a subgroup of cancer patients who may be cured with EGFR TKI monotherapy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAPK1', entrez:'5594'}), (a:Aberration { type:'snv', value:'E322K', gene_entrez:'5594', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1031', evidence:'direct', pmid:'http://bit.ly/QtmV66', evidence_text:\"A bi-allelic missense mutation of MAPK1(E322K) was detected in the tumor in the absence of other mutations reported to be correlated with EGFR TKI sensitivity in preclinical models or cancer patients. Combined bioinformatics analyses of genomics-drug databases revealed, for the first time, that loss-of-function of MAPK1 or MAPK1 mutation correlates with erlotinib sensitivity. Functional studies confirmed heightened sensitivity to erlotinib by loss-of-function of MAPK1 in HNSCC cells. Our preliminary findings suggest that this MAPK1 mutation, which is also found at low frequencies in HNSCC and cervical cancers, may identify a subgroup of cancer patients who may be cured with EGFR TKI monotherapy.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G12#', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1032', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15696205', evidence_text:\"We screened 60 lung adenocarcinomas defined as sensitive or refractory to gefitinib or erlotinib for mutations in EGFR and KRAS. We show that mutations in KRAS are associated with a lack of sensitivity to either drug.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G12C', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1033', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15696205', evidence_text:\"We screened 60 lung adenocarcinomas defined as sensitive or refractory to gefitinib or erlotinib for mutations in EGFR and KRAS. We show that mutations in KRAS are associated with a lack of sensitivity to either drug.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G12D', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1034', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15696205', evidence_text:\"We screened 60 lung adenocarcinomas defined as sensitive or refractory to gefitinib or erlotinib for mutations in EGFR and KRAS. We show that mutations in KRAS are associated with a lack of sensitivity to either drug.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G12S', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1035', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15696205', evidence_text:\"We screened 60 lung adenocarcinomas defined as sensitive or refractory to gefitinib or erlotinib for mutations in EGFR and KRAS. We show that mutations in KRAS are associated with a lack of sensitivity to either drug.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G12V', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1036', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15696205', evidence_text:\"We screened 60 lung adenocarcinomas defined as sensitive or refractory to gefitinib or erlotinib for mutations in EGFR and KRAS. We show that mutations in KRAS are associated with a lack of sensitivity to either drug.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G13#', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1037', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15696205', evidence_text:\"We screened 60 lung adenocarcinomas defined as sensitive or refractory to gefitinib or erlotinib for mutations in EGFR and KRAS. We show that mutations in KRAS are associated with a lack of sensitivity to either drug.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'snv', value:'G13C', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1038', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15696205', evidence_text:\"Table 2. KRAS Exon 2 Mutations Found in Non-Small-Cell Lung Cancers Refractory to Treatment with Gefitinib or erlotinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PTEN', entrez:'5728'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5728', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1039', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16282176', evidence_text:\"We found that the lack of PTEN in gliomas is associated with resistance to EGFR kinase inhibitors. Moreover, in four isogenic cell lines, loss of PTEN markedly diminished responsiveness to EGFR kinase inhibitors. Our data also suggest that downstream inhibition of the PI3K pathway, perhaps at the level of the mammalian target of rapamycin (a kinase related to PI3K) (Figure 1), could be combined with EGFR kinase inhibitors in patients with PTEN-deficient tumors to promote responsiveness\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PTEN', entrez:'5728'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'5728', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1040', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16282176', evidence_text:\"We found that the lack of PTEN in gliomas is associated with resistance to EGFR kinase inhibitors. Moreover, in four isogenic cell lines, loss of PTEN markedly diminished responsiveness to EGFR kinase inhibitors. Our data also suggest that downstream inhibition of the PI3K pathway, perhaps at the level of the mammalian target of rapamycin (a kinase related to PI3K) (Figure 1), could be combined with EGFR kinase inhibitors in patients with PTEN-deficient tumors to promote responsiveness\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PTEN', entrez:'5728'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'5728', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1041', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16282176', evidence_text:\"We found that the lack of PTEN in gliomas is associated with resistance to EGFR kinase inhibitors. Moreover, in four isogenic cell lines, loss of PTEN markedly diminished responsiveness to EGFR kinase inhibitors. Our data also suggest that downstream inhibition of the PI3K pathway, perhaps at the level of the mammalian target of rapamycin (a kinase related to PI3K) (Figure 1), could be combined with EGFR kinase inhibitors in patients with PTEN-deficient tumors to promote responsiveness\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PTEN', entrez:'5728'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'5728', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1042', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16282176', evidence_text:\"We found that the lack of PTEN in gliomas is associated with resistance to EGFR kinase inhibitors. Moreover, in four isogenic cell lines, loss of PTEN markedly diminished responsiveness to EGFR kinase inhibitors. Our data also suggest that downstream inhibition of the PI3K pathway, perhaps at the level of the mammalian target of rapamycin (a kinase related to PI3K) (Figure 1), could be combined with EGFR kinase inhibitors in patients with PTEN-deficient tumors to promote responsiveness\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERRFI1', entrez:'54206'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'54206', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1043', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16648858', evidence_text:\"Mig6 knockout (Errfi1_/_) mice exhibit hyperactivation of endogenous EGFR, resulting in hyperproliferation and impaired differentiation of epidermal keratinocytes. In addition, carcinogen-induced tumors in Errfi1_/_ mice are unusually sensitive to the EGFR TKI gefitinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERRFI1', entrez:'54206'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'54206', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1044', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16648858', evidence_text:\"Mig6 knockout (Errfi1_/_) mice exhibit hyperactivation of endogenous EGFR, resulting in hyperproliferation and impaired differentiation of epidermal keratinocytes. In addition, carcinogen-induced tumors in Errfi1_/_ mice are unusually sensitive to the EGFR TKI gefitinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ADAM17', entrez:'6868'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6868', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1045', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16843264', evidence_text:\"ADAM17 protein, a major ErbB ligand sheddase, is upregulated in NSCLC and is required not only for heregulin-dependent HER3 signaling, but also for EGFR ligand-dependent signaling in NSCLC cell lines. A selective ADAM inhibitor, INCB3619, prevents the processing and activation of multiple ErbB ligands, including heregulin. In addition, INCB3619 inhibits gefitinib-resistant HER3 signaling and enhances gefitinib inhibition of EGFR signaling in NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ADAM17', entrez:'6868'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6868', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1046', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16843264', evidence_text:\"ADAM17 protein, a major ErbB ligand sheddase, is upregulated in NSCLC and is required not only for heregulin-dependent HER3 signaling, but also for EGFR ligand-dependent signaling in NSCLC cell lines. A selective ADAM inhibitor, INCB3619, prevents the processing and activation of multiple ErbB ligands, including heregulin. In addition, INCB3619 inhibits gefitinib-resistant HER3 signaling and enhances gefitinib inhibition of EGFR signaling in NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRG1', entrez:'3084'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3084', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1047', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16843264', evidence_text:\"Heregulin contributes to gefitinib resistance in NSCLC cell lines\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRG1', entrez:'3084'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3084', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1048', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16843264', evidence_text:\"Heregulin contributes to gefitinib resistance in NSCLC cell lines\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIK3CA', entrez:'5290'}), (a:Aberration { type:'snv', value:'E545#', gene_entrez:'5290', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1049', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16906227', evidence_text:\"Continued activation of PI3K signaling by the PIK3CA oncogenic mutant, p110_ E545K, was sufficient to abrogate gefitinib-induced apoptosis. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIK3CA', entrez:'5290'}), (a:Aberration { type:'snv', value:'E545K', gene_entrez:'5290', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1050', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16906227', evidence_text:\"Continued activation of PI3K signaling by the PIK3CA oncogenic mutant, p110_ E545K, was sufficient to abrogate gefitinib-induced apoptosis. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'snv', value:'L726#', gene_entrez:'2064', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1051', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17638894', evidence_text:\"IR-5 bears a novel point mutation in the HER2/neu that corresponds to a L726I in the ATP-binding pocket and correlates with a log decrease in sensitivity to gefitinib, increased heterodimerization with EGFR and HER3, and impaired down-regulation.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'snv', value:'L726I', gene_entrez:'2064', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1052', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17638894', evidence_text:\"IR-5 bears a novel point mutation in the HER2/neu that corresponds to a L726I in the ATP-binding pocket and correlates with a log decrease in sensitivity to gefitinib, increased heterodimerization with EGFR and HER3, and impaired down-regulation.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CTTN', entrez:'2017'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2017', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1053', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17909038', evidence_text:\"Finally, we have determined that cortactin overexpression, either alone or in combination with cyclin D1 up-regulation, promotes resistance to the EGFR kinase inhibitor gefitinib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CTTN', entrez:'2017'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2017', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1054', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17909038', evidence_text:\"Finally, we have determined that cortactin overexpression, either alone or in combination with cyclin D1 up-regulation, promotes resistance to the EGFR kinase inhibitor gefitinib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2064', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1055', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"HER2 is amplified in both murine and human tumors with acquired resistance to erlotinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4233', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1056', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18093943', evidence_text:\"These data suggest that MET amplification occurs independently of EGFR(T790M) mutations and that MET may be a clinically relevant therapeutic target for some patients with acquired resistance to gefitinib or erlotinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4233', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1057', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18093943', evidence_text:\"These data suggest that MET amplification occurs independently of EGFR(T790M) mutations and that MET may be a clinically relevant therapeutic target for some patients with acquired resistance to gefitinib or erlotinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB3', entrez:'2065'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2065', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1058', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18483355', evidence_text:\"There are now several studies reporting a correlation between gefitinib sensitivity and erbB3 expression in NSCLC cell lines (25 _ 27). In fact, erbB3 expression analysis identified patients that most benefited from EGFR inhibitors (28). (Annotator note: some evidence this is reversed in TN BrCa.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB3', entrez:'2065'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2065', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1059', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18483355', evidence_text:\"There are now several studies reporting a correlation between gefitinib sensitivity and erbB3 expression in NSCLC cell lines (25 _ 27). In fact, erbB3 expression analysis identified patients that most benefited from EGFR inhibitors (28). (Annotator note: some evidence this is reversed in TN BrCa.)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGF1R', entrez:'3480'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3480', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1060', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18568074', evidence_text:\"We investigated mechanisms of acquired resistance to the EGFR TKI gefitinib by generating gefitinib-resistant (GR) A431 squamous cancer cells. In GR cells, gefitinib reduced phosphorylation of EGFR, ErbB-3, and Erk but not Akt. These cells also showed hyperphosphorylation of the IGFI receptor (IGFIR) and constitutive association of IRS-1 with PI3K. Inhibition of IGFIR signaling disrupted the association of IRS-1 with PI3K and restored the ability of gefitinib to downregulate PI3K/Akt signaling and to inhibit GR cell growth.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGF1R', entrez:'3480'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3480', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1061', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18568074', evidence_text:\"We investigated mechanisms of acquired resistance to the EGFR TKI gefitinib by generating gefitinib-resistant (GR) A431 squamous cancer cells. In GR cells, gefitinib reduced phosphorylation of EGFR, ErbB-3, and Erk but not Akt. These cells also showed hyperphosphorylation of the IGFI receptor (IGFIR) and constitutive association of IRS-1 with PI3K. Inhibition of IGFIR signaling disrupted the association of IRS-1 with PI3K and restored the ability of gefitinib to downregulate PI3K/Akt signaling and to inhibit GR cell growth.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AREG', entrez:'374'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'374', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1062', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18980991 ', evidence_text:\"Cell lines which produced >or=20 pmol/L amphiregulin, as detected by an ELISA, were significantly more likely to be growth inhibited by both gefitinib and cetuximab than those that produced minimal or no amphiregulin. In these cell lines, both cetuximab and gefitinib led to cell cycle arrest at the G(1)-S boundary and was associated with preferential inhibition of extracellular signal-regulated kinase 1/2 but not Akt signaling. Amphiregulin expression was significantly higher in NSCLC patients that developed stable disease compared with those that developed disease progression following gefitinib or erlotinib treatment.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AREG', entrez:'374'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'374', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1063', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18980991 ', evidence_text:\"Cell lines which produced >or=20 pmol/L amphiregulin, as detected by an ELISA, were significantly more likely to be growth inhibited by both gefitinib and cetuximab than those that produced minimal or no amphiregulin. In these cell lines, both cetuximab and gefitinib led to cell cycle arrest at the G(1)-S boundary and was associated with preferential inhibition of extracellular signal-regulated kinase 1/2 but not Akt signaling. Amphiregulin expression was significantly higher in NSCLC patients that developed stable disease compared with those that developed disease progression following gefitinib or erlotinib treatment.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIK3CA', entrez:'5290'}), (a:Aberration { type:'snv', value:'H1047#', gene_entrez:'5290', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1064', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010894', evidence_text:\"Here, we have identified the tumor suppressor PTEN as a modulator of lapatinib sensitivity in vitro and in vivo. In addition, we show that two dominant activating mutations in PIK3CA (E545K and H1047R), which are prevalent in breast cancer, also confer resistance to lapatinib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PIK3CA', entrez:'5290'}), (a:Aberration { type:'snv', value:'H1047R', gene_entrez:'5290', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1065', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010894', evidence_text:\"Here, we have identified the tumor suppressor PTEN as a modulator of lapatinib sensitivity in vitro and in vivo. In addition, we show that two dominant activating mutations in PIK3CA (E545K and H1047R), which are prevalent in breast cancer, also confer resistance to lapatinib. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAML2_CRTC1 ', entrez:'EMPTY'}), (a:Aberration { type:'EMPTY', value:'EMPTY', gene_entrez:'EMPTY', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1066', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19185385', evidence_text:\"In vitro data has shown that MEC cell-lines with t(11;19) are sensitive to gefitinib and that this may be mediated by the action of CRTC1-MAML2 in up-regulating the EGFR ligand, amphiregulin. Data also shows that gefitinib demonstrates amphiregulin-dependant activity in NSCLC cell-lines. As such, it may be speculated that MEC from lung and salivary glands expressing CRTC1-MAML2 present a valid target for treatment with gefitinib, even in the absence of sensitizing EGFR mutations.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAML2_CRTC1 ', entrez:'EMPTY'}), (a:Aberration { type:'EMPTY', value:'EMPTY', gene_entrez:'EMPTY', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1067', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19185385', evidence_text:\"In vitro data has shown that MEC cell-lines with t(11;19) are sensitive to gefitinib and that this may be mediated by the action of CRTC1-MAML2 in up-regulating the EGFR ligand, amphiregulin. Data also shows that gefitinib demonstrates amphiregulin-dependant activity in NSCLC cell-lines. As such, it may be speculated that MEC from lung and salivary glands expressing CRTC1-MAML2 present a valid target for treatment with gefitinib, even in the absence of sensitizing EGFR mutations.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EML4_ALK', entrez:'EMPTY'}), (a:Aberration { type:'EMPTY', value:'EMPTY', gene_entrez:'EMPTY', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1068', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19667264', evidence_text:\"EML4-ALK defines a molecular subset of NSCLC with distinct clinical characteristics. Patients who harbor this mutation do not benefit from EGFR TKIs and should be directed to trials of ALK-targeted agents.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2263', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1069', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21152424', evidence_text:\"Rapidly acquired resistance to EGFR tyrosine kinase inhibitors in NSCLC cell lines through de-repression of FGFR2 and FGFR3 expression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR2', entrez:'2263'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2263', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1070', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21152424', evidence_text:\"Rapidly acquired resistance to EGFR tyrosine kinase inhibitors in NSCLC cell lines through de-repression of FGFR2 and FGFR3 expression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HRAS', entrez:'3265'}), (a:Aberration { type:'snv', value:'G12#', gene_entrez:'3265', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1071', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21411223', evidence_text:\"To determine whether Harvey Ras (H-Ras) mutations also can confer resistance to EGFR-targeted therapy, we expressed a constitutively active H-Ras (Ras G12V) in A431 human vulvar squamous carcinoma cells. Compared with corresponding control cells, A431-Ras cells exhibited marked resistance to the EGFR inhibitors cetuximab and gefitinib, reducing inhibition of Akt and Erk phosphorylation, inhibition of HIF-1_ expression and transcriptional activity, and antitumor effects in vitro and in vivo. Our data indicate that constitutively active H-Ras can also confer resistance to anti-EGFR therapy in cancer cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HRAS', entrez:'3265'}), (a:Aberration { type:'snv', value:'G12V', gene_entrez:'3265', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1072', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21411223', evidence_text:\"To determine whether Harvey Ras (H-Ras) mutations also can confer resistance to EGFR-targeted therapy, we expressed a constitutively active H-Ras (Ras G12V) in A431 human vulvar squamous carcinoma cells. Compared with corresponding control cells, A431-Ras cells exhibited marked resistance to the EGFR inhibitors cetuximab and gefitinib, reducing inhibition of Akt and Erk phosphorylation, inhibition of HIF-1_ expression and transcriptional activity, and antitumor effects in vitro and in vivo. Our data indicate that constitutively active H-Ras can also confer resistance to anti-EGFR therapy in cancer cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NFKBIA', entrez:'4792'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4792', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1073', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21430781', evidence_text:\"Because reduced I_B levels were associated with erlotinib resistance in cell lines and tumour models, we asked if I_B expression correlated with EGFR TKI response in patients with EGFR-mutant lung cancers. Low I_B expression (high-NF-_B' activation state) was predictive of worse progression-free survival (Fig. 4c and Supplementary Table 4a) and decreased overall survival (Supplementary Fig. 16a and Supplementary Table 4b). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NFKBIA', entrez:'4792'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4792', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1074', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21430781', evidence_text:\"Because reduced I_B levels were associated with erlotinib resistance in cell lines and tumour models, we asked if I_B expression correlated with EGFR TKI response in patients with EGFR-mutant lung cancers. Low I_B expression (high-NF-_B' activation state) was predictive of worse progression-free survival (Fig. 4c and Supplementary Table 4a) and decreased overall survival (Supplementary Fig. 16a and Supplementary Table 4b). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PPP1R1B', entrez:'84152'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'84152', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1075', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21741919', evidence_text:\"DARPP-32 promotes resistance of gastric cancer cells to gefitinib by promoting interaction between EGFR and ERBB3 and activating phosphatidylinositol-3-kinase-AKT signaling\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PPP1R1B', entrez:'84152'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'84152', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1076', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21741919', evidence_text:\"DARPP-32 promotes resistance of gastric cancer cells to gefitinib by promoting interaction between EGFR and ERBB3 and activating phosphatidylinositol-3-kinase-AKT signaling\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PTPRS', entrez:'5802'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'5802', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1077', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22065749', evidence_text:\"PTPRS deletion in the absence of EGFR gene alterations can promote EGFR/PI3K pathway activation in HNSCC. We found that PTPRS knockdown increased resistance of the EGFR-dependent cell lines HCC827 (EGFR exon 19 deletion) and H3255 (EGFR L858R) to erlotinib treatment (Fig. 4A and Fig. S4). Similarly, knockdown of PTPRS in HNSCC cell lines sensitive to EGFR inhibition significantly modulated response to erlotinib. We found that PTPRS expression predicted response to cetuximab in HNSCC cell lines\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PTPRS', entrez:'5802'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'5802', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1078', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22065749', evidence_text:\"PTPRS deletion in the absence of EGFR gene alterations can promote EGFR/PI3K pathway activation in HNSCC. We found that PTPRS knockdown increased resistance of the EGFR-dependent cell lines HCC827 (EGFR exon 19 deletion) and H3255 (EGFR L858R) to erlotinib treatment (Fig. 4A and Fig. S4). Similarly, knockdown of PTPRS in HNSCC cell lines sensitive to EGFR inhibition significantly modulated response to erlotinib. We found that PTPRS expression predicted response to cetuximab in HNSCC cell lines\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BCL2L11', entrez:'10018'}), (a:Aberration { type:'small_deletion', value:'2.9 kb intronic deletion', gene_entrez:'10018', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1079', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22426421', evidence_text:\"Using paired-end DNA sequencing, we discovered a common intronic deletion polymorphism in the gene encoding BCL2-like 11 (BIM). This structural variation comprised an identical 2,903-bp genomic deletion that was common to all three resistant samples. The 2.9-kb deleted region contains cis elements that suppress the splicing of BIM exon 3, which, in cells harboring the deletion, results in preferential splicing of exon 3 over exon 4. This polymorphism has a profound effect on the TKI sensitivity of CML and EGFR NSCLC cells, such that one copy of the deleted allele is sufficient to render cells intrinsically TKI resistant. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DAPK', entrez:'1612'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'1612', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1080', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22487682', evidence_text:\"We found that death-associated protein kinase (DAPK) was hypermethylated in drug-resistant derivatives generated from both parental cell lines. Restoration of DAPK into the resistant NSCLC cells by stable transfection re-sensitized the cells to both erlotinib and cetuximab. Conversely, siRNA-mediated knockdown of DAPK induced resistance in the parental sensitive cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DAPK', entrez:'1612'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'1612', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1081', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22487682', evidence_text:\"We found that death-associated protein kinase (DAPK) was hypermethylated in drug-resistant derivatives generated from both parental cell lines. Restoration of DAPK into the resistant NSCLC cells by stable transfection re-sensitized the cells to both erlotinib and cetuximab. Conversely, siRNA-mediated knockdown of DAPK induced resistance in the parental sensitive cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CRKL', entrez:'1399'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1399', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1082', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22586683', evidence_text:\"Overexpression of CRKL in epidermal growth factor receptor (EGFR)-mutant cells induces resistance to gefitinib by activating extracellular signal-regulated kinase and AKT signaling. We identified CRKL amplification in an EGFR inhibitor-treated lung adenocarcinoma that was not present before treatment. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'CRKL', entrez:'1399'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1399', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1083', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22586683', evidence_text:\"Overexpression of CRKL in epidermal growth factor receptor (EGFR)-mutant cells induces resistance to gefitinib by activating extracellular signal-regulated kinase and AKT signaling. We identified CRKL amplification in an EGFR inhibitor-treated lung adenocarcinoma that was not present before treatment. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DIAPH3', entrez:'81624'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'81624', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1084', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22593025', evidence_text:\"DIAPH3 silencing in human carcinoma cells destabilized microtubules and induced defective endocytic trafficking, endosomal accumulation of EGFR, and hyperactivation of EGFR/MEK/ERK signalling. DIAPH3-silenced cells were sensitive to MEK inhibition, but showed reduced sensitize to EGFR inhibition. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'DIAPH3', entrez:'81624'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'81624', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1085', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22593025', evidence_text:\"DIAPH3 silencing in human carcinoma cells destabilized microtubules and induced defective endocytic trafficking, endosomal accumulation of EGFR, and hyperactivation of EGFR/MEK/ERK signalling. DIAPH3-silenced cells were sensitive to MEK inhibition, but showed reduced sensitize to EGFR inhibition. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'JAK2', entrez:'3717'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3717', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1086', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22712764', evidence_text:\"We established an erlotinib-resistant lung cancer cell line named PC-9/ER3 that harbors an EGFR mutation after continuously exposing PC-9 cells to erlotinib. PC-9/ER3 cells were 136-fold more resistant to erlotinib than the parental cells. Although the PC-9/ER3 cells did not carry the T790M mutation or MET amplification and had similar levels of phosphorylated (p) STAT3, pJAK2 increased in the resistant cells. It was found in the present study that 3-12 h of exposure to erlotinib in both cell lines did not affect pJAK2 expression, but did result in increased pSTAT3 expression. pAkt in PC-9/ER3 cells was less suppressed than in PC-9 cells, although pEGFR and pMAPK were markedly suppressed in both cell lines. The combined treatment of erlotinib plus a JAK2 inhibitor (JSI-124) suppressed pAkt in PC-9/ER3 cells. Similarly, the combination of erlotinib plus JSI-124 or siRNA against JAK2 restored sensitivity to erlotinib in PC-9/ER3 cells. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'JAK2', entrez:'3717'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3717', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1087', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22712764', evidence_text:\"We established an erlotinib-resistant lung cancer cell line named PC-9/ER3 that harbors an EGFR mutation after continuously exposing PC-9 cells to erlotinib. PC-9/ER3 cells were 136-fold more resistant to erlotinib than the parental cells. Although the PC-9/ER3 cells did not carry the T790M mutation or MET amplification and had similar levels of phosphorylated (p) STAT3, pJAK2 increased in the resistant cells. It was found in the present study that 3-12 h of exposure to erlotinib in both cell lines did not affect pJAK2 expression, but did result in increased pSTAT3 expression. pAkt in PC-9/ER3 cells was less suppressed than in PC-9 cells, although pEGFR and pMAPK were markedly suppressed in both cell lines. The combined treatment of erlotinib plus a JAK2 inhibitor (JSI-124) suppressed pAkt in PC-9/ER3 cells. Similarly, the combination of erlotinib plus JSI-124 or siRNA against JAK2 restored sensitivity to erlotinib in PC-9/ER3 cells. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TNKS', entrez:'8658'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'8658', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1088', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22738915', evidence_text:\"Using a loss-of-function, whole genome short hairpin RNA (shRNA) screen, we identified that the canonical Wnt pathway contributes to the maintenance of NSCLC cells during EGFR inhibition, particularly the poly-ADP-ribosylating enzymes tankyrase 1 and 2 that positively regulate canonical Wnt signaling. Inhibition of tankyrase and various other components of the Wnt pathway with shRNAs or small molecules significantly increased the efficacy of EGFR inhibitors both in vitro and in vivo. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TNKS', entrez:'8658'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'8658', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1089', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22738915', evidence_text:\"Using a loss-of-function, whole genome short hairpin RNA (shRNA) screen, we identified that the canonical Wnt pathway contributes to the maintenance of NSCLC cells during EGFR inhibition, particularly the poly-ADP-ribosylating enzymes tankyrase 1 and 2 that positively regulate canonical Wnt signaling. Inhibition of tankyrase and various other components of the Wnt pathway with shRNAs or small molecules significantly increased the efficacy of EGFR inhibitors both in vitro and in vivo. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GAS6', entrez:'2621'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2621', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1090', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22751098', evidence_text:\"Here, we report increased activation of AXL and evidence for epithelial-to-mesenchymal transition (EMT) in multiple in vitro and in vivo EGFR-mutant lung cancer models with acquired resistance to erlotinib in the absence of the EGFR p.Thr790Met alteration or MET activation. Genetic or pharmacological inhibition of AXL restored sensitivity to erlotinib in these tumor models. Increased expression of AXL and, in some cases, of its ligand GAS6 was found in EGFR-mutant lung cancers obtained from individuals with acquired resistance to TKIs.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'GAS6', entrez:'2621'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2621', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1091', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22751098', evidence_text:\"Here, we report increased activation of AXL and evidence for epithelial-to-mesenchymal transition (EMT) in multiple in vitro and in vivo EGFR-mutant lung cancer models with acquired resistance to erlotinib in the absence of the EGFR p.Thr790Met alteration or MET activation. Genetic or pharmacological inhibition of AXL restored sensitivity to erlotinib in these tumor models. Increased expression of AXL and, in some cases, of its ligand GAS6 was found in EGFR-mutant lung cancers obtained from individuals with acquired resistance to TKIs.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'673'}), (a:Aberration { type:'snv', value:'G469#', gene_entrez:'673', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1092', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22773810', evidence_text:\"A patient was found at diagnosis to have a tumor with an EGFR exon19 deletion that initially responded radiographically to erlotinib monotherapy.The patient demonstrated acquired resistance to erlotinib. The tumor harbored EGFR exon 19 deletion and BRAF G469A (Table 2 and Fig. S5). The BRAF G469A mutation was confirmed to be absent before treatment. 2 of 195 (1%) were found to have mutations in BRAF (G469A and V600E). Ectopic expression of mutant NRAS or BRAF in drug-sensitive EGFR-mutant cells conferred resistance to EGFR TKIs that was overcome by addition of a MEK inhibitor.  Transfectants with mutant BRAF were sensitive to the combination of EGFR and BRAF inhibition as well. |Evidence Level:  Case Study. Preclinical. Cell Line. |Disease:  NSCLC. |Annotator Note:  SNVs at this position are associated with activation and reduced drug sensitivity. However, not all changes at this position may result in activation and reduced drug sensitivity. This mutation was acquired during treatment.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'673'}), (a:Aberration { type:'snv', value:'G469A', gene_entrez:'673', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1093', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22773810', evidence_text:\"A patient was found at diagnosis to have a tumor with an EGFR exon19 deletion that initially responded radiographically to erlotinib monotherapy.The patient demonstrated acquired resistance to erlotinib. The tumor harbored EGFR exon 19 deletion and BRAF G469A (Table 2 and Fig. S5). The BRAF G469A mutation was confirmed to be absent before treatment. 2 of 195 (1%) were found to have mutations in BRAF (G469A and V600E). Ectopic expression of mutant NRAS or BRAF in drug-sensitive EGFR-mutant cells conferred resistance to EGFR TKIs that was overcome by addition of a MEK inhibitor.  Transfectants with mutant BRAF were sensitive to the combination of EGFR and BRAF inhibition as well. |Evidence Level:  Case Study. Preclinical. Cell Line. |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'673'}), (a:Aberration { type:'snv', value:'V600#', gene_entrez:'673', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1094', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22773810', evidence_text:\"A patient was found at diagnosis to have a tumor with an EGFR exon19 deletion that initially responded radiographically to erlotinib monotherapy.The patient demonstrated acquired resistance to erlotinib. The tumor harbored EGFR exon 19 deletion and BRAF G469A (Table 2 and Fig. S5). The BRAF G469A mutation was confirmed to be absent before treatment. 2 of 195 (1%) were found to have mutations in BRAF (G469A and V600E). Ectopic expression of mutant NRAS or BRAF in drug-sensitive EGFR-mutant cells conferred resistance to EGFR TKIs that was overcome by addition of a MEK inhibitor.  Transfectants with mutant BRAF were sensitive to the combination of EGFR and BRAF inhibition as well. |Evidence Level:  Case Study. Preclinical. Cell Line. |Disease:  NSCLC. |Annotator Note:  SNVs at this position are associated with activation and reduced drug sensitivity. However, not all changes at this position may result in activation and reduced drug sensitivity. This mutation was acquired during treatment.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'BRAF', entrez:'673'}), (a:Aberration { type:'snv', value:'V600E', gene_entrez:'673', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1095', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22773810', evidence_text:\"A patient was found at diagnosis to have a tumor with an EGFR exon19 deletion that initially responded radiographically to erlotinib monotherapy.The patient demonstrated acquired resistance to erlotinib. The tumor harbored EGFR exon 19 deletion and BRAF G469A (Table 2 and Fig. S5). The BRAF G469A mutation was confirmed to be absent before treatment. 2 of 195 (1%) were found to have mutations in BRAF (G469A and V600E). Ectopic expression of mutant NRAS or BRAF in drug-sensitive EGFR-mutant cells conferred resistance to EGFR TKIs that was overcome by addition of a MEK inhibitor.  Transfectants with mutant BRAF were sensitive to the combination of EGFR and BRAF inhibition as well. |Evidence Level:  Case Study. Preclinical. Cell Line. |Disease:  NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FAM83A', entrez:'84985'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'84985', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1096', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22886303', evidence_text:\"FAM83A overexpression in cancer cells increased proliferation and invasion and imparted EGFR-TKI resistance both in cultured cells and in animals. Conversely, FAM83A depletion in cancer cells caused reversion of the malignant phenotype, delayed tumor growth in mice, and rendered cells more sensitive to EGFR-TKI. Analyses of published clinical data revealed a correlation between high FAM83A expression and breast cancer patients' poor prognosis. We found that FAM83A interacted with and caused phosphorylation of c-RAF and PI3K p85, upstream of MAPK and downstream of EGFR. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FAM83A', entrez:'84985'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'84985', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1097', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22886303', evidence_text:\"FAM83A overexpression in cancer cells increased proliferation and invasion and imparted EGFR-TKI resistance both in cultured cells and in animals. Conversely, FAM83A depletion in cancer cells caused reversion of the malignant phenotype, delayed tumor growth in mice, and rendered cells more sensitive to EGFR-TKI. Analyses of published clinical data revealed a correlation between high FAM83A expression and breast cancer patients' poor prognosis. We found that FAM83A interacted with and caused phosphorylation of c-RAF and PI3K p85, upstream of MAPK and downstream of EGFR. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB2', entrez:'2064'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2064', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1098', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22956644', evidence_text:\"HER2 is amplified in both murine and human tumors with acquired resistance to erlotinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP2K1', entrez:'5604'}), (a:Aberration { type:'snv', value:'K57#', gene_entrez:'5604', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1099', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22961667', evidence_text:\"In addition, MEK1 K57N was sufficient to cause resistance to both WZ4002 (data not shown) and to gefitinib \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MAP2K1', entrez:'5604'}), (a:Aberration { type:'snv', value:'K57N', gene_entrez:'5604', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1100', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22961667', evidence_text:\"In addition, MEK1 K57N was sufficient to cause resistance to both WZ4002 (data not shown) and to gefitinib \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGFBP3', entrez:'3486'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'3486', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1101', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23172312', evidence_text:\"Neither PF299804-resistant nor WZ4002-resistant clones of PC9 harbor EGFR T790M. Instead, they have shown activated insulin-like growth factor receptor (IGF1R) signaling as a result of loss of expression of IGFBP3 with the IGF1R inhibitor, BMS 536924, restoring EGFR inhibitor sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'IGFBP3', entrez:'3486'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'3486', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1102', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23172312', evidence_text:\"Neither PF299804-resistant nor WZ4002-resistant clones of PC9 harbor EGFR T790M. Instead, they have shown activated insulin-like growth factor receptor (IGF1R) signaling as a result of loss of expression of IGFBP3 with the IGF1R inhibitor, BMS 536924, restoring EGFR inhibitor sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MED12', entrez:'9968'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'9968', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1103', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23178117', evidence_text:\"We used a large-scale RNAi screen to identify MED12, a component of the transcriptional MEDIATOR complex that is mutated in cancers, as a determinant of response to ALK and EGFR inhibitors. MED12 is in part cytoplasmic where it negatively regulates TGF-_R2 through physical interaction. MED12 suppression therefore results in activation of TGF-_R signaling, which is both necessary and sufficient for drug resistance. TGF-_ signaling causes MEK/ERK activation, and consequently MED12 suppression also confers resistance to MEK and BRAF inhibitors in other cancers.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MED12', entrez:'9968'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'9968', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1104', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23178117', evidence_text:\"We used a large-scale RNAi screen to identify MED12, a component of the transcriptional MEDIATOR complex that is mutated in cancers, as a determinant of response to ALK and EGFR inhibitors. MED12 is in part cytoplasmic where it negatively regulates TGF-_R2 through physical interaction. MED12 suppression therefore results in activation of TGF-_R signaling, which is both necessary and sufficient for drug resistance. TGF-_ signaling causes MEK/ERK activation, and consequently MED12 suppression also confers resistance to MEK and BRAF inhibitors in other cancers.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIAA1524', entrez:'57650'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'57650', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1105', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23178652', evidence_text:\"CIP2A has emerged as a novel oncoprotein and a growing number of reports have shown its overexpression in many human malignancies. Down-regulation of CIP2A, a cellular inhibitor of protein phosphatase 2A (PP2A), mediated the apoptotic effect of erlotinib in HCC. Overexpression of CIP2A upregulated phospho-Akt and protected Hep3B cells from erlotinib-induced apoptosis. In addition, silencing CIP2A by siRNA restored the effects of erlotinib in Huh-7 cells.  In conclusion, inhibition of CIP2A determines the effects of erlotinib on apoptosis in HCC. CIP2A may be useful as a therapeutic biomarker for predicting clinical response to erlotinib in HCC treatment. Moreover, a combination of the Akt inhibitor MK-2206 and erlotinib restored the sensitivity of resistant HCC cells to erlotinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KIAA1524', entrez:'57650'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'57650', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1106', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23178652', evidence_text:\"CIP2A has emerged as a novel oncoprotein and a growing number of reports have shown its overexpression in many human malignancies. Down-regulation of CIP2A, a cellular inhibitor of protein phosphatase 2A (PP2A), mediated the apoptotic effect of erlotinib in HCC. Overexpression of CIP2A upregulated phospho-Akt and protected Hep3B cells from erlotinib-induced apoptosis. In addition, silencing CIP2A by siRNA restored the effects of erlotinib in Huh-7 cells.  In conclusion, inhibition of CIP2A determines the effects of erlotinib on apoptosis in HCC. CIP2A may be useful as a therapeutic biomarker for predicting clinical response to erlotinib in HCC treatment. Moreover, a combination of the Akt inhibitor MK-2206 and erlotinib restored the sensitivity of resistant HCC cells to erlotinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATG12', entrez:'9140'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9140', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1107', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23307622', evidence_text:\"An analysis of the transcriptional status of ATG12 in > 50 breast cancer cell lines suggested that the ATG12 transcript is commonly upregulated in trastuzumab-unresponsive HER2-overexpressing breast cancer cells. A lentiviral-delivered small hairpin RNA stable knockdown of the ATG12 gene fully suppressed the refractoriness of JIMT1 cells to trastuzumab, erlotinib, gefitinib, and lapatinib in vitro. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ATG12', entrez:'9140'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9140', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1108', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23307622', evidence_text:\"An analysis of the transcriptional status of ATG12 in > 50 breast cancer cell lines suggested that the ATG12 transcript is commonly upregulated in trastuzumab-unresponsive HER2-overexpressing breast cancer cells. A lentiviral-delivered small hairpin RNA stable knockdown of the ATG12 gene fully suppressed the refractoriness of JIMT1 cells to trastuzumab, erlotinib, gefitinib, and lapatinib in vitro. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61K', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1109', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23515407', evidence_text:\"To identify potential therapies for patients with NRAS mutant tumors, we tested the sensitivity of 6 NRAS mutant NSCLC cell lines (Table S3) against a variety of kinase inhibitors in in vitro cell growth inhibition assays (Fig. 2A). None of the lines were sensitive (with lower than 1 micromolar IC50s) to the EGFR TKI, erlotinib, the ALK/MET/RON/ROS1 inhibitor, crizotinib, or the IGF-1R inhibitor, linsitinib. By contrast, 5 of 6 lines were sensitive to two different MEK inhibitors, selumetinib and trametinib. |Evidence Level:  Pre-clinical. Cell line. |Disease: NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61L', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1110', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23515407', evidence_text:\"To identify potential therapies for patients with NRAS mutant tumors, we tested the sensitivity of 6 NRAS mutant NSCLC cell lines (Table S3) against a variety of kinase inhibitors in in vitro cell growth inhibition assays (Fig. 2A). None of the lines were sensitive (with lower than 1 micromolar IC50s) to the EGFR TKI, erlotinib, the ALK/MET/RON/ROS1 inhibitor, crizotinib, or the IGF-1R inhibitor, linsitinib. By contrast, 5 of 6 lines were sensitive to two different MEK inhibitors, selumetinib and trametinib. |Evidence Level:  Pre-clinical. Cell line. |Disease: NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61R', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1111', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23515407', evidence_text:\"To identify potential therapies for patients with NRAS mutant tumors, we tested the sensitivity of 6 NRAS mutant NSCLC cell lines (Table S3) against a variety of kinase inhibitors in in vitro cell growth inhibition assays (Fig. 2A). None of the lines were sensitive (with lower than 1 micromolar IC50s) to the EGFR TKI, erlotinib, the ALK/MET/RON/ROS1 inhibitor, crizotinib, or the IGF-1R inhibitor, linsitinib. By contrast, 5 of 6 lines were sensitive to two different MEK inhibitors, selumetinib and trametinib. |Evidence Level:  Pre-clinical. Cell line. |Disease: NSCLC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NRAS', entrez:'4893'}), (a:Aberration { type:'snv', value:'Q61#', gene_entrez:'4893', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1112', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23515407', evidence_text:\"To identify potential therapies for patients with NRAS mutant tumors, we tested the sensitivity of 6 NRAS mutant NSCLC cell lines (Table S3) against a variety of kinase inhibitors in in vitro cell growth inhibition assays (Fig. 2A). None of the lines were sensitive (with lower than 1 micromolar IC50s) to the EGFR TKI, erlotinib, the ALK/MET/RON/ROS1 inhibitor, crizotinib, or the IGF-1R inhibitor, linsitinib. By contrast, 5 of 6 lines were sensitive to two different MEK inhibitors, selumetinib and trametinib. |Evidence Level:  Pre-clinical. Cell line. |Disease: NSCLC. |Annotator Note:  SNVs at this position are associated with activation and reduced drug sensitivity. However, not all changes at this position may result in activation and reduced drug sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGF2', entrez:'2247'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2247', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1113', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23536707', evidence_text:\"Expression analyses indicated that both FGFR1 and FGF2 were increased in PC9 gefitinib-resistant (PC9 GR) cells as compared with PC9 nave (PC9 na) cells. Importantly, proliferation of gefitinib-resistant cells was dependent on the FGF2 -FGFR1 pathway. Mechanistically, inhibition of either FGF2 or FGFR1 by siRNA or FGFR inhibitor (PD173074) restored gefitinib sensitivity in PC9 GR cells. These data suggest that FGF2 -FGFR1 activation through an autocrine loop is a novel mechanism of acquired resistance to EGFR-TKIs.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGF2', entrez:'2247'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2247', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1114', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23536707', evidence_text:\"Expression analyses indicated that both FGFR1 and FGF2 were increased in PC9 gefitinib-resistant (PC9 GR) cells as compared with PC9 nave (PC9 na) cells. Importantly, proliferation of gefitinib-resistant cells was dependent on the FGF2 -FGFR1 pathway. Mechanistically, inhibition of either FGF2 or FGFR1 by siRNA or FGFR inhibitor (PD173074) restored gefitinib sensitivity in PC9 GR cells. These data suggest that FGF2 -FGFR1 activation through an autocrine loop is a novel mechanism of acquired resistance to EGFR-TKIs.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR1', entrez:'2260'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2260', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1115', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23536707', evidence_text:\"Expression analyses indicated that both FGFR1 and FGF2 were increased in PC9 gefitinib-resistant (PC9 GR) cells as compared with PC9 nave (PC9 na) cells. Importantly, proliferation of gefitinib-resistant cells was dependent on the FGF2 -FGFR1 pathway. Mechanistically, inhibition of either FGF2 or FGFR1 by siRNA or FGFR inhibitor (PD173074) restored gefitinib sensitivity in PC9 GR cells. These data suggest that FGF2 -FGFR1 activation through an autocrine loop is a novel mechanism of acquired resistance to EGFR-TKIs.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'FGFR1', entrez:'2260'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2260', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1116', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23536707', evidence_text:\"Expression analyses indicated that both FGFR1 and FGF2 were increased in PC9 gefitinib-resistant (PC9 GR) cells as compared with PC9 nave (PC9 na) cells. Importantly, proliferation of gefitinib-resistant cells was dependent on the FGF2 -FGFR1 pathway. Mechanistically, inhibition of either FGF2 or FGFR1 by siRNA or FGFR inhibitor (PD173074) restored gefitinib sensitivity in PC9 GR cells. These data suggest that FGF2 -FGFR1 activation through an autocrine loop is a novel mechanism of acquired resistance to EGFR-TKIs.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HRAS', entrez:'3265'}), (a:Aberration { type:'snv', value:'Q61#', gene_entrez:'3265', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1117', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23559083', evidence_text:\"The poor cell response to gefitinib was associated with genetic alterations, leading to constitutive activation of the ERK pathway, including BRAF(V600E) and HRAS(G12A/Q61R) mutations and RET/PTC1 rearrangement. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HRAS', entrez:'3265'}), (a:Aberration { type:'snv', value:'G12A', gene_entrez:'3265', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1118', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23559083', evidence_text:\"The poor cell response to gefitinib was associated with genetic alterations, leading to constitutive activation of the ERK pathway, including BRAF(V600E) and HRAS(G12A/Q61R) mutations and RET/PTC1 rearrangement. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HRAS', entrez:'3265'}), (a:Aberration { type:'snv', value:'Q61R', gene_entrez:'3265', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1119', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23559083', evidence_text:\"The poor cell response to gefitinib was associated with genetic alterations, leading to constitutive activation of the ERK pathway, including BRAF(V600E) and HRAS(G12A/Q61R) mutations and RET/PTC1 rearrangement. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AXL', entrez:'558'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'558', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1120', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24026012', evidence_text:\"Our findings emphasize the role of Axl in promoting EMT, cell migration, and erlotinib resistance in HNC, and support the rationale for therapeutic blockade of Axl in EGFR inhibitor-resistant tumors, including HNC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AXL', entrez:'558'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'558', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1121', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24026012', evidence_text:\"Our findings emphasize the role of Axl in promoting EMT, cell migration, and erlotinib resistance in HNC, and support the rationale for therapeutic blockade of Axl in EGFR inhibitor-resistant tumors, including HNC.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1122', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24535670', evidence_text:\"We have used a functional genomic in vitro screening approach, together with a genetically modified mouse lung cancer model system, to investigate mechanisms of acquired resistance to erlotinib in EGFR-mutant lung adenocarcinoma, identifying NF1 as a gene whose loss of function is capable of causing EGFR TKI drug resistance in both settings. Our analyses of two independent sets of paired EGFR-mutant lung adenocarcinoma samples from patients treated with EGFR inhibitors confirm that downregulation of NF1 expres- sion at the time of TKI resistance is a common occurrence in the clinic. Furthermore, NF1 levels may influence the initial response to TKIs, as low neurofibromin in pretreatment specimens is strongly associated with reduced overall sur- vival for patients with EGFR-mutant lung adenocarcinomas treated with EGFR TKI. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1123', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24535670', evidence_text:\"We have used a functional genomic in vitro screening approach, together with a genetically modified mouse lung cancer model system, to investigate mechanisms of acquired resistance to erlotinib in EGFR-mutant lung adenocarcinoma, identifying NF1 as a gene whose loss of function is capable of causing EGFR TKI drug resistance in both settings. Our analyses of two independent sets of paired EGFR-mutant lung adenocarcinoma samples from patients treated with EGFR inhibitors confirm that downregulation of NF1 expres- sion at the time of TKI resistance is a common occurrence in the clinic. Furthermore, NF1 levels may influence the initial response to TKIs, as low neurofibromin in pretreatment specimens is strongly associated with reduced overall sur- vival for patients with EGFR-mutant lung adenocarcinomas treated with EGFR TKI. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'small_deletion', value:'change', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1124', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24535670', evidence_text:\"We have used a functional genomic in vitro screening approach, together with a genetically modified mouse lung cancer model system, to investigate mechanisms of acquired resistance to erlotinib in EGFR-mutant lung adenocarcinoma, identifying NF1 as a gene whose loss of function is capable of causing EGFR TKI drug resistance in both settings. Our analyses of two independent sets of paired EGFR-mutant lung adenocarcinoma samples from patients treated with EGFR inhibitors confirm that downregulation of NF1 expres- sion at the time of TKI resistance is a common occurrence in the clinic. Furthermore, NF1 levels may influence the initial response to TKIs, as low neurofibromin in pretreatment specimens is strongly associated with reduced overall sur- vival for patients with EGFR-mutant lung adenocarcinomas treated with EGFR TKI. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'NF1', entrez:'4763'}), (a:Aberration { type:'small_insertion', value:'change', gene_entrez:'4763', entity_class:'modifier' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1125', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24535670', evidence_text:\"We have used a functional genomic in vitro screening approach, together with a genetically modified mouse lung cancer model system, to investigate mechanisms of acquired resistance to erlotinib in EGFR-mutant lung adenocarcinoma, identifying NF1 as a gene whose loss of function is capable of causing EGFR TKI drug resistance in both settings. Our analyses of two independent sets of paired EGFR-mutant lung adenocarcinoma samples from patients treated with EGFR inhibitors confirm that downregulation of NF1 expres- sion at the time of TKI resistance is a common occurrence in the clinic. Furthermore, NF1 levels may influence the initial response to TKIs, as low neurofibromin in pretreatment specimens is strongly associated with reduced overall sur- vival for patients with EGFR-mutant lung adenocarcinomas treated with EGFR TKI. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747_P753>S', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1126', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15118073', evidence_text:\"Panel D also shows the mean (SD) inhibition of EGFR by gefitinib. Concentrations of phosphorylated EGFR were adjusted for total protein expression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'R776#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1127', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15329413', evidence_text:\"EGF receptor gene mutations are common in lung cancers from never smokers and are associated with sensitivity of tumors to gefitinib and erlotinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746_A750delELREA', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1128', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15329413', evidence_text:\"EGF receptor gene mutations are common in lung cancers from \\\"never smokers\\\" and are associated with sensitivity of tumors to gefitinib and erlotinib. From Table 1. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-T751>l', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1129', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15329413', evidence_text:\"EGF receptor gene mutations are common in lung cancers from \\\"never smokers\\\" and are associated with sensitivity of tumors to gefitinib and erlotinib. From Table 1. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747_S752delLREATS', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1130', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15329413', evidence_text:\"EGF receptor gene mutations are common in lung cancers from \\\"never smokers\\\" and are associated with sensitivity of tumors to gefitinib and erlotinib. From Table 1. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747-S752', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1131', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15329413', evidence_text:\"EGF receptor gene mutations are common in lung cancers from \\\"never smokers\\\" and are associated with sensitivity of tumors to gefitinib and erlotinib. From Table 1. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747-S752>Q', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1132', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15329413', evidence_text:\"EGF receptor gene mutations are common in lung cancers from \\\"never smokers\\\" and are associated with sensitivity of tumors to gefitinib and erlotinib. From Table 1. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747-S752insQ', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1133', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15329413', evidence_text:\"EGF receptor gene mutations are common in lung cancers from \\\"never smokers\\\" and are associated with sensitivity of tumors to gefitinib and erlotinib. From Table 1. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'R776C', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1134', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15329413', evidence_text:\"EGF receptor gene mutations are common in lung cancers from never smokers and are associated with sensitivity of tumors to gefitinib and erlotinib\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-A750del', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1135', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/15604279', evidence_text:\"The NSCLC cell line PC-9 was seen to be hypersensitive to gefitinib and ZD6474, and a small (15-bp) in-frame deletion of an ATP-binding site (exon 19) in the EGFR was detected (delE746-A750-type deletion)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G724#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1136', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16166444', evidence_text:\"in the current study, the patient whose colorectal cancer possessed the c.2170G>A (p.Gly724Ser) mutation did not experience any disease regression following treatment with gefitinib and chemotherapy; the patient achieved stable disease for only 6 months before developing disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G724S', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1137', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16166444', evidence_text:\"in the current study, the patient whose colorectal cancer possessed the c.2170G>A (p.Gly724Ser) mutation did not experience any disease regression following treatment with gefitinib and chemotherapy; the patient achieved stable disease for only 6 months before developing disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E709#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1138', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16205628', evidence_text:\"S768I and E709G mutant-expressing cells were slightly more resistant to gefitinib than the wild-type EGFR-expressing cells\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E709G', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1139', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/16205628', evidence_text:\"S768I and E709G mutant-expressing cells were slightly more resistant to gefitinib than the wild-type EGFR-expressing cells\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E884#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1140', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17052295', evidence_text:\"In vitro transfection and biochemical studies demonstrated that the novel E884K mutation confers opposite effects in sensitivity to the two EGFR inhibitors. EGFRE884K and EGFRL858R+E884K enhanced the sensitivity of the mutated receptor to gefitinib inhibition. Conversely, the E884K mutation resulted in decreased responsiveness of the receptor to erlotinib, and it significantly abrogated the drug sensitivity conferred by L858R (EGFRL858R+E884K). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E884K', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1141', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17052295', evidence_text:\"In vitro transfection and biochemical studies demonstrated that the novel E884K mutation confers opposite effects in sensitivity to the two EGFR inhibitors. EGFRE884K and EGFRL858R+E884K enhanced the sensitivity of the mutated receptor to gefitinib inhibition. Conversely, the E884K mutation resulted in decreased responsiveness of the receptor to erlotinib, and it significantly abrogated the drug sensitivity conferred by L858R (EGFRL858R+E884K). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G796#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1142', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17695517', evidence_text:\"In vitro, a stable clone of cells bearing the G796A mutation was approximately 50,000-fold less sensitive to gefitinib in comparison to cells carrying the delE746-A750 mutant EGFR.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G796A', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1143', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/17695517', evidence_text:\"In vitro, a stable clone of cells bearing the G796A mutation was approximately 50,000-fold less sensitive to gefitinib in comparison to cells carrying the delE746-A750 mutant EGFR.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747_A750', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1144', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"Table 2 from publication showing sensitivity.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747_A750InsS', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1145', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"Similarly, in the Ba/F3 cell models, PF00299804 was more effective than gefitinib at inhibiting the growth of cells containing the common EGFR-acti- vating mutations (Table 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747_A750InsV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1146', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"Similarly, in the Ba/F3 cell models, PF00299804 was more effective than gefitinib at inhibiting the growth of cells containing the common EGFR-acti- vating mutations (Table 3).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'L747_E749', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1147', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"The NSCLC cell lines (H3255, HCC4006, HCC827, and PC-9) containing the two common EGFR-activating mutations (L858R and exon 19 deletions), although all sensitive to gefitinib, also had lower IC50 values with PF00299804.Table 2\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'Ins S/T790M', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1148', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18089823', evidence_text:\"H1975 and H3255 GR are NSCLC cell lines that contain both EGFR L858R and T790M mutations, whereas H820 contains an EGFR exon 19 deletion (Del 747_L751, Ins S) and a T790M mutation (13, 23, 45). All three are resistant to gefitinib in vitro, but PF00299804 is able to effectively inhibit the growth of these cell lines (IC50 <1 Amol/L for all three cell lines; Table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A767#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1149', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18497962', evidence_text:\"LS174T cells with E762G or A767T were significantly (p<0.05) more resistant than the parental cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E749#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1150', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18497962', evidence_text:\"The LS174T cells with E749K were significantly (p<0.05) more responsive to gefitinib than the parental cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E762#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1151', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18497962', evidence_text:\"LS174T cells with E762G or A767T were significantly (p<0.05) more resistant than the parental cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E749K', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1152', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18497962', evidence_text:\"The LS174T cells with E749K were significantly (p<0.05) more responsive to gefitinib than the parental cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E762G', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1153', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18497962', evidence_text:\"LS174T cells with E762G or A767T were significantly (p<0.05) more resistant than the parental cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A767T', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1154', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18497962', evidence_text:\"LS174T cells with E762G or A767T were significantly (p<0.05) more resistant than the parental cells.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'T783#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1155', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"From Table 4 Responders.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V765#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1156', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"From Table 4 Responders.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V769#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1157', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"responders had G719C or L858R, whereas nonresponders expressed G719A or V769L\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insNPG', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1158', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"mutations in exon 20 (D770_N771insNPG) confer EGFR mutant cell resistance to gefitinib in contrast to mutations in other exons\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V769L', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1159', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"responders had G719C or L858R, whereas nonresponders expressed G719A or V769L\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'R776G', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1160', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"Case 3 had mutation R776G and did not respond to gefitinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'R776H', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1161', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"Case 4, the responder to gefitinib, had mutation R776H\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'T783A', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1162', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"From Table 4 Responders.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V765A', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1163', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18676761', evidence_text:\"From Table 4 Responders.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'T854#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1164', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010870', evidence_text:\"Acquired resistance to epidermal growth factor receptor kinase inhibitors associated with a novel T854A mutation in a patient with EGFR-mutant lung adenocarcinoma. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'T854A', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1165', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010870', evidence_text:\"Acquired resistance to epidermal growth factor receptor kinase inhibitors associated with a novel T854A mutation in a patient with EGFR-mutant lung adenocarcinoma. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'N826S', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1166', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19147750', evidence_text:\"EGFR-N826S required high concentrations of all three EGFR kinase inhibitors of _500 nmol/L for complete inhibition. Interestingly, EGFR-N826S was detected in a NSCLC patient who did not respond to gefitinib treatment and this lack of response might be explained by the high IC50 value\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'S768#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1167', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20522446', evidence_text:\"We reported a case of non-small cell lung cancer harboring a rare epidermal growth factor somatic mutation, codon 768 AGC > ATC in exon 20 (S768I), who showed a good clinical response to gefitinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'S768I', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1168', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20522446', evidence_text:\"We reported a case of non-small cell lung cancer harboring a rare epidermal growth factor somatic mutation, codon 768 AGC > ATC in exon 20 (S768I), who showed a good clinical response to gefitinib.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'HIS773_V774dupHV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1169', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'HIS773_V774insG', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1170', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'HIS773_V774insGH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1171', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'HIS773_V774insGNPH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1172', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'HIS773_V774insH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1173', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'HIS773_V774insNPH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1174', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'HIS773_V774insPH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1175', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'N771insGY', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1176', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"PF00299804 show that Ala767_ Val769dupAlaSerVal, Asp770_Asn771insAsnProGly, delAsn771insGlyTyr, and His773_Val774insHis have an IC50 to these compounds in similar dose ranges as EGFR Thr790Met models (table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:' D770>GY', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1177', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'A767_V769dupASV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1178', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"The few mutations that have been studied in vitro, including Ala767_Val769dupAlaSerVal,27,58 Asp770_ Asn771insAsnProGly,26,27,59 delAsn771insGlyTyr,50 and His773_Val774insHis,27 have been shown to be resistant to gefitinib and erlotinib; the insertions had 50% inhibitory concentrations (IC50) to gefitinib or erlotinib, of higher than 3 ?mol/L (table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'A767_V769insTLA', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1179', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insAPW', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1180', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insD', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1181', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insDG', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1182', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insG', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1183', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insGL', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1184', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insMATP', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1185', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insN', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1186', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insNPH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1187', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insSVD', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1188', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insSVP', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1189', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N771insSVQ', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1190', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'D770_N777insNPG', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1191', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"The few mutations that have been studied in vitro, including Ala767_Val769dupAlaSerVal,27,58 Asp770_ Asn771insAsnProGly,26,27,59 delAsn771insGlyTyr,50 and His773_Val774insHis,27 have been shown to be resistant to gefitinib and erlotinib; the insertions had 50% inhibitory concentrations (IC50) to gefitinib or erlotinib, of higher than 3 ?mol/L (table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'H773_V774dupHV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1192', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'H773_V774insG', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1193', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'H773_V774insGH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1194', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'H773_V774insGNPH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1195', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'H773_V774insH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1196', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"The few mutations that have been studied in vitro, including Ala767_Val769dupAlaSerVal,27,58 Asp770_ Asn771insAsnProGly,26,27,59 delAsn771insGlyTyr,50 and His773_Val774insHis,27 have been shown to be resistant to gefitinib and erlotinib; the insertions had 50% inhibitory concentrations (IC50) to gefitinib or erlotinib, of higher than 3 ?mol/L (table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'M766_A767insA', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1197', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"Two patients with tumours harbouring Tyr764_Val765insHisHis or Met766_Ala767insAIa had prolonged periods of disease control with reversible EGFR TKIs (table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'N770>GY', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1198', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In an initial phase 1 trial of PF00299804, six patients with EGFR exon 20 insertions were included and only one (with delAsn770insGlyTyr) had a response.51\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'N770insGY', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1199', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In an initial phase 1 trial of PF00299804, six patients with EGFR exon 20 insertions were included and only one (with delAsn770insGlyTyr) had a response.51\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'N771>GY', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1200', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"The few mutations that have been studied in vitro, including Ala767_Val769dupAlaSerVal,27,58 Asp770_ Asn771insAsnProGly,26,27,59 delAsn771insGlyTyr,50 and His773_Val774insHis,27 have been shown to be resistant to gefitinib and erlotinib; the insertions had 50% inhibitory concentrations (IC50) to gefitinib or erlotinib, of higher than 3 ?mol/L (table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'P772_H773insDNP', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1201', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'P772_H773insDPH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1202', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'P772_H773insN', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1203', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'P772_H773insPR', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1204', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'P772_H773insQV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1205', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'P772_H773insTPH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1206', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'P772_H773insYNP', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1207', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'P772_H773V', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1208', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"In summary, EGFR exon 20 insertion mutations affecting aminoacids Ala767, Ser768, Asp770, Pro772, and His773 are resistant to clinically achievable doses of EGFR inhibitors that have gained regulatory approval or entered late-stage clinical trials, such as gefitinib, erlotinib, neratinib, afatinib, and PF00299804.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'S768_V769insAWT', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1209', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"EGFR exon 20 insertions may account for up to 4% of EGFR mutations,22,23 occur in the same group of patients and tumours with with classic EGFR mutations (women, never smokers, adenocarcinomas),25 cluster around aminoacid positions Ser768 and Val774 located in the N-lobe of the kinase domain of EGFR after the C-helix (table 1, figure 1, and figure 2), lead to a pattern of in-vitro resistance to reversible (gefitinib, erlotinib) and irreversible (neratinib, afatinib, PF00299804) EGFR TKIs (table 2), and are rarely associated with meaningful clinical responses to EGFR inhibitors in patients given gefitinib, erlotinib, neratinib, afatinib, or PF00299804 (table 3 and table 4).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'S768_V769insVAS', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1210', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"EGFR exon 20 insertions may account for up to 4% of EGFR mutations,22,23 occur in the same group of patients and tumours with with classic EGFR mutations (women, never smokers, adenocarcinomas),25 cluster around aminoacid positions Ser768 and Val774 located in the N-lobe of the kinase domain of EGFR after the C-helix (table 1, figure 1, and figure 2), lead to a pattern of in-vitro resistance to reversible (gefitinib, erlotinib) and irreversible (neratinib, afatinib, PF00299804) EGFR TKIs (table 2), and are rarely associated with meaningful clinical responses to EGFR inhibitors in patients given gefitinib, erlotinib, neratinib, afatinib, or PF00299804 (table 3 and table 4).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'V774_C775insHV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1211', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"EGFR exon 20 insertions may account for up to 4% of EGFR mutations,22,23 occur in the same group of patients and tumours with with classic EGFR mutations (women, never smokers, adenocarcinomas),25 cluster around aminoacid positions Ser768 and Val774 located in the N-lobe of the kinase domain of EGFR after the C-helix (table 1, figure 1, and figure 2), lead to a pattern of in-vitro resistance to reversible (gefitinib, erlotinib) and irreversible (neratinib, afatinib, PF00299804) EGFR TKIs (table 2), and are rarely associated with meaningful clinical responses to EGFR inhibitors in patients given gefitinib, erlotinib, neratinib, afatinib, or PF00299804 (table 3 and table 4).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'Y764_V765insHH', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1212', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"Two patients with tumours harbouring Tyr764_Val765insHisHis or Met766_Ala767insAIa had prolonged periods of disease control with reversible EGFR TKIs (table 2).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'S768_D770dupASV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1213', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21764376', evidence_text:\"EGFR exon 20 insertions may account for up to 4% of EGFR mutations,22,23 occur in the same group of patients and tumours with with classic EGFR mutations (women, never smokers, adenocarcinomas),25 cluster around aminoacid positions Ser768 and Val774 located in the N-lobe of the kinase domain of EGFR after the C-helix (table 1, figure 1, and figure 2), lead to a pattern of in-vitro resistance to reversible (gefitinib, erlotinib) and irreversible (neratinib, afatinib, PF00299804) EGFR TKIs (table 2), and are rarely associated with meaningful clinical responses to EGFR inhibitors in patients given gefitinib, erlotinib, neratinib, afatinib, or PF00299804 (table 3 and table 4).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'I744_K745insKIPVAI', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1214', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22190593', evidence_text:\"The Ba/F3 cells transfected with exon 19 insertions were found to be sensitive to both TKIs, similar to Ba/F3 cells harboring the common EGFR exon 19 deletion (E746_A750del; Fig. 2A)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_insertion', value:'K745_E746insTPVAIK', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1215', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22190593', evidence_text:\"The Ba/F3 cells transfected with exon 19 insertions were found to be sensitive to both TKIs, similar to Ba/F3 cells harboring the common EGFR exon 19 deletion (E746_A750del; Fig. 2A)\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A289#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1216', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22588883', evidence_text:\"Lapatinib, but not erlotinib, induced cell death in all examined GBM cell lines with EGFR ectodomain mutants. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G598#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1217', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22588883', evidence_text:\"Lapatinib, but not erlotinib, induced cell death in all examined GBM cell lines with EGFR ectodomain mutants. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A289V', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1218', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22588883', evidence_text:\"Lapatinib, but not erlotinib, induced cell death in all examined GBM cell lines with EGFR ectodomain mutants. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G598V', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1219', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22588883', evidence_text:\"Lapatinib, but not erlotinib, induced cell death in all examined GBM cell lines with EGFR ectodomain mutants. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1220', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22869875', evidence_text:\"A biomarker analysis of this trial was later published, which found that only high EGFR copy number was predictive of survival benefit from erlotinib.10\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-S752insA', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1221', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-S752insI', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1222', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-S752insV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1223', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-T751insA', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1224', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-T751insD', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1225', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-T751insF', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1226', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-T751insG', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1227', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-T751insI', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1228', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-T751insV', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1229', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'E746-T751insVA', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1230', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23205292', evidence_text:\"Results of several preclinical studies demonstrated that dacomitinib displayed the ability to inhibit several different EGFR mutants, including the commonly found primary EGFR mutants (i.e. exon 19 deletion or L858R mutation) and the secondary EGFR T790M mutant (9,10). \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A702#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1231', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A839#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1232', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A871#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1233', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E746#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1234', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E868#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1235', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G719#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1236', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"We further confirm that G719X and L861Q mutations confer sensitivity to EGFR TKIs, although L861Q mutations were associated with a lower response rate. \" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G779#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1237', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G857#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1238', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G863#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1239', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G874#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1240', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'K708#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1241', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'K856#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1242', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'L692#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1243', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'L718#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1244', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'L838#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1245', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'L858#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1246', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"About 210 of the 235 therapy_response instances (89%) from 225 patients with a single L858R mutation achieved disease control with either gefitinib or erlotinib. L858R mutations were also frequently found with a second de novo mutation (n 1_4 31). Including all combinations of L858R mutation with concurrent de novo mutations, 237 of the 266 therapy_response instances (89%) from 256 unique patients achieved disease control. L858R mutations occur- ring in combination with either A871E, L747S, de novo G719S, de novo T790M, L861P, or R776G were associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'N700#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1247', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'N756#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1248', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'P691#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1249', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'P699#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1250', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'R748#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1251', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'R832#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1252', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'R836#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1253', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'S720#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1254', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'S921#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1255', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V689#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1256', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V726#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1257', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V740#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1258', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V742#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1259', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V819#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1260', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V834#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1261', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V843#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1262', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V851#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1263', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'Y727#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1264', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A736#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1265', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A859#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1266', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'D761#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1267', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"The other 3 patients (with acquired resistance mutations) developed a D761Y, L747S, or T854A resistance mutation in line with known acquired resistance mutations to EGFR TKI theray (9).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'E711#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1268', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G721#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1269', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'G729#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1270', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'I744#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1271', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'I758#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1272', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'K806#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1273', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'K860#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1274', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'L703#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1275', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'L747#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1276', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'L747#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1277', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"The other 3 patients (with acquired resistance mutations) developed a D761Y, L747S, or T854A resistance mutation in line with known acquired resistance mutations to EGFR TKI theray (9).\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'L777#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1278', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'N826#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1279', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'N842#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1280', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'S784#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1281', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'T847#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1282', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V774#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1283', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V802#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1284', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'V852#', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1285', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A702S', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1286', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A736V', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1287', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'A767-A769', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1288', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A839T', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1289', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Supplemental Table 1. EGFR Mutations Associated with Disease Control (Partial Response, Completer Response, or Stable Disease). These are rarer varients linked to response and should be double checked across other resources.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'snv', value:'A859T', gene_entrez:'1956', entity_class:'biomarker' }), (d:Drug {name:'erlotinib'} ) Merge (r:Rule {rulenum:'1290', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23344264', evidence_text:\"From Table 2. EGFR mutation associated with disease progression.\" }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) "
     ]
    }
   ],
   "source": [
    "import pandas as pd \n",
    "import numpy as np\n",
    "from py2neo import Graph, Node, Relationship, authenticate\n",
    "\n",
    "\n",
    "def createDrugs(graph,data):\n",
    "    uniqueDrugs = np.unique(data[\"drug\"])\n",
    "    print uniqueDrugs\n",
    "    nodeCount = 0\n",
    "    for drug in uniqueDrugs:\n",
    "        graph.cypher.execute(\"MERGE (d:Drug {{name:\\\"{0}\\\"}})\".format(drug))\n",
    "        nodeCount = nodeCount + 1\n",
    "\n",
    "def createGenes(graph,data):\n",
    "    for row_index, row in data.iterrows():\n",
    "        gene_name = \"\"\n",
    "        gene_entrez = \"\"\n",
    "        if row[\"entity_class\"] == \"biomarker\":\n",
    "            gene_name = row[\"biomarker_symbol\"]\n",
    "            gene_entrez = row[\"biomarker_entrez\"]\n",
    "        elif row[\"entity_class\"] == \"modifier\":\n",
    "            gene_name = row[\"modifier_symbol\"]\n",
    "            gene_entrez = row[\"modifier_entrez\"]\n",
    "        else:\n",
    "            print\"Error in getting genes\"\n",
    "            break\n",
    "        query = (\n",
    "            \"MATCH (d:Drug {{ name:'{2}'}})\"\n",
    "            \"MERGE (g:Gene {{ name:'{0}', entrez:'{1}'}})\"\n",
    "            \"CREATE UNIQUE (d)-[:TARGETS]->(g)\"\n",
    "        ).format(gene_name, gene_entrez, row[\"drug\"])\n",
    "        print query\n",
    "        graph.cypher.execute(query)\n",
    "\n",
    "def createAberrations(graph,data):\n",
    "    for row_index, row in data.iterrows():\n",
    "        gene_name = \"\"\n",
    "        gene_entrez = \"\"\n",
    "        ab_type = \"\"\n",
    "        ab_value = \"\"\n",
    "        \n",
    "        if row[\"entity_class\"] == \"biomarker\":\n",
    "            gene_name = row[\"biomarker_symbol\"]\n",
    "            gene_entrez = row[\"biomarker_entrez\"]\n",
    "            ab_type = row[\"biomarker_aberration_type\"]\n",
    "            ab_value = row[\"biomarker_aberration_value\"]\n",
    "            \n",
    "        elif row[\"entity_class\"] == \"modifier\":\n",
    "            gene_name = row[\"modifier_symbol\"]\n",
    "            gene_entrez = row[\"modifier_entrez\"]\n",
    "            ab_type = row[\"modifier_aberration_type\"]\n",
    "            ab_value = row[\"modifier_aberration_value\"]\n",
    "        else:\n",
    "            print\"Error in createAberrations()\"\n",
    "            break\n",
    "            \n",
    "        query = (\"MATCH (g:Gene {{name:'{0}', entrez:'{1}'}}) \"\n",
    "                \"MERGE (a:Aberration {{ type:'{2}', value:'{3}', gene_entrez:'{4}', entity_class:'{5}' }}) \"\n",
    "                \"CREATE UNIQUE (a)<-[:HAS_ABERRATION]-(g)\"\n",
    "                ).format(gene_name, gene_entrez, ab_type, ab_value, gene_entrez, row[\"entity_class\"])\n",
    "        print query\n",
    "        graph.cypher.execute(query)\n",
    "    \n",
    "def createRuleNodes(graph,data):\n",
    "    for row_index, row in data.iterrows():\n",
    "        gene_name = \"\"\n",
    "        gene_entrez = \"\"\n",
    "        ab_type = \"\"\n",
    "        ab_value = \"\"\n",
    "        \n",
    "        if row[\"entity_class\"] == \"biomarker\":\n",
    "            gene_name = row[\"biomarker_symbol\"]\n",
    "            gene_entrez = row[\"biomarker_entrez\"]\n",
    "            ab_type = row[\"biomarker_aberration_type\"]\n",
    "            ab_value = row[\"biomarker_aberration_value\"]\n",
    "            \n",
    "        elif row[\"entity_class\"] == \"modifier\":\n",
    "            gene_name = row[\"modifier_symbol\"]\n",
    "            gene_entrez = row[\"modifier_entrez\"]\n",
    "            ab_type = row[\"modifier_aberration_type\"]\n",
    "            ab_value = row[\"modifier_aberration_value\"]\n",
    "        else:\n",
    "            print\"Error in createAberrations()\"\n",
    "            break\n",
    "            \n",
    "        query = (\"MATCH (g:Gene {{name:'{0}', entrez:'{1}'}}), \"\n",
    "                 \"(a:Aberration {{ type:'{2}', value:'{3}', gene_entrez:'{4}', entity_class:'{5}' }}), \"\n",
    "                 \"(d:Drug {{name:'{6}'}} ) \"\n",
    "                 \"Merge (r:Rule {{rulenum:'{7}', evidence:'{8}', pmid:'{9}', evidence_text:\\\"{10}\\\" }}) \"\n",
    "                 \"CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) \"\n",
    "                 \"CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) \"\n",
    "                 \"CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \"\n",
    "                ).format(gene_name, gene_entrez, ab_type, ab_value, gene_entrez, row[\"entity_class\"], \n",
    "                         row[\"drug\"], row_index, row[\"Evidence\"], row[\"PMID\"], row[\"Evidence Text\"])\n",
    "        print query\n",
    "        graph.cypher.execute(query)\n",
    "    \n",
    "   \n",
    "        \n",
    "def main():\n",
    "    print \"Starting Main Function\"\n",
    "    data = pd.read_csv('./DrugRulesFixed.csv')\n",
    "\n",
    "    authenticate(\"localhost:7474\", \"neo4j\", \"qwerty1\")\n",
    "    graph = Graph()\n",
    "    \n",
    "    #createDrugs(graph,data)\n",
    "    #createGenes(graph,data)\n",
    "    #createAberrations(graph,data)\n",
    "    createRuleNodes(graph,data)\n",
    "\n",
    "main()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
