{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting Main Function\n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'Y156H', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'0', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18559266', evidence_text:'Thus, the two mutations in the catalytic cleft, Y156H and G160V, render Aurora B resistant to four classes of inhibitor, namely, ZM447439, VX680, Hesperadin, and MLN8054.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'Y156#', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'1', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18559266', evidence_text:'Thus, the two mutations in the catalytic cleft, Y156H and G160V, render Aurora B resistant to four classes of inhibitor, namely, ZM447439, VX680, Hesperadin, and MLN8054.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'G160V', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'2', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18559266', evidence_text:'Thus, the two mutations in the catalytic cleft, Y156H and G160V, render Aurora B resistant to four classes of inhibitor, namely, ZM447439, VX680, Hesperadin, and MLN8054.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'G160#', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'3', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18559266', evidence_text:'Thus, the two mutations in the catalytic cleft, Y156H and G160V, render Aurora B resistant to four classes of inhibitor, namely, ZM447439, VX680, Hesperadin, and MLN8054.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'4', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:'Results revealed that MLN8237 inhibits the phosphorylation of both AURKA and AURKB, though it is more specific to AURKA. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Melanoma.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'5', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:'Results revealed that MLN8237 inhibits the phosphorylation of both AURKA and AURKB, though it is more specific to AURKA. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Melanoma.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HMMR', entrez:'3161'}), (a:Aberration { type:'exp', value:'under', gene_entrez:'3161', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'6', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23328114', evidence_text:'Silencing of HMMR/RHAMM, but not TPX2, augments AURKA activity and sensitizes MPNST cells to AKI.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'HMMR', entrez:'3161'}), (a:Aberration { type:'cnv', value:'loss', gene_entrez:'3161', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'7', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23328114', evidence_text:'Silencing of HMMR/RHAMM, but not TPX2, augments AURKA activity and sensitizes MPNST cells to AKI.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'snv', value:'T217D', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'8', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20426425', evidence_text:'Using inducible human cell lines, we show that cells expressing near-physiological levels of a functional but partially drug-resistant Aurora A T217D mutant survive in the presence of MLN8054 or MLN8237' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'snv', value:'T217#', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'9', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20426425', evidence_text:'Using inducible human cell lines, we show that cells expressing near-physiological levels of a functional but partially drug-resistant Aurora A T217D mutant survive in the presence of MLN8054 or MLN8237' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'10', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:'Here, using a mouse model with orthotopic implantation of metastatic melanoma tumours taken from 19 patients, we observed that targeting aurora kinases with MLN8054/MLN8237 impaired mitosis, induced senescence and markedly blocked proliferation in patient tumour implants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Disease:  Melanoma' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'11', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:'Here, using a mouse model with orthotopic implantation of metastatic melanoma tumours taken from 19 patients, we observed that targeting aurora kinases with MLN8054/MLN8237 impaired mitosis, induced senescence and markedly blocked proliferation in patient tumour implants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Disease:  Melanoma' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKA', entrez:'6790'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'6790', entity_class:'biomarker' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'12', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:'Here, using a mouse model with orthotopic implantation of metastatic melanoma tumours taken from 19 patients, we observed that targeting aurora kinases with MLN8054/MLN8237 impaired mitosis, induced senescence and markedly blocked proliferation in patient tumour implants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Disease:  Melanoma |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'AURKB', entrez:'9212'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'9212', entity_class:'modifier' }), (d:Drug {name:'Alisertib'} ) Merge (r:Rule {rulenum:'13', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23180582', evidence_text:'Results revealed that MLN8237 inhibits the phosphorylation of both AURKA and AURKB, though it is more specific to AURKA. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Melanoma. |Annotator Note: Some mutations in AURKB cause resistance to related drug MLN8054.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'14', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:'We show that MET inhibition induces complete growth arrest of MET-amplified gastric cancer cells. |Evidence Level:  Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'D1228#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'15', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'D1228A', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'16', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'D1228L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'17', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'D1228N', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'18', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'E168#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'19', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19723643', evidence_text:'In contrast, the cells expressing E168D mutant showed higher susceptibility to the c-Met inhibitor at both concentrations tested (Fig. 3C). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Lung Cancer. |Annotator Note: Drug class relation with SU11274. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'E168D', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'20', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19723643', evidence_text:'In contrast, the cells expressing E168D mutant showed higher susceptibility to the c-Met inhibitor at both concentrations tested (Fig. 3C). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Lung Cancer. |Annotator Note: Drug class relation with SU11274. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'F1200#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'21', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'F1200I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'22', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 1. Mutations found through drug exporsure in cell line reporter system. Drug was NVP-BVU972 which is class I type MET inhibitor. Not sure what class AMG 337 is exactly. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class I MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'F1200L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'23', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'G1163#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'24', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'G1163E', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'25', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'G1163V', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'26', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'27', evidence:'direct', pmid:'http://bit.ly/1ziTsf4', evidence_text:'Responses were observed in a subset of pts with MET-amplified tumors. A dose-expansion phase will enroll up to 50 pts at the MTD (300 mg QD). |Evidence Level:  Phase I |Disease:  gastroesophageal cancer. |Annotator Note: Some studies suggest amplficiation of MET as an acquired resistance mechanism upon MET inhibitor treatment.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1094#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'28', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23213094', evidence_text:'The presence of a germline MET mutation was highly predictive of a response (five of 10 v five of 57 patients with and without germline MET mutations, respectively). |Evidence Level:  Phase II |Disease:  papillary renal cell carcinoma |Annotator Note:  Drug class relation with Foretinib. H1094R in this study was a germline mutation.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1094R', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'29', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23213094', evidence_text:'The presence of a germline MET mutation was highly predictive of a response (five of 10 v five of 57 patients with and without germline MET mutations, respectively). |Evidence Level:  Phase II |Disease:  papillary renal cell carcinoma |Annotator Note:  Drug class relation with Foretinib. H1094R in this study was a germline mutation.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1112#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'30', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'Remarkably, five days of EMD1214063 treatment resulted in a complete regression of the sensitive H1112L-derived tumors. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1112L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'31', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'Remarkably, five days of EMD1214063 treatment resulted in a complete regression of the sensitive H1112L-derived tumors. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H1112Y', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'32', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'Contrary to the observations with M1268T, H1112Y, H1112L, V1238I, and V1110I, EMD1214063 doses up to 75 nmol/L did not display any inhibition of the MET enzymatic activity in cells expressing the Y1248H, L1213V, and V1206L mutants as judged from the levels of phosphorylated MET. |Evidence Level:  Human. Pre-clinical.  Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'H112#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'33', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'Contrary to the observations with M1268T, H1112Y, H1112L, V1238I, and V1110I, EMD1214063 doses up to 75 nmol/L did not display any inhibition of the MET enzymatic activity in cells expressing the Y1248H, L1213V, and V1206L mutants as judged from the levels of phosphorylated MET. |Evidence Level:  Human. Pre-clinical.  Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'K1193#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'34', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'K1193I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'35', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1195#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'36', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1195I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'37', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1195V', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'38', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1213#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'39', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'On the other hand, no changes in cell-cycle progression in terms of G1 and S-phase populations were observed in Y1248H and L1213V cells treated with the EMD1214063, indicating a resistance of these MET variants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'L1213V', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'40', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'On the other hand, no changes in cell-cycle progression in terms of G1 and S-phase populations were observed in Y1248H and L1213V cells treated with the EMD1214063, indicating a resistance of these MET variants. |Evidence Level:  Human. Pre-clinical. Xenograft, Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1211#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'41', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1211L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'42', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1250#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'43', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1250T', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'44', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1268#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'45', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23610116', evidence_text:'The patient was confirmed as having a heterozygous MET mutation at M1268T. The patient had a reduction of 35% in the sum of one-dimensional measurements of target lesions after receiving treatment for 53 weeks, achieving a confirmed partial response by RECIST version 1.04. |Evidence Level:  Case Study |Disease:  papillary renal cell carcinoma' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'M1268T', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'46', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23610116', evidence_text:'The patient was confirmed as having a heterozygous MET mutation at M1268T. The patient had a reduction of 35% in the sum of one-dimensional measurements of target lesions after receiving treatment for 53 weeks, achieving a confirmed partial response by RECIST version 1.04. |Evidence Level:  Case Study |Disease:  papillary renal cell carcinoma' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'N375#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'47', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19723643', evidence_text:'The MET-N375S mutation seems to confer resistance to MET inhibition based on hepatocyte growth factor ligand binding, molecular modeling, and apoptotic susceptibility to MET inhibitor studies. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Lung Cancer. |Annotator Note: Drug class relation with SU11274. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'N375S', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'48', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19723643', evidence_text:'The MET-N375S mutation seems to confer resistance to MET inhibition based on hepatocyte growth factor ligand binding, molecular modeling, and apoptotic susceptibility to MET inhibitor studies. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Lung Cancer. |Annotator Note: Drug class relation with SU11274. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'49', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/23388504', evidence_text:'These data suggest that c-MET may be a promising target in the treatment of HCC and that c-MET overexpression may be a predictive biomarker of response. |Evidence Level: Pre-clinical. |Disease:  Hepatocellular Cancer.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Q1258#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'50', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Q1258L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'51', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1110#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'52', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'IC50 values for the studied mutants ranged between 2.2 and 43 nmol/L for the H1112L and V1110I variants, respectively. In this regard, EMD1214063 displays anti-MET IC50 values, which are in the range of crizotinib, a MET small molecule that is currently in phase III clinical trials.  |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1110I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'53', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'IC50 values for the studied mutants ranged between 2.2 and 43 nmol/L for the H1112L and V1110I variants, respectively. In this regard, EMD1214063 displays anti-MET IC50 values, which are in the range of crizotinib, a MET small molecule that is currently in phase III clinical trials.  |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1155#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'54', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. SNVs at this position associated with reduced drug sensitivity. However, not all SNVs at this position may be associated with reduced drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1155L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'55', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. |Evidence Level:  Pre-clinical. Cell line. |Annotator Note:  The mutations are for a class II MET inhibitor but annotator was not able to determine what class AMG337 is and if these mutations would convey to AMG337. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1206#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'56', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'Of major importance is our observation that the autophosphorylation status of the MET-mutated variants Y1248H, L1213V, and V1206L displays a rather resistant-like response towards EMD1214063. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1206L', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'57', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'Of major importance is our observation that the autophosphorylation status of the MET-mutated variants Y1248H, L1213V, and V1206L displays a rather resistant-like response towards EMD1214063. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1238#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'58', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'As the results of Fig. 3 suggest, variants V1238I, M1268T, and H1112L exhibited the highest sensitivity to 30 nmol/L of EMD1214063. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'V1238I', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'59', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'As the results of Fig. 3 suggest, variants V1238I, M1268T, and H1112L exhibited the highest sensitivity to 30 nmol/L of EMD1214063. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1159#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'60', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. Not sure what class AMG 337 is exactly.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1159N', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'61', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21697284', evidence_text:'Figure 3. Mutations found through drug exporsure in cell line reporter system. Drug was AMG458 which is class II type MET inhibitor. Not sure what class AMG 337 is exactly.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'62', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:'In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230C', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'63', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:'In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230D', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'64', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:'In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230H', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'65', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:'In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230N', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'66', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:'In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1230S', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'67', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:'In this study, we identified mutations in Y1230 as an acquired resis- tance mechanism to class I MET inhibitors. The occasional existence of MET Y1230 mutations in pretreatment cancers (24) is analogous to the observations that some lung cancers and leukemias harbor EGFR T790M and ABL T315I, respec- tively, prior to treatment (25, 26). In the case of MET, this is likely related because of increased MET activity conferred by the Y1230 mutation. The MET Y1230H mutation is located in the activation loop of the enzyme. Structural analyses suggest that the substitu- tion of Y1230 with histidine or cysteine has a lower affinity with PF-2341066 and PHA-665752. Indeed, these results are supported by previous in vitro kinase assays showing that these compounds have decreased inhibitory activity toward MET Y1230H as compared with wt MET in enzymatic and cellular assays (18). Although these and other class I MET inhibitors seem to have decreased activity against MET Y1230H (28), there have recently been reports of class II MET inhibitors that can potently inhibit Y1230H (29, 30). |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PHA-665752 and PF-2341066. ' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1248#', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'68', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'On the other hand, no changes in cell-cycle progression in terms of G1 and S-phase populations were observed in Y1248H and L1213V cells treated with the EMD1214063, indicating a resistance of these MET variants. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'MET', entrez:'4233'}), (a:Aberration { type:'snv', value:'Y1248H', gene_entrez:'4233', entity_class:'biomarker' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'69', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/24061647', evidence_text:'On the other hand, no changes in cell-cycle progression in terms of G1 and S-phase populations were observed in Y1248H and L1213V cells treated with the EMD1214063, indicating a resistance of these MET variants. |Evidence Level:  Human. Pre-clinical. Cell line. |Annotator Note:  Drug class relation with EMD1214063.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'70', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:'Our data provide a strong rationale for combined inhibition of MET and HER kinase signaling in patients with MET-amplified tumors that coexpress EGFR and/or HER-3. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'small_deletion', value:'del L747_S752', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'71', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:'Transfection of GTL-16 Cells with Mutant EGFR Harboring an Exon 19 Deletion (EGFR?L747-S752) Confers Resistance to PHA-665752 treatment.  |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'EGFR', entrez:'1956'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'1956', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'72', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:'Our data provide a strong rationale for combined inhibition of MET and HER kinase signaling in patients with MET-amplified tumors that coexpress EGFR and/or HER-3. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB3', entrez:'2065'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'2065', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'73', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:'Our data provide a strong rationale for combined inhibition of MET and HER kinase signaling in patients with MET-amplified tumors that coexpress EGFR and/or HER-3. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'ERBB3', entrez:'2065'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'2065', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'74', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/18974395', evidence_text:'Our data provide a strong rationale for combined inhibition of MET and HER kinase signaling in patients with MET-amplified tumors that coexpress EGFR and/or HER-3. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease: Gastric carcinoma.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'75', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20841479', evidence_text:'KRAS amplification is a general mechanism of resistance to specific MET inhibitors given that similar results were observed with two small inhibitors and in different cell lines of different histotypes. To our knowledge, this is the first report showing that overexpression of wt KRAS can overcome the inhibitory effect of a RTK inhibitor. |Evidence Level:  Human. Pre-clinical. Cell line.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'KRAS', entrez:'3845'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'3845', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'76', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/20841479', evidence_text:'KRAS amplification is a general mechanism of resistance to specific MET inhibitors given that similar results were observed with two small inhibitors and in different cell lines of different histotypes. To our knowledge, this is the first report showing that overexpression of wt KRAS can overcome the inhibitory effect of a RTK inhibitor. |Evidence Level:  Human. Pre-clinical. Cell line.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'TGFA', entrez:'7039'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'7039', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'77', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/21266357', evidence_text:'In this study, we found that SNU638 cells adapt to MET inhibition by overexpressing the EGFR ligand TGFa to promote resistance. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Carcinoma.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'SND1_BRAF', entrez:'EMPTY'}), (a:Aberration { type:'EMPTY', value:'EMPTY', gene_entrez:'EMPTY', entity_class:'modifier' }), (d:Drug {name:'AMG 337'} ) Merge (r:Rule {rulenum:'78', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/22745804', evidence_text:'We rendered a MET amplified gastric cancer cell line, GTL16, resistant to c-Met inhibition with prolonged exposure to a c-Met inhibitor, PF-04217903 (METi). Characterization of surviving cells identified an amplified chromosomal rearrangement between 7q32 and 7q34 which overexpresses a constitutively active SND1-BRAF fusion protein. In the resistant clones, hyperactivation of the downstream MAPK pathway via SND1-BRAF conferred resistance to c-Met receptor tyrosine kinase inhibition. Combination treatment with METi and a RAF inhibitor, PF-04880594 (RAFi) inhibited ERK activation and circumvented resistance to either single agent. Alternatively, treatment with a MEK inhibitor, PD-0325901 (MEKi) alone effectively blocked ERK phosphorylation and inhibited cell growth. Our results suggest that combination of a c-Met tyrosine kinase inhibitor with a BRAF or a MEK inhibitor may be effective in treating resistant tumors that use activated BRAF to escape suppression of c-Met signaling. |Evidence Level:  Human. Pre-clinical. Cell line. |Disease:  Gastric Cancer. |Annotator Note:  Drug class relation with PF=-4217903.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'79', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:'Axitinib inhibits PDGF-R-beta with IC50=2.9 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical. |Annotator Note:  SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'80', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:'Axitinib inhibits PDGF-R-beta with IC50=2.9 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRB', entrez:'5159'}), (a:Aberration { type:'exp', value:'over', gene_entrez:'5159', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'81', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:'Axitinib inhibits PDGF-R-beta with IC50=2.9 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Pre-clinical.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'snv', value:'change', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'82', evidence:'inferred', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:'Axitinib inhibits PDGF-R-alpha with IC50=5.0 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Human. Pre-clinical. |Annotator Note:  PDGFRA SNVs can be associated either with drug response or sensitivity depending on disease and position in protein. SNVs should be activating and associated with drug sensitivity. However, not all SNVs may result in activation and drug sensitivity.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n",
      "MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'83', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:'Axitinib inhibits PDGF-R-alpha with IC50=5.0 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Human. Pre-clinical. |Annotator Note:  'Is a target' relationship.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \n"
     ]
    },
    {
     "ename": "InvalidSyntax",
     "evalue": "Invalid input 'a': expected whitespace, comment, NULL or NOT (line 1, column 649 (offset: 648))\n\"MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'83', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:'Axitinib inhibits PDGF-R-alpha with IC50=5.0 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Human. Pre-clinical. |Annotator Note:  'Is a target' relationship.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \"\n                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         ^",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mInvalidSyntax\u001b[0m                             Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-36-35a8da6df5d3>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m    110\u001b[0m     \u001b[0mcreateRuleNodes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mgraph\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    111\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 112\u001b[0;31m \u001b[0mmain\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m<ipython-input-36-35a8da6df5d3>\u001b[0m in \u001b[0;36mmain\u001b[0;34m()\u001b[0m\n\u001b[1;32m    108\u001b[0m     \u001b[0;31m#createGenes(graph,data)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    109\u001b[0m     \u001b[0;31m#createAberrations(graph,data)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 110\u001b[0;31m     \u001b[0mcreateRuleNodes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mgraph\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    111\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    112\u001b[0m \u001b[0mmain\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m<ipython-input-36-35a8da6df5d3>\u001b[0m in \u001b[0;36mcreateRuleNodes\u001b[0;34m(graph, data)\u001b[0m\n\u001b[1;32m     94\u001b[0m                          row[\"drug\"], row_index, row[\"Evidence\"], row[\"PMID\"], row[\"Evidence Text\"])\n\u001b[1;32m     95\u001b[0m         \u001b[0;32mprint\u001b[0m \u001b[0mquery\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 96\u001b[0;31m         \u001b[0mgraph\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcypher\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mexecute\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mquery\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     97\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     98\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/py2neo/cypher/core.pyc\u001b[0m in \u001b[0;36mexecute\u001b[0;34m(self, statement, parameters, **kwparameters)\u001b[0m\n\u001b[1;32m    109\u001b[0m             \u001b[0mtx\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mCypherTransaction\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtransaction_uri\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    110\u001b[0m             \u001b[0mtx\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstatement\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mparameters\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwparameters\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 111\u001b[0;31m             \u001b[0mresults\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtx\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcommit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    112\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mresults\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    113\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/py2neo/cypher/core.pyc\u001b[0m in \u001b[0;36mcommit\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    304\u001b[0m         \u001b[0mlog\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0minfo\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"commit\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    305\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 306\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpost\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m__commit\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m__begin_commit\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    307\u001b[0m         \u001b[0;32mfinally\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    308\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m__finished\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mTrue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/py2neo/cypher/core.pyc\u001b[0m in \u001b[0;36mpost\u001b[0;34m(self, resource)\u001b[0m\n\u001b[1;32m    259\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0merrors\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m>=\u001b[0m \u001b[0;36m1\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    260\u001b[0m                 \u001b[0merror\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0merrors\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 261\u001b[0;31m                 \u001b[0;32mraise\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0merror_class\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mhydrate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0merror\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    262\u001b[0m         \u001b[0mout\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mRecordListList\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    263\u001b[0m         \u001b[0;32mfor\u001b[0m \u001b[0mresult\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mj\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"results\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mInvalidSyntax\u001b[0m: Invalid input 'a': expected whitespace, comment, NULL or NOT (line 1, column 649 (offset: 648))\n\"MATCH (g:Gene {name:'PDGFRA', entrez:'5156'}), (a:Aberration { type:'cnv', value:'gain', gene_entrez:'5156', entity_class:'biomarker' }), (d:Drug {name:'Axitinib'} ) Merge (r:Rule {rulenum:'83', evidence:'direct', pmid:'http://www.ncbi.nlm.nih.gov/pubmed/19010843', evidence_text:'Axitinib inhibits PDGF-R-alpha with IC50=5.0 nM. Although axitinib inhibits platelet-derived growth factor receptors and KIT with nanomolar in vitro potencies, based on pharmacokinetic/pharmacodynamic analysis, axitinib acts primarily as a VEGFR tyrosine kinase inhibitor at the current clinical exposure. |Evidence Level:  Human. Pre-clinical. |Annotator Note:  'Is a target' relationship.' }) CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \"\n                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         ^"
     ]
    }
   ],
   "source": [
    "import pandas as pd \n",
    "import numpy as np\n",
    "from py2neo import Graph, Node, Relationship, authenticate\n",
    "\n",
    "\n",
    "def createDrugs(graph,data):\n",
    "    uniqueDrugs = np.unique(data[\"drug\"])\n",
    "    print uniqueDrugs\n",
    "    nodeCount = 0\n",
    "    for drug in uniqueDrugs:\n",
    "        graph.cypher.execute(\"MERGE (d:Drug {{name:\\\"{0}\\\"}})\".format(drug))\n",
    "        nodeCount = nodeCount + 1\n",
    "\n",
    "def createGenes(graph,data):\n",
    "    for row_index, row in data.iterrows():\n",
    "        gene_name = \"\"\n",
    "        gene_entrez = \"\"\n",
    "        if row[\"entity_class\"] == \"biomarker\":\n",
    "            gene_name = row[\"biomarker_symbol\"]\n",
    "            gene_entrez = row[\"biomarker_entrez\"]\n",
    "        elif row[\"entity_class\"] == \"modifier\":\n",
    "            gene_name = row[\"modifier_symbol\"]\n",
    "            gene_entrez = row[\"modifier_entrez\"]\n",
    "        else:\n",
    "            print\"Error in getting genes\"\n",
    "            break\n",
    "        query = (\n",
    "            \"MATCH (d:Drug {{ name:'{2}'}})\"\n",
    "            \"MERGE (g:Gene {{ name:'{0}', entrez:'{1}'}})\"\n",
    "            \"CREATE UNIQUE (d)-[:TARGETS]->(g)\"\n",
    "        ).format(gene_name, gene_entrez, row[\"drug\"])\n",
    "        print query\n",
    "        graph.cypher.execute(query)\n",
    "\n",
    "def createAberrations(graph,data):\n",
    "    for row_index, row in data.iterrows():\n",
    "        gene_name = \"\"\n",
    "        gene_entrez = \"\"\n",
    "        ab_type = \"\"\n",
    "        ab_value = \"\"\n",
    "        \n",
    "        if row[\"entity_class\"] == \"biomarker\":\n",
    "            gene_name = row[\"biomarker_symbol\"]\n",
    "            gene_entrez = row[\"biomarker_entrez\"]\n",
    "            ab_type = row[\"biomarker_aberration_type\"]\n",
    "            ab_value = row[\"biomarker_aberration_value\"]\n",
    "            \n",
    "        elif row[\"entity_class\"] == \"modifier\":\n",
    "            gene_name = row[\"modifier_symbol\"]\n",
    "            gene_entrez = row[\"modifier_entrez\"]\n",
    "            ab_type = row[\"modifier_aberration_type\"]\n",
    "            ab_value = row[\"modifier_aberration_value\"]\n",
    "        else:\n",
    "            print\"Error in createAberrations()\"\n",
    "            break\n",
    "            \n",
    "        query = (\"MATCH (g:Gene {{name:'{0}', entrez:'{1}'}}) \"\n",
    "                \"MERGE (a:Aberration {{ type:'{2}', value:'{3}', gene_entrez:'{4}', entity_class:'{5}' }}) \"\n",
    "                \"CREATE UNIQUE (a)<-[:HAS_ABERRATION]-(g)\"\n",
    "                ).format(gene_name, gene_entrez, ab_type, ab_value, gene_entrez, row[\"entity_class\"])\n",
    "        print query\n",
    "        graph.cypher.execute(query)\n",
    "    \n",
    "def createRuleNodes(graph,data):\n",
    "    for row_index, row in data.iterrows():\n",
    "        gene_name = \"\"\n",
    "        gene_entrez = \"\"\n",
    "        ab_type = \"\"\n",
    "        ab_value = \"\"\n",
    "        \n",
    "        if row[\"entity_class\"] == \"biomarker\":\n",
    "            gene_name = row[\"biomarker_symbol\"]\n",
    "            gene_entrez = row[\"biomarker_entrez\"]\n",
    "            ab_type = row[\"biomarker_aberration_type\"]\n",
    "            ab_value = row[\"biomarker_aberration_value\"]\n",
    "            \n",
    "        elif row[\"entity_class\"] == \"modifier\":\n",
    "            gene_name = row[\"modifier_symbol\"]\n",
    "            gene_entrez = row[\"modifier_entrez\"]\n",
    "            ab_type = row[\"modifier_aberration_type\"]\n",
    "            ab_value = row[\"modifier_aberration_value\"]\n",
    "        else:\n",
    "            print\"Error in createAberrations()\"\n",
    "            break\n",
    "            \n",
    "        query = (\"MATCH (g:Gene {{name:'{0}', entrez:'{1}'}}), \"\n",
    "                 \"(a:Aberration {{ type:'{2}', value:'{3}', gene_entrez:'{4}', entity_class:'{5}' }}), \"\n",
    "                 \"(d:Drug {{name:'{6}'}} ) \"\n",
    "                 \"Merge (r:Rule {{rulenum:'{7}', evidence:'{8}', pmid:'{9}', evidence_text:\\\"{10}\\\" }}) \"\n",
    "                 \"CREATE UNIQUE (r)<-[:IS_PART_OF]-(g) \"\n",
    "                 \"CREATE UNIQUE (r)<-[:IS_PART_OF]-(a) \"\n",
    "                 \"CREATE UNIQUE (r)<-[:IS_PART_OF]-(d) \"\n",
    "                ).format(gene_name, gene_entrez, ab_type, ab_value, gene_entrez, row[\"entity_class\"], \n",
    "                         row[\"drug\"], row_index, row[\"Evidence\"], row[\"PMID\"], row[\"Evidence Text\"])\n",
    "        print query\n",
    "        graph.cypher.execute(query)\n",
    "    \n",
    "   \n",
    "        \n",
    "def main():\n",
    "    print \"Starting Main Function\"\n",
    "    data = pd.read_csv('./DrugRulesFixed.csv')\n",
    "\n",
    "    authenticate(\"localhost:7474\", \"neo4j\", \"qwerty1\")\n",
    "    graph = Graph()\n",
    "    \n",
    "    #createDrugs(graph,data)\n",
    "    #createGenes(graph,data)\n",
    "    #createAberrations(graph,data)\n",
    "    createRuleNodes(graph,data)\n",
    "\n",
    "main()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
